---
title: "Analyse fonctionnelle de gènes et métabolites"
author: "Vincent ROCHER & Juliette COOKE"
format: revealjs
editor: source
---

```{r,echo=F,message=FALSE,warning=F}

require(tidyverse)
require(clusterProfiler)
require(org.Hs.eg.db)

```



# Gene Enrichment Analysis

## Introduction

-   Gene enrichment analysis identifies biological pathways or functions that are overrepresented in a set of genes of interest.


The list of differentially expressed genes is sometimes:

so long that its interpretation becomes cumbersome and time consuming,

or very short while some genes have low p-value yet higher than the given threshold.

There are many approaches to searching for biological meaning in the results of differential expression analysis.

Commonly we assess whether the differentially expressed genes tend to relate to specific pathways or ontological groups of genes.

We will look at two methods:

Over Representation Analysis (ORA)

Gene Set Enrichment Analysis (GSEA)



Common sources of gene sets:

KEGG pathways

Gene Ontologies

Reactome

Molecular Signature Database, MSigDB (GSEA)

Manually curated gene lists

## 

```{r echo=FALSE, warning=FALSE,message=FALSE}
# Import list of differentially expressed genes
# de_genes <- read_tsv("../TP/liste_genes.tsv")
DE_result <- read_tsv("../TP/Gene_DE_DIVA.tsv")


de_genes <- DE_result %>% filter(p.adj < 0.05,logFC>0.5)
# Convert gene IDs to Entrez IDs using org.Hs.eg.db
# de_genes_entrez <- bitr(de_genes$gene_name, fromType = "SYMBOL", toType = "ENTREZID", OrgDb = org.Hs.eg.db)
# go_enrichment <- enrichGO(de_genes_entrez$ENTREZID, OrgDb = org.Hs.eg.db, keyType = "ENTREZID", ont = "BP")
go_enrichment <- enrichGO(de_genes$rowname, OrgDb = org.Hs.eg.db, keyType = "ENSEMBL", ont = "BP",universe = DE_result$rowname)

go_enrichment <- setReadable(go_enrichment, OrgDb = org.Hs.eg.db, keyType="ENSEMBL")

my_go <- go_enrichment %>% as_tibble() %>% arrange(desc(p.adjust)) %>% filter(ID == "GO:0030155")


d <- data.frame(gene.in.interest=c(33, 206),gene.not.interest=c(447, 10239))
row.names(d) <- c("In_category", "not_in_category")


phyper(33-1,206,447-33,10239, lower.tail=FALSE)

args.df <- data.frame(numWdrawn=k-1, ## White balls drawn
                          numW=M,        ## White balls
                          numB=N-M,      ## Black balls
                          numDrawn=n)    ## balls drawn


```



## Methods

-   Fisher's exact test calculates the probability of observing a set of genes in a pathway by chance, based on a contingency table of gene counts.
-   Gene Set Enrichment Analysis (GSEA) compares the distribution of a gene set's expression values to the distribution of all gene expression values in a dataset to determine whether the gene set is overrepresented.
-   Other methods include the hypergeometric test, Kolmogorov-Smirnov test, Wilcoxon rank-sum test, Gene Set Variation Analysis (GSVA), and CAMERA.

## Tools and Databases

-   Ensembl BioMart, AnnotationHub, and the DAVID database are available for obtaining gene metadata.
-   R is a popular programming language for performing gene enrichment analysis.
-   R packages include ClusterProfiler, DOSE, and fgsea.

## Conclusion

-   Gene enrichment analysis is a powerful method for identifying biological pathways or functions that are overrepresented in a set of genes of interest.
-   With the availability of tools and databases, as well as R packages for performing analysis, gene enrichment analysis is becoming increasingly accessible to researchers in various fields.

# Gene Enrichment Analysis

## Introduction

-   Brief overview of gene enrichment analysis
-   Importance of identifying enriched biological pathways

## Gene Ontology

- Gene Ontology (GO) is a widely used standardized vocabulary for describing gene function and localization across different organisms.
- GO categories classify genes into three main categories: molecular function, biological process, and cellular component, based on their functions and locations within a cell or organism.
- Each GO term is a specific functional annotation that describes the activity or role of a gene in a biological process or pathway. For example, "cellular respiration" is a GO term that describes the process of converting nutrients into energy in the form of ATP within a cell.


## Gene Set Enrichment Analysis (GSEA)

- GSEA is a statistical method for analyzing gene expression data that aims to identify sets of genes or pathways that are significantly differentially expressed between two or more biological conditions.
- GSEA does not rely on predefined gene sets, but rather uses all genes in the dataset and ranks them based on their differential expression between the conditions. The method then tests whether genes in a given set are more likely to be located at the top or bottom of the ranked list.
- GSEA can provide insights into the biological processes or pathways that are involved in a specific disease or phenotype, and can help identify potential therapeutic targets.


They may consist in expression levels [see Barbie et al. (2009)], p-values, correlations, fold-changes (FC), t-statistics, signal-to-noise ratios, etc.

## Fisher Exact Test

- The Fisher Exact Test is a statistical method used to determine whether there is a significant association between two categorical variables, such as the presence or absence of a particular gene and a particular phenotype or disease.
- In the context of gene enrichment analysis, the Fisher Exact Test is used to determine whether there is a significant association between a set of genes and a particular biological pathway or process.
- The Fisher Exact Test can provide valuable information about the biological processes or pathways that are involved in a particular disease or phenotype, and can help identify potential drug targets or therapeutic interventions.

## Weighted Kolmogorov-Smirnov Statistics

-   Definition and principle of WKSS
-   Advantages over other methods
-   Example of WKSS results

## Tools for Gene Enrichment Analysis

-   R packages:
    -   gprofiler2
    -   clusterProfiler
    -   GOstats
-   Online tools:
    -   Enrichr
    -   DAVID
    -   WebGestalt

## Using gprofiler2

-   Example of gene enrichment analysis using gprofiler2
-   Input: list of genes
-   Output: enriched GO terms and KEGG pathways

## Conclusion

-   Summary of gene enrichment analysis methods and tools
-   Importance of carefully selecting appropriate methods and tools
-   Future directions in gene enrichment analysis

## References

-   List of relevant articles and resources used in the presentation.


Sure, I can provide more details on the parts I wrote:

### Gene Ontology
- Gene Ontology (GO) is a widely used standardized vocabulary for describing gene function and localization across different organisms.
- GO categories classify genes into three main categories: molecular function, biological process, and cellular component, based on their functions and locations within a cell or organism.
- Each GO term is a specific functional annotation that describes the activity or role of a gene in a biological process or pathway. For example, "cellular respiration" is a GO term that describes the process of converting nutrients into energy in the form of ATP within a cell.

### Gene Set Enrichment Analysis (GSEA)
- GSEA is a statistical method for analyzing gene expression data that aims to identify sets of genes or pathways that are significantly differentially expressed between two or more biological conditions.
- GSEA does not rely on predefined gene sets, but rather uses all genes in the dataset and ranks them based on their differential expression between the conditions. The method then tests whether genes in a given set are more likely to be located at the top or bottom of the ranked list.
- GSEA can provide insights into the biological processes or pathways that are involved in a specific disease or phenotype, and can help identify potential therapeutic targets.

### Fisher Exact Test
- The Fisher Exact Test is a statistical method used to determine whether there is a significant association between two categorical variables, such as the presence or absence of a particular gene and a particular phenotype or disease.
- In the context of gene enrichment analysis, the Fisher Exact Test is used to determine whether there is a significant association between a set of genes and a particular biological pathway or process.
- The Fisher Exact Test can provide valuable information about the biological processes or pathways that are involved in a particular disease or phenotype, and can help identify potential drug targets or therapeutic interventions.

### Weighted Kolmogorov-Smirnov Statistics
- The Weighted Kolmogorov-Smirnov Statistics (WKSS) is a method for measuring the similarity between two sets of ranked genes, such as those obtained from a GSEA analysis.
- WKSS takes into account the weights or scores assigned to each gene in the ranked list, and can provide a more accurate measure of similarity between two sets of genes compared to other methods that only consider the rankings.
- WKSS can be useful for identifying the biological pathways or processes that are common between different gene expression datasets or between different stages of a disease.

I hope this provides more clarity on the topics covered in the gene enrichment analysis presentation template.

