<!DOCTYPE html>
<html lang="en"><head>
<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/tabby.min.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.2.335">

  <meta name="author" content="Vincent ROCHER &amp; Juliette COOKE">
  <title>Analyse fonctionnelle de gènes et métabolites</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
  </style>
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/theme/quarto.css" id="theme">
  <link href="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-captioned.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-captioned) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-captioned.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-captioned .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-captioned .callout-caption  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-captioned.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-captioned.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-caption {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-caption {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-caption {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-captioned .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-captioned .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-captioned) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-caption {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-caption {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-caption {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-caption {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-caption {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
  
  <link href="index_files/libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet">
  <script src="index_files/libs/pagedtable-1.1/js/pagedtable.js"></script>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">Analyse fonctionnelle de gènes et métabolites</h1>

<div class="quarto-title-authors">
<div class="quarto-title-author">
<div class="quarto-title-author-name">
Vincent ROCHER &amp; Juliette COOKE 
</div>
</div>
</div>

</section>
<section class="slide level2">

<div class="cell">

</div>
</section>
<section>
<section id="gene-enrichment-analysis" class="title-slide slide level1 center">
<h1>Gene Enrichment Analysis</h1>

</section>
<section id="introduction" class="slide level2">
<h2>Introduction</h2>
<p><strong>Definition</strong>: Gene enrichment analysis identifies biological pathways or functions that are overrepresented in a set of genes of interest.</p>
<p>The interpretation of the list of differentially expressed genes can be challenging due to two scenarios:</p>
<ol type="1">
<li><p>The list is so long that it becomes cumbersome and time-consuming to analyze and interpret each gene individually.</p></li>
<li><p>The list is very short, and some genes may have low p-values but not low enough to meet the given threshold for significance.</p></li>
</ol>
</section>
<section id="approches" class="slide level2">
<h2>Approches</h2>
<p>To derive biological meaning from the results of differential expression analysis, there are several approaches available. Two commonly used methods are:</p>
<ol type="1">
<li><p><strong>Over Representation Analysis (ORA)</strong>: This method determines whether the differentially expressed genes are enriched in specific pathways or ontological groups. It assesses whether the observed number of genes in a particular pathway or gene ontology term is higher than what would be expected by chance.</p></li>
<li><p><strong>Gene Set Enrichment Analysis (GSEA)</strong>: GSEA evaluates whether a pre-defined set of genes (gene set) shows statistically significant differences between two or more biological conditions. Instead of focusing on individual genes, GSEA considers the collective behavior of genes within a gene set to identify enriched biological pathways or functional categories.</p></li>
</ol>
</section>
<section id="common-sources-of-feature-sets" class="slide level2">
<h2>Common sources of feature sets</h2>
<p>KEGG pathways</p>
<p>Gene Ontologies</p>
<p>Reactome</p>
<p>Molecular Signature Database, MSigDB (GSEA)</p>
<p>Manually curated gene lists</p>
</section></section>
<section>
<section id="before-functionnal-analysis" class="title-slide slide level1 center">
<h1>Before Functionnal analysis</h1>

</section>
<section id="from-gene-symbols-to-gene-ids" class="slide level2">
<h2>From (Gene) Symbols to (Gene) IDs</h2>
<div style="font-size: 18px;">
<p>Working with gene IDs, such as <code>Ensembl IDs</code>, instead of gene symbols is important for reproducibility in gene enrichment analysis due to the following reasons:</p>
<ol type="1">
<li><p><strong>Gene Symbol Ambiguity:</strong> Gene symbols can be ambiguous, as multiple genes can have the same symbol. This can lead to confusion and errors when performing analyses, especially when using external databases or tools.</p></li>
<li><p><strong>Uniqueness:</strong> Gene IDs are unique identifiers assigned to specific genes. They provide a standardized and consistent way of referring to genes across different datasets and analyses, ensuring accuracy and reproducibility.</p></li>
<li><p><strong>Consistent Annotation:</strong> Gene IDs are linked to comprehensive and up-to-date gene annotation databases, such as Ensembl. These databases provide detailed information about gene features, genomic locations, functional annotations, and other relevant metadata.</p></li>
<li><p><strong>Cross-Species Comparisons:</strong> Gene IDs enable seamless comparisons of gene enrichment results across different species.</p></li>
<li><p><strong>Data Integration:</strong> Gene IDs facilitate data integration across multiple experiments or studies. They enable the merging of different datasets, such as gene expression data or functional annotations, based on a common identifier.</p></li>
<li><p><strong>Long-term Accessibility:</strong> Gene IDs are stable and persist over time.</p></li>
</ol>
<p>Using Gene IDs: enhances reproducibility in gene enrichment analysis by providing unique and standardized gene identifiers, enabling consistent annotation, supporting cross-species comparisons, facilitating data integration, and ensuring long-term accessibility of results.</p>
</div>
</section>
<section id="ensembl-biomart" class="slide level2">
<h2>Ensembl BioMart</h2>
<p>BioMart provides a flexible and intuitive interface to explore and retrieve genomic data from Ensembl. It allows you to filter and select specific data attributes, enabling you to extract the information that is most relevant to your research or analysis.</p>
<ol type="1">
<li><p><strong>Accessing BioMart</strong>: Go to the Ensembl website (<a href="https://www.ensembl.org" class="uri">https://www.ensembl.org</a>) and click on the “BioMart” option in the menu to access the BioMart interface.</p></li>
<li><p><strong>Choosing a Dataset</strong>: In the BioMart interface, you will see several tabs. Start by selecting the “CHOOSE DATABASE” tab. Here, you can choose the dataset you want to retrieve data from, such as the Ensembl genes, variation, or sequence databases.</p></li>
<li><p><strong>Selecting Filters</strong>: Once you’ve chosen the dataset, click on the “CHOOSE FILTERS” tab. Here, you can specify criteria to filter the data you are interested in. For example, you can filter by gene ID, chromosome location, gene biotype, or other attributes.</p></li>
<li><p><strong>Selecting Attributes</strong>: After setting your filters, switch to the “CHOOSE ATTRIBUTES” tab. Here, you can select the specific attributes or data fields you want to retrieve for the filtered data. These attributes can include gene names, chromosomal coordinates, functional annotations, or any other available information.</p></li>
<li><p><strong>Retrieving Results</strong>: Once you have chosen the desired attributes, switch to the “RESULTS” tab. Click on the “Results” button to retrieve the data based on your chosen filters and attributes. The results will be displayed in a table format.</p></li>
</ol>
</section>
<section id="biomart" class="slide level2">
<h2>BioMart</h2>
<p>Ensembl Biomart have an R api called <code>bioMart</code>:</p>
<div class="columns">
<div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="fu">require</span>(biomaRt)</span>
<span id="cb1-2"><a href="#cb1-2"></a><span class="fu">listMarts</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>               biomart                version
1 ENSEMBL_MART_ENSEMBL      Ensembl Genes 109
2   ENSEMBL_MART_MOUSE      Mouse strains 109
3     ENSEMBL_MART_SNP  Ensembl Variation 109
4 ENSEMBL_MART_FUNCGEN Ensembl Regulation 109</code></pre>
</div>
</div>
</div><div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>mart <span class="ot">&lt;-</span> biomaRt<span class="sc">::</span><span class="fu">useMart</span>(<span class="at">biomart=</span><span class="st">"ENSEMBL_MART_ENSEMBL"</span>)</span>
<span id="cb3-2"><a href="#cb3-2"></a><span class="fu">listEnsembl</span>(<span class="at">mart=</span>mart)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>        biomart                version
1         genes      Ensembl Genes 109
2 mouse_strains      Mouse strains 109
3          snps  Ensembl Variation 109
4    regulation Ensembl Regulation 109</code></pre>
</div>
</div>
</div>
</div>
<div class="fragment">
<p>Check datasets availables</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a>ensembl <span class="ot">&lt;-</span> <span class="fu">useEnsembl</span>(<span class="at">biomart =</span> <span class="st">"genes"</span>)</span>
<span id="cb5-2"><a href="#cb5-2"></a>datasets <span class="ot">&lt;-</span> <span class="fu">listDatasets</span>(ensembl)</span>
<span id="cb5-3"><a href="#cb5-3"></a>datasets <span class="sc">%&gt;%</span> <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span> rmarkdown<span class="sc">::</span><span class="fu">paged_table</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div style="font-size: 12px;">
<div class="cell">
<div class="cell-output-display">

<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["dataset"],"name":[1],"type":["I<chr>"],"align":["right"]},{"label":["description"],"name":[2],"type":["I<chr>"],"align":["right"]},{"label":["version"],"name":[3],"type":["I<chr>"],"align":["right"]}],"data":[{"1":"abrachyrhynchus_gene_ensembl","2":"Pink-footed goose genes (ASM259213v1)","3":"ASM259213v1"},{"1":"acalliptera_gene_ensembl","2":"Eastern happy genes (fAstCal1.2)","3":"fAstCal1.2"},{"1":"acarolinensis_gene_ensembl","2":"Green anole genes (AnoCar2.0v2)","3":"AnoCar2.0v2"},{"1":"acchrysaetos_gene_ensembl","2":"Golden eagle genes (bAquChr1.2)","3":"bAquChr1.2"},{"1":"acitrinellus_gene_ensembl","2":"Midas cichlid genes (Midas_v5)","3":"Midas_v5"},{"1":"amelanoleuca_gene_ensembl","2":"Giant panda genes (ASM200744v2)","3":"ASM200744v2"},{"1":"amexicanus_gene_ensembl","2":"Mexican tetra genes (Astyanax_mexicanus-2.0)","3":"Astyanax_mexicanus-2.0"},{"1":"anancymaae_gene_ensembl","2":"Ma's night monkey genes (Anan_2.0)","3":"Anan_2.0"},{"1":"aocellaris_gene_ensembl","2":"Clown anemonefish genes (AmpOce1.0)","3":"AmpOce1.0"},{"1":"apercula_gene_ensembl","2":"Orange clownfish genes (Nemo_v1)","3":"Nemo_v1"},{"1":"aplatyrhynchos_gene_ensembl","2":"Mallard genes (ASM874695v1)","3":"ASM874695v1"},{"1":"apolyacanthus_gene_ensembl","2":"Spiny chromis genes (ASM210954v1)","3":"ASM210954v1"},{"1":"applatyrhynchos_gene_ensembl","2":"Duck genes (CAU_duck1.0)","3":"CAU_duck1.0"},{"1":"atestudineus_gene_ensembl","2":"Climbing perch genes (fAnaTes1.2)","3":"fAnaTes1.2"},{"1":"bbbison_gene_ensembl","2":"American bison genes (Bison_UMD1.0)","3":"Bison_UMD1.0"},{"1":"bgrunniens_gene_ensembl","2":"Domestic yak genes (LU_Bosgru_v3.0)","3":"LU_Bosgru_v3.0"},{"1":"bihybrid_gene_ensembl","2":"Hybrid - Bos Indicus genes (UOA_Brahman_1)","3":"UOA_Brahman_1"},{"1":"bmusculus_gene_ensembl","2":"Blue whale genes (mBalMus1.v2)","3":"mBalMus1.v2"},{"1":"bmutus_gene_ensembl","2":"Wild yak genes (BosGru_v2.0)","3":"BosGru_v2.0"},{"1":"bsplendens_gene_ensembl","2":"Siamese fighting fish genes (fBetSpl5.2)","3":"fBetSpl5.2"},{"1":"btaurus_gene_ensembl","2":"Cow genes (ARS-UCD1.2)","3":"ARS-UCD1.2"},{"1":"cabingdonii_gene_ensembl","2":"Abingdon island giant tortoise genes (ASM359739v1)","3":"ASM359739v1"},{"1":"catys_gene_ensembl","2":"Sooty mangabey genes (Caty_1.0)","3":"Caty_1.0"},{"1":"cauratus_gene_ensembl","2":"Goldfish genes (ASM336829v1)","3":"ASM336829v1"},{"1":"cccarpio_gene_ensembl","2":"Common carp genes (Cypcar_WagV4.0)","3":"Cypcar_WagV4.0"},{"1":"cdromedarius_gene_ensembl","2":"Arabian camel genes (CamDro2)","3":"CamDro2"},{"1":"celegans_gene_ensembl","2":"Caenorhabditis elegans (PRJNA13758) genes (WBcel235)","3":"WBcel235"},{"1":"cgchok1gshd_gene_ensembl","2":"Chinese hamster CHOK1GS genes (CHOK1GS_HDv1)","3":"CHOK1GS_HDv1"},{"1":"cgobio_gene_ensembl","2":"Channel bull blenny genes (fCotGob3.1)","3":"fCotGob3.1"},{"1":"charengus_gene_ensembl","2":"Atlantic herring genes (Ch_v2.0.2)","3":"Ch_v2.0.2"},{"1":"chircus_gene_ensembl","2":"Goat genes (ARS1)","3":"ARS1"},{"1":"choffmanni_gene_ensembl","2":"Sloth genes (choHof1)","3":"choHof1"},{"1":"chyarkandensis_gene_ensembl","2":"Yarkand deer genes (CEY_v1)","3":"CEY_v1"},{"1":"cintestinalis_gene_ensembl","2":"C.intestinalis genes (KH)","3":"KH"},{"1":"cjacchus_gene_ensembl","2":"White-tufted-ear marmoset genes (mCalJac1.pat.X)","3":"mCalJac1.pat.X"},{"1":"cjaponica_gene_ensembl","2":"Japanese quail genes (Coturnix_japonica_2.0)","3":"Coturnix_japonica_2.0"},{"1":"clanigera_gene_ensembl","2":"Long-tailed chinchilla genes (ChiLan1.0)","3":"ChiLan1.0"},{"1":"cldingo_gene_ensembl","2":"Dingo genes (ASM325472v1)","3":"ASM325472v1"},{"1":"clfamiliaris_gene_ensembl","2":"Dog genes (ROS_Cfam_1.0)","3":"ROS_Cfam_1.0"},{"1":"clumpus_gene_ensembl","2":"Lumpfish genes (fCycLum1.pri)","3":"fCycLum1.pri"},{"1":"cmilii_gene_ensembl","2":"Elephant shark genes (Callorhinchus_milii-6.1.3)","3":"Callorhinchus_milii-6.1.3"},{"1":"cpbellii_gene_ensembl","2":"Painted turtle genes (Chrysemys_picta_bellii-3.0.3)","3":"Chrysemys_picta_bellii-3.0.3"},{"1":"cporcellus_gene_ensembl","2":"Guinea Pig genes (Cavpor3.0)","3":"Cavpor3.0"},{"1":"cporosus_gene_ensembl","2":"Australian saltwater crocodile genes (CroPor_comp1)","3":"CroPor_comp1"},{"1":"csabaeus_gene_ensembl","2":"Vervet-AGM genes (ChlSab1.1)","3":"ChlSab1.1"},{"1":"csavignyi_gene_ensembl","2":"C.savignyi genes (CSAV 2.0)","3":"CSAV 2.0"},{"1":"csemilaevis_gene_ensembl","2":"Tongue sole genes (Cse_v1.0)","3":"Cse_v1.0"},{"1":"csyrichta_gene_ensembl","2":"Tarsier genes (Tarsius_syrichta-2.0.1)","3":"Tarsius_syrichta-2.0.1"},{"1":"cvariegatus_gene_ensembl","2":"Sheepshead minnow genes (C_variegatus-1.0)","3":"C_variegatus-1.0"},{"1":"cwagneri_gene_ensembl","2":"Chacoan peccary genes (CatWag_v2_BIUU_UCD)","3":"CatWag_v2_BIUU_UCD"},{"1":"dclupeoides_gene_ensembl","2":"Denticle herring genes (fDenClu1.1)","3":"fDenClu1.1"},{"1":"dlabrax_gene_ensembl","2":"European seabass genes (dlabrax2021)","3":"dlabrax2021"},{"1":"dleucas_gene_ensembl","2":"Beluga whale genes (ASM228892v3)","3":"ASM228892v3"},{"1":"dmelanogaster_gene_ensembl","2":"Drosophila melanogaster genes (BDGP6.32)","3":"BDGP6.32"},{"1":"dnovemcinctus_gene_ensembl","2":"Armadillo genes (Dasnov3.0)","3":"Dasnov3.0"},{"1":"dordii_gene_ensembl","2":"Kangaroo rat genes (Dord_2.0)","3":"Dord_2.0"},{"1":"drerio_gene_ensembl","2":"Zebrafish genes (GRCz11)","3":"GRCz11"},{"1":"easinus_gene_ensembl","2":"Donkey genes (ASM1607732v2)","3":"ASM1607732v2"},{"1":"eburgeri_gene_ensembl","2":"Hagfish genes (Eburgeri_3.2)","3":"Eburgeri_3.2"},{"1":"ecaballus_gene_ensembl","2":"Horse genes (EquCab3.0)","3":"EquCab3.0"},{"1":"ecalabaricus_gene_ensembl","2":"Reedfish genes (fErpCal1.1)","3":"fErpCal1.1"},{"1":"eelectricus_gene_ensembl","2":"Electric eel genes (Ee_SOAP_WITH_SSPACE)","3":"Ee_SOAP_WITH_SSPACE"},{"1":"eeuropaeus_gene_ensembl","2":"Hedgehog genes (eriEur1)","3":"eriEur1"},{"1":"elucius_gene_ensembl","2":"Northern pike genes (Eluc_v4)","3":"Eluc_v4"},{"1":"etelfairi_gene_ensembl","2":"Lesser hedgehog tenrec genes (TENREC)","3":"TENREC"},{"1":"falbicollis_gene_ensembl","2":"Collared flycatcher genes (FicAlb1.5)","3":"FicAlb1.5"},{"1":"fcatus_gene_ensembl","2":"Cat genes (Felis_catus_9.0)","3":"Felis_catus_9.0"},{"1":"fheteroclitus_gene_ensembl","2":"Mummichog genes (Fundulus_heteroclitus-3.0.2)","3":"Fundulus_heteroclitus-3.0.2"},{"1":"gaculeatus_gene_ensembl","2":"Stickleback genes (BROAD S1)","3":"BROAD S1"},{"1":"gevgoodei_gene_ensembl","2":"Goodes thornscrub tortoise genes (rGopEvg1_v1.p)","3":"rGopEvg1_v1.p"},{"1":"gfortis_gene_ensembl","2":"Medium ground-finch genes (GeoFor_1.0)","3":"GeoFor_1.0"},{"1":"ggallus_gene_ensembl","2":"Chicken genes (bGalGal1.mat.broiler.GRCg7b)","3":"bGalGal1.mat.broiler.GRCg7b"},{"1":"ggorilla_gene_ensembl","2":"Gorilla genes (gorGor4)","3":"gorGor4"},{"1":"gmorhua_gene_ensembl","2":"Atlantic cod genes (gadMor3.0)","3":"gadMor3.0"},{"1":"hburtoni_gene_ensembl","2":"Burton's mouthbrooder genes (AstBur1.0)","3":"AstBur1.0"},{"1":"hcomes_gene_ensembl","2":"Tiger tail seahorse genes (H_comes_QL1_v1)","3":"H_comes_QL1_v1"},{"1":"hgfemale_gene_ensembl","2":"Naked mole-rat female genes (HetGla_female_1.0)","3":"HetGla_female_1.0"},{"1":"hhucho_gene_ensembl","2":"Huchen genes (ASM331708v1)","3":"ASM331708v1"},{"1":"hsapiens_gene_ensembl","2":"Human genes (GRCh38.p13)","3":"GRCh38.p13"},{"1":"ipunctatus_gene_ensembl","2":"Channel catfish genes (IpCoco_1.2)","3":"IpCoco_1.2"},{"1":"itridecemlineatus_gene_ensembl","2":"Squirrel genes (SpeTri2.0)","3":"SpeTri2.0"},{"1":"jjaculus_gene_ensembl","2":"Lesser Egyptian jerboa genes (JacJac1.0)","3":"JacJac1.0"},{"1":"kmarmoratus_gene_ensembl","2":"Mangrove rivulus genes (ASM164957v1)","3":"ASM164957v1"},{"1":"lafricana_gene_ensembl","2":"Elephant genes (Loxafr3.0)","3":"Loxafr3.0"},{"1":"lbergylta_gene_ensembl","2":"Ballan wrasse genes (BallGen_V1)","3":"BallGen_V1"},{"1":"lcalcarifer_gene_ensembl","2":"Barramundi perch genes (ASB_HGAPassembly_v1)","3":"ASB_HGAPassembly_v1"},{"1":"lchalumnae_gene_ensembl","2":"Coelacanth genes (LatCha1)","3":"LatCha1"},{"1":"lcrocea_gene_ensembl","2":"Large yellow croaker genes (L_crocea_2.0)","3":"L_crocea_2.0"},{"1":"llaticaudata_gene_ensembl","2":"Blue-ringed sea krait genes (latLat_1.0)","3":"latLat_1.0"},{"1":"lleishanense_gene_ensembl","2":"Leishan spiny toad genes (ASM966780v1)","3":"ASM966780v1"},{"1":"loculatus_gene_ensembl","2":"Spotted gar genes (LepOcu1)","3":"LepOcu1"},{"1":"marmatus_gene_ensembl","2":"Zig-zag eel genes (fMasArm1.2)","3":"fMasArm1.2"},{"1":"mauratus_gene_ensembl","2":"Golden Hamster genes (MesAur1.0)","3":"MesAur1.0"},{"1":"mcaroli_gene_ensembl","2":"Ryukyu mouse genes (CAROLI_EIJ_v1.1)","3":"CAROLI_EIJ_v1.1"},{"1":"mdomestica_gene_ensembl","2":"Opossum genes (ASM229v1)","3":"ASM229v1"},{"1":"mfascicularis_gene_ensembl","2":"Crab-eating macaque genes (Macaca_fascicularis_6.0)","3":"Macaca_fascicularis_6.0"},{"1":"mgallopavo_gene_ensembl","2":"Turkey genes (Turkey_5.1)","3":"Turkey_5.1"},{"1":"mleucophaeus_gene_ensembl","2":"Drill genes (Mleu.le_1.0)","3":"Mleu.le_1.0"},{"1":"mlucifugus_gene_ensembl","2":"Microbat genes (Myoluc2.0)","3":"Myoluc2.0"},{"1":"mmmarmota_gene_ensembl","2":"Alpine marmot genes (marMar2.1)","3":"marMar2.1"},{"1":"mmonoceros_gene_ensembl","2":"Narwhal genes (NGI_Narwhal_1)","3":"NGI_Narwhal_1"},{"1":"mmoschiferus_gene_ensembl","2":"Siberian musk deer genes (MosMos_v2_BIUU_UCD)","3":"MosMos_v2_BIUU_UCD"},{"1":"mmulatta_gene_ensembl","2":"Macaque genes (Mmul_10)","3":"Mmul_10"},{"1":"mmurdjan_gene_ensembl","2":"Pinecone soldierfish genes (fMyrMur1.1)","3":"fMyrMur1.1"},{"1":"mmurinus_gene_ensembl","2":"Mouse Lemur genes (Mmur_3.0)","3":"Mmur_3.0"},{"1":"mmusculus_gene_ensembl","2":"Mouse genes (GRCm39)","3":"GRCm39"},{"1":"mnemestrina_gene_ensembl","2":"Pig-tailed macaque genes (Mnem_1.0)","3":"Mnem_1.0"},{"1":"mochrogaster_gene_ensembl","2":"Prairie vole genes (MicOch1.0)","3":"MicOch1.0"},{"1":"mpahari_gene_ensembl","2":"Shrew mouse genes (PAHARI_EIJ_v1.1)","3":"PAHARI_EIJ_v1.1"},{"1":"mpfuro_gene_ensembl","2":"Ferret genes (MusPutFur1.0)","3":"MusPutFur1.0"},{"1":"mspicilegus_gene_ensembl","2":"Steppe mouse genes (MUSP714)","3":"MUSP714"},{"1":"mspretus_gene_ensembl","2":"Algerian mouse genes (SPRET_EiJ_v1)","3":"SPRET_EiJ_v1"},{"1":"mzebra_gene_ensembl","2":"Zebra mbuna genes (M_zebra_UMD2a)","3":"M_zebra_UMD2a"},{"1":"nbrichardi_gene_ensembl","2":"Lyretail cichlid genes (NeoBri1.0)","3":"NeoBri1.0"},{"1":"neugenii_gene_ensembl","2":"Wallaby genes (Meug_1.0)","3":"Meug_1.0"},{"1":"nfurzeri_gene_ensembl","2":"Turquoise killifish genes (Nfu_20140520)","3":"Nfu_20140520"},{"1":"ngalili_gene_ensembl","2":"Upper Galilee mountains blind mole rat genes (S.galili_v1.0)","3":"S.galili_v1.0"},{"1":"nleucogenys_gene_ensembl","2":"Gibbon genes (Nleu_3.0)","3":"Nleu_3.0"},{"1":"nnaja_gene_ensembl","2":"Indian cobra genes (Nana_v5)","3":"Nana_v5"},{"1":"nscutatus_gene_ensembl","2":"Mainland tiger snake genes (TS10Xv2-PRI)","3":"TS10Xv2-PRI"},{"1":"nvison_gene_ensembl","2":"American mink genes (NNQGG.v01)","3":"NNQGG.v01"},{"1":"oanatinus_gene_ensembl","2":"Platypus genes (mOrnAna1.p.v1)","3":"mOrnAna1.p.v1"},{"1":"oarambouillet_gene_ensembl","2":"Sheep genes (Oar_rambouillet_v1.0)","3":"Oar_rambouillet_v1.0"},{"1":"oaries_gene_ensembl","2":"Sheep (texel) genes (Oar_v3.1)","3":"Oar_v3.1"},{"1":"ocuniculus_gene_ensembl","2":"Rabbit genes (OryCun2.0)","3":"OryCun2.0"},{"1":"odegus_gene_ensembl","2":"Degu genes (OctDeg1.0)","3":"OctDeg1.0"},{"1":"ogarnettii_gene_ensembl","2":"Bushbaby genes (OtoGar3)","3":"OtoGar3"},{"1":"ojavanicus_gene_ensembl","2":"Javanese ricefish genes (OJAV_1.1)","3":"OJAV_1.1"},{"1":"okisutch_gene_ensembl","2":"Coho salmon genes (Okis_V2)","3":"Okis_V2"},{"1":"olatipes_gene_ensembl","2":"Japanese medaka HdrR genes (ASM223467v1)","3":"ASM223467v1"},{"1":"omelastigma_gene_ensembl","2":"Indian medaka genes (Om_v0.7.RACA)","3":"Om_v0.7.RACA"},{"1":"omykiss_gene_ensembl","2":"Rainbow trout genes (USDA_OmykA_1.1)","3":"USDA_OmykA_1.1"},{"1":"oniloticus_gene_ensembl","2":"Nile tilapia genes (O_niloticus_UMD_NMBU)","3":"O_niloticus_UMD_NMBU"},{"1":"oprinceps_gene_ensembl","2":"Pika genes (OchPri2.0-Ens)","3":"OchPri2.0-Ens"},{"1":"osinensis_gene_ensembl","2":"Chinese medaka genes (ASM858656v1)","3":"ASM858656v1"},{"1":"otshawytscha_gene_ensembl","2":"Chinook salmon genes (Otsh_v1.0)","3":"Otsh_v1.0"},{"1":"pabelii_gene_ensembl","2":"Sumatran orangutan genes (Susie_PABv2)","3":"Susie_PABv2"},{"1":"panubis_gene_ensembl","2":"Olive baboon genes (Panubis1.0)","3":"Panubis1.0"},{"1":"pcapensis_gene_ensembl","2":"Hyrax genes (proCap1)","3":"proCap1"},{"1":"pcatodon_gene_ensembl","2":"Sperm whale genes (ASM283717v2)","3":"ASM283717v2"},{"1":"pcinereus_gene_ensembl","2":"Koala genes (phaCin_unsw_v4.1)","3":"phaCin_unsw_v4.1"},{"1":"pcoquereli_gene_ensembl","2":"Coquerel's sifaka genes (Pcoq_1.0)","3":"Pcoq_1.0"},{"1":"pformosa_gene_ensembl","2":"Amazon molly genes (Poecilia_formosa-5.1.2)","3":"Poecilia_formosa-5.1.2"},{"1":"pkingsleyae_gene_ensembl","2":"Paramormyrops kingsleyae genes (PKINGS_0.1)","3":"PKINGS_0.1"},{"1":"platipinna_gene_ensembl","2":"Sailfin molly genes (P_latipinna-1.0)","3":"P_latipinna-1.0"},{"1":"pleo_gene_ensembl","2":"Lion genes (PanLeo1.0)","3":"PanLeo1.0"},{"1":"pmajor_gene_ensembl","2":"Great Tit genes (Parus_major1.1)","3":"Parus_major1.1"},{"1":"pmarinus_gene_ensembl","2":"Lamprey genes (Pmarinus_7.0)","3":"Pmarinus_7.0"},{"1":"pmbairdii_gene_ensembl","2":"Northern American deer mouse genes (HU_Pman_2.1)","3":"HU_Pman_2.1"},{"1":"pmuralis_gene_ensembl","2":"Common wall lizard genes (PodMur_1.0)","3":"PodMur_1.0"},{"1":"pnattereri_gene_ensembl","2":"Red-bellied piranha genes (Pygocentrus_nattereri-1.0.2)","3":"Pygocentrus_nattereri-1.0.2"},{"1":"pnyererei_gene_ensembl","2":"Makobe Island cichlid genes (PunNye1.0)","3":"PunNye1.0"},{"1":"ppaniscus_gene_ensembl","2":"Bonobo genes (panpan1.1)","3":"panpan1.1"},{"1":"ppardus_gene_ensembl","2":"Leopard genes (PanPar1.0)","3":"PanPar1.0"},{"1":"preticulata_gene_ensembl","2":"Guppy genes (Guppy_female_1.0_MT)","3":"Guppy_female_1.0_MT"},{"1":"psimus_gene_ensembl","2":"Greater bamboo lemur genes (Prosim_1.0)","3":"Prosim_1.0"},{"1":"psinensis_gene_ensembl","2":"Chinese softshell turtle genes (PelSin_1.0)","3":"PelSin_1.0"},{"1":"psinus_gene_ensembl","2":"Vaquita genes (mPhoSin1.pri)","3":"mPhoSin1.pri"},{"1":"ptaltaica_gene_ensembl","2":"Tiger genes (PanTig1.0)","3":"PanTig1.0"},{"1":"ptextilis_gene_ensembl","2":"Eastern brown snake genes (EBS10Xv2-PRI)","3":"EBS10Xv2-PRI"},{"1":"ptroglodytes_gene_ensembl","2":"Chimpanzee genes (Pan_tro_3.0)","3":"Pan_tro_3.0"},{"1":"pvampyrus_gene_ensembl","2":"Megabat genes (pteVam1)","3":"pteVam1"},{"1":"rbieti_gene_ensembl","2":"Black snub-nosed monkey genes (ASM169854v1)","3":"ASM169854v1"},{"1":"rferrumequinum_gene_ensembl","2":"Greater horseshoe bat genes (mRhiFer1_v1.p)","3":"mRhiFer1_v1.p"},{"1":"rnorvegicus_gene_ensembl","2":"Rat genes (mRatBN7.2)","3":"mRatBN7.2"},{"1":"rroxellana_gene_ensembl","2":"Golden snub-nosed monkey genes (Rrox_v1)","3":"Rrox_v1"},{"1":"saraneus_gene_ensembl","2":"Shrew genes (sorAra1)","3":"sorAra1"},{"1":"saurata_gene_ensembl","2":"Gilthead seabream genes (fSpaAur1.1)","3":"fSpaAur1.1"},{"1":"sbboliviensis_gene_ensembl","2":"Bolivian squirrel monkey genes (SaiBol1.0)","3":"SaiBol1.0"},{"1":"scanaria_gene_ensembl","2":"Common canary genes (SCA1)","3":"SCA1"},{"1":"scaustralis_gene_ensembl","2":"African ostrich genes (ASM69896v1)","3":"ASM69896v1"},{"1":"scerevisiae_gene_ensembl","2":"Saccharomyces cerevisiae genes (R64-1-1)","3":"R64-1-1"},{"1":"sdumerili_gene_ensembl","2":"Greater amberjack genes (Sdu_1.0)","3":"Sdu_1.0"},{"1":"sformosus_gene_ensembl","2":"Asian bonytongue genes (fSclFor1.1)","3":"fSclFor1.1"},{"1":"sgrahami_gene_ensembl","2":"Golden-line barbel genes (SAMN03320097.WGS_v1.1)","3":"SAMN03320097.WGS_v1.1"},{"1":"shabroptila_gene_ensembl","2":"Kakapo genes (bStrHab1_v1.p)","3":"bStrHab1_v1.p"},{"1":"sharrisii_gene_ensembl","2":"Tasmanian devil genes (mSarHar1.11)","3":"mSarHar1.11"},{"1":"sldorsalis_gene_ensembl","2":"Yellowtail amberjack genes (Sedor1)","3":"Sedor1"},{"1":"slucioperca_gene_ensembl","2":"Pike-perch genes (SLUC_FBN_1)","3":"SLUC_FBN_1"},{"1":"smaximus_gene_ensembl","2":"Turbot genes (ASM1334776v1)","3":"ASM1334776v1"},{"1":"smerianae_gene_ensembl","2":"Argentine black and white tegu genes (HLtupMer3)","3":"HLtupMer3"},{"1":"spartitus_gene_ensembl","2":"Bicolor damselfish genes (Stegastes_partitus-1.0.2)","3":"Stegastes_partitus-1.0.2"},{"1":"spunctatus_gene_ensembl","2":"Tuatara genes (ASM311381v1)","3":"ASM311381v1"},{"1":"ssalar_gene_ensembl","2":"Atlantic salmon genes (Ssal_v3.1)","3":"Ssal_v3.1"},{"1":"ssbamei_gene_ensembl","2":"Pig - Bamei genes (Bamei_pig_v1)","3":"Bamei_pig_v1"},{"1":"ssberkshire_gene_ensembl","2":"Pig - Berkshire genes (Berkshire_pig_v1)","3":"Berkshire_pig_v1"},{"1":"sscrofa_gene_ensembl","2":"Pig genes (Sscrofa11.1)","3":"Sscrofa11.1"},{"1":"sshampshire_gene_ensembl","2":"Pig - Hampshire genes (Hampshire_pig_v1)","3":"Hampshire_pig_v1"},{"1":"ssjinhua_gene_ensembl","2":"Pig - Jinhua genes (Jinhua_pig_v1)","3":"Jinhua_pig_v1"},{"1":"sslandrace_gene_ensembl","2":"Pig - Landrace genes (Landrace_pig_v1)","3":"Landrace_pig_v1"},{"1":"sslargewhite_gene_ensembl","2":"Pig - Largewhite genes (Large_White_v1)","3":"Large_White_v1"},{"1":"ssmeishan_gene_ensembl","2":"Pig - Meishan genes (Meishan_pig_v1)","3":"Meishan_pig_v1"},{"1":"sspietrain_gene_ensembl","2":"Pig - Pietrain genes (Pietrain_pig_v1)","3":"Pietrain_pig_v1"},{"1":"ssrongchang_gene_ensembl","2":"Pig - Rongchang genes (Rongchang_pig_v1)","3":"Rongchang_pig_v1"},{"1":"sstibetan_gene_ensembl","2":"Pig - Tibetan genes (Tibetan_Pig_v2)","3":"Tibetan_Pig_v2"},{"1":"ssusmarc_gene_ensembl","2":"Pig USMARC genes (USMARCv1.0)","3":"USMARCv1.0"},{"1":"sswuzhishan_gene_ensembl","2":"Pig - Wuzhishan genes (minipig_v1.0)","3":"minipig_v1.0"},{"1":"strutta_gene_ensembl","2":"Brown trout genes (fSalTru1.1)","3":"fSalTru1.1"},{"1":"svulgaris_gene_ensembl","2":"Eurasian red squirrel genes (mSciVul1.1)","3":"mSciVul1.1"},{"1":"tbelangeri_gene_ensembl","2":"Tree Shrew genes (tupBel1)","3":"tupBel1"},{"1":"tctriunguis_gene_ensembl","2":"Three-toed box turtle genes (T_m_triunguis-2.0)","3":"T_m_triunguis-2.0"},{"1":"tguttata_gene_ensembl","2":"Zebra finch genes (bTaeGut1_v1.p)","3":"bTaeGut1_v1.p"},{"1":"tnigroviridis_gene_ensembl","2":"Tetraodon genes (TETRAODON 8.0)","3":"TETRAODON 8.0"},{"1":"trubripes_gene_ensembl","2":"Fugu genes (fTakRub1.2)","3":"fTakRub1.2"},{"1":"ttruncatus_gene_ensembl","2":"Dolphin genes (turTru1)","3":"turTru1"},{"1":"uamericanus_gene_ensembl","2":"American black bear genes (ASM334442v1)","3":"ASM334442v1"},{"1":"umaritimus_gene_ensembl","2":"Polar bear genes (UrsMar_1.0)","3":"UrsMar_1.0"},{"1":"uparryii_gene_ensembl","2":"Arctic ground squirrel genes (ASM342692v1)","3":"ASM342692v1"},{"1":"vpacos_gene_ensembl","2":"Alpaca genes (vicPac1)","3":"vicPac1"},{"1":"vursinus_gene_ensembl","2":"Common wombat genes (bare-nosed_wombat_genome_assembly)","3":"bare-nosed_wombat_genome_assembly"},{"1":"vvulpes_gene_ensembl","2":"Red fox genes (VulVul2.2)","3":"VulVul2.2"},{"1":"xmaculatus_gene_ensembl","2":"Platyfish genes (X_maculatus-5.0-male)","3":"X_maculatus-5.0-male"},{"1":"xtropicalis_gene_ensembl","2":"Tropical clawed frog genes (UCB_Xtro_10.0)","3":"UCB_Xtro_10.0"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="biomart-1" class="slide level2">
<h2>BioMart</h2>
<h3 id="now-we-can-ask-for-the-same-thing-but-with-r">Now we can ask for the same thing but with <code>R</code> :</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a>ensembl <span class="ot">&lt;-</span> <span class="fu">useDataset</span>(<span class="at">dataset =</span> <span class="st">"hsapiens_gene_ensembl"</span>, <span class="at">mart =</span> ensembl)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<h3 id="if-the-dataset-is-known-in-advance">If the dataset is known in advance:</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a>ensembl <span class="ot">&lt;-</span> <span class="fu">useEnsembl</span>(<span class="at">biomart =</span> <span class="st">"genes"</span>, <span class="at">dataset =</span> <span class="st">"hsapiens_gene_ensembl"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="biomart-2" class="slide level2">
<h2>BioMart</h2>
<h3 id="query-in-biomart">Query in <code>biomaRt</code></h3>
<p>The <code>getBM()</code> function is the primary query function in biomaRt.</p>
<p>It has four main arguments:</p>
<ul>
<li><code>attributes</code>: is a vector of attributes that one wants to retrieve (= the output of the query).</li>
<li><code>filters</code>: is a vector of filters that one wil use as input to the query.</li>
<li><code>values</code>: a vector of values for the filters. In case multple filters are in use, the values argument requires a list of values where each position in the list corresponds to the position of the filters in the filters argument (see examples below).</li>
<li><code>mart</code>: is an object of class Mart, which is created by the useEnsembl() function.</li>
</ul>
</section>
<section id="biomart-3" class="slide level2">
<h2>BioMart</h2>
<h3 id="retrieve-attributes-and-filters-to-query-ensembl">Retrieve attributes and filters to query <code>ensembl</code>:</h3>
<div class="columns">
<div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>filters <span class="ot">=</span> <span class="fu">listFilters</span>(ensembl)</span>
<span id="cb8-2"><a href="#cb8-2"></a>filters  <span class="sc">%&gt;%</span> <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span> rmarkdown<span class="sc">::</span><span class="fu">paged_table</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">

<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["description"],"name":[2],"type":["chr"],"align":["left"]}],"data":[{"1":"chromosome_name","2":"Chromosome/scaffold name"},{"1":"start","2":"Start"},{"1":"end","2":"End"},{"1":"band_start","2":"Band Start"},{"1":"band_end","2":"Band End"},{"1":"marker_start","2":"Marker Start"},{"1":"marker_end","2":"Marker End"},{"1":"encode_region","2":"Encode region"},{"1":"strand","2":"Strand"},{"1":"chromosomal_region","2":"e.g. 1:100:10000:-1, 1:100000:200000:1"},{"1":"with_biogrid","2":"With BioGRID Interaction data, The General Repository for Interaction Datasets ID(s)"},{"1":"with_ccds","2":"With CCDS ID(s)"},{"1":"with_chembl","2":"With ChEMBL ID(s)"},{"1":"with_dbass3","2":"With DataBase of Aberrant 3' Splice Sites ID(s)"},{"1":"with_dbass5","2":"With DataBase of Aberrant 5' Splice Sites ID(s)"},{"1":"with_entrezgene_trans_name","2":"With EntrezGene transcript name ID(s)"},{"1":"with_embl","2":"With European Nucleotide Archive ID(s)"},{"1":"with_arrayexpress","2":"With Expression Atlas ID(s)"},{"1":"with_genecards","2":"With GeneCards ID(s)"},{"1":"with_go","2":"With GO ID(s)"},{"1":"with_goslim_goa","2":"With GOSlim GOA ID(s)"},{"1":"with_hgnc","2":"With HGNC Symbol ID(s)"},{"1":"with_hpa","2":"With Human Protein Atlas ID(s)"},{"1":"with_protein_id","2":"With INSDC protein ID ID(s)"},{"1":"with_ens_lrg_gene","2":"With LRG display in Ensembl gene ID(s)"},{"1":"with_ens_lrg_transcript","2":"With LRG display in Ensembl transcript ID(s)"},{"1":"with_merops","2":"With MEROPS - the Peptidase Database ID(s)"},{"1":"with_mim_gene","2":"With MIM gene ID(s)"},{"1":"with_mim_morbid","2":"With MIM morbid ID(s)"},{"1":"with_mirbase","2":"With miRBase ID(s)"},{"1":"with_mirbase_trans_name","2":"With miRBase transcript name ID(s)"},{"1":"with_entrezgene","2":"With NCBI gene (formerly Entrezgene) ID(s)"},{"1":"with_pdb","2":"With PDB ID(s)"},{"1":"with_reactome","2":"With Reactome ID(s)"},{"1":"with_reactome_gene","2":"With Reactome gene ID(s)"},{"1":"with_reactome_transcript","2":"With Reactome transcript ID(s)"},{"1":"with_refseq_mrna","2":"With RefSeq mRNA ID(s)"},{"1":"with_refseq_mrna_predicted","2":"With RefSeq mRNA predicted ID(s)"},{"1":"with_refseq_ncrna","2":"With RefSeq ncRNA ID(s)"},{"1":"with_refseq_ncrna_predicted","2":"With RefSeq ncRNA predicted ID(s)"},{"1":"with_refseq_peptide","2":"With RefSeq peptide ID(s)"},{"1":"with_refseq_peptide_predicted","2":"With RefSeq peptide predicted ID(s)"},{"1":"with_rfam","2":"With RFAM ID(s)"},{"1":"with_rfam_trans_name","2":"With RFAM transcript name ID(s)"},{"1":"with_rnacentral","2":"With RNAcentral ID(s)"},{"1":"with_hgnc_trans_name","2":"With Transcript name ID(s)"},{"1":"with_ucsc","2":"With UCSC Stable ID ID(s)"},{"1":"with_uniparc","2":"With UniParc ID(s)"},{"1":"with_uniprot_gn","2":"With UniProtKB Gene Name ID(s)"},{"1":"with_uniprot_isoform","2":"With UniProtKB isoform ID(s)"},{"1":"with_uniprotswissprot","2":"With UniProtKB/Swiss-Prot ID(s)"},{"1":"with_uniprotsptrembl","2":"With UniProtKB/TrEMBL ID(s)"},{"1":"with_wikigene","2":"With WikiGene ID(s)"},{"1":"ensembl_gene_id","2":"Gene stable ID(s) [e.g. ENSG00000000003]"},{"1":"ensembl_gene_id_version","2":"Gene stable ID(s) with version [e.g. ENSG00000000003.15]"},{"1":"ensembl_transcript_id","2":"Transcript stable ID(s) [e.g. ENST00000000233]"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID(s) with version [e.g. ENST00000000233.10]"},{"1":"ensembl_peptide_id","2":"Protein stable ID(s) [e.g. ENSP00000000233]"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID(s) with version [e.g. ENSP00000000233.5]"},{"1":"ensembl_exon_id","2":"Exon ID(s) [e.g. ENSE00000000001]"},{"1":"external_gene_name","2":"Gene Name(s) [e.g. MT-TF]"},{"1":"external_transcript_name","2":"Transcript Name(s) [e.g. MT-TF-201]"},{"1":"external_synonym","2":"Gene Synonym(s) [e.g. 0808y08y]"},{"1":"biogrid","2":"BioGRID Interaction data, The General Repository for Interaction Datasets ID(s) [e.g. 106523]"},{"1":"ccds","2":"CCDS ID(s) [e.g. CCDS10]"},{"1":"chembl","2":"ChEMBL ID(s) [e.g. CHEMBL1075092]"},{"1":"dbass3_name","2":"DataBase of Aberrant 3' Splice Sites name(s) [e.g. ABCR ]"},{"1":"dbass3_id","2":"DataBase of Aberrant 3' Splice Sites ID(s) [e.g. 1]"},{"1":"dbass5_name","2":"DataBase of Aberrant 5' Splice Sites name(s) [e.g. BRCA1]"},{"1":"dbass5_id","2":"DataBase of Aberrant 5' Splice Sites ID(s) [e.g. 382]"},{"1":"entrezgene_trans_name","2":"EntrezGene transcript name ID(s) [e.g. A2MP1-202]"},{"1":"embl","2":"European Nucleotide Archive ID(s) [e.g. A06800]"},{"1":"arrayexpress","2":"Expression Atlas ID(s) [e.g. ENSG00000000003]"},{"1":"genecards","2":"GeneCards ID(s) [e.g. 100]"},{"1":"go","2":"GO ID(s) [e.g. GO:0000002]"},{"1":"goslim_goa","2":"GOSlim GOA ID(s) [e.g. GO:0000228]"},{"1":"hgnc_id","2":"HGNC ID(s) [e.g. HGNC:100]"},{"1":"hgnc_symbol","2":"HGNC symbol(s) [e.g. A1BG]"},{"1":"hpa_accession","2":"Human Protein Atlas accession(s) [e.g. CAB000001]"},{"1":"hpa_id","2":"Human Protein Atlas ID(s) [e.g. 1]"},{"1":"protein_id","2":"INSDC protein ID(s) [e.g. AAA02489]"},{"1":"ens_lrg_gene","2":"LRG display in Ensembl gene ID(s) [e.g. LRG_1]"},{"1":"ens_lrg_transcript","2":"LRG display in Ensembl transcript ID(s) [e.g. LRG_1000t1]"},{"1":"merops","2":"MEROPS - the Peptidase Database ID(s) [e.g. A01.001]"},{"1":"mim_gene_accession","2":"MIM gene accession(s) [e.g. 100640]"},{"1":"mim_morbid_accession","2":"MIM morbid accession(s) [e.g. 100100]"},{"1":"mirbase_accession","2":"miRBase accession(s) [e.g. MI0000060]"},{"1":"mirbase_id","2":"miRBase ID(s) [e.g. hsa-let-7a-1]"},{"1":"mirbase_trans_name","2":"miRBase transcript name ID(s) [e.g. hsa-mir-1253.1-201]"},{"1":"entrezgene_accession","2":"NCBI gene (formerly Entrezgene) accession(s) [e.g. A1BG]"},{"1":"entrezgene_id","2":"NCBI gene (formerly Entrezgene) ID(s) [e.g. 1]"},{"1":"pdb","2":"PDB ID(s) [e.g. 10GS]"},{"1":"reactome","2":"Reactome ID(s) [e.g. R-HSA-1059683]"},{"1":"reactome_gene","2":"Reactome gene ID(s) [e.g. R-HSA-1059683]"},{"1":"reactome_transcript","2":"Reactome transcript ID(s) [e.g. R-HSA-1059683]"},{"1":"refseq_mrna","2":"RefSeq mRNA ID(s) [e.g. NM_000014]"},{"1":"refseq_mrna_predicted","2":"RefSeq mRNA predicted ID(s) [e.g. XM_003118719]"},{"1":"refseq_ncrna","2":"RefSeq ncRNA ID(s) [e.g. NR_000005]"},{"1":"refseq_ncrna_predicted","2":"RefSeq ncRNA predicted ID(s) [e.g. XR_001736900]"},{"1":"refseq_peptide","2":"RefSeq peptide ID(s) [e.g. NP_000005]"},{"1":"refseq_peptide_predicted","2":"RefSeq peptide predicted ID(s) [e.g. XP_003118767]"},{"1":"rfam","2":"RFAM ID(s) [e.g. RF00001]"},{"1":"rfam_trans_name","2":"RFAM transcript name ID(s) [e.g. 5S_rRNA.1-201]"},{"1":"rnacentral","2":"RNAcentral ID(s) [e.g. URS0000000055]"},{"1":"hgnc_trans_name","2":"Transcript name ID(s) [e.g. A1BG-201]"},{"1":"ucsc","2":"UCSC Stable ID(s) [e.g. uc001aak.4]"},{"1":"uniparc","2":"UniParc ID(s) [e.g. UPI000000003C]"},{"1":"uniprot_gn_symbol","2":"UniProtKB Gene Name symbol(s) [e.g. 5HT1A]"},{"1":"uniprot_gn_id","2":"UniProtKB Gene Name ID(s) [e.g. A0A023T6R1]"},{"1":"uniprot_isoform","2":"UniProtKB isoform ID(s) [e.g. A0A096LP49-1]"},{"1":"uniprotswissprot","2":"UniProtKB/Swiss-Prot ID(s) [e.g. A0A024RBG1]"},{"1":"uniprotsptrembl","2":"UniProtKB/TrEMBL ID(s) [e.g. A0A023T6R1]"},{"1":"wikigene_name","2":"WikiGene name(s) [e.g. A1BG]"},{"1":"wikigene_id","2":"WikiGene ID(s) [e.g. 1]"},{"1":"with_affy_hc_g110","2":"With AFFY HC G110 probe ID(s)"},{"1":"with_affy_hg_focus","2":"With AFFY HG Focus probe ID(s)"},{"1":"with_affy_hg_u133a_2","2":"With AFFY HG U133A 2 probe ID(s)"},{"1":"with_affy_hg_u133b","2":"With AFFY HG U133B probe ID(s)"},{"1":"with_affy_hg_u133_plus_2","2":"With AFFY HG U133 Plus 2 probe ID(s)"},{"1":"with_affy_hg_u95a","2":"With AFFY HG U95A probe ID(s)"},{"1":"with_affy_hg_u95av2","2":"With AFFY HG U95Av2 probe ID(s)"},{"1":"with_affy_hg_u95b","2":"With AFFY HG U95B probe ID(s)"},{"1":"with_affy_hg_u95c","2":"With AFFY HG U95C probe ID(s)"},{"1":"with_affy_hg_u95d","2":"With AFFY HG U95D probe ID(s)"},{"1":"with_affy_hg_u95e","2":"With AFFY HG U95E probe ID(s)"},{"1":"with_affy_hta_2_0","2":"With AFFY HTA 2 0 probe ID(s)"},{"1":"with_affy_ht_hg_u133_plus_pm","2":"With AFFY HT HG U133 Plus PM probe ID(s)"},{"1":"with_affy_huex_1_0_st_v2","2":"With AFFY HuEx 1 0 st v2 probe ID(s)"},{"1":"with_affy_hugenefl","2":"With AFFY HuGeneFL probe ID(s)"},{"1":"with_affy_hugene_1_0_st_v1","2":"With AFFY HuGene 1 0 st v1 probe ID(s)"},{"1":"with_affy_hugene_2_0_st_v1","2":"With AFFY HuGene 2 0 st v1 probe ID(s)"},{"1":"with_affy_hugene_2_1_st_v1","2":"With AFFY HuGene 2 1 st v1 probe ID(s)"},{"1":"with_affy_primeview","2":"With AFFY PrimeView probe ID(s)"},{"1":"with_affy_u133_x3p","2":"With AFFY U133 X3P probe ID(s)"},{"1":"with_agilent_cgh_44b","2":"With AGILENT CGH 44b probe ID(s)"},{"1":"with_agilent_gpl19072","2":"With AGILENT GPL19072 probe ID(s)"},{"1":"with_agilent_gpl26966","2":"With AGILENT GPL26966 probe ID(s)"},{"1":"with_agilent_gpl6848","2":"With AGILENT GPL6848 probe ID(s)"},{"1":"with_agilent_sureprint_g3_ge_8x60k","2":"With AGILENT SurePrint G3 GE 8x60k probe ID(s)"},{"1":"with_agilent_sureprint_g3_ge_8x60k_v2","2":"With AGILENT SurePrint G3 GE 8x60k v2 probe ID(s)"},{"1":"with_agilent_wholegenome","2":"With AGILENT WholeGenome probe ID(s)"},{"1":"with_agilent_wholegenome_4x44k_v1","2":"With AGILENT WholeGenome 4x44k v1 probe ID(s)"},{"1":"with_agilent_wholegenome_4x44k_v2","2":"With AGILENT WholeGenome 4x44k v2 probe ID(s)"},{"1":"with_codelink_codelink","2":"With CODELINK CODELINK probe ID(s)"},{"1":"with_illumina_humanref_8_v3","2":"With ILLUMINA HumanRef 8 V3 probe ID(s)"},{"1":"with_illumina_humanwg_6_v3","2":"With ILLUMINA HumanWG 6 V3 probe ID(s)"},{"1":"with_phalanx_onearray","2":"With PHALANX OneArray probe ID(s)"},{"1":"affy_hc_g110","2":"AFFY HC G110 probe ID(s) [e.g. 737_at]"},{"1":"affy_hg_focus","2":"AFFY HG Focus probe ID(s) [e.g. 220771_at]"},{"1":"affy_hg_u133a_2","2":"AFFY HG U133A 2 probe ID(s) [e.g. 211600_at]"},{"1":"affy_hg_u133b","2":"AFFY HG U133B probe ID(s) [e.g. 224373_s_at]"},{"1":"affy_hg_u133_plus_2","2":"AFFY HG U133 Plus 2 probe ID(s) [e.g. 1553551_s_at]"},{"1":"affy_hg_u95a","2":"AFFY HG U95A probe ID(s) [e.g. 34279_at]"},{"1":"affy_hg_u95av2","2":"AFFY HG U95Av2 probe ID(s) [e.g. 34279_at]"},{"1":"affy_hg_u95b","2":"AFFY HG U95B probe ID(s) [e.g. 49072_at]"},{"1":"affy_hg_u95c","2":"AFFY HG U95C probe ID(s) [e.g. 64410_at]"},{"1":"affy_hg_u95d","2":"AFFY HG U95D probe ID(s) [e.g. 76451_at]"},{"1":"affy_hg_u95e","2":"AFFY HG U95E probe ID(s) [e.g. 88289_at]"},{"1":"affy_hta_2_0","2":"AFFY HTA 2 0 probe ID(s) [e.g. TC07000959.hg]"},{"1":"affy_ht_hg_u133_plus_pm","2":"AFFY HT HG U133 Plus PM probe ID(s) [e.g. 1553551_PM_s_at]"},{"1":"affy_huex_1_0_st_v2","2":"AFFY HuEx 1 0 st v2 probe ID(s) [e.g. 4037576]"},{"1":"affy_hugenefl","2":"AFFY HuGeneFL probe ID(s) [e.g. Z70759_at]"},{"1":"affy_hugene_1_0_st_v1","2":"AFFY HuGene 1 0 st v1 probe ID(s) [e.g. 8137008]"},{"1":"affy_hugene_2_0_st_v1","2":"AFFY HuGene 2 0 st v1 probe ID(s) [e.g. 17100639]"},{"1":"affy_hugene_2_1_st_v1","2":"AFFY HuGene 2 1 st v1 probe ID(s) [e.g. 17100639]"},{"1":"affy_primeview","2":"AFFY PrimeView probe ID(s) [e.g. 11761514_at]"},{"1":"affy_u133_x3p","2":"AFFY U133 X3P probe ID(s) [e.g. 1553551_3p_s_at]"},{"1":"agilent_cgh_44b","2":"AGILENT CGH 44b probe ID(s) [e.g. A_14_P129616]"},{"1":"agilent_gpl19072","2":"AGILENT GPL19072 probe ID(s) [e.g. RNA143570|rRNA_9_954]"},{"1":"agilent_gpl26966","2":"AGILENT GPL26966 probe ID(s) [e.g. HMNXSV003012672]"},{"1":"agilent_gpl6848","2":"AGILENT GPL6848 probe ID(s) [e.g. A_24_P179339]"},{"1":"agilent_sureprint_g3_ge_8x60k","2":"AGILENT SurePrint G3 GE 8x60k probe ID(s) [e.g. A_24_P182122]"},{"1":"agilent_sureprint_g3_ge_8x60k_v2","2":"AGILENT SurePrint G3 GE 8x60k v2 probe ID(s) [e.g. A_24_P182122]"},{"1":"agilent_wholegenome","2":"AGILENT WholeGenome probe ID(s) [e.g. A_24_P42453]"},{"1":"agilent_wholegenome_4x44k_v1","2":"AGILENT WholeGenome 4x44k v1 probe ID(s) [e.g. A_24_P179339]"},{"1":"agilent_wholegenome_4x44k_v2","2":"AGILENT WholeGenome 4x44k v2 probe ID(s) [e.g. A_24_P182122]"},{"1":"codelink_codelink","2":"CODELINK CODELINK probe ID(s) [e.g. GE84207]"},{"1":"illumina_humanref_8_v3","2":"ILLUMINA HumanRef 8 V3 probe ID(s) [e.g. ILMN_2295987]"},{"1":"illumina_humanwg_6_v3","2":"ILLUMINA HumanWG 6 V3 probe ID(s) [e.g. ILMN_1779625]"},{"1":"phalanx_onearray","2":"PHALANX OneArray probe ID(s) [e.g. PH_hs_0033947]"},{"1":"transcript_count_greater_than","2":"Transcript count >="},{"1":"transcript_count_less_than","2":"Transcript count <="},{"1":"biotype","2":"Type"},{"1":"transcript_biotype","2":"Transcript Type"},{"1":"source","2":"Source (gene)"},{"1":"transcript_source","2":"Source (transcript)"},{"1":"transcript_tsl","2":"Transcript Support Level (TSL)"},{"1":"transcript_gencode_basic","2":"GENCODE basic annotation"},{"1":"transcript_appris","2":"APPRIS annotation"},{"1":"transcript_is_canonical","2":"Ensembl Canonical"},{"1":"mane_select","2":"MANE Select"},{"1":"mane_plus_clinical","2":"MANE Plus clinical"},{"1":"phenotype_description","2":"Phenotype description"},{"1":"phenotype_source","2":"Phenotype source"},{"1":"go_parent_term","2":"Parent term accession"},{"1":"go_parent_name","2":"Parent term name"},{"1":"go_evidence_code","2":"GO Evidence code"},{"1":"with_hsapiens_paralog","2":"Paralogous Human Genes"},{"1":"with_cabingdonii_homolog","2":"Orthologous Abingdon island giant tortoise Genes"},{"1":"with_scaustralis_homolog","2":"Orthologous African ostrich Genes"},{"1":"with_mspretus_homolog","2":"Orthologous Algerian mouse Genes"},{"1":"with_vpacos_homolog","2":"Orthologous Alpaca Genes"},{"1":"with_mmmarmota_homolog","2":"Orthologous Alpine marmot Genes"},{"1":"with_pformosa_homolog","2":"Orthologous Amazon molly Genes"},{"1":"with_bbbison_homolog","2":"Orthologous American bison Genes"},{"1":"with_uamericanus_homolog","2":"Orthologous American black bear Genes"},{"1":"with_nvison_homolog","2":"Orthologous American mink Genes"},{"1":"with_cdromedarius_homolog","2":"Orthologous Arabian camel Genes"},{"1":"with_uparryii_homolog","2":"Orthologous Arctic ground squirrel Genes"},{"1":"with_smerianae_homolog","2":"Orthologous Argentine black and white tegu Genes"},{"1":"with_dnovemcinctus_homolog","2":"Orthologous Armadillo Genes"},{"1":"with_sformosus_homolog","2":"Orthologous Asian bonytongue Genes"},{"1":"with_gmorhua_homolog","2":"Orthologous Atlantic cod Genes"},{"1":"with_charengus_homolog","2":"Orthologous Atlantic herring Genes"},{"1":"with_ssalar_homolog","2":"Orthologous Atlantic salmon Genes"},{"1":"with_cporosus_homolog","2":"Orthologous Australian saltwater crocodile Genes"},{"1":"with_lbergylta_homolog","2":"Orthologous Ballan wrasse Genes"},{"1":"with_lcalcarifer_homolog","2":"Orthologous Barramundi perch Genes"},{"1":"with_dleucas_homolog","2":"Orthologous Beluga whale Genes"},{"1":"with_spartitus_homolog","2":"Orthologous Bicolor damselfish Genes"},{"1":"with_rbieti_homolog","2":"Orthologous Black snub-nosed monkey Genes"},{"1":"with_bmusculus_homolog","2":"Orthologous Blue whale Genes"},{"1":"with_llaticaudata_homolog","2":"Orthologous Blue-ringed sea krait Genes"},{"1":"with_sbboliviensis_homolog","2":"Orthologous Bolivian squirrel monkey Genes"},{"1":"with_ppaniscus_homolog","2":"Orthologous Bonobo Genes"},{"1":"with_strutta_homolog","2":"Orthologous Brown trout Genes"},{"1":"with_hburtoni_homolog","2":"Orthologous Burton's mouthbrooder Genes"},{"1":"with_ogarnettii_homolog","2":"Orthologous Bushbaby Genes"},{"1":"with_cintestinalis_homolog","2":"Orthologous C.intestinalis Genes"},{"1":"with_csavignyi_homolog","2":"Orthologous C.savignyi Genes"},{"1":"with_celegans_homolog","2":"Orthologous Caenorhabditis elegans (PRJNA13758) Genes"},{"1":"with_fcatus_homolog","2":"Orthologous Cat Genes"},{"1":"with_cwagneri_homolog","2":"Orthologous Chacoan peccary Genes"},{"1":"with_cgobio_homolog","2":"Orthologous Channel bull blenny Genes"},{"1":"with_ipunctatus_homolog","2":"Orthologous Channel catfish Genes"},{"1":"with_ggallus_homolog","2":"Orthologous Chicken Genes"},{"1":"with_ptroglodytes_homolog","2":"Orthologous Chimpanzee Genes"},{"1":"with_cgchok1gshd_homolog","2":"Orthologous Chinese hamster CHOK1GS Genes"},{"1":"with_osinensis_homolog","2":"Orthologous Chinese medaka Genes"},{"1":"with_psinensis_homolog","2":"Orthologous Chinese softshell turtle Genes"},{"1":"with_otshawytscha_homolog","2":"Orthologous Chinook salmon Genes"},{"1":"with_atestudineus_homolog","2":"Orthologous Climbing perch Genes"},{"1":"with_aocellaris_homolog","2":"Orthologous Clown anemonefish Genes"},{"1":"with_lchalumnae_homolog","2":"Orthologous Coelacanth Genes"},{"1":"with_okisutch_homolog","2":"Orthologous Coho salmon Genes"},{"1":"with_falbicollis_homolog","2":"Orthologous Collared flycatcher Genes"},{"1":"with_scanaria_homolog","2":"Orthologous Common canary Genes"},{"1":"with_cccarpio_homolog","2":"Orthologous Common carp Genes"},{"1":"with_pmuralis_homolog","2":"Orthologous Common wall lizard Genes"},{"1":"with_vursinus_homolog","2":"Orthologous Common wombat Genes"},{"1":"with_pcoquereli_homolog","2":"Orthologous Coquerel's sifaka Genes"},{"1":"with_btaurus_homolog","2":"Orthologous Cow Genes"},{"1":"with_mfascicularis_homolog","2":"Orthologous Crab-eating macaque Genes"},{"1":"with_odegus_homolog","2":"Orthologous Degu Genes"},{"1":"with_dclupeoides_homolog","2":"Orthologous Denticle herring Genes"},{"1":"with_cldingo_homolog","2":"Orthologous Dingo Genes"},{"1":"with_clfamiliaris_homolog","2":"Orthologous Dog Genes"},{"1":"with_ttruncatus_homolog","2":"Orthologous Dolphin Genes"},{"1":"with_bgrunniens_homolog","2":"Orthologous Domestic yak Genes"},{"1":"with_easinus_homolog","2":"Orthologous Donkey Genes"},{"1":"with_mleucophaeus_homolog","2":"Orthologous Drill Genes"},{"1":"with_dmelanogaster_homolog","2":"Orthologous Drosophila melanogaster Genes"},{"1":"with_applatyrhynchos_homolog","2":"Orthologous Duck Genes"},{"1":"with_ptextilis_homolog","2":"Orthologous Eastern brown snake Genes"},{"1":"with_acalliptera_homolog","2":"Orthologous Eastern happy Genes"},{"1":"with_eelectricus_homolog","2":"Orthologous Electric eel Genes"},{"1":"with_lafricana_homolog","2":"Orthologous Elephant Genes"},{"1":"with_cmilii_homolog","2":"Orthologous Elephant shark Genes"},{"1":"with_svulgaris_homolog","2":"Orthologous Eurasian red squirrel Genes"},{"1":"with_dlabrax_homolog","2":"Orthologous European seabass Genes"},{"1":"with_mpfuro_homolog","2":"Orthologous Ferret Genes"},{"1":"with_trubripes_homolog","2":"Orthologous Fugu Genes"},{"1":"with_amelanoleuca_homolog","2":"Orthologous Giant panda Genes"},{"1":"with_nleucogenys_homolog","2":"Orthologous Gibbon Genes"},{"1":"with_saurata_homolog","2":"Orthologous Gilthead seabream Genes"},{"1":"with_chircus_homolog","2":"Orthologous Goat Genes"},{"1":"with_mauratus_homolog","2":"Orthologous Golden Hamster Genes"},{"1":"with_acchrysaetos_homolog","2":"Orthologous Golden eagle Genes"},{"1":"with_rroxellana_homolog","2":"Orthologous Golden snub-nosed monkey Genes"},{"1":"with_sgrahami_homolog","2":"Orthologous Golden-line barbel Genes"},{"1":"with_cauratus_homolog","2":"Orthologous Goldfish Genes"},{"1":"with_gevgoodei_homolog","2":"Orthologous Goodes thornscrub tortoise Genes"},{"1":"with_ggorilla_homolog","2":"Orthologous Gorilla Genes"},{"1":"with_pmajor_homolog","2":"Orthologous Great Tit Genes"},{"1":"with_sdumerili_homolog","2":"Orthologous Greater amberjack Genes"},{"1":"with_psimus_homolog","2":"Orthologous Greater bamboo lemur Genes"},{"1":"with_rferrumequinum_homolog","2":"Orthologous Greater horseshoe bat Genes"},{"1":"with_acarolinensis_homolog","2":"Orthologous Green anole Genes"},{"1":"with_cporcellus_homolog","2":"Orthologous Guinea Pig Genes"},{"1":"with_preticulata_homolog","2":"Orthologous Guppy Genes"},{"1":"with_eburgeri_homolog","2":"Orthologous Hagfish Genes"},{"1":"with_eeuropaeus_homolog","2":"Orthologous Hedgehog Genes"},{"1":"with_ecaballus_homolog","2":"Orthologous Horse Genes"},{"1":"with_hhucho_homolog","2":"Orthologous Huchen Genes"},{"1":"with_bihybrid_homolog","2":"Orthologous Hybrid - Bos Indicus Genes"},{"1":"with_pcapensis_homolog","2":"Orthologous Hyrax Genes"},{"1":"with_nnaja_homolog","2":"Orthologous Indian cobra Genes"},{"1":"with_omelastigma_homolog","2":"Orthologous Indian medaka Genes"},{"1":"with_olatipes_homolog","2":"Orthologous Japanese medaka HdrR Genes"},{"1":"with_cjaponica_homolog","2":"Orthologous Japanese quail Genes"},{"1":"with_ojavanicus_homolog","2":"Orthologous Javanese ricefish Genes"},{"1":"with_shabroptila_homolog","2":"Orthologous Kakapo Genes"},{"1":"with_dordii_homolog","2":"Orthologous Kangaroo rat Genes"},{"1":"with_pcinereus_homolog","2":"Orthologous Koala Genes"},{"1":"with_pmarinus_homolog","2":"Orthologous Lamprey Genes"},{"1":"with_lcrocea_homolog","2":"Orthologous Large yellow croaker Genes"},{"1":"with_lleishanense_homolog","2":"Orthologous Leishan spiny toad Genes"},{"1":"with_ppardus_homolog","2":"Orthologous Leopard Genes"},{"1":"with_jjaculus_homolog","2":"Orthologous Lesser Egyptian jerboa Genes"},{"1":"with_etelfairi_homolog","2":"Orthologous Lesser hedgehog tenrec Genes"},{"1":"with_pleo_homolog","2":"Orthologous Lion Genes"},{"1":"with_clanigera_homolog","2":"Orthologous Long-tailed chinchilla Genes"},{"1":"with_clumpus_homolog","2":"Orthologous Lumpfish Genes"},{"1":"with_nbrichardi_homolog","2":"Orthologous Lyretail cichlid Genes"},{"1":"with_anancymaae_homolog","2":"Orthologous Ma's night monkey Genes"},{"1":"with_mmulatta_homolog","2":"Orthologous Macaque Genes"},{"1":"with_nscutatus_homolog","2":"Orthologous Mainland tiger snake Genes"},{"1":"with_pnyererei_homolog","2":"Orthologous Makobe Island cichlid Genes"},{"1":"with_kmarmoratus_homolog","2":"Orthologous Mangrove rivulus Genes"},{"1":"with_gfortis_homolog","2":"Orthologous Medium ground-finch Genes"},{"1":"with_pvampyrus_homolog","2":"Orthologous Megabat Genes"},{"1":"with_amexicanus_homolog","2":"Orthologous Mexican tetra Genes"},{"1":"with_mlucifugus_homolog","2":"Orthologous Microbat Genes"},{"1":"with_acitrinellus_homolog","2":"Orthologous Midas cichlid Genes"},{"1":"with_mmusculus_homolog","2":"Orthologous Mouse Genes"},{"1":"with_mmurinus_homolog","2":"Orthologous Mouse Lemur Genes"},{"1":"with_fheteroclitus_homolog","2":"Orthologous Mummichog Genes"},{"1":"with_hgfemale_homolog","2":"Orthologous Naked mole-rat female Genes"},{"1":"with_mmonoceros_homolog","2":"Orthologous Narwhal Genes"},{"1":"with_oniloticus_homolog","2":"Orthologous Nile tilapia Genes"},{"1":"with_pmbairdii_homolog","2":"Orthologous Northern American deer mouse Genes"},{"1":"with_elucius_homolog","2":"Orthologous Northern pike Genes"},{"1":"with_panubis_homolog","2":"Orthologous Olive baboon Genes"},{"1":"with_mdomestica_homolog","2":"Orthologous Opossum Genes"},{"1":"with_apercula_homolog","2":"Orthologous Orange clownfish Genes"},{"1":"with_cpbellii_homolog","2":"Orthologous Painted turtle Genes"},{"1":"with_pkingsleyae_homolog","2":"Orthologous Paramormyrops kingsleyae Genes"},{"1":"with_sscrofa_homolog","2":"Orthologous Pig Genes"},{"1":"with_mnemestrina_homolog","2":"Orthologous Pig-tailed macaque Genes"},{"1":"with_oprinceps_homolog","2":"Orthologous Pika Genes"},{"1":"with_slucioperca_homolog","2":"Orthologous Pike-perch Genes"},{"1":"with_mmurdjan_homolog","2":"Orthologous Pinecone soldierfish Genes"},{"1":"with_abrachyrhynchus_homolog","2":"Orthologous Pink-footed goose Genes"},{"1":"with_xmaculatus_homolog","2":"Orthologous Platyfish Genes"},{"1":"with_oanatinus_homolog","2":"Orthologous Platypus Genes"},{"1":"with_umaritimus_homolog","2":"Orthologous Polar bear Genes"},{"1":"with_mochrogaster_homolog","2":"Orthologous Prairie vole Genes"},{"1":"with_ocuniculus_homolog","2":"Orthologous Rabbit Genes"},{"1":"with_omykiss_homolog","2":"Orthologous Rainbow trout Genes"},{"1":"with_rnorvegicus_homolog","2":"Orthologous Rat Genes"},{"1":"with_vvulpes_homolog","2":"Orthologous Red fox Genes"},{"1":"with_pnattereri_homolog","2":"Orthologous Red-bellied piranha Genes"},{"1":"with_ecalabaricus_homolog","2":"Orthologous Reedfish Genes"},{"1":"with_mcaroli_homolog","2":"Orthologous Ryukyu mouse Genes"},{"1":"with_scerevisiae_homolog","2":"Orthologous Saccharomyces cerevisiae Genes"},{"1":"with_platipinna_homolog","2":"Orthologous Sailfin molly Genes"},{"1":"with_oarambouillet_homolog","2":"Orthologous Sheep Genes"},{"1":"with_cvariegatus_homolog","2":"Orthologous Sheepshead minnow Genes"},{"1":"with_saraneus_homolog","2":"Orthologous Shrew Genes"},{"1":"with_mpahari_homolog","2":"Orthologous Shrew mouse Genes"},{"1":"with_bsplendens_homolog","2":"Orthologous Siamese fighting fish Genes"},{"1":"with_mmoschiferus_homolog","2":"Orthologous Siberian musk deer Genes"},{"1":"with_choffmanni_homolog","2":"Orthologous Sloth Genes"},{"1":"with_catys_homolog","2":"Orthologous Sooty mangabey Genes"},{"1":"with_pcatodon_homolog","2":"Orthologous Sperm whale Genes"},{"1":"with_apolyacanthus_homolog","2":"Orthologous Spiny chromis Genes"},{"1":"with_loculatus_homolog","2":"Orthologous Spotted gar Genes"},{"1":"with_itridecemlineatus_homolog","2":"Orthologous Squirrel Genes"},{"1":"with_mspicilegus_homolog","2":"Orthologous Steppe mouse Genes"},{"1":"with_gaculeatus_homolog","2":"Orthologous Stickleback Genes"},{"1":"with_pabelii_homolog","2":"Orthologous Sumatran orangutan Genes"},{"1":"with_csyrichta_homolog","2":"Orthologous Tarsier Genes"},{"1":"with_sharrisii_homolog","2":"Orthologous Tasmanian devil Genes"},{"1":"with_tnigroviridis_homolog","2":"Orthologous Tetraodon Genes"},{"1":"with_tctriunguis_homolog","2":"Orthologous Three-toed box turtle Genes"},{"1":"with_ptaltaica_homolog","2":"Orthologous Tiger Genes"},{"1":"with_hcomes_homolog","2":"Orthologous Tiger tail seahorse Genes"},{"1":"with_csemilaevis_homolog","2":"Orthologous Tongue sole Genes"},{"1":"with_tbelangeri_homolog","2":"Orthologous Tree Shrew Genes"},{"1":"with_xtropicalis_homolog","2":"Orthologous Tropical clawed frog Genes"},{"1":"with_spunctatus_homolog","2":"Orthologous Tuatara Genes"},{"1":"with_smaximus_homolog","2":"Orthologous Turbot Genes"},{"1":"with_mgallopavo_homolog","2":"Orthologous Turkey Genes"},{"1":"with_nfurzeri_homolog","2":"Orthologous Turquoise killifish Genes"},{"1":"with_ngalili_homolog","2":"Orthologous Upper Galilee mountains blind mole rat Genes"},{"1":"with_psinus_homolog","2":"Orthologous Vaquita Genes"},{"1":"with_csabaeus_homolog","2":"Orthologous Vervet-AGM Genes"},{"1":"with_neugenii_homolog","2":"Orthologous Wallaby Genes"},{"1":"with_cjacchus_homolog","2":"Orthologous White-tufted-ear marmoset Genes"},{"1":"with_bmutus_homolog","2":"Orthologous Wild yak Genes"},{"1":"with_chyarkandensis_homolog","2":"Orthologous Yarkand deer Genes"},{"1":"with_sldorsalis_homolog","2":"Orthologous Yellowtail amberjack Genes"},{"1":"with_tguttata_homolog","2":"Orthologous Zebra finch Genes"},{"1":"with_mzebra_homolog","2":"Orthologous Zebra mbuna Genes"},{"1":"with_drerio_homolog","2":"Orthologous Zebrafish Genes"},{"1":"with_marmatus_homolog","2":"Orthologous Zig-zag eel Genes"},{"1":"with_interpro","2":"With Interpro ID(s)"},{"1":"with_alphafold_import","2":"With AFDB-ENSP mappings"},{"1":"with_cdd","2":"With CDD ID(s)"},{"1":"with_gene3d","2":"With Gene3D ID(s)"},{"1":"with_hamap","2":"With HAMAP ID(s)"},{"1":"with_mobidblite","2":"With MobiDB lite"},{"1":"with_ncoils","2":"With Coiled-coils (Ncoils)"},{"1":"with_hmmpanther","2":"With PANTHER ID(s)"},{"1":"with_pfam","2":"With Pfam ID(s)"},{"1":"with_pirsf","2":"With PIRSF ID(s)"},{"1":"with_prints","2":"With Prints ID(s)"},{"1":"with_scanprosite","2":"With PROSITE patterns ID(s)"},{"1":"with_pfscan","2":"With PROSITE profiles ID(s)"},{"1":"with_seg","2":"With Low complexity (Seg)"},{"1":"with_sfld","2":"With SFLD ID(s)"},{"1":"with_sifts_import","2":"With PDB-ENSP mappings"},{"1":"with_signalp","2":"With Cleavage site (Signalp)"},{"1":"with_smart","2":"With SMART ID(s)"},{"1":"with_superfamily","2":"With Superfamily ID(s)"},{"1":"with_tigrfam","2":"With TIGRFAM ID(s)"},{"1":"with_tmhmm","2":"With Transmembrane helices"},{"1":"interpro","2":"Interpro ID(s) [e.g. IPR000001]"},{"1":"cdd","2":"CDD ID(s) [e.g. cd00009]"},{"1":"gene3d","2":"Gene3D ID(s) [e.g. 1.10.10.10]"},{"1":"hamap","2":"HAMAP ID(s) [e.g. MF_00001]"},{"1":"hmmpanther","2":"PANTHER ID(s) [e.g. PTHR10003]"},{"1":"pfam","2":"Pfam ID(s) [e.g. PF00001]"},{"1":"pirsf","2":"PIRSF ID(s) [e.g. PIRSF000019]"},{"1":"prints","2":"Prints ID(s) [e.g. PR00001]"},{"1":"scanprosite","2":"PROSITE patterns ID(s) [e.g. PS00010]"},{"1":"pfscan","2":"PROSITE profiles ID(s) [e.g. PS01031]"},{"1":"sfld","2":"SFLD ID(s) [e.g. SFLDF00002]"},{"1":"smart","2":"SMART ID(s) [e.g. SM00002]"},{"1":"superfamily","2":"Superfamily ID(s) [e.g. SSF100879]"},{"1":"tigrfam","2":"TIGRFAM ID(s) [e.g. TIGR00002]"},{"1":"germ_line_variation_source","2":"limit to genes with germline variant data sources"},{"1":"somatic_variation_source","2":"limit to genes with somatic variant sources"},{"1":"with_validated_snp","2":"Variant supporting evidence"},{"1":"so_mini_parent_name","2":"Parent term name"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div><div class="column" style="width:50%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a>attributes <span class="ot">=</span> <span class="fu">listAttributes</span>(ensembl)</span>
<span id="cb9-2"><a href="#cb9-2"></a>attributes <span class="sc">%&gt;%</span> <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span> rmarkdown<span class="sc">::</span><span class="fu">paged_table</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">

<div data-pagedtable="false">
  <script data-pagedtable-source="" type="application/json">
{"columns":[{"label":["name"],"name":[1],"type":["chr"],"align":["left"]},{"label":["description"],"name":[2],"type":["chr"],"align":["left"]},{"label":["page"],"name":[3],"type":["chr"],"align":["left"]}],"data":[{"1":"ensembl_gene_id","2":"Gene stable ID","3":"feature_page"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"feature_page"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"feature_page"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"feature_page"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"feature_page"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"feature_page"},{"1":"ensembl_exon_id","2":"Exon stable ID","3":"feature_page"},{"1":"description","2":"Gene description","3":"feature_page"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"feature_page"},{"1":"start_position","2":"Gene start (bp)","3":"feature_page"},{"1":"end_position","2":"Gene end (bp)","3":"feature_page"},{"1":"strand","2":"Strand","3":"feature_page"},{"1":"band","2":"Karyotype band","3":"feature_page"},{"1":"transcript_start","2":"Transcript start (bp)","3":"feature_page"},{"1":"transcript_end","2":"Transcript end (bp)","3":"feature_page"},{"1":"transcription_start_site","2":"Transcription start site (TSS)","3":"feature_page"},{"1":"transcript_length","2":"Transcript length (including UTRs and CDS)","3":"feature_page"},{"1":"transcript_tsl","2":"Transcript support level (TSL)","3":"feature_page"},{"1":"transcript_gencode_basic","2":"GENCODE basic annotation","3":"feature_page"},{"1":"transcript_appris","2":"APPRIS annotation","3":"feature_page"},{"1":"transcript_is_canonical","2":"Ensembl Canonical","3":"feature_page"},{"1":"transcript_mane_select","2":"RefSeq match transcript (MANE Select)","3":"feature_page"},{"1":"transcript_mane_plus_clinical","2":"RefSeq match transcript (MANE Plus Clinical)","3":"feature_page"},{"1":"external_gene_name","2":"Gene name","3":"feature_page"},{"1":"external_gene_source","2":"Source of gene name","3":"feature_page"},{"1":"external_transcript_name","2":"Transcript name","3":"feature_page"},{"1":"external_transcript_source_name","2":"Source of transcript name","3":"feature_page"},{"1":"transcript_count","2":"Transcript count","3":"feature_page"},{"1":"percentage_gene_gc_content","2":"Gene % GC content","3":"feature_page"},{"1":"gene_biotype","2":"Gene type","3":"feature_page"},{"1":"transcript_biotype","2":"Transcript type","3":"feature_page"},{"1":"source","2":"Source (gene)","3":"feature_page"},{"1":"transcript_source","2":"Source (transcript)","3":"feature_page"},{"1":"version","2":"Version (gene)","3":"feature_page"},{"1":"transcript_version","2":"Version (transcript)","3":"feature_page"},{"1":"peptide_version","2":"Version (protein)","3":"feature_page"},{"1":"external_synonym","2":"Gene Synonym","3":"feature_page"},{"1":"phenotype_description","2":"Phenotype description","3":"feature_page"},{"1":"source_name","2":"Source name","3":"feature_page"},{"1":"study_external_id","2":"Study external reference","3":"feature_page"},{"1":"strain_name","2":"Strain name","3":"feature_page"},{"1":"strain_gender","2":"Strain gender","3":"feature_page"},{"1":"p_value","2":"P value","3":"feature_page"},{"1":"go_id","2":"GO term accession","3":"feature_page"},{"1":"name_1006","2":"GO term name","3":"feature_page"},{"1":"definition_1006","2":"GO term definition","3":"feature_page"},{"1":"go_linkage_type","2":"GO term evidence code","3":"feature_page"},{"1":"namespace_1003","2":"GO domain","3":"feature_page"},{"1":"goslim_goa_accession","2":"GOSlim GOA Accession(s)","3":"feature_page"},{"1":"goslim_goa_description","2":"GOSlim GOA Description","3":"feature_page"},{"1":"biogrid","2":"BioGRID Interaction data, The General Repository for Interaction Datasets ID","3":"feature_page"},{"1":"ccds","2":"CCDS ID","3":"feature_page"},{"1":"chembl","2":"ChEMBL ID","3":"feature_page"},{"1":"dbass3_name","2":"DataBase of Aberrant 3' Splice Sites name","3":"feature_page"},{"1":"dbass3_id","2":"DataBase of Aberrant 3' Splice Sites ID","3":"feature_page"},{"1":"dbass5_name","2":"DataBase of Aberrant 5' Splice Sites name","3":"feature_page"},{"1":"dbass5_id","2":"DataBase of Aberrant 5' Splice Sites ID","3":"feature_page"},{"1":"entrezgene_trans_name","2":"EntrezGene transcript name ID","3":"feature_page"},{"1":"embl","2":"European Nucleotide Archive ID","3":"feature_page"},{"1":"arrayexpress","2":"Expression Atlas ID","3":"feature_page"},{"1":"genecards","2":"GeneCards ID","3":"feature_page"},{"1":"hgnc_id","2":"HGNC ID","3":"feature_page"},{"1":"hgnc_symbol","2":"HGNC symbol","3":"feature_page"},{"1":"hpa_accession","2":"Human Protein Atlas accession","3":"feature_page"},{"1":"hpa_id","2":"Human Protein Atlas ID","3":"feature_page"},{"1":"protein_id","2":"INSDC protein ID","3":"feature_page"},{"1":"ens_lrg_gene","2":"LRG display in Ensembl gene ID","3":"feature_page"},{"1":"ens_lrg_transcript","2":"LRG display in Ensembl transcript ID","3":"feature_page"},{"1":"merops","2":"MEROPS - the Peptidase Database ID","3":"feature_page"},{"1":"mim_gene_description","2":"MIM gene description","3":"feature_page"},{"1":"mim_gene_accession","2":"MIM gene accession","3":"feature_page"},{"1":"mim_morbid_description","2":"MIM morbid description","3":"feature_page"},{"1":"mim_morbid_accession","2":"MIM morbid accession","3":"feature_page"},{"1":"mirbase_accession","2":"miRBase accession","3":"feature_page"},{"1":"mirbase_id","2":"miRBase ID","3":"feature_page"},{"1":"mirbase_trans_name","2":"miRBase transcript name ID","3":"feature_page"},{"1":"entrezgene_description","2":"NCBI gene (formerly Entrezgene) description","3":"feature_page"},{"1":"entrezgene_accession","2":"NCBI gene (formerly Entrezgene) accession","3":"feature_page"},{"1":"entrezgene_id","2":"NCBI gene (formerly Entrezgene) ID","3":"feature_page"},{"1":"pdb","2":"PDB ID","3":"feature_page"},{"1":"reactome","2":"Reactome ID","3":"feature_page"},{"1":"reactome_gene","2":"Reactome gene ID","3":"feature_page"},{"1":"reactome_transcript","2":"Reactome transcript ID","3":"feature_page"},{"1":"refseq_mrna","2":"RefSeq mRNA ID","3":"feature_page"},{"1":"refseq_mrna_predicted","2":"RefSeq mRNA predicted ID","3":"feature_page"},{"1":"refseq_ncrna","2":"RefSeq ncRNA ID","3":"feature_page"},{"1":"refseq_ncrna_predicted","2":"RefSeq ncRNA predicted ID","3":"feature_page"},{"1":"refseq_peptide","2":"RefSeq peptide ID","3":"feature_page"},{"1":"refseq_peptide_predicted","2":"RefSeq peptide predicted ID","3":"feature_page"},{"1":"rfam","2":"RFAM ID","3":"feature_page"},{"1":"rfam_trans_name","2":"RFAM transcript name ID","3":"feature_page"},{"1":"rnacentral","2":"RNAcentral ID","3":"feature_page"},{"1":"hgnc_trans_name","2":"Transcript name ID","3":"feature_page"},{"1":"ucsc","2":"UCSC Stable ID","3":"feature_page"},{"1":"uniparc","2":"UniParc ID","3":"feature_page"},{"1":"uniprot_gn_symbol","2":"UniProtKB Gene Name symbol","3":"feature_page"},{"1":"uniprot_gn_id","2":"UniProtKB Gene Name ID","3":"feature_page"},{"1":"uniprot_isoform","2":"UniProtKB isoform ID","3":"feature_page"},{"1":"uniprotswissprot","2":"UniProtKB/Swiss-Prot ID","3":"feature_page"},{"1":"uniprotsptrembl","2":"UniProtKB/TrEMBL ID","3":"feature_page"},{"1":"wikigene_description","2":"WikiGene description","3":"feature_page"},{"1":"wikigene_name","2":"WikiGene name","3":"feature_page"},{"1":"wikigene_id","2":"WikiGene ID","3":"feature_page"},{"1":"affy_hc_g110","2":"AFFY HC G110 probe","3":"feature_page"},{"1":"affy_hg_focus","2":"AFFY HG Focus probe","3":"feature_page"},{"1":"affy_hg_u133a_2","2":"AFFY HG U133A 2 probe","3":"feature_page"},{"1":"affy_hg_u133b","2":"AFFY HG U133B probe","3":"feature_page"},{"1":"affy_hg_u133_plus_2","2":"AFFY HG U133 Plus 2 probe","3":"feature_page"},{"1":"affy_hg_u95a","2":"AFFY HG U95A probe","3":"feature_page"},{"1":"affy_hg_u95av2","2":"AFFY HG U95Av2 probe","3":"feature_page"},{"1":"affy_hg_u95b","2":"AFFY HG U95B probe","3":"feature_page"},{"1":"affy_hg_u95c","2":"AFFY HG U95C probe","3":"feature_page"},{"1":"affy_hg_u95d","2":"AFFY HG U95D probe","3":"feature_page"},{"1":"affy_hg_u95e","2":"AFFY HG U95E probe","3":"feature_page"},{"1":"affy_hta_2_0","2":"AFFY HTA 2 0 probe","3":"feature_page"},{"1":"affy_ht_hg_u133_plus_pm","2":"AFFY HT HG U133 Plus PM probe","3":"feature_page"},{"1":"affy_huex_1_0_st_v2","2":"AFFY HuEx 1 0 st v2 probe","3":"feature_page"},{"1":"affy_hugenefl","2":"AFFY HuGeneFL probe","3":"feature_page"},{"1":"affy_hugene_1_0_st_v1","2":"AFFY HuGene 1 0 st v1 probe","3":"feature_page"},{"1":"affy_hugene_2_0_st_v1","2":"AFFY HuGene 2 0 st v1 probe","3":"feature_page"},{"1":"affy_hugene_2_1_st_v1","2":"AFFY HuGene 2 1 st v1 probe","3":"feature_page"},{"1":"affy_primeview","2":"AFFY PrimeView probe","3":"feature_page"},{"1":"affy_u133_x3p","2":"AFFY U133 X3P probe","3":"feature_page"},{"1":"agilent_cgh_44b","2":"AGILENT CGH 44b probe","3":"feature_page"},{"1":"agilent_gpl19072","2":"AGILENT GPL19072 probe","3":"feature_page"},{"1":"agilent_gpl26966","2":"AGILENT GPL26966 probe","3":"feature_page"},{"1":"agilent_gpl6848","2":"AGILENT GPL6848 probe","3":"feature_page"},{"1":"agilent_sureprint_g3_ge_8x60k","2":"AGILENT SurePrint G3 GE 8x60k probe","3":"feature_page"},{"1":"agilent_sureprint_g3_ge_8x60k_v2","2":"AGILENT SurePrint G3 GE 8x60k v2 probe","3":"feature_page"},{"1":"agilent_wholegenome","2":"AGILENT WholeGenome probe","3":"feature_page"},{"1":"agilent_wholegenome_4x44k_v1","2":"AGILENT WholeGenome 4x44k v1 probe","3":"feature_page"},{"1":"agilent_wholegenome_4x44k_v2","2":"AGILENT WholeGenome 4x44k v2 probe","3":"feature_page"},{"1":"codelink_codelink","2":"CODELINK CODELINK probe","3":"feature_page"},{"1":"illumina_humanref_8_v3","2":"ILLUMINA HumanRef 8 V3 probe","3":"feature_page"},{"1":"illumina_humanwg_6_v3","2":"ILLUMINA HumanWG 6 V3 probe","3":"feature_page"},{"1":"phalanx_onearray","2":"PHALANX OneArray probe","3":"feature_page"},{"1":"cdd","2":"CDD ID","3":"feature_page"},{"1":"cdd_start","2":"CDD start","3":"feature_page"},{"1":"cdd_end","2":"CDD end","3":"feature_page"},{"1":"gene3d","2":"Gene3D ID","3":"feature_page"},{"1":"gene3d_start","2":"Gene3D start","3":"feature_page"},{"1":"gene3d_end","2":"Gene3D end","3":"feature_page"},{"1":"hamap","2":"HAMAP ID","3":"feature_page"},{"1":"hamap_start","2":"HAMAP start","3":"feature_page"},{"1":"hamap_end","2":"HAMAP end","3":"feature_page"},{"1":"hmmpanther","2":"PANTHER ID","3":"feature_page"},{"1":"hmmpanther_start","2":"PANTHER start","3":"feature_page"},{"1":"hmmpanther_end","2":"PANTHER end","3":"feature_page"},{"1":"pfam","2":"Pfam ID","3":"feature_page"},{"1":"pfam_start","2":"Pfam start","3":"feature_page"},{"1":"pfam_end","2":"Pfam end","3":"feature_page"},{"1":"pirsf","2":"PIRSF ID","3":"feature_page"},{"1":"pirsf_start","2":"PIRSF start","3":"feature_page"},{"1":"pirsf_end","2":"PIRSF end","3":"feature_page"},{"1":"prints","2":"Prints ID","3":"feature_page"},{"1":"prints_start","2":"Prints start","3":"feature_page"},{"1":"prints_end","2":"Prints end","3":"feature_page"},{"1":"scanprosite","2":"PROSITE patterns ID","3":"feature_page"},{"1":"scanprosite_start","2":"PROSITE patterns start","3":"feature_page"},{"1":"scanprosite_end","2":"PROSITE patterns end","3":"feature_page"},{"1":"pfscan","2":"PROSITE profiles ID","3":"feature_page"},{"1":"pfscan_start","2":"PROSITE profiles start","3":"feature_page"},{"1":"pfscan_end","2":"PROSITE profiles end","3":"feature_page"},{"1":"sfld","2":"SFLD ID","3":"feature_page"},{"1":"sfld_start","2":"SFLD start","3":"feature_page"},{"1":"sfld_end","2":"SFLD end","3":"feature_page"},{"1":"smart","2":"SMART ID","3":"feature_page"},{"1":"smart_start","2":"SMART start","3":"feature_page"},{"1":"smart_end","2":"SMART end","3":"feature_page"},{"1":"superfamily","2":"Superfamily ID","3":"feature_page"},{"1":"superfamily_start","2":"Superfamily start","3":"feature_page"},{"1":"superfamily_end","2":"Superfamily end","3":"feature_page"},{"1":"tigrfam","2":"TIGRFAM ID","3":"feature_page"},{"1":"tigrfam_start","2":"TIGRFAM start","3":"feature_page"},{"1":"tigrfam_end","2":"TIGRFAM end","3":"feature_page"},{"1":"interpro","2":"Interpro ID","3":"feature_page"},{"1":"interpro_short_description","2":"Interpro Short Description","3":"feature_page"},{"1":"interpro_description","2":"Interpro Description","3":"feature_page"},{"1":"interpro_start","2":"Interpro start","3":"feature_page"},{"1":"interpro_end","2":"Interpro end","3":"feature_page"},{"1":"alphafold_import","2":"AFDB-ENSP mappings","3":"feature_page"},{"1":"alphafold_import_start","2":"AFDB-ENSP mappings start","3":"feature_page"},{"1":"alphafold_import_end","2":"AFDB-ENSP mappings end","3":"feature_page"},{"1":"mobidblite","2":"MobiDB lite","3":"feature_page"},{"1":"mobidblite_start","2":"MobiDB lite start","3":"feature_page"},{"1":"mobidblite_end","2":"MobiDB lite end","3":"feature_page"},{"1":"ncoils","2":"Coiled-coils (Ncoils)","3":"feature_page"},{"1":"ncoils_start","2":"Coiled-coils (Ncoils) start","3":"feature_page"},{"1":"ncoils_end","2":"Coiled-coils (Ncoils) end","3":"feature_page"},{"1":"seg","2":"Low complexity (Seg)","3":"feature_page"},{"1":"seg_start","2":"Low complexity (Seg) start","3":"feature_page"},{"1":"seg_end","2":"Low complexity (Seg) end","3":"feature_page"},{"1":"sifts_import","2":"PDB-ENSP mappings","3":"feature_page"},{"1":"sifts_import_start","2":"PDB-ENSP mappings start","3":"feature_page"},{"1":"sifts_import_end","2":"PDB-ENSP mappings end","3":"feature_page"},{"1":"signalp","2":"Cleavage site (Signalp)","3":"feature_page"},{"1":"signalp_start","2":"Cleavage site (Signalp) start","3":"feature_page"},{"1":"signalp_end","2":"Cleavage site (Signalp) end","3":"feature_page"},{"1":"tmhmm","2":"Transmembrane helices","3":"feature_page"},{"1":"tmhmm_start","2":"Transmembrane helices start","3":"feature_page"},{"1":"tmhmm_end","2":"Transmembrane helices end","3":"feature_page"},{"1":"ensembl_gene_id","2":"Gene stable ID","3":"structure"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"structure"},{"1":"version","2":"Version (gene)","3":"structure"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"structure"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"structure"},{"1":"transcript_version","2":"Version (transcript)","3":"structure"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"structure"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"structure"},{"1":"peptide_version","2":"Version (protein)","3":"structure"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"structure"},{"1":"start_position","2":"Gene start (bp)","3":"structure"},{"1":"end_position","2":"Gene end (bp)","3":"structure"},{"1":"transcript_start","2":"Transcript start (bp)","3":"structure"},{"1":"transcript_end","2":"Transcript end (bp)","3":"structure"},{"1":"transcription_start_site","2":"Transcription start site (TSS)","3":"structure"},{"1":"transcript_length","2":"Transcript length (including UTRs and CDS)","3":"structure"},{"1":"strand","2":"Strand","3":"structure"},{"1":"external_gene_name","2":"Gene name","3":"structure"},{"1":"external_gene_source","2":"Source of gene name","3":"structure"},{"1":"5_utr_start","2":"5' UTR start","3":"structure"},{"1":"5_utr_end","2":"5' UTR end","3":"structure"},{"1":"3_utr_start","2":"3' UTR start","3":"structure"},{"1":"3_utr_end","2":"3' UTR end","3":"structure"},{"1":"cds_length","2":"CDS Length","3":"structure"},{"1":"transcript_count","2":"Transcript count","3":"structure"},{"1":"description","2":"Gene description","3":"structure"},{"1":"gene_biotype","2":"Gene type","3":"structure"},{"1":"exon_chrom_start","2":"Exon region start (bp)","3":"structure"},{"1":"exon_chrom_end","2":"Exon region end (bp)","3":"structure"},{"1":"is_constitutive","2":"Constitutive exon","3":"structure"},{"1":"rank","2":"Exon rank in transcript","3":"structure"},{"1":"phase","2":"Start phase","3":"structure"},{"1":"end_phase","2":"End phase","3":"structure"},{"1":"cdna_coding_start","2":"cDNA coding start","3":"structure"},{"1":"cdna_coding_end","2":"cDNA coding end","3":"structure"},{"1":"genomic_coding_start","2":"Genomic coding start","3":"structure"},{"1":"genomic_coding_end","2":"Genomic coding end","3":"structure"},{"1":"ensembl_exon_id","2":"Exon stable ID","3":"structure"},{"1":"cds_start","2":"CDS start","3":"structure"},{"1":"cds_end","2":"CDS end","3":"structure"},{"1":"ensembl_gene_id","2":"Gene stable ID","3":"homologs"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"homologs"},{"1":"version","2":"Version (gene)","3":"homologs"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"homologs"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"homologs"},{"1":"transcript_version","2":"Version (transcript)","3":"homologs"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"homologs"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"homologs"},{"1":"peptide_version","2":"Version (protein)","3":"homologs"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"homologs"},{"1":"start_position","2":"Gene start (bp)","3":"homologs"},{"1":"end_position","2":"Gene end (bp)","3":"homologs"},{"1":"strand","2":"Strand","3":"homologs"},{"1":"band","2":"Karyotype band","3":"homologs"},{"1":"external_gene_name","2":"Gene name","3":"homologs"},{"1":"external_gene_source","2":"Source of gene name","3":"homologs"},{"1":"transcript_count","2":"Transcript count","3":"homologs"},{"1":"percentage_gene_gc_content","2":"Gene % GC content","3":"homologs"},{"1":"description","2":"Gene description","3":"homologs"},{"1":"cabingdonii_homolog_ensembl_gene","2":"Abingdon island giant tortoise gene stable ID","3":"homologs"},{"1":"cabingdonii_homolog_associated_gene_name","2":"Abingdon island giant tortoise gene name","3":"homologs"},{"1":"cabingdonii_homolog_ensembl_peptide","2":"Abingdon island giant tortoise protein or transcript stable ID","3":"homologs"},{"1":"cabingdonii_homolog_chromosome","2":"Abingdon island giant tortoise chromosome/scaffold name","3":"homologs"},{"1":"cabingdonii_homolog_chrom_start","2":"Abingdon island giant tortoise chromosome/scaffold start (bp)","3":"homologs"},{"1":"cabingdonii_homolog_chrom_end","2":"Abingdon island giant tortoise chromosome/scaffold end (bp)","3":"homologs"},{"1":"cabingdonii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cabingdonii_homolog_subtype","2":"Last common ancestor with Abingdon island giant tortoise","3":"homologs"},{"1":"cabingdonii_homolog_orthology_type","2":"Abingdon island giant tortoise homology type","3":"homologs"},{"1":"cabingdonii_homolog_perc_id","2":"%id. target Abingdon island giant tortoise gene identical to query gene","3":"homologs"},{"1":"cabingdonii_homolog_perc_id_r1","2":"%id. query gene identical to target Abingdon island giant tortoise gene","3":"homologs"},{"1":"cabingdonii_homolog_goc_score","2":"Abingdon island giant tortoise Gene-order conservation score","3":"homologs"},{"1":"cabingdonii_homolog_wga_coverage","2":"Abingdon island giant tortoise Whole-genome alignment coverage","3":"homologs"},{"1":"cabingdonii_homolog_orthology_confidence","2":"Abingdon island giant tortoise orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"scaustralis_homolog_ensembl_gene","2":"African ostrich gene stable ID","3":"homologs"},{"1":"scaustralis_homolog_associated_gene_name","2":"African ostrich gene name","3":"homologs"},{"1":"scaustralis_homolog_ensembl_peptide","2":"African ostrich protein or transcript stable ID","3":"homologs"},{"1":"scaustralis_homolog_chromosome","2":"African ostrich chromosome/scaffold name","3":"homologs"},{"1":"scaustralis_homolog_chrom_start","2":"African ostrich chromosome/scaffold start (bp)","3":"homologs"},{"1":"scaustralis_homolog_chrom_end","2":"African ostrich chromosome/scaffold end (bp)","3":"homologs"},{"1":"scaustralis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"scaustralis_homolog_subtype","2":"Last common ancestor with African ostrich","3":"homologs"},{"1":"scaustralis_homolog_orthology_type","2":"African ostrich homology type","3":"homologs"},{"1":"scaustralis_homolog_perc_id","2":"%id. target African ostrich gene identical to query gene","3":"homologs"},{"1":"scaustralis_homolog_perc_id_r1","2":"%id. query gene identical to target African ostrich gene","3":"homologs"},{"1":"scaustralis_homolog_goc_score","2":"African ostrich Gene-order conservation score","3":"homologs"},{"1":"scaustralis_homolog_wga_coverage","2":"African ostrich Whole-genome alignment coverage","3":"homologs"},{"1":"scaustralis_homolog_orthology_confidence","2":"African ostrich orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mspretus_homolog_ensembl_gene","2":"Algerian mouse gene stable ID","3":"homologs"},{"1":"mspretus_homolog_associated_gene_name","2":"Algerian mouse gene name","3":"homologs"},{"1":"mspretus_homolog_ensembl_peptide","2":"Algerian mouse protein or transcript stable ID","3":"homologs"},{"1":"mspretus_homolog_chromosome","2":"Algerian mouse chromosome/scaffold name","3":"homologs"},{"1":"mspretus_homolog_chrom_start","2":"Algerian mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"mspretus_homolog_chrom_end","2":"Algerian mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"mspretus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mspretus_homolog_subtype","2":"Last common ancestor with Algerian mouse","3":"homologs"},{"1":"mspretus_homolog_orthology_type","2":"Algerian mouse homology type","3":"homologs"},{"1":"mspretus_homolog_perc_id","2":"%id. target Algerian mouse gene identical to query gene","3":"homologs"},{"1":"mspretus_homolog_perc_id_r1","2":"%id. query gene identical to target Algerian mouse gene","3":"homologs"},{"1":"mspretus_homolog_goc_score","2":"Algerian mouse Gene-order conservation score","3":"homologs"},{"1":"mspretus_homolog_wga_coverage","2":"Algerian mouse Whole-genome alignment coverage","3":"homologs"},{"1":"mspretus_homolog_orthology_confidence","2":"Algerian mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"vpacos_homolog_ensembl_gene","2":"Alpaca gene stable ID","3":"homologs"},{"1":"vpacos_homolog_associated_gene_name","2":"Alpaca gene name","3":"homologs"},{"1":"vpacos_homolog_ensembl_peptide","2":"Alpaca protein or transcript stable ID","3":"homologs"},{"1":"vpacos_homolog_chromosome","2":"Alpaca chromosome/scaffold name","3":"homologs"},{"1":"vpacos_homolog_chrom_start","2":"Alpaca chromosome/scaffold start (bp)","3":"homologs"},{"1":"vpacos_homolog_chrom_end","2":"Alpaca chromosome/scaffold end (bp)","3":"homologs"},{"1":"vpacos_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"vpacos_homolog_subtype","2":"Last common ancestor with Alpaca","3":"homologs"},{"1":"vpacos_homolog_orthology_type","2":"Alpaca homology type","3":"homologs"},{"1":"vpacos_homolog_perc_id","2":"%id. target Alpaca gene identical to query gene","3":"homologs"},{"1":"vpacos_homolog_perc_id_r1","2":"%id. query gene identical to target Alpaca gene","3":"homologs"},{"1":"vpacos_homolog_goc_score","2":"Alpaca Gene-order conservation score","3":"homologs"},{"1":"vpacos_homolog_wga_coverage","2":"Alpaca Whole-genome alignment coverage","3":"homologs"},{"1":"vpacos_homolog_orthology_confidence","2":"Alpaca orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmmarmota_homolog_ensembl_gene","2":"Alpine marmot gene stable ID","3":"homologs"},{"1":"mmmarmota_homolog_associated_gene_name","2":"Alpine marmot gene name","3":"homologs"},{"1":"mmmarmota_homolog_ensembl_peptide","2":"Alpine marmot protein or transcript stable ID","3":"homologs"},{"1":"mmmarmota_homolog_chromosome","2":"Alpine marmot chromosome/scaffold name","3":"homologs"},{"1":"mmmarmota_homolog_chrom_start","2":"Alpine marmot chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmmarmota_homolog_chrom_end","2":"Alpine marmot chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmmarmota_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmmarmota_homolog_subtype","2":"Last common ancestor with Alpine marmot","3":"homologs"},{"1":"mmmarmota_homolog_orthology_type","2":"Alpine marmot homology type","3":"homologs"},{"1":"mmmarmota_homolog_perc_id","2":"%id. target Alpine marmot gene identical to query gene","3":"homologs"},{"1":"mmmarmota_homolog_perc_id_r1","2":"%id. query gene identical to target Alpine marmot gene","3":"homologs"},{"1":"mmmarmota_homolog_goc_score","2":"Alpine marmot Gene-order conservation score","3":"homologs"},{"1":"mmmarmota_homolog_wga_coverage","2":"Alpine marmot Whole-genome alignment coverage","3":"homologs"},{"1":"mmmarmota_homolog_orthology_confidence","2":"Alpine marmot orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pformosa_homolog_ensembl_gene","2":"Amazon molly gene stable ID","3":"homologs"},{"1":"pformosa_homolog_associated_gene_name","2":"Amazon molly gene name","3":"homologs"},{"1":"pformosa_homolog_ensembl_peptide","2":"Amazon molly protein or transcript stable ID","3":"homologs"},{"1":"pformosa_homolog_chromosome","2":"Amazon molly chromosome/scaffold name","3":"homologs"},{"1":"pformosa_homolog_chrom_start","2":"Amazon molly chromosome/scaffold start (bp)","3":"homologs"},{"1":"pformosa_homolog_chrom_end","2":"Amazon molly chromosome/scaffold end (bp)","3":"homologs"},{"1":"pformosa_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pformosa_homolog_subtype","2":"Last common ancestor with Amazon molly","3":"homologs"},{"1":"pformosa_homolog_orthology_type","2":"Amazon molly homology type","3":"homologs"},{"1":"pformosa_homolog_perc_id","2":"%id. target Amazon molly gene identical to query gene","3":"homologs"},{"1":"pformosa_homolog_perc_id_r1","2":"%id. query gene identical to target Amazon molly gene","3":"homologs"},{"1":"pformosa_homolog_goc_score","2":"Amazon molly Gene-order conservation score","3":"homologs"},{"1":"pformosa_homolog_wga_coverage","2":"Amazon molly Whole-genome alignment coverage","3":"homologs"},{"1":"pformosa_homolog_orthology_confidence","2":"Amazon molly orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bbbison_homolog_ensembl_gene","2":"American bison gene stable ID","3":"homologs"},{"1":"bbbison_homolog_associated_gene_name","2":"American bison gene name","3":"homologs"},{"1":"bbbison_homolog_ensembl_peptide","2":"American bison protein or transcript stable ID","3":"homologs"},{"1":"bbbison_homolog_chromosome","2":"American bison chromosome/scaffold name","3":"homologs"},{"1":"bbbison_homolog_chrom_start","2":"American bison chromosome/scaffold start (bp)","3":"homologs"},{"1":"bbbison_homolog_chrom_end","2":"American bison chromosome/scaffold end (bp)","3":"homologs"},{"1":"bbbison_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bbbison_homolog_subtype","2":"Last common ancestor with American bison","3":"homologs"},{"1":"bbbison_homolog_orthology_type","2":"American bison homology type","3":"homologs"},{"1":"bbbison_homolog_perc_id","2":"%id. target American bison gene identical to query gene","3":"homologs"},{"1":"bbbison_homolog_perc_id_r1","2":"%id. query gene identical to target American bison gene","3":"homologs"},{"1":"bbbison_homolog_goc_score","2":"American bison Gene-order conservation score","3":"homologs"},{"1":"bbbison_homolog_wga_coverage","2":"American bison Whole-genome alignment coverage","3":"homologs"},{"1":"bbbison_homolog_orthology_confidence","2":"American bison orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"uamericanus_homolog_ensembl_gene","2":"American black bear gene stable ID","3":"homologs"},{"1":"uamericanus_homolog_associated_gene_name","2":"American black bear gene name","3":"homologs"},{"1":"uamericanus_homolog_ensembl_peptide","2":"American black bear protein or transcript stable ID","3":"homologs"},{"1":"uamericanus_homolog_chromosome","2":"American black bear chromosome/scaffold name","3":"homologs"},{"1":"uamericanus_homolog_chrom_start","2":"American black bear chromosome/scaffold start (bp)","3":"homologs"},{"1":"uamericanus_homolog_chrom_end","2":"American black bear chromosome/scaffold end (bp)","3":"homologs"},{"1":"uamericanus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"uamericanus_homolog_subtype","2":"Last common ancestor with American black bear","3":"homologs"},{"1":"uamericanus_homolog_orthology_type","2":"American black bear homology type","3":"homologs"},{"1":"uamericanus_homolog_perc_id","2":"%id. target American black bear gene identical to query gene","3":"homologs"},{"1":"uamericanus_homolog_perc_id_r1","2":"%id. query gene identical to target American black bear gene","3":"homologs"},{"1":"uamericanus_homolog_goc_score","2":"American black bear Gene-order conservation score","3":"homologs"},{"1":"uamericanus_homolog_wga_coverage","2":"American black bear Whole-genome alignment coverage","3":"homologs"},{"1":"uamericanus_homolog_orthology_confidence","2":"American black bear orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nvison_homolog_ensembl_gene","2":"American mink gene stable ID","3":"homologs"},{"1":"nvison_homolog_associated_gene_name","2":"American mink gene name","3":"homologs"},{"1":"nvison_homolog_ensembl_peptide","2":"American mink protein or transcript stable ID","3":"homologs"},{"1":"nvison_homolog_chromosome","2":"American mink chromosome/scaffold name","3":"homologs"},{"1":"nvison_homolog_chrom_start","2":"American mink chromosome/scaffold start (bp)","3":"homologs"},{"1":"nvison_homolog_chrom_end","2":"American mink chromosome/scaffold end (bp)","3":"homologs"},{"1":"nvison_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nvison_homolog_subtype","2":"Last common ancestor with American mink","3":"homologs"},{"1":"nvison_homolog_orthology_type","2":"American mink homology type","3":"homologs"},{"1":"nvison_homolog_perc_id","2":"%id. target American mink gene identical to query gene","3":"homologs"},{"1":"nvison_homolog_perc_id_r1","2":"%id. query gene identical to target American mink gene","3":"homologs"},{"1":"nvison_homolog_goc_score","2":"American mink Gene-order conservation score","3":"homologs"},{"1":"nvison_homolog_wga_coverage","2":"American mink Whole-genome alignment coverage","3":"homologs"},{"1":"nvison_homolog_orthology_confidence","2":"American mink orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cdromedarius_homolog_ensembl_gene","2":"Arabian camel gene stable ID","3":"homologs"},{"1":"cdromedarius_homolog_associated_gene_name","2":"Arabian camel gene name","3":"homologs"},{"1":"cdromedarius_homolog_ensembl_peptide","2":"Arabian camel protein or transcript stable ID","3":"homologs"},{"1":"cdromedarius_homolog_chromosome","2":"Arabian camel chromosome/scaffold name","3":"homologs"},{"1":"cdromedarius_homolog_chrom_start","2":"Arabian camel chromosome/scaffold start (bp)","3":"homologs"},{"1":"cdromedarius_homolog_chrom_end","2":"Arabian camel chromosome/scaffold end (bp)","3":"homologs"},{"1":"cdromedarius_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cdromedarius_homolog_subtype","2":"Last common ancestor with Arabian camel","3":"homologs"},{"1":"cdromedarius_homolog_orthology_type","2":"Arabian camel homology type","3":"homologs"},{"1":"cdromedarius_homolog_perc_id","2":"%id. target Arabian camel gene identical to query gene","3":"homologs"},{"1":"cdromedarius_homolog_perc_id_r1","2":"%id. query gene identical to target Arabian camel gene","3":"homologs"},{"1":"cdromedarius_homolog_goc_score","2":"Arabian camel Gene-order conservation score","3":"homologs"},{"1":"cdromedarius_homolog_wga_coverage","2":"Arabian camel Whole-genome alignment coverage","3":"homologs"},{"1":"cdromedarius_homolog_orthology_confidence","2":"Arabian camel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"uparryii_homolog_ensembl_gene","2":"Arctic ground squirrel gene stable ID","3":"homologs"},{"1":"uparryii_homolog_associated_gene_name","2":"Arctic ground squirrel gene name","3":"homologs"},{"1":"uparryii_homolog_ensembl_peptide","2":"Arctic ground squirrel protein or transcript stable ID","3":"homologs"},{"1":"uparryii_homolog_chromosome","2":"Arctic ground squirrel chromosome/scaffold name","3":"homologs"},{"1":"uparryii_homolog_chrom_start","2":"Arctic ground squirrel chromosome/scaffold start (bp)","3":"homologs"},{"1":"uparryii_homolog_chrom_end","2":"Arctic ground squirrel chromosome/scaffold end (bp)","3":"homologs"},{"1":"uparryii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"uparryii_homolog_subtype","2":"Last common ancestor with Arctic ground squirrel","3":"homologs"},{"1":"uparryii_homolog_orthology_type","2":"Arctic ground squirrel homology type","3":"homologs"},{"1":"uparryii_homolog_perc_id","2":"%id. target Arctic ground squirrel gene identical to query gene","3":"homologs"},{"1":"uparryii_homolog_perc_id_r1","2":"%id. query gene identical to target Arctic ground squirrel gene","3":"homologs"},{"1":"uparryii_homolog_goc_score","2":"Arctic ground squirrel Gene-order conservation score","3":"homologs"},{"1":"uparryii_homolog_wga_coverage","2":"Arctic ground squirrel Whole-genome alignment coverage","3":"homologs"},{"1":"uparryii_homolog_orthology_confidence","2":"Arctic ground squirrel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"smerianae_homolog_ensembl_gene","2":"Argentine black and white tegu gene stable ID","3":"homologs"},{"1":"smerianae_homolog_associated_gene_name","2":"Argentine black and white tegu gene name","3":"homologs"},{"1":"smerianae_homolog_ensembl_peptide","2":"Argentine black and white tegu protein or transcript stable ID","3":"homologs"},{"1":"smerianae_homolog_chromosome","2":"Argentine black and white tegu chromosome/scaffold name","3":"homologs"},{"1":"smerianae_homolog_chrom_start","2":"Argentine black and white tegu chromosome/scaffold start (bp)","3":"homologs"},{"1":"smerianae_homolog_chrom_end","2":"Argentine black and white tegu chromosome/scaffold end (bp)","3":"homologs"},{"1":"smerianae_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"smerianae_homolog_subtype","2":"Last common ancestor with Argentine black and white tegu","3":"homologs"},{"1":"smerianae_homolog_orthology_type","2":"Argentine black and white tegu homology type","3":"homologs"},{"1":"smerianae_homolog_perc_id","2":"%id. target Argentine black and white tegu gene identical to query gene","3":"homologs"},{"1":"smerianae_homolog_perc_id_r1","2":"%id. query gene identical to target Argentine black and white tegu gene","3":"homologs"},{"1":"smerianae_homolog_goc_score","2":"Argentine black and white tegu Gene-order conservation score","3":"homologs"},{"1":"smerianae_homolog_wga_coverage","2":"Argentine black and white tegu Whole-genome alignment coverage","3":"homologs"},{"1":"smerianae_homolog_orthology_confidence","2":"Argentine black and white tegu orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dnovemcinctus_homolog_ensembl_gene","2":"Armadillo gene stable ID","3":"homologs"},{"1":"dnovemcinctus_homolog_associated_gene_name","2":"Armadillo gene name","3":"homologs"},{"1":"dnovemcinctus_homolog_ensembl_peptide","2":"Armadillo protein or transcript stable ID","3":"homologs"},{"1":"dnovemcinctus_homolog_chromosome","2":"Armadillo chromosome/scaffold name","3":"homologs"},{"1":"dnovemcinctus_homolog_chrom_start","2":"Armadillo chromosome/scaffold start (bp)","3":"homologs"},{"1":"dnovemcinctus_homolog_chrom_end","2":"Armadillo chromosome/scaffold end (bp)","3":"homologs"},{"1":"dnovemcinctus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dnovemcinctus_homolog_subtype","2":"Last common ancestor with Armadillo","3":"homologs"},{"1":"dnovemcinctus_homolog_orthology_type","2":"Armadillo homology type","3":"homologs"},{"1":"dnovemcinctus_homolog_perc_id","2":"%id. target Armadillo gene identical to query gene","3":"homologs"},{"1":"dnovemcinctus_homolog_perc_id_r1","2":"%id. query gene identical to target Armadillo gene","3":"homologs"},{"1":"dnovemcinctus_homolog_goc_score","2":"Armadillo Gene-order conservation score","3":"homologs"},{"1":"dnovemcinctus_homolog_wga_coverage","2":"Armadillo Whole-genome alignment coverage","3":"homologs"},{"1":"dnovemcinctus_homolog_orthology_confidence","2":"Armadillo orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sformosus_homolog_ensembl_gene","2":"Asian bonytongue gene stable ID","3":"homologs"},{"1":"sformosus_homolog_associated_gene_name","2":"Asian bonytongue gene name","3":"homologs"},{"1":"sformosus_homolog_ensembl_peptide","2":"Asian bonytongue protein or transcript stable ID","3":"homologs"},{"1":"sformosus_homolog_chromosome","2":"Asian bonytongue chromosome/scaffold name","3":"homologs"},{"1":"sformosus_homolog_chrom_start","2":"Asian bonytongue chromosome/scaffold start (bp)","3":"homologs"},{"1":"sformosus_homolog_chrom_end","2":"Asian bonytongue chromosome/scaffold end (bp)","3":"homologs"},{"1":"sformosus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sformosus_homolog_subtype","2":"Last common ancestor with Asian bonytongue","3":"homologs"},{"1":"sformosus_homolog_orthology_type","2":"Asian bonytongue homology type","3":"homologs"},{"1":"sformosus_homolog_perc_id","2":"%id. target Asian bonytongue gene identical to query gene","3":"homologs"},{"1":"sformosus_homolog_perc_id_r1","2":"%id. query gene identical to target Asian bonytongue gene","3":"homologs"},{"1":"sformosus_homolog_goc_score","2":"Asian bonytongue Gene-order conservation score","3":"homologs"},{"1":"sformosus_homolog_wga_coverage","2":"Asian bonytongue Whole-genome alignment coverage","3":"homologs"},{"1":"sformosus_homolog_orthology_confidence","2":"Asian bonytongue orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"gmorhua_homolog_ensembl_gene","2":"Atlantic cod gene stable ID","3":"homologs"},{"1":"gmorhua_homolog_associated_gene_name","2":"Atlantic cod gene name","3":"homologs"},{"1":"gmorhua_homolog_ensembl_peptide","2":"Atlantic cod protein or transcript stable ID","3":"homologs"},{"1":"gmorhua_homolog_chromosome","2":"Atlantic cod chromosome/scaffold name","3":"homologs"},{"1":"gmorhua_homolog_chrom_start","2":"Atlantic cod chromosome/scaffold start (bp)","3":"homologs"},{"1":"gmorhua_homolog_chrom_end","2":"Atlantic cod chromosome/scaffold end (bp)","3":"homologs"},{"1":"gmorhua_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"gmorhua_homolog_subtype","2":"Last common ancestor with Atlantic cod","3":"homologs"},{"1":"gmorhua_homolog_orthology_type","2":"Atlantic cod homology type","3":"homologs"},{"1":"gmorhua_homolog_perc_id","2":"%id. target Atlantic cod gene identical to query gene","3":"homologs"},{"1":"gmorhua_homolog_perc_id_r1","2":"%id. query gene identical to target Atlantic cod gene","3":"homologs"},{"1":"gmorhua_homolog_goc_score","2":"Atlantic cod Gene-order conservation score","3":"homologs"},{"1":"gmorhua_homolog_orthology_confidence","2":"Atlantic cod orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"charengus_homolog_ensembl_gene","2":"Atlantic herring gene stable ID","3":"homologs"},{"1":"charengus_homolog_associated_gene_name","2":"Atlantic herring gene name","3":"homologs"},{"1":"charengus_homolog_ensembl_peptide","2":"Atlantic herring protein or transcript stable ID","3":"homologs"},{"1":"charengus_homolog_chromosome","2":"Atlantic herring chromosome/scaffold name","3":"homologs"},{"1":"charengus_homolog_chrom_start","2":"Atlantic herring chromosome/scaffold start (bp)","3":"homologs"},{"1":"charengus_homolog_chrom_end","2":"Atlantic herring chromosome/scaffold end (bp)","3":"homologs"},{"1":"charengus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"charengus_homolog_subtype","2":"Last common ancestor with Atlantic herring","3":"homologs"},{"1":"charengus_homolog_orthology_type","2":"Atlantic herring homology type","3":"homologs"},{"1":"charengus_homolog_perc_id","2":"%id. target Atlantic herring gene identical to query gene","3":"homologs"},{"1":"charengus_homolog_perc_id_r1","2":"%id. query gene identical to target Atlantic herring gene","3":"homologs"},{"1":"charengus_homolog_goc_score","2":"Atlantic herring Gene-order conservation score","3":"homologs"},{"1":"charengus_homolog_wga_coverage","2":"Atlantic herring Whole-genome alignment coverage","3":"homologs"},{"1":"charengus_homolog_orthology_confidence","2":"Atlantic herring orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ssalar_homolog_ensembl_gene","2":"Atlantic salmon gene stable ID","3":"homologs"},{"1":"ssalar_homolog_associated_gene_name","2":"Atlantic salmon gene name","3":"homologs"},{"1":"ssalar_homolog_ensembl_peptide","2":"Atlantic salmon protein or transcript stable ID","3":"homologs"},{"1":"ssalar_homolog_chromosome","2":"Atlantic salmon chromosome/scaffold name","3":"homologs"},{"1":"ssalar_homolog_chrom_start","2":"Atlantic salmon chromosome/scaffold start (bp)","3":"homologs"},{"1":"ssalar_homolog_chrom_end","2":"Atlantic salmon chromosome/scaffold end (bp)","3":"homologs"},{"1":"ssalar_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ssalar_homolog_subtype","2":"Last common ancestor with Atlantic salmon","3":"homologs"},{"1":"ssalar_homolog_orthology_type","2":"Atlantic salmon homology type","3":"homologs"},{"1":"ssalar_homolog_perc_id","2":"%id. target Atlantic salmon gene identical to query gene","3":"homologs"},{"1":"ssalar_homolog_perc_id_r1","2":"%id. query gene identical to target Atlantic salmon gene","3":"homologs"},{"1":"ssalar_homolog_goc_score","2":"Atlantic salmon Gene-order conservation score","3":"homologs"},{"1":"ssalar_homolog_orthology_confidence","2":"Atlantic salmon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cporosus_homolog_ensembl_gene","2":"Australian saltwater crocodile gene stable ID","3":"homologs"},{"1":"cporosus_homolog_associated_gene_name","2":"Australian saltwater crocodile gene name","3":"homologs"},{"1":"cporosus_homolog_ensembl_peptide","2":"Australian saltwater crocodile protein or transcript stable ID","3":"homologs"},{"1":"cporosus_homolog_chromosome","2":"Australian saltwater crocodile chromosome/scaffold name","3":"homologs"},{"1":"cporosus_homolog_chrom_start","2":"Australian saltwater crocodile chromosome/scaffold start (bp)","3":"homologs"},{"1":"cporosus_homolog_chrom_end","2":"Australian saltwater crocodile chromosome/scaffold end (bp)","3":"homologs"},{"1":"cporosus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cporosus_homolog_subtype","2":"Last common ancestor with Australian saltwater crocodile","3":"homologs"},{"1":"cporosus_homolog_orthology_type","2":"Australian saltwater crocodile homology type","3":"homologs"},{"1":"cporosus_homolog_perc_id","2":"%id. target Australian saltwater crocodile gene identical to query gene","3":"homologs"},{"1":"cporosus_homolog_perc_id_r1","2":"%id. query gene identical to target Australian saltwater crocodile gene","3":"homologs"},{"1":"cporosus_homolog_goc_score","2":"Australian saltwater crocodile Gene-order conservation score","3":"homologs"},{"1":"cporosus_homolog_wga_coverage","2":"Australian saltwater crocodile Whole-genome alignment coverage","3":"homologs"},{"1":"cporosus_homolog_orthology_confidence","2":"Australian saltwater crocodile orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lbergylta_homolog_ensembl_gene","2":"Ballan wrasse gene stable ID","3":"homologs"},{"1":"lbergylta_homolog_associated_gene_name","2":"Ballan wrasse gene name","3":"homologs"},{"1":"lbergylta_homolog_ensembl_peptide","2":"Ballan wrasse protein or transcript stable ID","3":"homologs"},{"1":"lbergylta_homolog_chromosome","2":"Ballan wrasse chromosome/scaffold name","3":"homologs"},{"1":"lbergylta_homolog_chrom_start","2":"Ballan wrasse chromosome/scaffold start (bp)","3":"homologs"},{"1":"lbergylta_homolog_chrom_end","2":"Ballan wrasse chromosome/scaffold end (bp)","3":"homologs"},{"1":"lbergylta_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lbergylta_homolog_subtype","2":"Last common ancestor with Ballan wrasse","3":"homologs"},{"1":"lbergylta_homolog_orthology_type","2":"Ballan wrasse homology type","3":"homologs"},{"1":"lbergylta_homolog_perc_id","2":"%id. target Ballan wrasse gene identical to query gene","3":"homologs"},{"1":"lbergylta_homolog_perc_id_r1","2":"%id. query gene identical to target Ballan wrasse gene","3":"homologs"},{"1":"lbergylta_homolog_goc_score","2":"Ballan wrasse Gene-order conservation score","3":"homologs"},{"1":"lbergylta_homolog_wga_coverage","2":"Ballan wrasse Whole-genome alignment coverage","3":"homologs"},{"1":"lbergylta_homolog_orthology_confidence","2":"Ballan wrasse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lcalcarifer_homolog_ensembl_gene","2":"Barramundi perch gene stable ID","3":"homologs"},{"1":"lcalcarifer_homolog_associated_gene_name","2":"Barramundi perch gene name","3":"homologs"},{"1":"lcalcarifer_homolog_ensembl_peptide","2":"Barramundi perch protein or transcript stable ID","3":"homologs"},{"1":"lcalcarifer_homolog_chromosome","2":"Barramundi perch chromosome/scaffold name","3":"homologs"},{"1":"lcalcarifer_homolog_chrom_start","2":"Barramundi perch chromosome/scaffold start (bp)","3":"homologs"},{"1":"lcalcarifer_homolog_chrom_end","2":"Barramundi perch chromosome/scaffold end (bp)","3":"homologs"},{"1":"lcalcarifer_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lcalcarifer_homolog_subtype","2":"Last common ancestor with Barramundi perch","3":"homologs"},{"1":"lcalcarifer_homolog_orthology_type","2":"Barramundi perch homology type","3":"homologs"},{"1":"lcalcarifer_homolog_perc_id","2":"%id. target Barramundi perch gene identical to query gene","3":"homologs"},{"1":"lcalcarifer_homolog_perc_id_r1","2":"%id. query gene identical to target Barramundi perch gene","3":"homologs"},{"1":"lcalcarifer_homolog_goc_score","2":"Barramundi perch Gene-order conservation score","3":"homologs"},{"1":"lcalcarifer_homolog_wga_coverage","2":"Barramundi perch Whole-genome alignment coverage","3":"homologs"},{"1":"lcalcarifer_homolog_orthology_confidence","2":"Barramundi perch orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dleucas_homolog_ensembl_gene","2":"Beluga whale gene stable ID","3":"homologs"},{"1":"dleucas_homolog_associated_gene_name","2":"Beluga whale gene name","3":"homologs"},{"1":"dleucas_homolog_ensembl_peptide","2":"Beluga whale protein or transcript stable ID","3":"homologs"},{"1":"dleucas_homolog_chromosome","2":"Beluga whale chromosome/scaffold name","3":"homologs"},{"1":"dleucas_homolog_chrom_start","2":"Beluga whale chromosome/scaffold start (bp)","3":"homologs"},{"1":"dleucas_homolog_chrom_end","2":"Beluga whale chromosome/scaffold end (bp)","3":"homologs"},{"1":"dleucas_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dleucas_homolog_subtype","2":"Last common ancestor with Beluga whale","3":"homologs"},{"1":"dleucas_homolog_orthology_type","2":"Beluga whale homology type","3":"homologs"},{"1":"dleucas_homolog_perc_id","2":"%id. target Beluga whale gene identical to query gene","3":"homologs"},{"1":"dleucas_homolog_perc_id_r1","2":"%id. query gene identical to target Beluga whale gene","3":"homologs"},{"1":"dleucas_homolog_goc_score","2":"Beluga whale Gene-order conservation score","3":"homologs"},{"1":"dleucas_homolog_wga_coverage","2":"Beluga whale Whole-genome alignment coverage","3":"homologs"},{"1":"dleucas_homolog_orthology_confidence","2":"Beluga whale orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"spartitus_homolog_ensembl_gene","2":"Bicolor damselfish gene stable ID","3":"homologs"},{"1":"spartitus_homolog_associated_gene_name","2":"Bicolor damselfish gene name","3":"homologs"},{"1":"spartitus_homolog_ensembl_peptide","2":"Bicolor damselfish protein or transcript stable ID","3":"homologs"},{"1":"spartitus_homolog_chromosome","2":"Bicolor damselfish chromosome/scaffold name","3":"homologs"},{"1":"spartitus_homolog_chrom_start","2":"Bicolor damselfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"spartitus_homolog_chrom_end","2":"Bicolor damselfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"spartitus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"spartitus_homolog_subtype","2":"Last common ancestor with Bicolor damselfish","3":"homologs"},{"1":"spartitus_homolog_orthology_type","2":"Bicolor damselfish homology type","3":"homologs"},{"1":"spartitus_homolog_perc_id","2":"%id. target Bicolor damselfish gene identical to query gene","3":"homologs"},{"1":"spartitus_homolog_perc_id_r1","2":"%id. query gene identical to target Bicolor damselfish gene","3":"homologs"},{"1":"spartitus_homolog_goc_score","2":"Bicolor damselfish Gene-order conservation score","3":"homologs"},{"1":"spartitus_homolog_wga_coverage","2":"Bicolor damselfish Whole-genome alignment coverage","3":"homologs"},{"1":"spartitus_homolog_orthology_confidence","2":"Bicolor damselfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"rbieti_homolog_ensembl_gene","2":"Black snub-nosed monkey gene stable ID","3":"homologs"},{"1":"rbieti_homolog_associated_gene_name","2":"Black snub-nosed monkey gene name","3":"homologs"},{"1":"rbieti_homolog_ensembl_peptide","2":"Black snub-nosed monkey protein or transcript stable ID","3":"homologs"},{"1":"rbieti_homolog_chromosome","2":"Black snub-nosed monkey chromosome/scaffold name","3":"homologs"},{"1":"rbieti_homolog_chrom_start","2":"Black snub-nosed monkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"rbieti_homolog_chrom_end","2":"Black snub-nosed monkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"rbieti_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"rbieti_homolog_subtype","2":"Last common ancestor with Black snub-nosed monkey","3":"homologs"},{"1":"rbieti_homolog_orthology_type","2":"Black snub-nosed monkey homology type","3":"homologs"},{"1":"rbieti_homolog_perc_id","2":"%id. target Black snub-nosed monkey gene identical to query gene","3":"homologs"},{"1":"rbieti_homolog_perc_id_r1","2":"%id. query gene identical to target Black snub-nosed monkey gene","3":"homologs"},{"1":"rbieti_homolog_goc_score","2":"Black snub-nosed monkey Gene-order conservation score","3":"homologs"},{"1":"rbieti_homolog_wga_coverage","2":"Black snub-nosed monkey Whole-genome alignment coverage","3":"homologs"},{"1":"rbieti_homolog_orthology_confidence","2":"Black snub-nosed monkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bmusculus_homolog_ensembl_gene","2":"Blue whale gene stable ID","3":"homologs"},{"1":"bmusculus_homolog_associated_gene_name","2":"Blue whale gene name","3":"homologs"},{"1":"bmusculus_homolog_ensembl_peptide","2":"Blue whale protein or transcript stable ID","3":"homologs"},{"1":"bmusculus_homolog_chromosome","2":"Blue whale chromosome/scaffold name","3":"homologs"},{"1":"bmusculus_homolog_chrom_start","2":"Blue whale chromosome/scaffold start (bp)","3":"homologs"},{"1":"bmusculus_homolog_chrom_end","2":"Blue whale chromosome/scaffold end (bp)","3":"homologs"},{"1":"bmusculus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bmusculus_homolog_subtype","2":"Last common ancestor with Blue whale","3":"homologs"},{"1":"bmusculus_homolog_orthology_type","2":"Blue whale homology type","3":"homologs"},{"1":"bmusculus_homolog_perc_id","2":"%id. target Blue whale gene identical to query gene","3":"homologs"},{"1":"bmusculus_homolog_perc_id_r1","2":"%id. query gene identical to target Blue whale gene","3":"homologs"},{"1":"bmusculus_homolog_goc_score","2":"Blue whale Gene-order conservation score","3":"homologs"},{"1":"bmusculus_homolog_wga_coverage","2":"Blue whale Whole-genome alignment coverage","3":"homologs"},{"1":"bmusculus_homolog_orthology_confidence","2":"Blue whale orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"llaticaudata_homolog_ensembl_gene","2":"Blue-ringed sea krait gene stable ID","3":"homologs"},{"1":"llaticaudata_homolog_associated_gene_name","2":"Blue-ringed sea krait gene name","3":"homologs"},{"1":"llaticaudata_homolog_ensembl_peptide","2":"Blue-ringed sea krait protein or transcript stable ID","3":"homologs"},{"1":"llaticaudata_homolog_chromosome","2":"Blue-ringed sea krait chromosome/scaffold name","3":"homologs"},{"1":"llaticaudata_homolog_chrom_start","2":"Blue-ringed sea krait chromosome/scaffold start (bp)","3":"homologs"},{"1":"llaticaudata_homolog_chrom_end","2":"Blue-ringed sea krait chromosome/scaffold end (bp)","3":"homologs"},{"1":"llaticaudata_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"llaticaudata_homolog_subtype","2":"Last common ancestor with Blue-ringed sea krait","3":"homologs"},{"1":"llaticaudata_homolog_orthology_type","2":"Blue-ringed sea krait homology type","3":"homologs"},{"1":"llaticaudata_homolog_perc_id","2":"%id. target Blue-ringed sea krait gene identical to query gene","3":"homologs"},{"1":"llaticaudata_homolog_perc_id_r1","2":"%id. query gene identical to target Blue-ringed sea krait gene","3":"homologs"},{"1":"llaticaudata_homolog_goc_score","2":"Blue-ringed sea krait Gene-order conservation score","3":"homologs"},{"1":"llaticaudata_homolog_wga_coverage","2":"Blue-ringed sea krait Whole-genome alignment coverage","3":"homologs"},{"1":"llaticaudata_homolog_orthology_confidence","2":"Blue-ringed sea krait orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sbboliviensis_homolog_ensembl_gene","2":"Bolivian squirrel monkey gene stable ID","3":"homologs"},{"1":"sbboliviensis_homolog_associated_gene_name","2":"Bolivian squirrel monkey gene name","3":"homologs"},{"1":"sbboliviensis_homolog_ensembl_peptide","2":"Bolivian squirrel monkey protein or transcript stable ID","3":"homologs"},{"1":"sbboliviensis_homolog_chromosome","2":"Bolivian squirrel monkey chromosome/scaffold name","3":"homologs"},{"1":"sbboliviensis_homolog_chrom_start","2":"Bolivian squirrel monkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"sbboliviensis_homolog_chrom_end","2":"Bolivian squirrel monkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"sbboliviensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sbboliviensis_homolog_subtype","2":"Last common ancestor with Bolivian squirrel monkey","3":"homologs"},{"1":"sbboliviensis_homolog_orthology_type","2":"Bolivian squirrel monkey homology type","3":"homologs"},{"1":"sbboliviensis_homolog_perc_id","2":"%id. target Bolivian squirrel monkey gene identical to query gene","3":"homologs"},{"1":"sbboliviensis_homolog_perc_id_r1","2":"%id. query gene identical to target Bolivian squirrel monkey gene","3":"homologs"},{"1":"sbboliviensis_homolog_goc_score","2":"Bolivian squirrel monkey Gene-order conservation score","3":"homologs"},{"1":"sbboliviensis_homolog_wga_coverage","2":"Bolivian squirrel monkey Whole-genome alignment coverage","3":"homologs"},{"1":"sbboliviensis_homolog_orthology_confidence","2":"Bolivian squirrel monkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ppaniscus_homolog_ensembl_gene","2":"Bonobo gene stable ID","3":"homologs"},{"1":"ppaniscus_homolog_associated_gene_name","2":"Bonobo gene name","3":"homologs"},{"1":"ppaniscus_homolog_ensembl_peptide","2":"Bonobo protein or transcript stable ID","3":"homologs"},{"1":"ppaniscus_homolog_chromosome","2":"Bonobo chromosome/scaffold name","3":"homologs"},{"1":"ppaniscus_homolog_chrom_start","2":"Bonobo chromosome/scaffold start (bp)","3":"homologs"},{"1":"ppaniscus_homolog_chrom_end","2":"Bonobo chromosome/scaffold end (bp)","3":"homologs"},{"1":"ppaniscus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ppaniscus_homolog_subtype","2":"Last common ancestor with Bonobo","3":"homologs"},{"1":"ppaniscus_homolog_orthology_type","2":"Bonobo homology type","3":"homologs"},{"1":"ppaniscus_homolog_perc_id","2":"%id. target Bonobo gene identical to query gene","3":"homologs"},{"1":"ppaniscus_homolog_perc_id_r1","2":"%id. query gene identical to target Bonobo gene","3":"homologs"},{"1":"ppaniscus_homolog_goc_score","2":"Bonobo Gene-order conservation score","3":"homologs"},{"1":"ppaniscus_homolog_wga_coverage","2":"Bonobo Whole-genome alignment coverage","3":"homologs"},{"1":"ppaniscus_homolog_orthology_confidence","2":"Bonobo orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"strutta_homolog_ensembl_gene","2":"Brown trout gene stable ID","3":"homologs"},{"1":"strutta_homolog_associated_gene_name","2":"Brown trout gene name","3":"homologs"},{"1":"strutta_homolog_ensembl_peptide","2":"Brown trout protein or transcript stable ID","3":"homologs"},{"1":"strutta_homolog_chromosome","2":"Brown trout chromosome/scaffold name","3":"homologs"},{"1":"strutta_homolog_chrom_start","2":"Brown trout chromosome/scaffold start (bp)","3":"homologs"},{"1":"strutta_homolog_chrom_end","2":"Brown trout chromosome/scaffold end (bp)","3":"homologs"},{"1":"strutta_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"strutta_homolog_subtype","2":"Last common ancestor with Brown trout","3":"homologs"},{"1":"strutta_homolog_orthology_type","2":"Brown trout homology type","3":"homologs"},{"1":"strutta_homolog_perc_id","2":"%id. target Brown trout gene identical to query gene","3":"homologs"},{"1":"strutta_homolog_perc_id_r1","2":"%id. query gene identical to target Brown trout gene","3":"homologs"},{"1":"strutta_homolog_goc_score","2":"Brown trout Gene-order conservation score","3":"homologs"},{"1":"strutta_homolog_wga_coverage","2":"Brown trout Whole-genome alignment coverage","3":"homologs"},{"1":"strutta_homolog_orthology_confidence","2":"Brown trout orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"hburtoni_homolog_ensembl_gene","2":"Burton's mouthbrooder gene stable ID","3":"homologs"},{"1":"hburtoni_homolog_associated_gene_name","2":"Burton's mouthbrooder gene name","3":"homologs"},{"1":"hburtoni_homolog_ensembl_peptide","2":"Burton's mouthbrooder protein or transcript stable ID","3":"homologs"},{"1":"hburtoni_homolog_chromosome","2":"Burton's mouthbrooder chromosome/scaffold name","3":"homologs"},{"1":"hburtoni_homolog_chrom_start","2":"Burton's mouthbrooder chromosome/scaffold start (bp)","3":"homologs"},{"1":"hburtoni_homolog_chrom_end","2":"Burton's mouthbrooder chromosome/scaffold end (bp)","3":"homologs"},{"1":"hburtoni_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"hburtoni_homolog_subtype","2":"Last common ancestor with Burton's mouthbrooder","3":"homologs"},{"1":"hburtoni_homolog_orthology_type","2":"Burton's mouthbrooder homology type","3":"homologs"},{"1":"hburtoni_homolog_perc_id","2":"%id. target Burton's mouthbrooder gene identical to query gene","3":"homologs"},{"1":"hburtoni_homolog_perc_id_r1","2":"%id. query gene identical to target Burton's mouthbrooder gene","3":"homologs"},{"1":"hburtoni_homolog_goc_score","2":"Burton's mouthbrooder Gene-order conservation score","3":"homologs"},{"1":"hburtoni_homolog_wga_coverage","2":"Burton's mouthbrooder Whole-genome alignment coverage","3":"homologs"},{"1":"hburtoni_homolog_orthology_confidence","2":"Burton's mouthbrooder orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ogarnettii_homolog_ensembl_gene","2":"Bushbaby gene stable ID","3":"homologs"},{"1":"ogarnettii_homolog_associated_gene_name","2":"Bushbaby gene name","3":"homologs"},{"1":"ogarnettii_homolog_ensembl_peptide","2":"Bushbaby protein or transcript stable ID","3":"homologs"},{"1":"ogarnettii_homolog_chromosome","2":"Bushbaby chromosome/scaffold name","3":"homologs"},{"1":"ogarnettii_homolog_chrom_start","2":"Bushbaby chromosome/scaffold start (bp)","3":"homologs"},{"1":"ogarnettii_homolog_chrom_end","2":"Bushbaby chromosome/scaffold end (bp)","3":"homologs"},{"1":"ogarnettii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ogarnettii_homolog_subtype","2":"Last common ancestor with Bushbaby","3":"homologs"},{"1":"ogarnettii_homolog_orthology_type","2":"Bushbaby homology type","3":"homologs"},{"1":"ogarnettii_homolog_perc_id","2":"%id. target Bushbaby gene identical to query gene","3":"homologs"},{"1":"ogarnettii_homolog_perc_id_r1","2":"%id. query gene identical to target Bushbaby gene","3":"homologs"},{"1":"ogarnettii_homolog_goc_score","2":"Bushbaby Gene-order conservation score","3":"homologs"},{"1":"ogarnettii_homolog_wga_coverage","2":"Bushbaby Whole-genome alignment coverage","3":"homologs"},{"1":"ogarnettii_homolog_orthology_confidence","2":"Bushbaby orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cintestinalis_homolog_ensembl_gene","2":"C.intestinalis gene stable ID","3":"homologs"},{"1":"cintestinalis_homolog_associated_gene_name","2":"C.intestinalis gene name","3":"homologs"},{"1":"cintestinalis_homolog_ensembl_peptide","2":"C.intestinalis protein or transcript stable ID","3":"homologs"},{"1":"cintestinalis_homolog_chromosome","2":"C.intestinalis chromosome/scaffold name","3":"homologs"},{"1":"cintestinalis_homolog_chrom_start","2":"C.intestinalis chromosome/scaffold start (bp)","3":"homologs"},{"1":"cintestinalis_homolog_chrom_end","2":"C.intestinalis chromosome/scaffold end (bp)","3":"homologs"},{"1":"cintestinalis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cintestinalis_homolog_subtype","2":"Last common ancestor with C.intestinalis","3":"homologs"},{"1":"cintestinalis_homolog_orthology_type","2":"C.intestinalis homology type","3":"homologs"},{"1":"cintestinalis_homolog_perc_id","2":"%id. target C.intestinalis gene identical to query gene","3":"homologs"},{"1":"cintestinalis_homolog_perc_id_r1","2":"%id. query gene identical to target C.intestinalis gene","3":"homologs"},{"1":"cintestinalis_homolog_wga_coverage","2":"C.intestinalis Whole-genome alignment coverage","3":"homologs"},{"1":"cintestinalis_homolog_orthology_confidence","2":"C.intestinalis orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"csavignyi_homolog_ensembl_gene","2":"C.savignyi gene stable ID","3":"homologs"},{"1":"csavignyi_homolog_associated_gene_name","2":"C.savignyi gene name","3":"homologs"},{"1":"csavignyi_homolog_ensembl_peptide","2":"C.savignyi protein or transcript stable ID","3":"homologs"},{"1":"csavignyi_homolog_chromosome","2":"C.savignyi chromosome/scaffold name","3":"homologs"},{"1":"csavignyi_homolog_chrom_start","2":"C.savignyi chromosome/scaffold start (bp)","3":"homologs"},{"1":"csavignyi_homolog_chrom_end","2":"C.savignyi chromosome/scaffold end (bp)","3":"homologs"},{"1":"csavignyi_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"csavignyi_homolog_subtype","2":"Last common ancestor with C.savignyi","3":"homologs"},{"1":"csavignyi_homolog_orthology_type","2":"C.savignyi homology type","3":"homologs"},{"1":"csavignyi_homolog_perc_id","2":"%id. target C.savignyi gene identical to query gene","3":"homologs"},{"1":"csavignyi_homolog_perc_id_r1","2":"%id. query gene identical to target C.savignyi gene","3":"homologs"},{"1":"csavignyi_homolog_wga_coverage","2":"C.savignyi Whole-genome alignment coverage","3":"homologs"},{"1":"csavignyi_homolog_orthology_confidence","2":"C.savignyi orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"celegans_homolog_ensembl_gene","2":"Caenorhabditis elegans (PRJNA13758) gene stable ID","3":"homologs"},{"1":"celegans_homolog_associated_gene_name","2":"Caenorhabditis elegans (PRJNA13758) gene name","3":"homologs"},{"1":"celegans_homolog_ensembl_peptide","2":"Caenorhabditis elegans (PRJNA13758) protein or transcript stable ID","3":"homologs"},{"1":"celegans_homolog_chromosome","2":"Caenorhabditis elegans (PRJNA13758) chromosome/scaffold name","3":"homologs"},{"1":"celegans_homolog_chrom_start","2":"Caenorhabditis elegans (PRJNA13758) chromosome/scaffold start (bp)","3":"homologs"},{"1":"celegans_homolog_chrom_end","2":"Caenorhabditis elegans (PRJNA13758) chromosome/scaffold end (bp)","3":"homologs"},{"1":"celegans_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"celegans_homolog_subtype","2":"Last common ancestor with Caenorhabditis elegans (PRJNA13758)","3":"homologs"},{"1":"celegans_homolog_orthology_type","2":"Caenorhabditis elegans (PRJNA13758) homology type","3":"homologs"},{"1":"celegans_homolog_perc_id","2":"%id. target Caenorhabditis elegans (PRJNA13758) gene identical to query gene","3":"homologs"},{"1":"celegans_homolog_perc_id_r1","2":"%id. query gene identical to target Caenorhabditis elegans (PRJNA13758) gene","3":"homologs"},{"1":"celegans_homolog_orthology_confidence","2":"Caenorhabditis elegans (PRJNA13758) orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"fcatus_homolog_ensembl_gene","2":"Cat gene stable ID","3":"homologs"},{"1":"fcatus_homolog_associated_gene_name","2":"Cat gene name","3":"homologs"},{"1":"fcatus_homolog_ensembl_peptide","2":"Cat protein or transcript stable ID","3":"homologs"},{"1":"fcatus_homolog_chromosome","2":"Cat chromosome/scaffold name","3":"homologs"},{"1":"fcatus_homolog_chrom_start","2":"Cat chromosome/scaffold start (bp)","3":"homologs"},{"1":"fcatus_homolog_chrom_end","2":"Cat chromosome/scaffold end (bp)","3":"homologs"},{"1":"fcatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"fcatus_homolog_subtype","2":"Last common ancestor with Cat","3":"homologs"},{"1":"fcatus_homolog_orthology_type","2":"Cat homology type","3":"homologs"},{"1":"fcatus_homolog_perc_id","2":"%id. target Cat gene identical to query gene","3":"homologs"},{"1":"fcatus_homolog_perc_id_r1","2":"%id. query gene identical to target Cat gene","3":"homologs"},{"1":"fcatus_homolog_goc_score","2":"Cat Gene-order conservation score","3":"homologs"},{"1":"fcatus_homolog_wga_coverage","2":"Cat Whole-genome alignment coverage","3":"homologs"},{"1":"fcatus_homolog_orthology_confidence","2":"Cat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cwagneri_homolog_ensembl_gene","2":"Chacoan peccary gene stable ID","3":"homologs"},{"1":"cwagneri_homolog_associated_gene_name","2":"Chacoan peccary gene name","3":"homologs"},{"1":"cwagneri_homolog_ensembl_peptide","2":"Chacoan peccary protein or transcript stable ID","3":"homologs"},{"1":"cwagneri_homolog_chromosome","2":"Chacoan peccary chromosome/scaffold name","3":"homologs"},{"1":"cwagneri_homolog_chrom_start","2":"Chacoan peccary chromosome/scaffold start (bp)","3":"homologs"},{"1":"cwagneri_homolog_chrom_end","2":"Chacoan peccary chromosome/scaffold end (bp)","3":"homologs"},{"1":"cwagneri_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cwagneri_homolog_subtype","2":"Last common ancestor with Chacoan peccary","3":"homologs"},{"1":"cwagneri_homolog_orthology_type","2":"Chacoan peccary homology type","3":"homologs"},{"1":"cwagneri_homolog_perc_id","2":"%id. target Chacoan peccary gene identical to query gene","3":"homologs"},{"1":"cwagneri_homolog_perc_id_r1","2":"%id. query gene identical to target Chacoan peccary gene","3":"homologs"},{"1":"cwagneri_homolog_goc_score","2":"Chacoan peccary Gene-order conservation score","3":"homologs"},{"1":"cwagneri_homolog_wga_coverage","2":"Chacoan peccary Whole-genome alignment coverage","3":"homologs"},{"1":"cwagneri_homolog_orthology_confidence","2":"Chacoan peccary orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cgobio_homolog_ensembl_gene","2":"Channel bull blenny gene stable ID","3":"homologs"},{"1":"cgobio_homolog_associated_gene_name","2":"Channel bull blenny gene name","3":"homologs"},{"1":"cgobio_homolog_ensembl_peptide","2":"Channel bull blenny protein or transcript stable ID","3":"homologs"},{"1":"cgobio_homolog_chromosome","2":"Channel bull blenny chromosome/scaffold name","3":"homologs"},{"1":"cgobio_homolog_chrom_start","2":"Channel bull blenny chromosome/scaffold start (bp)","3":"homologs"},{"1":"cgobio_homolog_chrom_end","2":"Channel bull blenny chromosome/scaffold end (bp)","3":"homologs"},{"1":"cgobio_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cgobio_homolog_subtype","2":"Last common ancestor with Channel bull blenny","3":"homologs"},{"1":"cgobio_homolog_orthology_type","2":"Channel bull blenny homology type","3":"homologs"},{"1":"cgobio_homolog_perc_id","2":"%id. target Channel bull blenny gene identical to query gene","3":"homologs"},{"1":"cgobio_homolog_perc_id_r1","2":"%id. query gene identical to target Channel bull blenny gene","3":"homologs"},{"1":"cgobio_homolog_goc_score","2":"Channel bull blenny Gene-order conservation score","3":"homologs"},{"1":"cgobio_homolog_wga_coverage","2":"Channel bull blenny Whole-genome alignment coverage","3":"homologs"},{"1":"cgobio_homolog_orthology_confidence","2":"Channel bull blenny orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ipunctatus_homolog_ensembl_gene","2":"Channel catfish gene stable ID","3":"homologs"},{"1":"ipunctatus_homolog_associated_gene_name","2":"Channel catfish gene name","3":"homologs"},{"1":"ipunctatus_homolog_ensembl_peptide","2":"Channel catfish protein or transcript stable ID","3":"homologs"},{"1":"ipunctatus_homolog_chromosome","2":"Channel catfish chromosome/scaffold name","3":"homologs"},{"1":"ipunctatus_homolog_chrom_start","2":"Channel catfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"ipunctatus_homolog_chrom_end","2":"Channel catfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"ipunctatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ipunctatus_homolog_subtype","2":"Last common ancestor with Channel catfish","3":"homologs"},{"1":"ipunctatus_homolog_orthology_type","2":"Channel catfish homology type","3":"homologs"},{"1":"ipunctatus_homolog_perc_id","2":"%id. target Channel catfish gene identical to query gene","3":"homologs"},{"1":"ipunctatus_homolog_perc_id_r1","2":"%id. query gene identical to target Channel catfish gene","3":"homologs"},{"1":"ipunctatus_homolog_goc_score","2":"Channel catfish Gene-order conservation score","3":"homologs"},{"1":"ipunctatus_homolog_wga_coverage","2":"Channel catfish Whole-genome alignment coverage","3":"homologs"},{"1":"ipunctatus_homolog_orthology_confidence","2":"Channel catfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ggallus_homolog_ensembl_gene","2":"Chicken gene stable ID","3":"homologs"},{"1":"ggallus_homolog_associated_gene_name","2":"Chicken gene name","3":"homologs"},{"1":"ggallus_homolog_ensembl_peptide","2":"Chicken protein or transcript stable ID","3":"homologs"},{"1":"ggallus_homolog_chromosome","2":"Chicken chromosome/scaffold name","3":"homologs"},{"1":"ggallus_homolog_chrom_start","2":"Chicken chromosome/scaffold start (bp)","3":"homologs"},{"1":"ggallus_homolog_chrom_end","2":"Chicken chromosome/scaffold end (bp)","3":"homologs"},{"1":"ggallus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ggallus_homolog_subtype","2":"Last common ancestor with Chicken","3":"homologs"},{"1":"ggallus_homolog_orthology_type","2":"Chicken homology type","3":"homologs"},{"1":"ggallus_homolog_perc_id","2":"%id. target Chicken gene identical to query gene","3":"homologs"},{"1":"ggallus_homolog_perc_id_r1","2":"%id. query gene identical to target Chicken gene","3":"homologs"},{"1":"ggallus_homolog_goc_score","2":"Chicken Gene-order conservation score","3":"homologs"},{"1":"ggallus_homolog_wga_coverage","2":"Chicken Whole-genome alignment coverage","3":"homologs"},{"1":"ggallus_homolog_orthology_confidence","2":"Chicken orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ptroglodytes_homolog_ensembl_gene","2":"Chimpanzee gene stable ID","3":"homologs"},{"1":"ptroglodytes_homolog_associated_gene_name","2":"Chimpanzee gene name","3":"homologs"},{"1":"ptroglodytes_homolog_ensembl_peptide","2":"Chimpanzee protein or transcript stable ID","3":"homologs"},{"1":"ptroglodytes_homolog_chromosome","2":"Chimpanzee chromosome/scaffold name","3":"homologs"},{"1":"ptroglodytes_homolog_chrom_start","2":"Chimpanzee chromosome/scaffold start (bp)","3":"homologs"},{"1":"ptroglodytes_homolog_chrom_end","2":"Chimpanzee chromosome/scaffold end (bp)","3":"homologs"},{"1":"ptroglodytes_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ptroglodytes_homolog_subtype","2":"Last common ancestor with Chimpanzee","3":"homologs"},{"1":"ptroglodytes_homolog_orthology_type","2":"Chimpanzee homology type","3":"homologs"},{"1":"ptroglodytes_homolog_perc_id","2":"%id. target Chimpanzee gene identical to query gene","3":"homologs"},{"1":"ptroglodytes_homolog_perc_id_r1","2":"%id. query gene identical to target Chimpanzee gene","3":"homologs"},{"1":"ptroglodytes_homolog_goc_score","2":"Chimpanzee Gene-order conservation score","3":"homologs"},{"1":"ptroglodytes_homolog_wga_coverage","2":"Chimpanzee Whole-genome alignment coverage","3":"homologs"},{"1":"ptroglodytes_homolog_orthology_confidence","2":"Chimpanzee orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cgchok1gshd_homolog_ensembl_gene","2":"Chinese hamster CHOK1GS gene stable ID","3":"homologs"},{"1":"cgchok1gshd_homolog_associated_gene_name","2":"Chinese hamster CHOK1GS gene name","3":"homologs"},{"1":"cgchok1gshd_homolog_ensembl_peptide","2":"Chinese hamster CHOK1GS protein or transcript stable ID","3":"homologs"},{"1":"cgchok1gshd_homolog_chromosome","2":"Chinese hamster CHOK1GS chromosome/scaffold name","3":"homologs"},{"1":"cgchok1gshd_homolog_chrom_start","2":"Chinese hamster CHOK1GS chromosome/scaffold start (bp)","3":"homologs"},{"1":"cgchok1gshd_homolog_chrom_end","2":"Chinese hamster CHOK1GS chromosome/scaffold end (bp)","3":"homologs"},{"1":"cgchok1gshd_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cgchok1gshd_homolog_subtype","2":"Last common ancestor with Chinese hamster CHOK1GS","3":"homologs"},{"1":"cgchok1gshd_homolog_orthology_type","2":"Chinese hamster CHOK1GS homology type","3":"homologs"},{"1":"cgchok1gshd_homolog_perc_id","2":"%id. target Chinese hamster CHOK1GS gene identical to query gene","3":"homologs"},{"1":"cgchok1gshd_homolog_perc_id_r1","2":"%id. query gene identical to target Chinese hamster CHOK1GS gene","3":"homologs"},{"1":"cgchok1gshd_homolog_goc_score","2":"Chinese hamster CHOK1GS Gene-order conservation score","3":"homologs"},{"1":"cgchok1gshd_homolog_wga_coverage","2":"Chinese hamster CHOK1GS Whole-genome alignment coverage","3":"homologs"},{"1":"cgchok1gshd_homolog_orthology_confidence","2":"Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"osinensis_homolog_ensembl_gene","2":"Chinese medaka gene stable ID","3":"homologs"},{"1":"osinensis_homolog_associated_gene_name","2":"Chinese medaka gene name","3":"homologs"},{"1":"osinensis_homolog_ensembl_peptide","2":"Chinese medaka protein or transcript stable ID","3":"homologs"},{"1":"osinensis_homolog_chromosome","2":"Chinese medaka chromosome/scaffold name","3":"homologs"},{"1":"osinensis_homolog_chrom_start","2":"Chinese medaka chromosome/scaffold start (bp)","3":"homologs"},{"1":"osinensis_homolog_chrom_end","2":"Chinese medaka chromosome/scaffold end (bp)","3":"homologs"},{"1":"osinensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"osinensis_homolog_subtype","2":"Last common ancestor with Chinese medaka","3":"homologs"},{"1":"osinensis_homolog_orthology_type","2":"Chinese medaka homology type","3":"homologs"},{"1":"osinensis_homolog_perc_id","2":"%id. target Chinese medaka gene identical to query gene","3":"homologs"},{"1":"osinensis_homolog_perc_id_r1","2":"%id. query gene identical to target Chinese medaka gene","3":"homologs"},{"1":"osinensis_homolog_goc_score","2":"Chinese medaka Gene-order conservation score","3":"homologs"},{"1":"osinensis_homolog_wga_coverage","2":"Chinese medaka Whole-genome alignment coverage","3":"homologs"},{"1":"osinensis_homolog_orthology_confidence","2":"Chinese medaka orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"psinensis_homolog_ensembl_gene","2":"Chinese softshell turtle gene stable ID","3":"homologs"},{"1":"psinensis_homolog_associated_gene_name","2":"Chinese softshell turtle gene name","3":"homologs"},{"1":"psinensis_homolog_ensembl_peptide","2":"Chinese softshell turtle protein or transcript stable ID","3":"homologs"},{"1":"psinensis_homolog_chromosome","2":"Chinese softshell turtle chromosome/scaffold name","3":"homologs"},{"1":"psinensis_homolog_chrom_start","2":"Chinese softshell turtle chromosome/scaffold start (bp)","3":"homologs"},{"1":"psinensis_homolog_chrom_end","2":"Chinese softshell turtle chromosome/scaffold end (bp)","3":"homologs"},{"1":"psinensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"psinensis_homolog_subtype","2":"Last common ancestor with Chinese softshell turtle","3":"homologs"},{"1":"psinensis_homolog_orthology_type","2":"Chinese softshell turtle homology type","3":"homologs"},{"1":"psinensis_homolog_perc_id","2":"%id. target Chinese softshell turtle gene identical to query gene","3":"homologs"},{"1":"psinensis_homolog_perc_id_r1","2":"%id. query gene identical to target Chinese softshell turtle gene","3":"homologs"},{"1":"psinensis_homolog_goc_score","2":"Chinese softshell turtle Gene-order conservation score","3":"homologs"},{"1":"psinensis_homolog_wga_coverage","2":"Chinese softshell turtle Whole-genome alignment coverage","3":"homologs"},{"1":"psinensis_homolog_orthology_confidence","2":"Chinese softshell turtle orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"otshawytscha_homolog_ensembl_gene","2":"Chinook salmon gene stable ID","3":"homologs"},{"1":"otshawytscha_homolog_associated_gene_name","2":"Chinook salmon gene name","3":"homologs"},{"1":"otshawytscha_homolog_ensembl_peptide","2":"Chinook salmon protein or transcript stable ID","3":"homologs"},{"1":"otshawytscha_homolog_chromosome","2":"Chinook salmon chromosome/scaffold name","3":"homologs"},{"1":"otshawytscha_homolog_chrom_start","2":"Chinook salmon chromosome/scaffold start (bp)","3":"homologs"},{"1":"otshawytscha_homolog_chrom_end","2":"Chinook salmon chromosome/scaffold end (bp)","3":"homologs"},{"1":"otshawytscha_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"otshawytscha_homolog_subtype","2":"Last common ancestor with Chinook salmon","3":"homologs"},{"1":"otshawytscha_homolog_orthology_type","2":"Chinook salmon homology type","3":"homologs"},{"1":"otshawytscha_homolog_perc_id","2":"%id. target Chinook salmon gene identical to query gene","3":"homologs"},{"1":"otshawytscha_homolog_perc_id_r1","2":"%id. query gene identical to target Chinook salmon gene","3":"homologs"},{"1":"otshawytscha_homolog_goc_score","2":"Chinook salmon Gene-order conservation score","3":"homologs"},{"1":"otshawytscha_homolog_wga_coverage","2":"Chinook salmon Whole-genome alignment coverage","3":"homologs"},{"1":"otshawytscha_homolog_orthology_confidence","2":"Chinook salmon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"atestudineus_homolog_ensembl_gene","2":"Climbing perch gene stable ID","3":"homologs"},{"1":"atestudineus_homolog_associated_gene_name","2":"Climbing perch gene name","3":"homologs"},{"1":"atestudineus_homolog_ensembl_peptide","2":"Climbing perch protein or transcript stable ID","3":"homologs"},{"1":"atestudineus_homolog_chromosome","2":"Climbing perch chromosome/scaffold name","3":"homologs"},{"1":"atestudineus_homolog_chrom_start","2":"Climbing perch chromosome/scaffold start (bp)","3":"homologs"},{"1":"atestudineus_homolog_chrom_end","2":"Climbing perch chromosome/scaffold end (bp)","3":"homologs"},{"1":"atestudineus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"atestudineus_homolog_subtype","2":"Last common ancestor with Climbing perch","3":"homologs"},{"1":"atestudineus_homolog_orthology_type","2":"Climbing perch homology type","3":"homologs"},{"1":"atestudineus_homolog_perc_id","2":"%id. target Climbing perch gene identical to query gene","3":"homologs"},{"1":"atestudineus_homolog_perc_id_r1","2":"%id. query gene identical to target Climbing perch gene","3":"homologs"},{"1":"atestudineus_homolog_goc_score","2":"Climbing perch Gene-order conservation score","3":"homologs"},{"1":"atestudineus_homolog_orthology_confidence","2":"Climbing perch orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"aocellaris_homolog_ensembl_gene","2":"Clown anemonefish gene stable ID","3":"homologs"},{"1":"aocellaris_homolog_associated_gene_name","2":"Clown anemonefish gene name","3":"homologs"},{"1":"aocellaris_homolog_ensembl_peptide","2":"Clown anemonefish protein or transcript stable ID","3":"homologs"},{"1":"aocellaris_homolog_chromosome","2":"Clown anemonefish chromosome/scaffold name","3":"homologs"},{"1":"aocellaris_homolog_chrom_start","2":"Clown anemonefish chromosome/scaffold start (bp)","3":"homologs"},{"1":"aocellaris_homolog_chrom_end","2":"Clown anemonefish chromosome/scaffold end (bp)","3":"homologs"},{"1":"aocellaris_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"aocellaris_homolog_subtype","2":"Last common ancestor with Clown anemonefish","3":"homologs"},{"1":"aocellaris_homolog_orthology_type","2":"Clown anemonefish homology type","3":"homologs"},{"1":"aocellaris_homolog_perc_id","2":"%id. target Clown anemonefish gene identical to query gene","3":"homologs"},{"1":"aocellaris_homolog_perc_id_r1","2":"%id. query gene identical to target Clown anemonefish gene","3":"homologs"},{"1":"aocellaris_homolog_goc_score","2":"Clown anemonefish Gene-order conservation score","3":"homologs"},{"1":"aocellaris_homolog_wga_coverage","2":"Clown anemonefish Whole-genome alignment coverage","3":"homologs"},{"1":"aocellaris_homolog_orthology_confidence","2":"Clown anemonefish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lchalumnae_homolog_ensembl_gene","2":"Coelacanth gene stable ID","3":"homologs"},{"1":"lchalumnae_homolog_associated_gene_name","2":"Coelacanth gene name","3":"homologs"},{"1":"lchalumnae_homolog_ensembl_peptide","2":"Coelacanth protein or transcript stable ID","3":"homologs"},{"1":"lchalumnae_homolog_chromosome","2":"Coelacanth chromosome/scaffold name","3":"homologs"},{"1":"lchalumnae_homolog_chrom_start","2":"Coelacanth chromosome/scaffold start (bp)","3":"homologs"},{"1":"lchalumnae_homolog_chrom_end","2":"Coelacanth chromosome/scaffold end (bp)","3":"homologs"},{"1":"lchalumnae_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lchalumnae_homolog_subtype","2":"Last common ancestor with Coelacanth","3":"homologs"},{"1":"lchalumnae_homolog_orthology_type","2":"Coelacanth homology type","3":"homologs"},{"1":"lchalumnae_homolog_perc_id","2":"%id. target Coelacanth gene identical to query gene","3":"homologs"},{"1":"lchalumnae_homolog_perc_id_r1","2":"%id. query gene identical to target Coelacanth gene","3":"homologs"},{"1":"lchalumnae_homolog_goc_score","2":"Coelacanth Gene-order conservation score","3":"homologs"},{"1":"lchalumnae_homolog_wga_coverage","2":"Coelacanth Whole-genome alignment coverage","3":"homologs"},{"1":"lchalumnae_homolog_orthology_confidence","2":"Coelacanth orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"okisutch_homolog_ensembl_gene","2":"Coho salmon gene stable ID","3":"homologs"},{"1":"okisutch_homolog_associated_gene_name","2":"Coho salmon gene name","3":"homologs"},{"1":"okisutch_homolog_ensembl_peptide","2":"Coho salmon protein or transcript stable ID","3":"homologs"},{"1":"okisutch_homolog_chromosome","2":"Coho salmon chromosome/scaffold name","3":"homologs"},{"1":"okisutch_homolog_chrom_start","2":"Coho salmon chromosome/scaffold start (bp)","3":"homologs"},{"1":"okisutch_homolog_chrom_end","2":"Coho salmon chromosome/scaffold end (bp)","3":"homologs"},{"1":"okisutch_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"okisutch_homolog_subtype","2":"Last common ancestor with Coho salmon","3":"homologs"},{"1":"okisutch_homolog_orthology_type","2":"Coho salmon homology type","3":"homologs"},{"1":"okisutch_homolog_perc_id","2":"%id. target Coho salmon gene identical to query gene","3":"homologs"},{"1":"okisutch_homolog_perc_id_r1","2":"%id. query gene identical to target Coho salmon gene","3":"homologs"},{"1":"okisutch_homolog_goc_score","2":"Coho salmon Gene-order conservation score","3":"homologs"},{"1":"okisutch_homolog_wga_coverage","2":"Coho salmon Whole-genome alignment coverage","3":"homologs"},{"1":"okisutch_homolog_orthology_confidence","2":"Coho salmon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"falbicollis_homolog_ensembl_gene","2":"Collared flycatcher gene stable ID","3":"homologs"},{"1":"falbicollis_homolog_associated_gene_name","2":"Collared flycatcher gene name","3":"homologs"},{"1":"falbicollis_homolog_ensembl_peptide","2":"Collared flycatcher protein or transcript stable ID","3":"homologs"},{"1":"falbicollis_homolog_chromosome","2":"Collared flycatcher chromosome/scaffold name","3":"homologs"},{"1":"falbicollis_homolog_chrom_start","2":"Collared flycatcher chromosome/scaffold start (bp)","3":"homologs"},{"1":"falbicollis_homolog_chrom_end","2":"Collared flycatcher chromosome/scaffold end (bp)","3":"homologs"},{"1":"falbicollis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"falbicollis_homolog_subtype","2":"Last common ancestor with Collared flycatcher","3":"homologs"},{"1":"falbicollis_homolog_orthology_type","2":"Collared flycatcher homology type","3":"homologs"},{"1":"falbicollis_homolog_perc_id","2":"%id. target Collared flycatcher gene identical to query gene","3":"homologs"},{"1":"falbicollis_homolog_perc_id_r1","2":"%id. query gene identical to target Collared flycatcher gene","3":"homologs"},{"1":"falbicollis_homolog_goc_score","2":"Collared flycatcher Gene-order conservation score","3":"homologs"},{"1":"falbicollis_homolog_orthology_confidence","2":"Collared flycatcher orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"scanaria_homolog_ensembl_gene","2":"Common canary gene stable ID","3":"homologs"},{"1":"scanaria_homolog_associated_gene_name","2":"Common canary gene name","3":"homologs"},{"1":"scanaria_homolog_ensembl_peptide","2":"Common canary protein or transcript stable ID","3":"homologs"},{"1":"scanaria_homolog_chromosome","2":"Common canary chromosome/scaffold name","3":"homologs"},{"1":"scanaria_homolog_chrom_start","2":"Common canary chromosome/scaffold start (bp)","3":"homologs"},{"1":"scanaria_homolog_chrom_end","2":"Common canary chromosome/scaffold end (bp)","3":"homologs"},{"1":"scanaria_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"scanaria_homolog_subtype","2":"Last common ancestor with Common canary","3":"homologs"},{"1":"scanaria_homolog_orthology_type","2":"Common canary homology type","3":"homologs"},{"1":"scanaria_homolog_perc_id","2":"%id. target Common canary gene identical to query gene","3":"homologs"},{"1":"scanaria_homolog_perc_id_r1","2":"%id. query gene identical to target Common canary gene","3":"homologs"},{"1":"scanaria_homolog_goc_score","2":"Common canary Gene-order conservation score","3":"homologs"},{"1":"scanaria_homolog_wga_coverage","2":"Common canary Whole-genome alignment coverage","3":"homologs"},{"1":"scanaria_homolog_orthology_confidence","2":"Common canary orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cccarpio_homolog_ensembl_gene","2":"Common carp gene stable ID","3":"homologs"},{"1":"cccarpio_homolog_associated_gene_name","2":"Common carp gene name","3":"homologs"},{"1":"cccarpio_homolog_ensembl_peptide","2":"Common carp protein or transcript stable ID","3":"homologs"},{"1":"cccarpio_homolog_chromosome","2":"Common carp chromosome/scaffold name","3":"homologs"},{"1":"cccarpio_homolog_chrom_start","2":"Common carp chromosome/scaffold start (bp)","3":"homologs"},{"1":"cccarpio_homolog_chrom_end","2":"Common carp chromosome/scaffold end (bp)","3":"homologs"},{"1":"cccarpio_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cccarpio_homolog_subtype","2":"Last common ancestor with Common carp","3":"homologs"},{"1":"cccarpio_homolog_orthology_type","2":"Common carp homology type","3":"homologs"},{"1":"cccarpio_homolog_perc_id","2":"%id. target Common carp gene identical to query gene","3":"homologs"},{"1":"cccarpio_homolog_perc_id_r1","2":"%id. query gene identical to target Common carp gene","3":"homologs"},{"1":"cccarpio_homolog_goc_score","2":"Common carp Gene-order conservation score","3":"homologs"},{"1":"cccarpio_homolog_orthology_confidence","2":"Common carp orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pmuralis_homolog_ensembl_gene","2":"Common wall lizard gene stable ID","3":"homologs"},{"1":"pmuralis_homolog_associated_gene_name","2":"Common wall lizard gene name","3":"homologs"},{"1":"pmuralis_homolog_ensembl_peptide","2":"Common wall lizard protein or transcript stable ID","3":"homologs"},{"1":"pmuralis_homolog_chromosome","2":"Common wall lizard chromosome/scaffold name","3":"homologs"},{"1":"pmuralis_homolog_chrom_start","2":"Common wall lizard chromosome/scaffold start (bp)","3":"homologs"},{"1":"pmuralis_homolog_chrom_end","2":"Common wall lizard chromosome/scaffold end (bp)","3":"homologs"},{"1":"pmuralis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pmuralis_homolog_subtype","2":"Last common ancestor with Common wall lizard","3":"homologs"},{"1":"pmuralis_homolog_orthology_type","2":"Common wall lizard homology type","3":"homologs"},{"1":"pmuralis_homolog_perc_id","2":"%id. target Common wall lizard gene identical to query gene","3":"homologs"},{"1":"pmuralis_homolog_perc_id_r1","2":"%id. query gene identical to target Common wall lizard gene","3":"homologs"},{"1":"pmuralis_homolog_goc_score","2":"Common wall lizard Gene-order conservation score","3":"homologs"},{"1":"pmuralis_homolog_wga_coverage","2":"Common wall lizard Whole-genome alignment coverage","3":"homologs"},{"1":"pmuralis_homolog_orthology_confidence","2":"Common wall lizard orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"vursinus_homolog_ensembl_gene","2":"Common wombat gene stable ID","3":"homologs"},{"1":"vursinus_homolog_associated_gene_name","2":"Common wombat gene name","3":"homologs"},{"1":"vursinus_homolog_ensembl_peptide","2":"Common wombat protein or transcript stable ID","3":"homologs"},{"1":"vursinus_homolog_chromosome","2":"Common wombat chromosome/scaffold name","3":"homologs"},{"1":"vursinus_homolog_chrom_start","2":"Common wombat chromosome/scaffold start (bp)","3":"homologs"},{"1":"vursinus_homolog_chrom_end","2":"Common wombat chromosome/scaffold end (bp)","3":"homologs"},{"1":"vursinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"vursinus_homolog_subtype","2":"Last common ancestor with Common wombat","3":"homologs"},{"1":"vursinus_homolog_orthology_type","2":"Common wombat homology type","3":"homologs"},{"1":"vursinus_homolog_perc_id","2":"%id. target Common wombat gene identical to query gene","3":"homologs"},{"1":"vursinus_homolog_perc_id_r1","2":"%id. query gene identical to target Common wombat gene","3":"homologs"},{"1":"vursinus_homolog_goc_score","2":"Common wombat Gene-order conservation score","3":"homologs"},{"1":"vursinus_homolog_wga_coverage","2":"Common wombat Whole-genome alignment coverage","3":"homologs"},{"1":"vursinus_homolog_orthology_confidence","2":"Common wombat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pcoquereli_homolog_ensembl_gene","2":"Coquerel's sifaka gene stable ID","3":"homologs"},{"1":"pcoquereli_homolog_associated_gene_name","2":"Coquerel's sifaka gene name","3":"homologs"},{"1":"pcoquereli_homolog_ensembl_peptide","2":"Coquerel's sifaka protein or transcript stable ID","3":"homologs"},{"1":"pcoquereli_homolog_chromosome","2":"Coquerel's sifaka chromosome/scaffold name","3":"homologs"},{"1":"pcoquereli_homolog_chrom_start","2":"Coquerel's sifaka chromosome/scaffold start (bp)","3":"homologs"},{"1":"pcoquereli_homolog_chrom_end","2":"Coquerel's sifaka chromosome/scaffold end (bp)","3":"homologs"},{"1":"pcoquereli_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pcoquereli_homolog_subtype","2":"Last common ancestor with Coquerel's sifaka","3":"homologs"},{"1":"pcoquereli_homolog_orthology_type","2":"Coquerel's sifaka homology type","3":"homologs"},{"1":"pcoquereli_homolog_perc_id","2":"%id. target Coquerel's sifaka gene identical to query gene","3":"homologs"},{"1":"pcoquereli_homolog_perc_id_r1","2":"%id. query gene identical to target Coquerel's sifaka gene","3":"homologs"},{"1":"pcoquereli_homolog_goc_score","2":"Coquerel's sifaka Gene-order conservation score","3":"homologs"},{"1":"pcoquereli_homolog_wga_coverage","2":"Coquerel's sifaka Whole-genome alignment coverage","3":"homologs"},{"1":"pcoquereli_homolog_orthology_confidence","2":"Coquerel's sifaka orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"btaurus_homolog_ensembl_gene","2":"Cow gene stable ID","3":"homologs"},{"1":"btaurus_homolog_associated_gene_name","2":"Cow gene name","3":"homologs"},{"1":"btaurus_homolog_ensembl_peptide","2":"Cow protein or transcript stable ID","3":"homologs"},{"1":"btaurus_homolog_chromosome","2":"Cow chromosome/scaffold name","3":"homologs"},{"1":"btaurus_homolog_chrom_start","2":"Cow chromosome/scaffold start (bp)","3":"homologs"},{"1":"btaurus_homolog_chrom_end","2":"Cow chromosome/scaffold end (bp)","3":"homologs"},{"1":"btaurus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"btaurus_homolog_subtype","2":"Last common ancestor with Cow","3":"homologs"},{"1":"btaurus_homolog_orthology_type","2":"Cow homology type","3":"homologs"},{"1":"btaurus_homolog_perc_id","2":"%id. target Cow gene identical to query gene","3":"homologs"},{"1":"btaurus_homolog_perc_id_r1","2":"%id. query gene identical to target Cow gene","3":"homologs"},{"1":"btaurus_homolog_goc_score","2":"Cow Gene-order conservation score","3":"homologs"},{"1":"btaurus_homolog_wga_coverage","2":"Cow Whole-genome alignment coverage","3":"homologs"},{"1":"btaurus_homolog_orthology_confidence","2":"Cow orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mfascicularis_homolog_ensembl_gene","2":"Crab-eating macaque gene stable ID","3":"homologs"},{"1":"mfascicularis_homolog_associated_gene_name","2":"Crab-eating macaque gene name","3":"homologs"},{"1":"mfascicularis_homolog_ensembl_peptide","2":"Crab-eating macaque protein or transcript stable ID","3":"homologs"},{"1":"mfascicularis_homolog_chromosome","2":"Crab-eating macaque chromosome/scaffold name","3":"homologs"},{"1":"mfascicularis_homolog_chrom_start","2":"Crab-eating macaque chromosome/scaffold start (bp)","3":"homologs"},{"1":"mfascicularis_homolog_chrom_end","2":"Crab-eating macaque chromosome/scaffold end (bp)","3":"homologs"},{"1":"mfascicularis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mfascicularis_homolog_subtype","2":"Last common ancestor with Crab-eating macaque","3":"homologs"},{"1":"mfascicularis_homolog_orthology_type","2":"Crab-eating macaque homology type","3":"homologs"},{"1":"mfascicularis_homolog_perc_id","2":"%id. target Crab-eating macaque gene identical to query gene","3":"homologs"},{"1":"mfascicularis_homolog_perc_id_r1","2":"%id. query gene identical to target Crab-eating macaque gene","3":"homologs"},{"1":"mfascicularis_homolog_goc_score","2":"Crab-eating macaque Gene-order conservation score","3":"homologs"},{"1":"mfascicularis_homolog_wga_coverage","2":"Crab-eating macaque Whole-genome alignment coverage","3":"homologs"},{"1":"mfascicularis_homolog_orthology_confidence","2":"Crab-eating macaque orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"odegus_homolog_ensembl_gene","2":"Degu gene stable ID","3":"homologs"},{"1":"odegus_homolog_associated_gene_name","2":"Degu gene name","3":"homologs"},{"1":"odegus_homolog_ensembl_peptide","2":"Degu protein or transcript stable ID","3":"homologs"},{"1":"odegus_homolog_chromosome","2":"Degu chromosome/scaffold name","3":"homologs"},{"1":"odegus_homolog_chrom_start","2":"Degu chromosome/scaffold start (bp)","3":"homologs"},{"1":"odegus_homolog_chrom_end","2":"Degu chromosome/scaffold end (bp)","3":"homologs"},{"1":"odegus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"odegus_homolog_subtype","2":"Last common ancestor with Degu","3":"homologs"},{"1":"odegus_homolog_orthology_type","2":"Degu homology type","3":"homologs"},{"1":"odegus_homolog_perc_id","2":"%id. target Degu gene identical to query gene","3":"homologs"},{"1":"odegus_homolog_perc_id_r1","2":"%id. query gene identical to target Degu gene","3":"homologs"},{"1":"odegus_homolog_goc_score","2":"Degu Gene-order conservation score","3":"homologs"},{"1":"odegus_homolog_wga_coverage","2":"Degu Whole-genome alignment coverage","3":"homologs"},{"1":"odegus_homolog_orthology_confidence","2":"Degu orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dclupeoides_homolog_ensembl_gene","2":"Denticle herring gene stable ID","3":"homologs"},{"1":"dclupeoides_homolog_associated_gene_name","2":"Denticle herring gene name","3":"homologs"},{"1":"dclupeoides_homolog_ensembl_peptide","2":"Denticle herring protein or transcript stable ID","3":"homologs"},{"1":"dclupeoides_homolog_chromosome","2":"Denticle herring chromosome/scaffold name","3":"homologs"},{"1":"dclupeoides_homolog_chrom_start","2":"Denticle herring chromosome/scaffold start (bp)","3":"homologs"},{"1":"dclupeoides_homolog_chrom_end","2":"Denticle herring chromosome/scaffold end (bp)","3":"homologs"},{"1":"dclupeoides_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dclupeoides_homolog_subtype","2":"Last common ancestor with Denticle herring","3":"homologs"},{"1":"dclupeoides_homolog_orthology_type","2":"Denticle herring homology type","3":"homologs"},{"1":"dclupeoides_homolog_perc_id","2":"%id. target Denticle herring gene identical to query gene","3":"homologs"},{"1":"dclupeoides_homolog_perc_id_r1","2":"%id. query gene identical to target Denticle herring gene","3":"homologs"},{"1":"dclupeoides_homolog_goc_score","2":"Denticle herring Gene-order conservation score","3":"homologs"},{"1":"dclupeoides_homolog_wga_coverage","2":"Denticle herring Whole-genome alignment coverage","3":"homologs"},{"1":"dclupeoides_homolog_orthology_confidence","2":"Denticle herring orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cldingo_homolog_ensembl_gene","2":"Dingo gene stable ID","3":"homologs"},{"1":"cldingo_homolog_associated_gene_name","2":"Dingo gene name","3":"homologs"},{"1":"cldingo_homolog_ensembl_peptide","2":"Dingo protein or transcript stable ID","3":"homologs"},{"1":"cldingo_homolog_chromosome","2":"Dingo chromosome/scaffold name","3":"homologs"},{"1":"cldingo_homolog_chrom_start","2":"Dingo chromosome/scaffold start (bp)","3":"homologs"},{"1":"cldingo_homolog_chrom_end","2":"Dingo chromosome/scaffold end (bp)","3":"homologs"},{"1":"cldingo_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cldingo_homolog_subtype","2":"Last common ancestor with Dingo","3":"homologs"},{"1":"cldingo_homolog_orthology_type","2":"Dingo homology type","3":"homologs"},{"1":"cldingo_homolog_perc_id","2":"%id. target Dingo gene identical to query gene","3":"homologs"},{"1":"cldingo_homolog_perc_id_r1","2":"%id. query gene identical to target Dingo gene","3":"homologs"},{"1":"cldingo_homolog_goc_score","2":"Dingo Gene-order conservation score","3":"homologs"},{"1":"cldingo_homolog_wga_coverage","2":"Dingo Whole-genome alignment coverage","3":"homologs"},{"1":"cldingo_homolog_orthology_confidence","2":"Dingo orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"clfamiliaris_homolog_ensembl_gene","2":"Dog gene stable ID","3":"homologs"},{"1":"clfamiliaris_homolog_associated_gene_name","2":"Dog gene name","3":"homologs"},{"1":"clfamiliaris_homolog_ensembl_peptide","2":"Dog protein or transcript stable ID","3":"homologs"},{"1":"clfamiliaris_homolog_chromosome","2":"Dog chromosome/scaffold name","3":"homologs"},{"1":"clfamiliaris_homolog_chrom_start","2":"Dog chromosome/scaffold start (bp)","3":"homologs"},{"1":"clfamiliaris_homolog_chrom_end","2":"Dog chromosome/scaffold end (bp)","3":"homologs"},{"1":"clfamiliaris_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"clfamiliaris_homolog_subtype","2":"Last common ancestor with Dog","3":"homologs"},{"1":"clfamiliaris_homolog_orthology_type","2":"Dog homology type","3":"homologs"},{"1":"clfamiliaris_homolog_perc_id","2":"%id. target Dog gene identical to query gene","3":"homologs"},{"1":"clfamiliaris_homolog_perc_id_r1","2":"%id. query gene identical to target Dog gene","3":"homologs"},{"1":"clfamiliaris_homolog_goc_score","2":"Dog Gene-order conservation score","3":"homologs"},{"1":"clfamiliaris_homolog_wga_coverage","2":"Dog Whole-genome alignment coverage","3":"homologs"},{"1":"clfamiliaris_homolog_orthology_confidence","2":"Dog orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ttruncatus_homolog_ensembl_gene","2":"Dolphin gene stable ID","3":"homologs"},{"1":"ttruncatus_homolog_associated_gene_name","2":"Dolphin gene name","3":"homologs"},{"1":"ttruncatus_homolog_ensembl_peptide","2":"Dolphin protein or transcript stable ID","3":"homologs"},{"1":"ttruncatus_homolog_chromosome","2":"Dolphin chromosome/scaffold name","3":"homologs"},{"1":"ttruncatus_homolog_chrom_start","2":"Dolphin chromosome/scaffold start (bp)","3":"homologs"},{"1":"ttruncatus_homolog_chrom_end","2":"Dolphin chromosome/scaffold end (bp)","3":"homologs"},{"1":"ttruncatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ttruncatus_homolog_subtype","2":"Last common ancestor with Dolphin","3":"homologs"},{"1":"ttruncatus_homolog_orthology_type","2":"Dolphin homology type","3":"homologs"},{"1":"ttruncatus_homolog_perc_id","2":"%id. target Dolphin gene identical to query gene","3":"homologs"},{"1":"ttruncatus_homolog_perc_id_r1","2":"%id. query gene identical to target Dolphin gene","3":"homologs"},{"1":"ttruncatus_homolog_goc_score","2":"Dolphin Gene-order conservation score","3":"homologs"},{"1":"ttruncatus_homolog_wga_coverage","2":"Dolphin Whole-genome alignment coverage","3":"homologs"},{"1":"ttruncatus_homolog_orthology_confidence","2":"Dolphin orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bgrunniens_homolog_ensembl_gene","2":"Domestic yak gene stable ID","3":"homologs"},{"1":"bgrunniens_homolog_associated_gene_name","2":"Domestic yak gene name","3":"homologs"},{"1":"bgrunniens_homolog_ensembl_peptide","2":"Domestic yak protein or transcript stable ID","3":"homologs"},{"1":"bgrunniens_homolog_chromosome","2":"Domestic yak chromosome/scaffold name","3":"homologs"},{"1":"bgrunniens_homolog_chrom_start","2":"Domestic yak chromosome/scaffold start (bp)","3":"homologs"},{"1":"bgrunniens_homolog_chrom_end","2":"Domestic yak chromosome/scaffold end (bp)","3":"homologs"},{"1":"bgrunniens_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bgrunniens_homolog_subtype","2":"Last common ancestor with Domestic yak","3":"homologs"},{"1":"bgrunniens_homolog_orthology_type","2":"Domestic yak homology type","3":"homologs"},{"1":"bgrunniens_homolog_perc_id","2":"%id. target Domestic yak gene identical to query gene","3":"homologs"},{"1":"bgrunniens_homolog_perc_id_r1","2":"%id. query gene identical to target Domestic yak gene","3":"homologs"},{"1":"bgrunniens_homolog_goc_score","2":"Domestic yak Gene-order conservation score","3":"homologs"},{"1":"bgrunniens_homolog_wga_coverage","2":"Domestic yak Whole-genome alignment coverage","3":"homologs"},{"1":"bgrunniens_homolog_orthology_confidence","2":"Domestic yak orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"easinus_homolog_ensembl_gene","2":"Donkey gene stable ID","3":"homologs"},{"1":"easinus_homolog_associated_gene_name","2":"Donkey gene name","3":"homologs"},{"1":"easinus_homolog_ensembl_peptide","2":"Donkey protein or transcript stable ID","3":"homologs"},{"1":"easinus_homolog_chromosome","2":"Donkey chromosome/scaffold name","3":"homologs"},{"1":"easinus_homolog_chrom_start","2":"Donkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"easinus_homolog_chrom_end","2":"Donkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"easinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"easinus_homolog_subtype","2":"Last common ancestor with Donkey","3":"homologs"},{"1":"easinus_homolog_orthology_type","2":"Donkey homology type","3":"homologs"},{"1":"easinus_homolog_perc_id","2":"%id. target Donkey gene identical to query gene","3":"homologs"},{"1":"easinus_homolog_perc_id_r1","2":"%id. query gene identical to target Donkey gene","3":"homologs"},{"1":"easinus_homolog_goc_score","2":"Donkey Gene-order conservation score","3":"homologs"},{"1":"easinus_homolog_orthology_confidence","2":"Donkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mleucophaeus_homolog_ensembl_gene","2":"Drill gene stable ID","3":"homologs"},{"1":"mleucophaeus_homolog_associated_gene_name","2":"Drill gene name","3":"homologs"},{"1":"mleucophaeus_homolog_ensembl_peptide","2":"Drill protein or transcript stable ID","3":"homologs"},{"1":"mleucophaeus_homolog_chromosome","2":"Drill chromosome/scaffold name","3":"homologs"},{"1":"mleucophaeus_homolog_chrom_start","2":"Drill chromosome/scaffold start (bp)","3":"homologs"},{"1":"mleucophaeus_homolog_chrom_end","2":"Drill chromosome/scaffold end (bp)","3":"homologs"},{"1":"mleucophaeus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mleucophaeus_homolog_subtype","2":"Last common ancestor with Drill","3":"homologs"},{"1":"mleucophaeus_homolog_orthology_type","2":"Drill homology type","3":"homologs"},{"1":"mleucophaeus_homolog_perc_id","2":"%id. target Drill gene identical to query gene","3":"homologs"},{"1":"mleucophaeus_homolog_perc_id_r1","2":"%id. query gene identical to target Drill gene","3":"homologs"},{"1":"mleucophaeus_homolog_goc_score","2":"Drill Gene-order conservation score","3":"homologs"},{"1":"mleucophaeus_homolog_wga_coverage","2":"Drill Whole-genome alignment coverage","3":"homologs"},{"1":"mleucophaeus_homolog_orthology_confidence","2":"Drill orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dmelanogaster_homolog_ensembl_gene","2":"Drosophila melanogaster gene stable ID","3":"homologs"},{"1":"dmelanogaster_homolog_associated_gene_name","2":"Drosophila melanogaster gene name","3":"homologs"},{"1":"dmelanogaster_homolog_ensembl_peptide","2":"Drosophila melanogaster protein or transcript stable ID","3":"homologs"},{"1":"dmelanogaster_homolog_chromosome","2":"Drosophila melanogaster chromosome/scaffold name","3":"homologs"},{"1":"dmelanogaster_homolog_chrom_start","2":"Drosophila melanogaster chromosome/scaffold start (bp)","3":"homologs"},{"1":"dmelanogaster_homolog_chrom_end","2":"Drosophila melanogaster chromosome/scaffold end (bp)","3":"homologs"},{"1":"dmelanogaster_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dmelanogaster_homolog_subtype","2":"Last common ancestor with Drosophila melanogaster","3":"homologs"},{"1":"dmelanogaster_homolog_orthology_type","2":"Drosophila melanogaster homology type","3":"homologs"},{"1":"dmelanogaster_homolog_perc_id","2":"%id. target Drosophila melanogaster gene identical to query gene","3":"homologs"},{"1":"dmelanogaster_homolog_perc_id_r1","2":"%id. query gene identical to target Drosophila melanogaster gene","3":"homologs"},{"1":"dmelanogaster_homolog_orthology_confidence","2":"Drosophila melanogaster orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"applatyrhynchos_homolog_ensembl_gene","2":"Duck gene stable ID","3":"homologs"},{"1":"applatyrhynchos_homolog_associated_gene_name","2":"Duck gene name","3":"homologs"},{"1":"applatyrhynchos_homolog_ensembl_peptide","2":"Duck protein or transcript stable ID","3":"homologs"},{"1":"applatyrhynchos_homolog_chromosome","2":"Duck chromosome/scaffold name","3":"homologs"},{"1":"applatyrhynchos_homolog_chrom_start","2":"Duck chromosome/scaffold start (bp)","3":"homologs"},{"1":"applatyrhynchos_homolog_chrom_end","2":"Duck chromosome/scaffold end (bp)","3":"homologs"},{"1":"applatyrhynchos_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"applatyrhynchos_homolog_subtype","2":"Last common ancestor with Duck","3":"homologs"},{"1":"applatyrhynchos_homolog_orthology_type","2":"Duck homology type","3":"homologs"},{"1":"applatyrhynchos_homolog_perc_id","2":"%id. target Duck gene identical to query gene","3":"homologs"},{"1":"applatyrhynchos_homolog_perc_id_r1","2":"%id. query gene identical to target Duck gene","3":"homologs"},{"1":"applatyrhynchos_homolog_goc_score","2":"Duck Gene-order conservation score","3":"homologs"},{"1":"applatyrhynchos_homolog_wga_coverage","2":"Duck Whole-genome alignment coverage","3":"homologs"},{"1":"applatyrhynchos_homolog_orthology_confidence","2":"Duck orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ptextilis_homolog_ensembl_gene","2":"Eastern brown snake gene stable ID","3":"homologs"},{"1":"ptextilis_homolog_associated_gene_name","2":"Eastern brown snake gene name","3":"homologs"},{"1":"ptextilis_homolog_ensembl_peptide","2":"Eastern brown snake protein or transcript stable ID","3":"homologs"},{"1":"ptextilis_homolog_chromosome","2":"Eastern brown snake chromosome/scaffold name","3":"homologs"},{"1":"ptextilis_homolog_chrom_start","2":"Eastern brown snake chromosome/scaffold start (bp)","3":"homologs"},{"1":"ptextilis_homolog_chrom_end","2":"Eastern brown snake chromosome/scaffold end (bp)","3":"homologs"},{"1":"ptextilis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ptextilis_homolog_subtype","2":"Last common ancestor with Eastern brown snake","3":"homologs"},{"1":"ptextilis_homolog_orthology_type","2":"Eastern brown snake homology type","3":"homologs"},{"1":"ptextilis_homolog_perc_id","2":"%id. target Eastern brown snake gene identical to query gene","3":"homologs"},{"1":"ptextilis_homolog_perc_id_r1","2":"%id. query gene identical to target Eastern brown snake gene","3":"homologs"},{"1":"ptextilis_homolog_goc_score","2":"Eastern brown snake Gene-order conservation score","3":"homologs"},{"1":"ptextilis_homolog_wga_coverage","2":"Eastern brown snake Whole-genome alignment coverage","3":"homologs"},{"1":"ptextilis_homolog_orthology_confidence","2":"Eastern brown snake orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"acalliptera_homolog_ensembl_gene","2":"Eastern happy gene stable ID","3":"homologs"},{"1":"acalliptera_homolog_associated_gene_name","2":"Eastern happy gene name","3":"homologs"},{"1":"acalliptera_homolog_ensembl_peptide","2":"Eastern happy protein or transcript stable ID","3":"homologs"},{"1":"acalliptera_homolog_chromosome","2":"Eastern happy chromosome/scaffold name","3":"homologs"},{"1":"acalliptera_homolog_chrom_start","2":"Eastern happy chromosome/scaffold start (bp)","3":"homologs"},{"1":"acalliptera_homolog_chrom_end","2":"Eastern happy chromosome/scaffold end (bp)","3":"homologs"},{"1":"acalliptera_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"acalliptera_homolog_subtype","2":"Last common ancestor with Eastern happy","3":"homologs"},{"1":"acalliptera_homolog_orthology_type","2":"Eastern happy homology type","3":"homologs"},{"1":"acalliptera_homolog_perc_id","2":"%id. target Eastern happy gene identical to query gene","3":"homologs"},{"1":"acalliptera_homolog_perc_id_r1","2":"%id. query gene identical to target Eastern happy gene","3":"homologs"},{"1":"acalliptera_homolog_goc_score","2":"Eastern happy Gene-order conservation score","3":"homologs"},{"1":"acalliptera_homolog_wga_coverage","2":"Eastern happy Whole-genome alignment coverage","3":"homologs"},{"1":"acalliptera_homolog_orthology_confidence","2":"Eastern happy orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"eelectricus_homolog_ensembl_gene","2":"Electric eel gene stable ID","3":"homologs"},{"1":"eelectricus_homolog_associated_gene_name","2":"Electric eel gene name","3":"homologs"},{"1":"eelectricus_homolog_ensembl_peptide","2":"Electric eel protein or transcript stable ID","3":"homologs"},{"1":"eelectricus_homolog_chromosome","2":"Electric eel chromosome/scaffold name","3":"homologs"},{"1":"eelectricus_homolog_chrom_start","2":"Electric eel chromosome/scaffold start (bp)","3":"homologs"},{"1":"eelectricus_homolog_chrom_end","2":"Electric eel chromosome/scaffold end (bp)","3":"homologs"},{"1":"eelectricus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"eelectricus_homolog_subtype","2":"Last common ancestor with Electric eel","3":"homologs"},{"1":"eelectricus_homolog_orthology_type","2":"Electric eel homology type","3":"homologs"},{"1":"eelectricus_homolog_perc_id","2":"%id. target Electric eel gene identical to query gene","3":"homologs"},{"1":"eelectricus_homolog_perc_id_r1","2":"%id. query gene identical to target Electric eel gene","3":"homologs"},{"1":"eelectricus_homolog_goc_score","2":"Electric eel Gene-order conservation score","3":"homologs"},{"1":"eelectricus_homolog_wga_coverage","2":"Electric eel Whole-genome alignment coverage","3":"homologs"},{"1":"eelectricus_homolog_orthology_confidence","2":"Electric eel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lafricana_homolog_ensembl_gene","2":"Elephant gene stable ID","3":"homologs"},{"1":"lafricana_homolog_associated_gene_name","2":"Elephant gene name","3":"homologs"},{"1":"lafricana_homolog_ensembl_peptide","2":"Elephant protein or transcript stable ID","3":"homologs"},{"1":"lafricana_homolog_chromosome","2":"Elephant chromosome/scaffold name","3":"homologs"},{"1":"lafricana_homolog_chrom_start","2":"Elephant chromosome/scaffold start (bp)","3":"homologs"},{"1":"lafricana_homolog_chrom_end","2":"Elephant chromosome/scaffold end (bp)","3":"homologs"},{"1":"lafricana_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lafricana_homolog_subtype","2":"Last common ancestor with Elephant","3":"homologs"},{"1":"lafricana_homolog_orthology_type","2":"Elephant homology type","3":"homologs"},{"1":"lafricana_homolog_perc_id","2":"%id. target Elephant gene identical to query gene","3":"homologs"},{"1":"lafricana_homolog_perc_id_r1","2":"%id. query gene identical to target Elephant gene","3":"homologs"},{"1":"lafricana_homolog_goc_score","2":"Elephant Gene-order conservation score","3":"homologs"},{"1":"lafricana_homolog_wga_coverage","2":"Elephant Whole-genome alignment coverage","3":"homologs"},{"1":"lafricana_homolog_orthology_confidence","2":"Elephant orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cmilii_homolog_ensembl_gene","2":"Elephant shark gene stable ID","3":"homologs"},{"1":"cmilii_homolog_associated_gene_name","2":"Elephant shark gene name","3":"homologs"},{"1":"cmilii_homolog_ensembl_peptide","2":"Elephant shark protein or transcript stable ID","3":"homologs"},{"1":"cmilii_homolog_chromosome","2":"Elephant shark chromosome/scaffold name","3":"homologs"},{"1":"cmilii_homolog_chrom_start","2":"Elephant shark chromosome/scaffold start (bp)","3":"homologs"},{"1":"cmilii_homolog_chrom_end","2":"Elephant shark chromosome/scaffold end (bp)","3":"homologs"},{"1":"cmilii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cmilii_homolog_subtype","2":"Last common ancestor with Elephant shark","3":"homologs"},{"1":"cmilii_homolog_orthology_type","2":"Elephant shark homology type","3":"homologs"},{"1":"cmilii_homolog_perc_id","2":"%id. target Elephant shark gene identical to query gene","3":"homologs"},{"1":"cmilii_homolog_perc_id_r1","2":"%id. query gene identical to target Elephant shark gene","3":"homologs"},{"1":"cmilii_homolog_wga_coverage","2":"Elephant shark Whole-genome alignment coverage","3":"homologs"},{"1":"cmilii_homolog_orthology_confidence","2":"Elephant shark orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"svulgaris_homolog_ensembl_gene","2":"Eurasian red squirrel gene stable ID","3":"homologs"},{"1":"svulgaris_homolog_associated_gene_name","2":"Eurasian red squirrel gene name","3":"homologs"},{"1":"svulgaris_homolog_ensembl_peptide","2":"Eurasian red squirrel protein or transcript stable ID","3":"homologs"},{"1":"svulgaris_homolog_chromosome","2":"Eurasian red squirrel chromosome/scaffold name","3":"homologs"},{"1":"svulgaris_homolog_chrom_start","2":"Eurasian red squirrel chromosome/scaffold start (bp)","3":"homologs"},{"1":"svulgaris_homolog_chrom_end","2":"Eurasian red squirrel chromosome/scaffold end (bp)","3":"homologs"},{"1":"svulgaris_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"svulgaris_homolog_subtype","2":"Last common ancestor with Eurasian red squirrel","3":"homologs"},{"1":"svulgaris_homolog_orthology_type","2":"Eurasian red squirrel homology type","3":"homologs"},{"1":"svulgaris_homolog_perc_id","2":"%id. target Eurasian red squirrel gene identical to query gene","3":"homologs"},{"1":"svulgaris_homolog_perc_id_r1","2":"%id. query gene identical to target Eurasian red squirrel gene","3":"homologs"},{"1":"svulgaris_homolog_goc_score","2":"Eurasian red squirrel Gene-order conservation score","3":"homologs"},{"1":"svulgaris_homolog_wga_coverage","2":"Eurasian red squirrel Whole-genome alignment coverage","3":"homologs"},{"1":"svulgaris_homolog_orthology_confidence","2":"Eurasian red squirrel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dlabrax_homolog_ensembl_gene","2":"European seabass gene stable ID","3":"homologs"},{"1":"dlabrax_homolog_associated_gene_name","2":"European seabass gene name","3":"homologs"},{"1":"dlabrax_homolog_ensembl_peptide","2":"European seabass protein or transcript stable ID","3":"homologs"},{"1":"dlabrax_homolog_chromosome","2":"European seabass chromosome/scaffold name","3":"homologs"},{"1":"dlabrax_homolog_chrom_start","2":"European seabass chromosome/scaffold start (bp)","3":"homologs"},{"1":"dlabrax_homolog_chrom_end","2":"European seabass chromosome/scaffold end (bp)","3":"homologs"},{"1":"dlabrax_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dlabrax_homolog_subtype","2":"Last common ancestor with European seabass","3":"homologs"},{"1":"dlabrax_homolog_orthology_type","2":"European seabass homology type","3":"homologs"},{"1":"dlabrax_homolog_perc_id","2":"%id. target European seabass gene identical to query gene","3":"homologs"},{"1":"dlabrax_homolog_perc_id_r1","2":"%id. query gene identical to target European seabass gene","3":"homologs"},{"1":"dlabrax_homolog_goc_score","2":"European seabass Gene-order conservation score","3":"homologs"},{"1":"dlabrax_homolog_orthology_confidence","2":"European seabass orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mpfuro_homolog_ensembl_gene","2":"Ferret gene stable ID","3":"homologs"},{"1":"mpfuro_homolog_associated_gene_name","2":"Ferret gene name","3":"homologs"},{"1":"mpfuro_homolog_ensembl_peptide","2":"Ferret protein or transcript stable ID","3":"homologs"},{"1":"mpfuro_homolog_chromosome","2":"Ferret chromosome/scaffold name","3":"homologs"},{"1":"mpfuro_homolog_chrom_start","2":"Ferret chromosome/scaffold start (bp)","3":"homologs"},{"1":"mpfuro_homolog_chrom_end","2":"Ferret chromosome/scaffold end (bp)","3":"homologs"},{"1":"mpfuro_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mpfuro_homolog_subtype","2":"Last common ancestor with Ferret","3":"homologs"},{"1":"mpfuro_homolog_orthology_type","2":"Ferret homology type","3":"homologs"},{"1":"mpfuro_homolog_perc_id","2":"%id. target Ferret gene identical to query gene","3":"homologs"},{"1":"mpfuro_homolog_perc_id_r1","2":"%id. query gene identical to target Ferret gene","3":"homologs"},{"1":"mpfuro_homolog_goc_score","2":"Ferret Gene-order conservation score","3":"homologs"},{"1":"mpfuro_homolog_wga_coverage","2":"Ferret Whole-genome alignment coverage","3":"homologs"},{"1":"mpfuro_homolog_orthology_confidence","2":"Ferret orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"trubripes_homolog_ensembl_gene","2":"Fugu gene stable ID","3":"homologs"},{"1":"trubripes_homolog_associated_gene_name","2":"Fugu gene name","3":"homologs"},{"1":"trubripes_homolog_ensembl_peptide","2":"Fugu protein or transcript stable ID","3":"homologs"},{"1":"trubripes_homolog_chromosome","2":"Fugu chromosome/scaffold name","3":"homologs"},{"1":"trubripes_homolog_chrom_start","2":"Fugu chromosome/scaffold start (bp)","3":"homologs"},{"1":"trubripes_homolog_chrom_end","2":"Fugu chromosome/scaffold end (bp)","3":"homologs"},{"1":"trubripes_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"trubripes_homolog_subtype","2":"Last common ancestor with Fugu","3":"homologs"},{"1":"trubripes_homolog_orthology_type","2":"Fugu homology type","3":"homologs"},{"1":"trubripes_homolog_perc_id","2":"%id. target Fugu gene identical to query gene","3":"homologs"},{"1":"trubripes_homolog_perc_id_r1","2":"%id. query gene identical to target Fugu gene","3":"homologs"},{"1":"trubripes_homolog_goc_score","2":"Fugu Gene-order conservation score","3":"homologs"},{"1":"trubripes_homolog_wga_coverage","2":"Fugu Whole-genome alignment coverage","3":"homologs"},{"1":"trubripes_homolog_orthology_confidence","2":"Fugu orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"amelanoleuca_homolog_ensembl_gene","2":"Giant panda gene stable ID","3":"homologs"},{"1":"amelanoleuca_homolog_associated_gene_name","2":"Giant panda gene name","3":"homologs"},{"1":"amelanoleuca_homolog_ensembl_peptide","2":"Giant panda protein or transcript stable ID","3":"homologs"},{"1":"amelanoleuca_homolog_chromosome","2":"Giant panda chromosome/scaffold name","3":"homologs"},{"1":"amelanoleuca_homolog_chrom_start","2":"Giant panda chromosome/scaffold start (bp)","3":"homologs"},{"1":"amelanoleuca_homolog_chrom_end","2":"Giant panda chromosome/scaffold end (bp)","3":"homologs"},{"1":"amelanoleuca_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"amelanoleuca_homolog_subtype","2":"Last common ancestor with Giant panda","3":"homologs"},{"1":"amelanoleuca_homolog_orthology_type","2":"Giant panda homology type","3":"homologs"},{"1":"amelanoleuca_homolog_perc_id","2":"%id. target Giant panda gene identical to query gene","3":"homologs"},{"1":"amelanoleuca_homolog_perc_id_r1","2":"%id. query gene identical to target Giant panda gene","3":"homologs"},{"1":"amelanoleuca_homolog_goc_score","2":"Giant panda Gene-order conservation score","3":"homologs"},{"1":"amelanoleuca_homolog_orthology_confidence","2":"Giant panda orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nleucogenys_homolog_ensembl_gene","2":"Gibbon gene stable ID","3":"homologs"},{"1":"nleucogenys_homolog_associated_gene_name","2":"Gibbon gene name","3":"homologs"},{"1":"nleucogenys_homolog_ensembl_peptide","2":"Gibbon protein or transcript stable ID","3":"homologs"},{"1":"nleucogenys_homolog_chromosome","2":"Gibbon chromosome/scaffold name","3":"homologs"},{"1":"nleucogenys_homolog_chrom_start","2":"Gibbon chromosome/scaffold start (bp)","3":"homologs"},{"1":"nleucogenys_homolog_chrom_end","2":"Gibbon chromosome/scaffold end (bp)","3":"homologs"},{"1":"nleucogenys_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nleucogenys_homolog_subtype","2":"Last common ancestor with Gibbon","3":"homologs"},{"1":"nleucogenys_homolog_orthology_type","2":"Gibbon homology type","3":"homologs"},{"1":"nleucogenys_homolog_perc_id","2":"%id. target Gibbon gene identical to query gene","3":"homologs"},{"1":"nleucogenys_homolog_perc_id_r1","2":"%id. query gene identical to target Gibbon gene","3":"homologs"},{"1":"nleucogenys_homolog_goc_score","2":"Gibbon Gene-order conservation score","3":"homologs"},{"1":"nleucogenys_homolog_wga_coverage","2":"Gibbon Whole-genome alignment coverage","3":"homologs"},{"1":"nleucogenys_homolog_orthology_confidence","2":"Gibbon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"saurata_homolog_ensembl_gene","2":"Gilthead seabream gene stable ID","3":"homologs"},{"1":"saurata_homolog_associated_gene_name","2":"Gilthead seabream gene name","3":"homologs"},{"1":"saurata_homolog_ensembl_peptide","2":"Gilthead seabream protein or transcript stable ID","3":"homologs"},{"1":"saurata_homolog_chromosome","2":"Gilthead seabream chromosome/scaffold name","3":"homologs"},{"1":"saurata_homolog_chrom_start","2":"Gilthead seabream chromosome/scaffold start (bp)","3":"homologs"},{"1":"saurata_homolog_chrom_end","2":"Gilthead seabream chromosome/scaffold end (bp)","3":"homologs"},{"1":"saurata_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"saurata_homolog_subtype","2":"Last common ancestor with Gilthead seabream","3":"homologs"},{"1":"saurata_homolog_orthology_type","2":"Gilthead seabream homology type","3":"homologs"},{"1":"saurata_homolog_perc_id","2":"%id. target Gilthead seabream gene identical to query gene","3":"homologs"},{"1":"saurata_homolog_perc_id_r1","2":"%id. query gene identical to target Gilthead seabream gene","3":"homologs"},{"1":"saurata_homolog_goc_score","2":"Gilthead seabream Gene-order conservation score","3":"homologs"},{"1":"saurata_homolog_wga_coverage","2":"Gilthead seabream Whole-genome alignment coverage","3":"homologs"},{"1":"saurata_homolog_orthology_confidence","2":"Gilthead seabream orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"chircus_homolog_ensembl_gene","2":"Goat gene stable ID","3":"homologs"},{"1":"chircus_homolog_associated_gene_name","2":"Goat gene name","3":"homologs"},{"1":"chircus_homolog_ensembl_peptide","2":"Goat protein or transcript stable ID","3":"homologs"},{"1":"chircus_homolog_chromosome","2":"Goat chromosome/scaffold name","3":"homologs"},{"1":"chircus_homolog_chrom_start","2":"Goat chromosome/scaffold start (bp)","3":"homologs"},{"1":"chircus_homolog_chrom_end","2":"Goat chromosome/scaffold end (bp)","3":"homologs"},{"1":"chircus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"chircus_homolog_subtype","2":"Last common ancestor with Goat","3":"homologs"},{"1":"chircus_homolog_orthology_type","2":"Goat homology type","3":"homologs"},{"1":"chircus_homolog_perc_id","2":"%id. target Goat gene identical to query gene","3":"homologs"},{"1":"chircus_homolog_perc_id_r1","2":"%id. query gene identical to target Goat gene","3":"homologs"},{"1":"chircus_homolog_goc_score","2":"Goat Gene-order conservation score","3":"homologs"},{"1":"chircus_homolog_wga_coverage","2":"Goat Whole-genome alignment coverage","3":"homologs"},{"1":"chircus_homolog_orthology_confidence","2":"Goat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mauratus_homolog_ensembl_gene","2":"Golden Hamster gene stable ID","3":"homologs"},{"1":"mauratus_homolog_associated_gene_name","2":"Golden Hamster gene name","3":"homologs"},{"1":"mauratus_homolog_ensembl_peptide","2":"Golden Hamster protein or transcript stable ID","3":"homologs"},{"1":"mauratus_homolog_chromosome","2":"Golden Hamster chromosome/scaffold name","3":"homologs"},{"1":"mauratus_homolog_chrom_start","2":"Golden Hamster chromosome/scaffold start (bp)","3":"homologs"},{"1":"mauratus_homolog_chrom_end","2":"Golden Hamster chromosome/scaffold end (bp)","3":"homologs"},{"1":"mauratus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mauratus_homolog_subtype","2":"Last common ancestor with Golden Hamster","3":"homologs"},{"1":"mauratus_homolog_orthology_type","2":"Golden Hamster homology type","3":"homologs"},{"1":"mauratus_homolog_perc_id","2":"%id. target Golden Hamster gene identical to query gene","3":"homologs"},{"1":"mauratus_homolog_perc_id_r1","2":"%id. query gene identical to target Golden Hamster gene","3":"homologs"},{"1":"mauratus_homolog_goc_score","2":"Golden Hamster Gene-order conservation score","3":"homologs"},{"1":"mauratus_homolog_wga_coverage","2":"Golden Hamster Whole-genome alignment coverage","3":"homologs"},{"1":"mauratus_homolog_orthology_confidence","2":"Golden Hamster orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"acchrysaetos_homolog_ensembl_gene","2":"Golden eagle gene stable ID","3":"homologs"},{"1":"acchrysaetos_homolog_associated_gene_name","2":"Golden eagle gene name","3":"homologs"},{"1":"acchrysaetos_homolog_ensembl_peptide","2":"Golden eagle protein or transcript stable ID","3":"homologs"},{"1":"acchrysaetos_homolog_chromosome","2":"Golden eagle chromosome/scaffold name","3":"homologs"},{"1":"acchrysaetos_homolog_chrom_start","2":"Golden eagle chromosome/scaffold start (bp)","3":"homologs"},{"1":"acchrysaetos_homolog_chrom_end","2":"Golden eagle chromosome/scaffold end (bp)","3":"homologs"},{"1":"acchrysaetos_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"acchrysaetos_homolog_subtype","2":"Last common ancestor with Golden eagle","3":"homologs"},{"1":"acchrysaetos_homolog_orthology_type","2":"Golden eagle homology type","3":"homologs"},{"1":"acchrysaetos_homolog_perc_id","2":"%id. target Golden eagle gene identical to query gene","3":"homologs"},{"1":"acchrysaetos_homolog_perc_id_r1","2":"%id. query gene identical to target Golden eagle gene","3":"homologs"},{"1":"acchrysaetos_homolog_goc_score","2":"Golden eagle Gene-order conservation score","3":"homologs"},{"1":"acchrysaetos_homolog_wga_coverage","2":"Golden eagle Whole-genome alignment coverage","3":"homologs"},{"1":"acchrysaetos_homolog_orthology_confidence","2":"Golden eagle orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"rroxellana_homolog_ensembl_gene","2":"Golden snub-nosed monkey gene stable ID","3":"homologs"},{"1":"rroxellana_homolog_associated_gene_name","2":"Golden snub-nosed monkey gene name","3":"homologs"},{"1":"rroxellana_homolog_ensembl_peptide","2":"Golden snub-nosed monkey protein or transcript stable ID","3":"homologs"},{"1":"rroxellana_homolog_chromosome","2":"Golden snub-nosed monkey chromosome/scaffold name","3":"homologs"},{"1":"rroxellana_homolog_chrom_start","2":"Golden snub-nosed monkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"rroxellana_homolog_chrom_end","2":"Golden snub-nosed monkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"rroxellana_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"rroxellana_homolog_subtype","2":"Last common ancestor with Golden snub-nosed monkey","3":"homologs"},{"1":"rroxellana_homolog_orthology_type","2":"Golden snub-nosed monkey homology type","3":"homologs"},{"1":"rroxellana_homolog_perc_id","2":"%id. target Golden snub-nosed monkey gene identical to query gene","3":"homologs"},{"1":"rroxellana_homolog_perc_id_r1","2":"%id. query gene identical to target Golden snub-nosed monkey gene","3":"homologs"},{"1":"rroxellana_homolog_goc_score","2":"Golden snub-nosed monkey Gene-order conservation score","3":"homologs"},{"1":"rroxellana_homolog_wga_coverage","2":"Golden snub-nosed monkey Whole-genome alignment coverage","3":"homologs"},{"1":"rroxellana_homolog_orthology_confidence","2":"Golden snub-nosed monkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sgrahami_homolog_ensembl_gene","2":"Golden-line barbel gene stable ID","3":"homologs"},{"1":"sgrahami_homolog_associated_gene_name","2":"Golden-line barbel gene name","3":"homologs"},{"1":"sgrahami_homolog_ensembl_peptide","2":"Golden-line barbel protein or transcript stable ID","3":"homologs"},{"1":"sgrahami_homolog_chromosome","2":"Golden-line barbel chromosome/scaffold name","3":"homologs"},{"1":"sgrahami_homolog_chrom_start","2":"Golden-line barbel chromosome/scaffold start (bp)","3":"homologs"},{"1":"sgrahami_homolog_chrom_end","2":"Golden-line barbel chromosome/scaffold end (bp)","3":"homologs"},{"1":"sgrahami_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sgrahami_homolog_subtype","2":"Last common ancestor with Golden-line barbel","3":"homologs"},{"1":"sgrahami_homolog_orthology_type","2":"Golden-line barbel homology type","3":"homologs"},{"1":"sgrahami_homolog_perc_id","2":"%id. target Golden-line barbel gene identical to query gene","3":"homologs"},{"1":"sgrahami_homolog_perc_id_r1","2":"%id. query gene identical to target Golden-line barbel gene","3":"homologs"},{"1":"sgrahami_homolog_goc_score","2":"Golden-line barbel Gene-order conservation score","3":"homologs"},{"1":"sgrahami_homolog_wga_coverage","2":"Golden-line barbel Whole-genome alignment coverage","3":"homologs"},{"1":"sgrahami_homolog_orthology_confidence","2":"Golden-line barbel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cauratus_homolog_ensembl_gene","2":"Goldfish gene stable ID","3":"homologs"},{"1":"cauratus_homolog_associated_gene_name","2":"Goldfish gene name","3":"homologs"},{"1":"cauratus_homolog_ensembl_peptide","2":"Goldfish protein or transcript stable ID","3":"homologs"},{"1":"cauratus_homolog_chromosome","2":"Goldfish chromosome/scaffold name","3":"homologs"},{"1":"cauratus_homolog_chrom_start","2":"Goldfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"cauratus_homolog_chrom_end","2":"Goldfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"cauratus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cauratus_homolog_subtype","2":"Last common ancestor with Goldfish","3":"homologs"},{"1":"cauratus_homolog_orthology_type","2":"Goldfish homology type","3":"homologs"},{"1":"cauratus_homolog_perc_id","2":"%id. target Goldfish gene identical to query gene","3":"homologs"},{"1":"cauratus_homolog_perc_id_r1","2":"%id. query gene identical to target Goldfish gene","3":"homologs"},{"1":"cauratus_homolog_goc_score","2":"Goldfish Gene-order conservation score","3":"homologs"},{"1":"cauratus_homolog_wga_coverage","2":"Goldfish Whole-genome alignment coverage","3":"homologs"},{"1":"cauratus_homolog_orthology_confidence","2":"Goldfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"gevgoodei_homolog_ensembl_gene","2":"Goodes thornscrub tortoise gene stable ID","3":"homologs"},{"1":"gevgoodei_homolog_associated_gene_name","2":"Goodes thornscrub tortoise gene name","3":"homologs"},{"1":"gevgoodei_homolog_ensembl_peptide","2":"Goodes thornscrub tortoise protein or transcript stable ID","3":"homologs"},{"1":"gevgoodei_homolog_chromosome","2":"Goodes thornscrub tortoise chromosome/scaffold name","3":"homologs"},{"1":"gevgoodei_homolog_chrom_start","2":"Goodes thornscrub tortoise chromosome/scaffold start (bp)","3":"homologs"},{"1":"gevgoodei_homolog_chrom_end","2":"Goodes thornscrub tortoise chromosome/scaffold end (bp)","3":"homologs"},{"1":"gevgoodei_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"gevgoodei_homolog_subtype","2":"Last common ancestor with Goodes thornscrub tortoise","3":"homologs"},{"1":"gevgoodei_homolog_orthology_type","2":"Goodes thornscrub tortoise homology type","3":"homologs"},{"1":"gevgoodei_homolog_perc_id","2":"%id. target Goodes thornscrub tortoise gene identical to query gene","3":"homologs"},{"1":"gevgoodei_homolog_perc_id_r1","2":"%id. query gene identical to target Goodes thornscrub tortoise gene","3":"homologs"},{"1":"gevgoodei_homolog_goc_score","2":"Goodes thornscrub tortoise Gene-order conservation score","3":"homologs"},{"1":"gevgoodei_homolog_wga_coverage","2":"Goodes thornscrub tortoise Whole-genome alignment coverage","3":"homologs"},{"1":"gevgoodei_homolog_orthology_confidence","2":"Goodes thornscrub tortoise orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ggorilla_homolog_ensembl_gene","2":"Gorilla gene stable ID","3":"homologs"},{"1":"ggorilla_homolog_associated_gene_name","2":"Gorilla gene name","3":"homologs"},{"1":"ggorilla_homolog_ensembl_peptide","2":"Gorilla protein or transcript stable ID","3":"homologs"},{"1":"ggorilla_homolog_chromosome","2":"Gorilla chromosome/scaffold name","3":"homologs"},{"1":"ggorilla_homolog_chrom_start","2":"Gorilla chromosome/scaffold start (bp)","3":"homologs"},{"1":"ggorilla_homolog_chrom_end","2":"Gorilla chromosome/scaffold end (bp)","3":"homologs"},{"1":"ggorilla_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ggorilla_homolog_subtype","2":"Last common ancestor with Gorilla","3":"homologs"},{"1":"ggorilla_homolog_orthology_type","2":"Gorilla homology type","3":"homologs"},{"1":"ggorilla_homolog_perc_id","2":"%id. target Gorilla gene identical to query gene","3":"homologs"},{"1":"ggorilla_homolog_perc_id_r1","2":"%id. query gene identical to target Gorilla gene","3":"homologs"},{"1":"ggorilla_homolog_goc_score","2":"Gorilla Gene-order conservation score","3":"homologs"},{"1":"ggorilla_homolog_wga_coverage","2":"Gorilla Whole-genome alignment coverage","3":"homologs"},{"1":"ggorilla_homolog_orthology_confidence","2":"Gorilla orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pmajor_homolog_ensembl_gene","2":"Great Tit gene stable ID","3":"homologs"},{"1":"pmajor_homolog_associated_gene_name","2":"Great Tit gene name","3":"homologs"},{"1":"pmajor_homolog_ensembl_peptide","2":"Great Tit protein or transcript stable ID","3":"homologs"},{"1":"pmajor_homolog_chromosome","2":"Great Tit chromosome/scaffold name","3":"homologs"},{"1":"pmajor_homolog_chrom_start","2":"Great Tit chromosome/scaffold start (bp)","3":"homologs"},{"1":"pmajor_homolog_chrom_end","2":"Great Tit chromosome/scaffold end (bp)","3":"homologs"},{"1":"pmajor_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pmajor_homolog_subtype","2":"Last common ancestor with Great Tit","3":"homologs"},{"1":"pmajor_homolog_orthology_type","2":"Great Tit homology type","3":"homologs"},{"1":"pmajor_homolog_perc_id","2":"%id. target Great Tit gene identical to query gene","3":"homologs"},{"1":"pmajor_homolog_perc_id_r1","2":"%id. query gene identical to target Great Tit gene","3":"homologs"},{"1":"pmajor_homolog_goc_score","2":"Great Tit Gene-order conservation score","3":"homologs"},{"1":"pmajor_homolog_wga_coverage","2":"Great Tit Whole-genome alignment coverage","3":"homologs"},{"1":"pmajor_homolog_orthology_confidence","2":"Great Tit orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sdumerili_homolog_ensembl_gene","2":"Greater amberjack gene stable ID","3":"homologs"},{"1":"sdumerili_homolog_associated_gene_name","2":"Greater amberjack gene name","3":"homologs"},{"1":"sdumerili_homolog_ensembl_peptide","2":"Greater amberjack protein or transcript stable ID","3":"homologs"},{"1":"sdumerili_homolog_chromosome","2":"Greater amberjack chromosome/scaffold name","3":"homologs"},{"1":"sdumerili_homolog_chrom_start","2":"Greater amberjack chromosome/scaffold start (bp)","3":"homologs"},{"1":"sdumerili_homolog_chrom_end","2":"Greater amberjack chromosome/scaffold end (bp)","3":"homologs"},{"1":"sdumerili_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sdumerili_homolog_subtype","2":"Last common ancestor with Greater amberjack","3":"homologs"},{"1":"sdumerili_homolog_orthology_type","2":"Greater amberjack homology type","3":"homologs"},{"1":"sdumerili_homolog_perc_id","2":"%id. target Greater amberjack gene identical to query gene","3":"homologs"},{"1":"sdumerili_homolog_perc_id_r1","2":"%id. query gene identical to target Greater amberjack gene","3":"homologs"},{"1":"sdumerili_homolog_goc_score","2":"Greater amberjack Gene-order conservation score","3":"homologs"},{"1":"sdumerili_homolog_wga_coverage","2":"Greater amberjack Whole-genome alignment coverage","3":"homologs"},{"1":"sdumerili_homolog_orthology_confidence","2":"Greater amberjack orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"psimus_homolog_ensembl_gene","2":"Greater bamboo lemur gene stable ID","3":"homologs"},{"1":"psimus_homolog_associated_gene_name","2":"Greater bamboo lemur gene name","3":"homologs"},{"1":"psimus_homolog_ensembl_peptide","2":"Greater bamboo lemur protein or transcript stable ID","3":"homologs"},{"1":"psimus_homolog_chromosome","2":"Greater bamboo lemur chromosome/scaffold name","3":"homologs"},{"1":"psimus_homolog_chrom_start","2":"Greater bamboo lemur chromosome/scaffold start (bp)","3":"homologs"},{"1":"psimus_homolog_chrom_end","2":"Greater bamboo lemur chromosome/scaffold end (bp)","3":"homologs"},{"1":"psimus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"psimus_homolog_subtype","2":"Last common ancestor with Greater bamboo lemur","3":"homologs"},{"1":"psimus_homolog_orthology_type","2":"Greater bamboo lemur homology type","3":"homologs"},{"1":"psimus_homolog_perc_id","2":"%id. target Greater bamboo lemur gene identical to query gene","3":"homologs"},{"1":"psimus_homolog_perc_id_r1","2":"%id. query gene identical to target Greater bamboo lemur gene","3":"homologs"},{"1":"psimus_homolog_goc_score","2":"Greater bamboo lemur Gene-order conservation score","3":"homologs"},{"1":"psimus_homolog_wga_coverage","2":"Greater bamboo lemur Whole-genome alignment coverage","3":"homologs"},{"1":"psimus_homolog_orthology_confidence","2":"Greater bamboo lemur orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"rferrumequinum_homolog_ensembl_gene","2":"Greater horseshoe bat gene stable ID","3":"homologs"},{"1":"rferrumequinum_homolog_associated_gene_name","2":"Greater horseshoe bat gene name","3":"homologs"},{"1":"rferrumequinum_homolog_ensembl_peptide","2":"Greater horseshoe bat protein or transcript stable ID","3":"homologs"},{"1":"rferrumequinum_homolog_chromosome","2":"Greater horseshoe bat chromosome/scaffold name","3":"homologs"},{"1":"rferrumequinum_homolog_chrom_start","2":"Greater horseshoe bat chromosome/scaffold start (bp)","3":"homologs"},{"1":"rferrumequinum_homolog_chrom_end","2":"Greater horseshoe bat chromosome/scaffold end (bp)","3":"homologs"},{"1":"rferrumequinum_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"rferrumequinum_homolog_subtype","2":"Last common ancestor with Greater horseshoe bat","3":"homologs"},{"1":"rferrumequinum_homolog_orthology_type","2":"Greater horseshoe bat homology type","3":"homologs"},{"1":"rferrumequinum_homolog_perc_id","2":"%id. target Greater horseshoe bat gene identical to query gene","3":"homologs"},{"1":"rferrumequinum_homolog_perc_id_r1","2":"%id. query gene identical to target Greater horseshoe bat gene","3":"homologs"},{"1":"rferrumequinum_homolog_goc_score","2":"Greater horseshoe bat Gene-order conservation score","3":"homologs"},{"1":"rferrumequinum_homolog_wga_coverage","2":"Greater horseshoe bat Whole-genome alignment coverage","3":"homologs"},{"1":"rferrumequinum_homolog_orthology_confidence","2":"Greater horseshoe bat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"acarolinensis_homolog_ensembl_gene","2":"Green anole gene stable ID","3":"homologs"},{"1":"acarolinensis_homolog_associated_gene_name","2":"Green anole gene name","3":"homologs"},{"1":"acarolinensis_homolog_ensembl_peptide","2":"Green anole protein or transcript stable ID","3":"homologs"},{"1":"acarolinensis_homolog_chromosome","2":"Green anole chromosome/scaffold name","3":"homologs"},{"1":"acarolinensis_homolog_chrom_start","2":"Green anole chromosome/scaffold start (bp)","3":"homologs"},{"1":"acarolinensis_homolog_chrom_end","2":"Green anole chromosome/scaffold end (bp)","3":"homologs"},{"1":"acarolinensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"acarolinensis_homolog_subtype","2":"Last common ancestor with Green anole","3":"homologs"},{"1":"acarolinensis_homolog_orthology_type","2":"Green anole homology type","3":"homologs"},{"1":"acarolinensis_homolog_perc_id","2":"%id. target Green anole gene identical to query gene","3":"homologs"},{"1":"acarolinensis_homolog_perc_id_r1","2":"%id. query gene identical to target Green anole gene","3":"homologs"},{"1":"acarolinensis_homolog_goc_score","2":"Green anole Gene-order conservation score","3":"homologs"},{"1":"acarolinensis_homolog_orthology_confidence","2":"Green anole orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cporcellus_homolog_ensembl_gene","2":"Guinea Pig gene stable ID","3":"homologs"},{"1":"cporcellus_homolog_associated_gene_name","2":"Guinea Pig gene name","3":"homologs"},{"1":"cporcellus_homolog_ensembl_peptide","2":"Guinea Pig protein or transcript stable ID","3":"homologs"},{"1":"cporcellus_homolog_chromosome","2":"Guinea Pig chromosome/scaffold name","3":"homologs"},{"1":"cporcellus_homolog_chrom_start","2":"Guinea Pig chromosome/scaffold start (bp)","3":"homologs"},{"1":"cporcellus_homolog_chrom_end","2":"Guinea Pig chromosome/scaffold end (bp)","3":"homologs"},{"1":"cporcellus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cporcellus_homolog_subtype","2":"Last common ancestor with Guinea Pig","3":"homologs"},{"1":"cporcellus_homolog_orthology_type","2":"Guinea Pig homology type","3":"homologs"},{"1":"cporcellus_homolog_perc_id","2":"%id. target Guinea Pig gene identical to query gene","3":"homologs"},{"1":"cporcellus_homolog_perc_id_r1","2":"%id. query gene identical to target Guinea Pig gene","3":"homologs"},{"1":"cporcellus_homolog_goc_score","2":"Guinea Pig Gene-order conservation score","3":"homologs"},{"1":"cporcellus_homolog_wga_coverage","2":"Guinea Pig Whole-genome alignment coverage","3":"homologs"},{"1":"cporcellus_homolog_orthology_confidence","2":"Guinea Pig orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"preticulata_homolog_ensembl_gene","2":"Guppy gene stable ID","3":"homologs"},{"1":"preticulata_homolog_associated_gene_name","2":"Guppy gene name","3":"homologs"},{"1":"preticulata_homolog_ensembl_peptide","2":"Guppy protein or transcript stable ID","3":"homologs"},{"1":"preticulata_homolog_chromosome","2":"Guppy chromosome/scaffold name","3":"homologs"},{"1":"preticulata_homolog_chrom_start","2":"Guppy chromosome/scaffold start (bp)","3":"homologs"},{"1":"preticulata_homolog_chrom_end","2":"Guppy chromosome/scaffold end (bp)","3":"homologs"},{"1":"preticulata_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"preticulata_homolog_subtype","2":"Last common ancestor with Guppy","3":"homologs"},{"1":"preticulata_homolog_orthology_type","2":"Guppy homology type","3":"homologs"},{"1":"preticulata_homolog_perc_id","2":"%id. target Guppy gene identical to query gene","3":"homologs"},{"1":"preticulata_homolog_perc_id_r1","2":"%id. query gene identical to target Guppy gene","3":"homologs"},{"1":"preticulata_homolog_goc_score","2":"Guppy Gene-order conservation score","3":"homologs"},{"1":"preticulata_homolog_wga_coverage","2":"Guppy Whole-genome alignment coverage","3":"homologs"},{"1":"preticulata_homolog_orthology_confidence","2":"Guppy orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"eburgeri_homolog_ensembl_gene","2":"Hagfish gene stable ID","3":"homologs"},{"1":"eburgeri_homolog_associated_gene_name","2":"Hagfish gene name","3":"homologs"},{"1":"eburgeri_homolog_ensembl_peptide","2":"Hagfish protein or transcript stable ID","3":"homologs"},{"1":"eburgeri_homolog_chromosome","2":"Hagfish chromosome/scaffold name","3":"homologs"},{"1":"eburgeri_homolog_chrom_start","2":"Hagfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"eburgeri_homolog_chrom_end","2":"Hagfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"eburgeri_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"eburgeri_homolog_subtype","2":"Last common ancestor with Hagfish","3":"homologs"},{"1":"eburgeri_homolog_orthology_type","2":"Hagfish homology type","3":"homologs"},{"1":"eburgeri_homolog_perc_id","2":"%id. target Hagfish gene identical to query gene","3":"homologs"},{"1":"eburgeri_homolog_perc_id_r1","2":"%id. query gene identical to target Hagfish gene","3":"homologs"},{"1":"eburgeri_homolog_wga_coverage","2":"Hagfish Whole-genome alignment coverage","3":"homologs"},{"1":"eburgeri_homolog_orthology_confidence","2":"Hagfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"eeuropaeus_homolog_ensembl_gene","2":"Hedgehog gene stable ID","3":"homologs"},{"1":"eeuropaeus_homolog_associated_gene_name","2":"Hedgehog gene name","3":"homologs"},{"1":"eeuropaeus_homolog_ensembl_peptide","2":"Hedgehog protein or transcript stable ID","3":"homologs"},{"1":"eeuropaeus_homolog_chromosome","2":"Hedgehog chromosome/scaffold name","3":"homologs"},{"1":"eeuropaeus_homolog_chrom_start","2":"Hedgehog chromosome/scaffold start (bp)","3":"homologs"},{"1":"eeuropaeus_homolog_chrom_end","2":"Hedgehog chromosome/scaffold end (bp)","3":"homologs"},{"1":"eeuropaeus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"eeuropaeus_homolog_subtype","2":"Last common ancestor with Hedgehog","3":"homologs"},{"1":"eeuropaeus_homolog_orthology_type","2":"Hedgehog homology type","3":"homologs"},{"1":"eeuropaeus_homolog_perc_id","2":"%id. target Hedgehog gene identical to query gene","3":"homologs"},{"1":"eeuropaeus_homolog_perc_id_r1","2":"%id. query gene identical to target Hedgehog gene","3":"homologs"},{"1":"eeuropaeus_homolog_goc_score","2":"Hedgehog Gene-order conservation score","3":"homologs"},{"1":"eeuropaeus_homolog_wga_coverage","2":"Hedgehog Whole-genome alignment coverage","3":"homologs"},{"1":"eeuropaeus_homolog_orthology_confidence","2":"Hedgehog orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ecaballus_homolog_ensembl_gene","2":"Horse gene stable ID","3":"homologs"},{"1":"ecaballus_homolog_associated_gene_name","2":"Horse gene name","3":"homologs"},{"1":"ecaballus_homolog_ensembl_peptide","2":"Horse protein or transcript stable ID","3":"homologs"},{"1":"ecaballus_homolog_chromosome","2":"Horse chromosome/scaffold name","3":"homologs"},{"1":"ecaballus_homolog_chrom_start","2":"Horse chromosome/scaffold start (bp)","3":"homologs"},{"1":"ecaballus_homolog_chrom_end","2":"Horse chromosome/scaffold end (bp)","3":"homologs"},{"1":"ecaballus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ecaballus_homolog_subtype","2":"Last common ancestor with Horse","3":"homologs"},{"1":"ecaballus_homolog_orthology_type","2":"Horse homology type","3":"homologs"},{"1":"ecaballus_homolog_perc_id","2":"%id. target Horse gene identical to query gene","3":"homologs"},{"1":"ecaballus_homolog_perc_id_r1","2":"%id. query gene identical to target Horse gene","3":"homologs"},{"1":"ecaballus_homolog_goc_score","2":"Horse Gene-order conservation score","3":"homologs"},{"1":"ecaballus_homolog_wga_coverage","2":"Horse Whole-genome alignment coverage","3":"homologs"},{"1":"ecaballus_homolog_orthology_confidence","2":"Horse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"hhucho_homolog_ensembl_gene","2":"Huchen gene stable ID","3":"homologs"},{"1":"hhucho_homolog_associated_gene_name","2":"Huchen gene name","3":"homologs"},{"1":"hhucho_homolog_ensembl_peptide","2":"Huchen protein or transcript stable ID","3":"homologs"},{"1":"hhucho_homolog_chromosome","2":"Huchen chromosome/scaffold name","3":"homologs"},{"1":"hhucho_homolog_chrom_start","2":"Huchen chromosome/scaffold start (bp)","3":"homologs"},{"1":"hhucho_homolog_chrom_end","2":"Huchen chromosome/scaffold end (bp)","3":"homologs"},{"1":"hhucho_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"hhucho_homolog_subtype","2":"Last common ancestor with Huchen","3":"homologs"},{"1":"hhucho_homolog_orthology_type","2":"Huchen homology type","3":"homologs"},{"1":"hhucho_homolog_perc_id","2":"%id. target Huchen gene identical to query gene","3":"homologs"},{"1":"hhucho_homolog_perc_id_r1","2":"%id. query gene identical to target Huchen gene","3":"homologs"},{"1":"hhucho_homolog_goc_score","2":"Huchen Gene-order conservation score","3":"homologs"},{"1":"hhucho_homolog_wga_coverage","2":"Huchen Whole-genome alignment coverage","3":"homologs"},{"1":"hhucho_homolog_orthology_confidence","2":"Huchen orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bihybrid_homolog_ensembl_gene","2":"Hybrid - Bos Indicus gene stable ID","3":"homologs"},{"1":"bihybrid_homolog_associated_gene_name","2":"Hybrid - Bos Indicus gene name","3":"homologs"},{"1":"bihybrid_homolog_ensembl_peptide","2":"Hybrid - Bos Indicus protein or transcript stable ID","3":"homologs"},{"1":"bihybrid_homolog_chromosome","2":"Hybrid - Bos Indicus chromosome/scaffold name","3":"homologs"},{"1":"bihybrid_homolog_chrom_start","2":"Hybrid - Bos Indicus chromosome/scaffold start (bp)","3":"homologs"},{"1":"bihybrid_homolog_chrom_end","2":"Hybrid - Bos Indicus chromosome/scaffold end (bp)","3":"homologs"},{"1":"bihybrid_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bihybrid_homolog_subtype","2":"Last common ancestor with Hybrid - Bos Indicus","3":"homologs"},{"1":"bihybrid_homolog_orthology_type","2":"Hybrid - Bos Indicus homology type","3":"homologs"},{"1":"bihybrid_homolog_perc_id","2":"%id. target Hybrid - Bos Indicus gene identical to query gene","3":"homologs"},{"1":"bihybrid_homolog_perc_id_r1","2":"%id. query gene identical to target Hybrid - Bos Indicus gene","3":"homologs"},{"1":"bihybrid_homolog_goc_score","2":"Hybrid - Bos Indicus Gene-order conservation score","3":"homologs"},{"1":"bihybrid_homolog_wga_coverage","2":"Hybrid - Bos Indicus Whole-genome alignment coverage","3":"homologs"},{"1":"bihybrid_homolog_orthology_confidence","2":"Hybrid - Bos Indicus orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pcapensis_homolog_ensembl_gene","2":"Hyrax gene stable ID","3":"homologs"},{"1":"pcapensis_homolog_associated_gene_name","2":"Hyrax gene name","3":"homologs"},{"1":"pcapensis_homolog_ensembl_peptide","2":"Hyrax protein or transcript stable ID","3":"homologs"},{"1":"pcapensis_homolog_chromosome","2":"Hyrax chromosome/scaffold name","3":"homologs"},{"1":"pcapensis_homolog_chrom_start","2":"Hyrax chromosome/scaffold start (bp)","3":"homologs"},{"1":"pcapensis_homolog_chrom_end","2":"Hyrax chromosome/scaffold end (bp)","3":"homologs"},{"1":"pcapensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pcapensis_homolog_subtype","2":"Last common ancestor with Hyrax","3":"homologs"},{"1":"pcapensis_homolog_orthology_type","2":"Hyrax homology type","3":"homologs"},{"1":"pcapensis_homolog_perc_id","2":"%id. target Hyrax gene identical to query gene","3":"homologs"},{"1":"pcapensis_homolog_perc_id_r1","2":"%id. query gene identical to target Hyrax gene","3":"homologs"},{"1":"pcapensis_homolog_goc_score","2":"Hyrax Gene-order conservation score","3":"homologs"},{"1":"pcapensis_homolog_wga_coverage","2":"Hyrax Whole-genome alignment coverage","3":"homologs"},{"1":"pcapensis_homolog_orthology_confidence","2":"Hyrax orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nnaja_homolog_ensembl_gene","2":"Indian cobra gene stable ID","3":"homologs"},{"1":"nnaja_homolog_associated_gene_name","2":"Indian cobra gene name","3":"homologs"},{"1":"nnaja_homolog_ensembl_peptide","2":"Indian cobra protein or transcript stable ID","3":"homologs"},{"1":"nnaja_homolog_chromosome","2":"Indian cobra chromosome/scaffold name","3":"homologs"},{"1":"nnaja_homolog_chrom_start","2":"Indian cobra chromosome/scaffold start (bp)","3":"homologs"},{"1":"nnaja_homolog_chrom_end","2":"Indian cobra chromosome/scaffold end (bp)","3":"homologs"},{"1":"nnaja_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nnaja_homolog_subtype","2":"Last common ancestor with Indian cobra","3":"homologs"},{"1":"nnaja_homolog_orthology_type","2":"Indian cobra homology type","3":"homologs"},{"1":"nnaja_homolog_perc_id","2":"%id. target Indian cobra gene identical to query gene","3":"homologs"},{"1":"nnaja_homolog_perc_id_r1","2":"%id. query gene identical to target Indian cobra gene","3":"homologs"},{"1":"nnaja_homolog_goc_score","2":"Indian cobra Gene-order conservation score","3":"homologs"},{"1":"nnaja_homolog_wga_coverage","2":"Indian cobra Whole-genome alignment coverage","3":"homologs"},{"1":"nnaja_homolog_orthology_confidence","2":"Indian cobra orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"omelastigma_homolog_ensembl_gene","2":"Indian medaka gene stable ID","3":"homologs"},{"1":"omelastigma_homolog_associated_gene_name","2":"Indian medaka gene name","3":"homologs"},{"1":"omelastigma_homolog_ensembl_peptide","2":"Indian medaka protein or transcript stable ID","3":"homologs"},{"1":"omelastigma_homolog_chromosome","2":"Indian medaka chromosome/scaffold name","3":"homologs"},{"1":"omelastigma_homolog_chrom_start","2":"Indian medaka chromosome/scaffold start (bp)","3":"homologs"},{"1":"omelastigma_homolog_chrom_end","2":"Indian medaka chromosome/scaffold end (bp)","3":"homologs"},{"1":"omelastigma_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"omelastigma_homolog_subtype","2":"Last common ancestor with Indian medaka","3":"homologs"},{"1":"omelastigma_homolog_orthology_type","2":"Indian medaka homology type","3":"homologs"},{"1":"omelastigma_homolog_perc_id","2":"%id. target Indian medaka gene identical to query gene","3":"homologs"},{"1":"omelastigma_homolog_perc_id_r1","2":"%id. query gene identical to target Indian medaka gene","3":"homologs"},{"1":"omelastigma_homolog_goc_score","2":"Indian medaka Gene-order conservation score","3":"homologs"},{"1":"omelastigma_homolog_wga_coverage","2":"Indian medaka Whole-genome alignment coverage","3":"homologs"},{"1":"omelastigma_homolog_orthology_confidence","2":"Indian medaka orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"olatipes_homolog_ensembl_gene","2":"Japanese medaka HdrR gene stable ID","3":"homologs"},{"1":"olatipes_homolog_associated_gene_name","2":"Japanese medaka HdrR gene name","3":"homologs"},{"1":"olatipes_homolog_ensembl_peptide","2":"Japanese medaka HdrR protein or transcript stable ID","3":"homologs"},{"1":"olatipes_homolog_chromosome","2":"Japanese medaka HdrR chromosome/scaffold name","3":"homologs"},{"1":"olatipes_homolog_chrom_start","2":"Japanese medaka HdrR chromosome/scaffold start (bp)","3":"homologs"},{"1":"olatipes_homolog_chrom_end","2":"Japanese medaka HdrR chromosome/scaffold end (bp)","3":"homologs"},{"1":"olatipes_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"olatipes_homolog_subtype","2":"Last common ancestor with Japanese medaka HdrR","3":"homologs"},{"1":"olatipes_homolog_orthology_type","2":"Japanese medaka HdrR homology type","3":"homologs"},{"1":"olatipes_homolog_perc_id","2":"%id. target Japanese medaka HdrR gene identical to query gene","3":"homologs"},{"1":"olatipes_homolog_perc_id_r1","2":"%id. query gene identical to target Japanese medaka HdrR gene","3":"homologs"},{"1":"olatipes_homolog_goc_score","2":"Japanese medaka HdrR Gene-order conservation score","3":"homologs"},{"1":"olatipes_homolog_wga_coverage","2":"Japanese medaka HdrR Whole-genome alignment coverage","3":"homologs"},{"1":"olatipes_homolog_orthology_confidence","2":"Japanese medaka HdrR orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cjaponica_homolog_ensembl_gene","2":"Japanese quail gene stable ID","3":"homologs"},{"1":"cjaponica_homolog_associated_gene_name","2":"Japanese quail gene name","3":"homologs"},{"1":"cjaponica_homolog_ensembl_peptide","2":"Japanese quail protein or transcript stable ID","3":"homologs"},{"1":"cjaponica_homolog_chromosome","2":"Japanese quail chromosome/scaffold name","3":"homologs"},{"1":"cjaponica_homolog_chrom_start","2":"Japanese quail chromosome/scaffold start (bp)","3":"homologs"},{"1":"cjaponica_homolog_chrom_end","2":"Japanese quail chromosome/scaffold end (bp)","3":"homologs"},{"1":"cjaponica_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cjaponica_homolog_subtype","2":"Last common ancestor with Japanese quail","3":"homologs"},{"1":"cjaponica_homolog_orthology_type","2":"Japanese quail homology type","3":"homologs"},{"1":"cjaponica_homolog_perc_id","2":"%id. target Japanese quail gene identical to query gene","3":"homologs"},{"1":"cjaponica_homolog_perc_id_r1","2":"%id. query gene identical to target Japanese quail gene","3":"homologs"},{"1":"cjaponica_homolog_goc_score","2":"Japanese quail Gene-order conservation score","3":"homologs"},{"1":"cjaponica_homolog_wga_coverage","2":"Japanese quail Whole-genome alignment coverage","3":"homologs"},{"1":"cjaponica_homolog_orthology_confidence","2":"Japanese quail orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ojavanicus_homolog_ensembl_gene","2":"Javanese ricefish gene stable ID","3":"homologs"},{"1":"ojavanicus_homolog_associated_gene_name","2":"Javanese ricefish gene name","3":"homologs"},{"1":"ojavanicus_homolog_ensembl_peptide","2":"Javanese ricefish protein or transcript stable ID","3":"homologs"},{"1":"ojavanicus_homolog_chromosome","2":"Javanese ricefish chromosome/scaffold name","3":"homologs"},{"1":"ojavanicus_homolog_chrom_start","2":"Javanese ricefish chromosome/scaffold start (bp)","3":"homologs"},{"1":"ojavanicus_homolog_chrom_end","2":"Javanese ricefish chromosome/scaffold end (bp)","3":"homologs"},{"1":"ojavanicus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ojavanicus_homolog_subtype","2":"Last common ancestor with Javanese ricefish","3":"homologs"},{"1":"ojavanicus_homolog_orthology_type","2":"Javanese ricefish homology type","3":"homologs"},{"1":"ojavanicus_homolog_perc_id","2":"%id. target Javanese ricefish gene identical to query gene","3":"homologs"},{"1":"ojavanicus_homolog_perc_id_r1","2":"%id. query gene identical to target Javanese ricefish gene","3":"homologs"},{"1":"ojavanicus_homolog_goc_score","2":"Javanese ricefish Gene-order conservation score","3":"homologs"},{"1":"ojavanicus_homolog_wga_coverage","2":"Javanese ricefish Whole-genome alignment coverage","3":"homologs"},{"1":"ojavanicus_homolog_orthology_confidence","2":"Javanese ricefish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"shabroptila_homolog_ensembl_gene","2":"Kakapo gene stable ID","3":"homologs"},{"1":"shabroptila_homolog_associated_gene_name","2":"Kakapo gene name","3":"homologs"},{"1":"shabroptila_homolog_ensembl_peptide","2":"Kakapo protein or transcript stable ID","3":"homologs"},{"1":"shabroptila_homolog_chromosome","2":"Kakapo chromosome/scaffold name","3":"homologs"},{"1":"shabroptila_homolog_chrom_start","2":"Kakapo chromosome/scaffold start (bp)","3":"homologs"},{"1":"shabroptila_homolog_chrom_end","2":"Kakapo chromosome/scaffold end (bp)","3":"homologs"},{"1":"shabroptila_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"shabroptila_homolog_subtype","2":"Last common ancestor with Kakapo","3":"homologs"},{"1":"shabroptila_homolog_orthology_type","2":"Kakapo homology type","3":"homologs"},{"1":"shabroptila_homolog_perc_id","2":"%id. target Kakapo gene identical to query gene","3":"homologs"},{"1":"shabroptila_homolog_perc_id_r1","2":"%id. query gene identical to target Kakapo gene","3":"homologs"},{"1":"shabroptila_homolog_goc_score","2":"Kakapo Gene-order conservation score","3":"homologs"},{"1":"shabroptila_homolog_wga_coverage","2":"Kakapo Whole-genome alignment coverage","3":"homologs"},{"1":"shabroptila_homolog_orthology_confidence","2":"Kakapo orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"dordii_homolog_ensembl_gene","2":"Kangaroo rat gene stable ID","3":"homologs"},{"1":"dordii_homolog_associated_gene_name","2":"Kangaroo rat gene name","3":"homologs"},{"1":"dordii_homolog_ensembl_peptide","2":"Kangaroo rat protein or transcript stable ID","3":"homologs"},{"1":"dordii_homolog_chromosome","2":"Kangaroo rat chromosome/scaffold name","3":"homologs"},{"1":"dordii_homolog_chrom_start","2":"Kangaroo rat chromosome/scaffold start (bp)","3":"homologs"},{"1":"dordii_homolog_chrom_end","2":"Kangaroo rat chromosome/scaffold end (bp)","3":"homologs"},{"1":"dordii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"dordii_homolog_subtype","2":"Last common ancestor with Kangaroo rat","3":"homologs"},{"1":"dordii_homolog_orthology_type","2":"Kangaroo rat homology type","3":"homologs"},{"1":"dordii_homolog_perc_id","2":"%id. target Kangaroo rat gene identical to query gene","3":"homologs"},{"1":"dordii_homolog_perc_id_r1","2":"%id. query gene identical to target Kangaroo rat gene","3":"homologs"},{"1":"dordii_homolog_goc_score","2":"Kangaroo rat Gene-order conservation score","3":"homologs"},{"1":"dordii_homolog_wga_coverage","2":"Kangaroo rat Whole-genome alignment coverage","3":"homologs"},{"1":"dordii_homolog_orthology_confidence","2":"Kangaroo rat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pcinereus_homolog_ensembl_gene","2":"Koala gene stable ID","3":"homologs"},{"1":"pcinereus_homolog_associated_gene_name","2":"Koala gene name","3":"homologs"},{"1":"pcinereus_homolog_ensembl_peptide","2":"Koala protein or transcript stable ID","3":"homologs"},{"1":"pcinereus_homolog_chromosome","2":"Koala chromosome/scaffold name","3":"homologs"},{"1":"pcinereus_homolog_chrom_start","2":"Koala chromosome/scaffold start (bp)","3":"homologs"},{"1":"pcinereus_homolog_chrom_end","2":"Koala chromosome/scaffold end (bp)","3":"homologs"},{"1":"pcinereus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pcinereus_homolog_subtype","2":"Last common ancestor with Koala","3":"homologs"},{"1":"pcinereus_homolog_orthology_type","2":"Koala homology type","3":"homologs"},{"1":"pcinereus_homolog_perc_id","2":"%id. target Koala gene identical to query gene","3":"homologs"},{"1":"pcinereus_homolog_perc_id_r1","2":"%id. query gene identical to target Koala gene","3":"homologs"},{"1":"pcinereus_homolog_goc_score","2":"Koala Gene-order conservation score","3":"homologs"},{"1":"pcinereus_homolog_wga_coverage","2":"Koala Whole-genome alignment coverage","3":"homologs"},{"1":"pcinereus_homolog_orthology_confidence","2":"Koala orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pmarinus_homolog_ensembl_gene","2":"Lamprey gene stable ID","3":"homologs"},{"1":"pmarinus_homolog_associated_gene_name","2":"Lamprey gene name","3":"homologs"},{"1":"pmarinus_homolog_ensembl_peptide","2":"Lamprey protein or transcript stable ID","3":"homologs"},{"1":"pmarinus_homolog_chromosome","2":"Lamprey chromosome/scaffold name","3":"homologs"},{"1":"pmarinus_homolog_chrom_start","2":"Lamprey chromosome/scaffold start (bp)","3":"homologs"},{"1":"pmarinus_homolog_chrom_end","2":"Lamprey chromosome/scaffold end (bp)","3":"homologs"},{"1":"pmarinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pmarinus_homolog_subtype","2":"Last common ancestor with Lamprey","3":"homologs"},{"1":"pmarinus_homolog_orthology_type","2":"Lamprey homology type","3":"homologs"},{"1":"pmarinus_homolog_perc_id","2":"%id. target Lamprey gene identical to query gene","3":"homologs"},{"1":"pmarinus_homolog_perc_id_r1","2":"%id. query gene identical to target Lamprey gene","3":"homologs"},{"1":"pmarinus_homolog_wga_coverage","2":"Lamprey Whole-genome alignment coverage","3":"homologs"},{"1":"pmarinus_homolog_orthology_confidence","2":"Lamprey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lcrocea_homolog_ensembl_gene","2":"Large yellow croaker gene stable ID","3":"homologs"},{"1":"lcrocea_homolog_associated_gene_name","2":"Large yellow croaker gene name","3":"homologs"},{"1":"lcrocea_homolog_ensembl_peptide","2":"Large yellow croaker protein or transcript stable ID","3":"homologs"},{"1":"lcrocea_homolog_chromosome","2":"Large yellow croaker chromosome/scaffold name","3":"homologs"},{"1":"lcrocea_homolog_chrom_start","2":"Large yellow croaker chromosome/scaffold start (bp)","3":"homologs"},{"1":"lcrocea_homolog_chrom_end","2":"Large yellow croaker chromosome/scaffold end (bp)","3":"homologs"},{"1":"lcrocea_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lcrocea_homolog_subtype","2":"Last common ancestor with Large yellow croaker","3":"homologs"},{"1":"lcrocea_homolog_orthology_type","2":"Large yellow croaker homology type","3":"homologs"},{"1":"lcrocea_homolog_perc_id","2":"%id. target Large yellow croaker gene identical to query gene","3":"homologs"},{"1":"lcrocea_homolog_perc_id_r1","2":"%id. query gene identical to target Large yellow croaker gene","3":"homologs"},{"1":"lcrocea_homolog_goc_score","2":"Large yellow croaker Gene-order conservation score","3":"homologs"},{"1":"lcrocea_homolog_wga_coverage","2":"Large yellow croaker Whole-genome alignment coverage","3":"homologs"},{"1":"lcrocea_homolog_orthology_confidence","2":"Large yellow croaker orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"lleishanense_homolog_ensembl_gene","2":"Leishan spiny toad gene stable ID","3":"homologs"},{"1":"lleishanense_homolog_associated_gene_name","2":"Leishan spiny toad gene name","3":"homologs"},{"1":"lleishanense_homolog_ensembl_peptide","2":"Leishan spiny toad protein or transcript stable ID","3":"homologs"},{"1":"lleishanense_homolog_chromosome","2":"Leishan spiny toad chromosome/scaffold name","3":"homologs"},{"1":"lleishanense_homolog_chrom_start","2":"Leishan spiny toad chromosome/scaffold start (bp)","3":"homologs"},{"1":"lleishanense_homolog_chrom_end","2":"Leishan spiny toad chromosome/scaffold end (bp)","3":"homologs"},{"1":"lleishanense_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"lleishanense_homolog_subtype","2":"Last common ancestor with Leishan spiny toad","3":"homologs"},{"1":"lleishanense_homolog_orthology_type","2":"Leishan spiny toad homology type","3":"homologs"},{"1":"lleishanense_homolog_perc_id","2":"%id. target Leishan spiny toad gene identical to query gene","3":"homologs"},{"1":"lleishanense_homolog_perc_id_r1","2":"%id. query gene identical to target Leishan spiny toad gene","3":"homologs"},{"1":"lleishanense_homolog_goc_score","2":"Leishan spiny toad Gene-order conservation score","3":"homologs"},{"1":"lleishanense_homolog_wga_coverage","2":"Leishan spiny toad Whole-genome alignment coverage","3":"homologs"},{"1":"lleishanense_homolog_orthology_confidence","2":"Leishan spiny toad orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ppardus_homolog_ensembl_gene","2":"Leopard gene stable ID","3":"homologs"},{"1":"ppardus_homolog_associated_gene_name","2":"Leopard gene name","3":"homologs"},{"1":"ppardus_homolog_ensembl_peptide","2":"Leopard protein or transcript stable ID","3":"homologs"},{"1":"ppardus_homolog_chromosome","2":"Leopard chromosome/scaffold name","3":"homologs"},{"1":"ppardus_homolog_chrom_start","2":"Leopard chromosome/scaffold start (bp)","3":"homologs"},{"1":"ppardus_homolog_chrom_end","2":"Leopard chromosome/scaffold end (bp)","3":"homologs"},{"1":"ppardus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ppardus_homolog_subtype","2":"Last common ancestor with Leopard","3":"homologs"},{"1":"ppardus_homolog_orthology_type","2":"Leopard homology type","3":"homologs"},{"1":"ppardus_homolog_perc_id","2":"%id. target Leopard gene identical to query gene","3":"homologs"},{"1":"ppardus_homolog_perc_id_r1","2":"%id. query gene identical to target Leopard gene","3":"homologs"},{"1":"ppardus_homolog_goc_score","2":"Leopard Gene-order conservation score","3":"homologs"},{"1":"ppardus_homolog_wga_coverage","2":"Leopard Whole-genome alignment coverage","3":"homologs"},{"1":"ppardus_homolog_orthology_confidence","2":"Leopard orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"jjaculus_homolog_ensembl_gene","2":"Lesser Egyptian jerboa gene stable ID","3":"homologs"},{"1":"jjaculus_homolog_associated_gene_name","2":"Lesser Egyptian jerboa gene name","3":"homologs"},{"1":"jjaculus_homolog_ensembl_peptide","2":"Lesser Egyptian jerboa protein or transcript stable ID","3":"homologs"},{"1":"jjaculus_homolog_chromosome","2":"Lesser Egyptian jerboa chromosome/scaffold name","3":"homologs"},{"1":"jjaculus_homolog_chrom_start","2":"Lesser Egyptian jerboa chromosome/scaffold start (bp)","3":"homologs"},{"1":"jjaculus_homolog_chrom_end","2":"Lesser Egyptian jerboa chromosome/scaffold end (bp)","3":"homologs"},{"1":"jjaculus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"jjaculus_homolog_subtype","2":"Last common ancestor with Lesser Egyptian jerboa","3":"homologs"},{"1":"jjaculus_homolog_orthology_type","2":"Lesser Egyptian jerboa homology type","3":"homologs"},{"1":"jjaculus_homolog_perc_id","2":"%id. target Lesser Egyptian jerboa gene identical to query gene","3":"homologs"},{"1":"jjaculus_homolog_perc_id_r1","2":"%id. query gene identical to target Lesser Egyptian jerboa gene","3":"homologs"},{"1":"jjaculus_homolog_goc_score","2":"Lesser Egyptian jerboa Gene-order conservation score","3":"homologs"},{"1":"jjaculus_homolog_wga_coverage","2":"Lesser Egyptian jerboa Whole-genome alignment coverage","3":"homologs"},{"1":"jjaculus_homolog_orthology_confidence","2":"Lesser Egyptian jerboa orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"etelfairi_homolog_ensembl_gene","2":"Lesser hedgehog tenrec gene stable ID","3":"homologs"},{"1":"etelfairi_homolog_associated_gene_name","2":"Lesser hedgehog tenrec gene name","3":"homologs"},{"1":"etelfairi_homolog_ensembl_peptide","2":"Lesser hedgehog tenrec protein or transcript stable ID","3":"homologs"},{"1":"etelfairi_homolog_chromosome","2":"Lesser hedgehog tenrec chromosome/scaffold name","3":"homologs"},{"1":"etelfairi_homolog_chrom_start","2":"Lesser hedgehog tenrec chromosome/scaffold start (bp)","3":"homologs"},{"1":"etelfairi_homolog_chrom_end","2":"Lesser hedgehog tenrec chromosome/scaffold end (bp)","3":"homologs"},{"1":"etelfairi_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"etelfairi_homolog_subtype","2":"Last common ancestor with Lesser hedgehog tenrec","3":"homologs"},{"1":"etelfairi_homolog_orthology_type","2":"Lesser hedgehog tenrec homology type","3":"homologs"},{"1":"etelfairi_homolog_perc_id","2":"%id. target Lesser hedgehog tenrec gene identical to query gene","3":"homologs"},{"1":"etelfairi_homolog_perc_id_r1","2":"%id. query gene identical to target Lesser hedgehog tenrec gene","3":"homologs"},{"1":"etelfairi_homolog_goc_score","2":"Lesser hedgehog tenrec Gene-order conservation score","3":"homologs"},{"1":"etelfairi_homolog_wga_coverage","2":"Lesser hedgehog tenrec Whole-genome alignment coverage","3":"homologs"},{"1":"etelfairi_homolog_orthology_confidence","2":"Lesser hedgehog tenrec orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pleo_homolog_ensembl_gene","2":"Lion gene stable ID","3":"homologs"},{"1":"pleo_homolog_associated_gene_name","2":"Lion gene name","3":"homologs"},{"1":"pleo_homolog_ensembl_peptide","2":"Lion protein or transcript stable ID","3":"homologs"},{"1":"pleo_homolog_chromosome","2":"Lion chromosome/scaffold name","3":"homologs"},{"1":"pleo_homolog_chrom_start","2":"Lion chromosome/scaffold start (bp)","3":"homologs"},{"1":"pleo_homolog_chrom_end","2":"Lion chromosome/scaffold end (bp)","3":"homologs"},{"1":"pleo_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pleo_homolog_subtype","2":"Last common ancestor with Lion","3":"homologs"},{"1":"pleo_homolog_orthology_type","2":"Lion homology type","3":"homologs"},{"1":"pleo_homolog_perc_id","2":"%id. target Lion gene identical to query gene","3":"homologs"},{"1":"pleo_homolog_perc_id_r1","2":"%id. query gene identical to target Lion gene","3":"homologs"},{"1":"pleo_homolog_goc_score","2":"Lion Gene-order conservation score","3":"homologs"},{"1":"pleo_homolog_wga_coverage","2":"Lion Whole-genome alignment coverage","3":"homologs"},{"1":"pleo_homolog_orthology_confidence","2":"Lion orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"clanigera_homolog_ensembl_gene","2":"Long-tailed chinchilla gene stable ID","3":"homologs"},{"1":"clanigera_homolog_associated_gene_name","2":"Long-tailed chinchilla gene name","3":"homologs"},{"1":"clanigera_homolog_ensembl_peptide","2":"Long-tailed chinchilla protein or transcript stable ID","3":"homologs"},{"1":"clanigera_homolog_chromosome","2":"Long-tailed chinchilla chromosome/scaffold name","3":"homologs"},{"1":"clanigera_homolog_chrom_start","2":"Long-tailed chinchilla chromosome/scaffold start (bp)","3":"homologs"},{"1":"clanigera_homolog_chrom_end","2":"Long-tailed chinchilla chromosome/scaffold end (bp)","3":"homologs"},{"1":"clanigera_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"clanigera_homolog_subtype","2":"Last common ancestor with Long-tailed chinchilla","3":"homologs"},{"1":"clanigera_homolog_orthology_type","2":"Long-tailed chinchilla homology type","3":"homologs"},{"1":"clanigera_homolog_perc_id","2":"%id. target Long-tailed chinchilla gene identical to query gene","3":"homologs"},{"1":"clanigera_homolog_perc_id_r1","2":"%id. query gene identical to target Long-tailed chinchilla gene","3":"homologs"},{"1":"clanigera_homolog_goc_score","2":"Long-tailed chinchilla Gene-order conservation score","3":"homologs"},{"1":"clanigera_homolog_wga_coverage","2":"Long-tailed chinchilla Whole-genome alignment coverage","3":"homologs"},{"1":"clanigera_homolog_orthology_confidence","2":"Long-tailed chinchilla orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"clumpus_homolog_ensembl_gene","2":"Lumpfish gene stable ID","3":"homologs"},{"1":"clumpus_homolog_associated_gene_name","2":"Lumpfish gene name","3":"homologs"},{"1":"clumpus_homolog_ensembl_peptide","2":"Lumpfish protein or transcript stable ID","3":"homologs"},{"1":"clumpus_homolog_chromosome","2":"Lumpfish chromosome/scaffold name","3":"homologs"},{"1":"clumpus_homolog_chrom_start","2":"Lumpfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"clumpus_homolog_chrom_end","2":"Lumpfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"clumpus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"clumpus_homolog_subtype","2":"Last common ancestor with Lumpfish","3":"homologs"},{"1":"clumpus_homolog_orthology_type","2":"Lumpfish homology type","3":"homologs"},{"1":"clumpus_homolog_perc_id","2":"%id. target Lumpfish gene identical to query gene","3":"homologs"},{"1":"clumpus_homolog_perc_id_r1","2":"%id. query gene identical to target Lumpfish gene","3":"homologs"},{"1":"clumpus_homolog_goc_score","2":"Lumpfish Gene-order conservation score","3":"homologs"},{"1":"clumpus_homolog_wga_coverage","2":"Lumpfish Whole-genome alignment coverage","3":"homologs"},{"1":"clumpus_homolog_orthology_confidence","2":"Lumpfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nbrichardi_homolog_ensembl_gene","2":"Lyretail cichlid gene stable ID","3":"homologs"},{"1":"nbrichardi_homolog_associated_gene_name","2":"Lyretail cichlid gene name","3":"homologs"},{"1":"nbrichardi_homolog_ensembl_peptide","2":"Lyretail cichlid protein or transcript stable ID","3":"homologs"},{"1":"nbrichardi_homolog_chromosome","2":"Lyretail cichlid chromosome/scaffold name","3":"homologs"},{"1":"nbrichardi_homolog_chrom_start","2":"Lyretail cichlid chromosome/scaffold start (bp)","3":"homologs"},{"1":"nbrichardi_homolog_chrom_end","2":"Lyretail cichlid chromosome/scaffold end (bp)","3":"homologs"},{"1":"nbrichardi_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nbrichardi_homolog_subtype","2":"Last common ancestor with Lyretail cichlid","3":"homologs"},{"1":"nbrichardi_homolog_orthology_type","2":"Lyretail cichlid homology type","3":"homologs"},{"1":"nbrichardi_homolog_perc_id","2":"%id. target Lyretail cichlid gene identical to query gene","3":"homologs"},{"1":"nbrichardi_homolog_perc_id_r1","2":"%id. query gene identical to target Lyretail cichlid gene","3":"homologs"},{"1":"nbrichardi_homolog_goc_score","2":"Lyretail cichlid Gene-order conservation score","3":"homologs"},{"1":"nbrichardi_homolog_wga_coverage","2":"Lyretail cichlid Whole-genome alignment coverage","3":"homologs"},{"1":"nbrichardi_homolog_orthology_confidence","2":"Lyretail cichlid orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"anancymaae_homolog_ensembl_gene","2":"Ma's night monkey gene stable ID","3":"homologs"},{"1":"anancymaae_homolog_associated_gene_name","2":"Ma's night monkey gene name","3":"homologs"},{"1":"anancymaae_homolog_ensembl_peptide","2":"Ma's night monkey protein or transcript stable ID","3":"homologs"},{"1":"anancymaae_homolog_chromosome","2":"Ma's night monkey chromosome/scaffold name","3":"homologs"},{"1":"anancymaae_homolog_chrom_start","2":"Ma's night monkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"anancymaae_homolog_chrom_end","2":"Ma's night monkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"anancymaae_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"anancymaae_homolog_subtype","2":"Last common ancestor with Ma's night monkey","3":"homologs"},{"1":"anancymaae_homolog_orthology_type","2":"Ma's night monkey homology type","3":"homologs"},{"1":"anancymaae_homolog_perc_id","2":"%id. target Ma's night monkey gene identical to query gene","3":"homologs"},{"1":"anancymaae_homolog_perc_id_r1","2":"%id. query gene identical to target Ma's night monkey gene","3":"homologs"},{"1":"anancymaae_homolog_goc_score","2":"Ma's night monkey Gene-order conservation score","3":"homologs"},{"1":"anancymaae_homolog_wga_coverage","2":"Ma's night monkey Whole-genome alignment coverage","3":"homologs"},{"1":"anancymaae_homolog_orthology_confidence","2":"Ma's night monkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmulatta_homolog_ensembl_gene","2":"Macaque gene stable ID","3":"homologs"},{"1":"mmulatta_homolog_associated_gene_name","2":"Macaque gene name","3":"homologs"},{"1":"mmulatta_homolog_ensembl_peptide","2":"Macaque protein or transcript stable ID","3":"homologs"},{"1":"mmulatta_homolog_chromosome","2":"Macaque chromosome/scaffold name","3":"homologs"},{"1":"mmulatta_homolog_chrom_start","2":"Macaque chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmulatta_homolog_chrom_end","2":"Macaque chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmulatta_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmulatta_homolog_subtype","2":"Last common ancestor with Macaque","3":"homologs"},{"1":"mmulatta_homolog_orthology_type","2":"Macaque homology type","3":"homologs"},{"1":"mmulatta_homolog_perc_id","2":"%id. target Macaque gene identical to query gene","3":"homologs"},{"1":"mmulatta_homolog_perc_id_r1","2":"%id. query gene identical to target Macaque gene","3":"homologs"},{"1":"mmulatta_homolog_goc_score","2":"Macaque Gene-order conservation score","3":"homologs"},{"1":"mmulatta_homolog_wga_coverage","2":"Macaque Whole-genome alignment coverage","3":"homologs"},{"1":"mmulatta_homolog_orthology_confidence","2":"Macaque orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nscutatus_homolog_ensembl_gene","2":"Mainland tiger snake gene stable ID","3":"homologs"},{"1":"nscutatus_homolog_associated_gene_name","2":"Mainland tiger snake gene name","3":"homologs"},{"1":"nscutatus_homolog_ensembl_peptide","2":"Mainland tiger snake protein or transcript stable ID","3":"homologs"},{"1":"nscutatus_homolog_chromosome","2":"Mainland tiger snake chromosome/scaffold name","3":"homologs"},{"1":"nscutatus_homolog_chrom_start","2":"Mainland tiger snake chromosome/scaffold start (bp)","3":"homologs"},{"1":"nscutatus_homolog_chrom_end","2":"Mainland tiger snake chromosome/scaffold end (bp)","3":"homologs"},{"1":"nscutatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nscutatus_homolog_subtype","2":"Last common ancestor with Mainland tiger snake","3":"homologs"},{"1":"nscutatus_homolog_orthology_type","2":"Mainland tiger snake homology type","3":"homologs"},{"1":"nscutatus_homolog_perc_id","2":"%id. target Mainland tiger snake gene identical to query gene","3":"homologs"},{"1":"nscutatus_homolog_perc_id_r1","2":"%id. query gene identical to target Mainland tiger snake gene","3":"homologs"},{"1":"nscutatus_homolog_goc_score","2":"Mainland tiger snake Gene-order conservation score","3":"homologs"},{"1":"nscutatus_homolog_wga_coverage","2":"Mainland tiger snake Whole-genome alignment coverage","3":"homologs"},{"1":"nscutatus_homolog_orthology_confidence","2":"Mainland tiger snake orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pnyererei_homolog_ensembl_gene","2":"Makobe Island cichlid gene stable ID","3":"homologs"},{"1":"pnyererei_homolog_associated_gene_name","2":"Makobe Island cichlid gene name","3":"homologs"},{"1":"pnyererei_homolog_ensembl_peptide","2":"Makobe Island cichlid protein or transcript stable ID","3":"homologs"},{"1":"pnyererei_homolog_chromosome","2":"Makobe Island cichlid chromosome/scaffold name","3":"homologs"},{"1":"pnyererei_homolog_chrom_start","2":"Makobe Island cichlid chromosome/scaffold start (bp)","3":"homologs"},{"1":"pnyererei_homolog_chrom_end","2":"Makobe Island cichlid chromosome/scaffold end (bp)","3":"homologs"},{"1":"pnyererei_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pnyererei_homolog_subtype","2":"Last common ancestor with Makobe Island cichlid","3":"homologs"},{"1":"pnyererei_homolog_orthology_type","2":"Makobe Island cichlid homology type","3":"homologs"},{"1":"pnyererei_homolog_perc_id","2":"%id. target Makobe Island cichlid gene identical to query gene","3":"homologs"},{"1":"pnyererei_homolog_perc_id_r1","2":"%id. query gene identical to target Makobe Island cichlid gene","3":"homologs"},{"1":"pnyererei_homolog_goc_score","2":"Makobe Island cichlid Gene-order conservation score","3":"homologs"},{"1":"pnyererei_homolog_wga_coverage","2":"Makobe Island cichlid Whole-genome alignment coverage","3":"homologs"},{"1":"pnyererei_homolog_orthology_confidence","2":"Makobe Island cichlid orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"kmarmoratus_homolog_ensembl_gene","2":"Mangrove rivulus gene stable ID","3":"homologs"},{"1":"kmarmoratus_homolog_associated_gene_name","2":"Mangrove rivulus gene name","3":"homologs"},{"1":"kmarmoratus_homolog_ensembl_peptide","2":"Mangrove rivulus protein or transcript stable ID","3":"homologs"},{"1":"kmarmoratus_homolog_chromosome","2":"Mangrove rivulus chromosome/scaffold name","3":"homologs"},{"1":"kmarmoratus_homolog_chrom_start","2":"Mangrove rivulus chromosome/scaffold start (bp)","3":"homologs"},{"1":"kmarmoratus_homolog_chrom_end","2":"Mangrove rivulus chromosome/scaffold end (bp)","3":"homologs"},{"1":"kmarmoratus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"kmarmoratus_homolog_subtype","2":"Last common ancestor with Mangrove rivulus","3":"homologs"},{"1":"kmarmoratus_homolog_orthology_type","2":"Mangrove rivulus homology type","3":"homologs"},{"1":"kmarmoratus_homolog_perc_id","2":"%id. target Mangrove rivulus gene identical to query gene","3":"homologs"},{"1":"kmarmoratus_homolog_perc_id_r1","2":"%id. query gene identical to target Mangrove rivulus gene","3":"homologs"},{"1":"kmarmoratus_homolog_goc_score","2":"Mangrove rivulus Gene-order conservation score","3":"homologs"},{"1":"kmarmoratus_homolog_wga_coverage","2":"Mangrove rivulus Whole-genome alignment coverage","3":"homologs"},{"1":"kmarmoratus_homolog_orthology_confidence","2":"Mangrove rivulus orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"gfortis_homolog_ensembl_gene","2":"Medium ground-finch gene stable ID","3":"homologs"},{"1":"gfortis_homolog_associated_gene_name","2":"Medium ground-finch gene name","3":"homologs"},{"1":"gfortis_homolog_ensembl_peptide","2":"Medium ground-finch protein or transcript stable ID","3":"homologs"},{"1":"gfortis_homolog_chromosome","2":"Medium ground-finch chromosome/scaffold name","3":"homologs"},{"1":"gfortis_homolog_chrom_start","2":"Medium ground-finch chromosome/scaffold start (bp)","3":"homologs"},{"1":"gfortis_homolog_chrom_end","2":"Medium ground-finch chromosome/scaffold end (bp)","3":"homologs"},{"1":"gfortis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"gfortis_homolog_subtype","2":"Last common ancestor with Medium ground-finch","3":"homologs"},{"1":"gfortis_homolog_orthology_type","2":"Medium ground-finch homology type","3":"homologs"},{"1":"gfortis_homolog_perc_id","2":"%id. target Medium ground-finch gene identical to query gene","3":"homologs"},{"1":"gfortis_homolog_perc_id_r1","2":"%id. query gene identical to target Medium ground-finch gene","3":"homologs"},{"1":"gfortis_homolog_goc_score","2":"Medium ground-finch Gene-order conservation score","3":"homologs"},{"1":"gfortis_homolog_wga_coverage","2":"Medium ground-finch Whole-genome alignment coverage","3":"homologs"},{"1":"gfortis_homolog_orthology_confidence","2":"Medium ground-finch orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pvampyrus_homolog_ensembl_gene","2":"Megabat gene stable ID","3":"homologs"},{"1":"pvampyrus_homolog_associated_gene_name","2":"Megabat gene name","3":"homologs"},{"1":"pvampyrus_homolog_ensembl_peptide","2":"Megabat protein or transcript stable ID","3":"homologs"},{"1":"pvampyrus_homolog_chromosome","2":"Megabat chromosome/scaffold name","3":"homologs"},{"1":"pvampyrus_homolog_chrom_start","2":"Megabat chromosome/scaffold start (bp)","3":"homologs"},{"1":"pvampyrus_homolog_chrom_end","2":"Megabat chromosome/scaffold end (bp)","3":"homologs"},{"1":"pvampyrus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pvampyrus_homolog_subtype","2":"Last common ancestor with Megabat","3":"homologs"},{"1":"pvampyrus_homolog_orthology_type","2":"Megabat homology type","3":"homologs"},{"1":"pvampyrus_homolog_perc_id","2":"%id. target Megabat gene identical to query gene","3":"homologs"},{"1":"pvampyrus_homolog_perc_id_r1","2":"%id. query gene identical to target Megabat gene","3":"homologs"},{"1":"pvampyrus_homolog_goc_score","2":"Megabat Gene-order conservation score","3":"homologs"},{"1":"pvampyrus_homolog_wga_coverage","2":"Megabat Whole-genome alignment coverage","3":"homologs"},{"1":"pvampyrus_homolog_orthology_confidence","2":"Megabat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"amexicanus_homolog_ensembl_gene","2":"Mexican tetra gene stable ID","3":"homologs"},{"1":"amexicanus_homolog_associated_gene_name","2":"Mexican tetra gene name","3":"homologs"},{"1":"amexicanus_homolog_ensembl_peptide","2":"Mexican tetra protein or transcript stable ID","3":"homologs"},{"1":"amexicanus_homolog_chromosome","2":"Mexican tetra chromosome/scaffold name","3":"homologs"},{"1":"amexicanus_homolog_chrom_start","2":"Mexican tetra chromosome/scaffold start (bp)","3":"homologs"},{"1":"amexicanus_homolog_chrom_end","2":"Mexican tetra chromosome/scaffold end (bp)","3":"homologs"},{"1":"amexicanus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"amexicanus_homolog_subtype","2":"Last common ancestor with Mexican tetra","3":"homologs"},{"1":"amexicanus_homolog_orthology_type","2":"Mexican tetra homology type","3":"homologs"},{"1":"amexicanus_homolog_perc_id","2":"%id. target Mexican tetra gene identical to query gene","3":"homologs"},{"1":"amexicanus_homolog_perc_id_r1","2":"%id. query gene identical to target Mexican tetra gene","3":"homologs"},{"1":"amexicanus_homolog_goc_score","2":"Mexican tetra Gene-order conservation score","3":"homologs"},{"1":"amexicanus_homolog_wga_coverage","2":"Mexican tetra Whole-genome alignment coverage","3":"homologs"},{"1":"amexicanus_homolog_orthology_confidence","2":"Mexican tetra orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mlucifugus_homolog_ensembl_gene","2":"Microbat gene stable ID","3":"homologs"},{"1":"mlucifugus_homolog_associated_gene_name","2":"Microbat gene name","3":"homologs"},{"1":"mlucifugus_homolog_ensembl_peptide","2":"Microbat protein or transcript stable ID","3":"homologs"},{"1":"mlucifugus_homolog_chromosome","2":"Microbat chromosome/scaffold name","3":"homologs"},{"1":"mlucifugus_homolog_chrom_start","2":"Microbat chromosome/scaffold start (bp)","3":"homologs"},{"1":"mlucifugus_homolog_chrom_end","2":"Microbat chromosome/scaffold end (bp)","3":"homologs"},{"1":"mlucifugus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mlucifugus_homolog_subtype","2":"Last common ancestor with Microbat","3":"homologs"},{"1":"mlucifugus_homolog_orthology_type","2":"Microbat homology type","3":"homologs"},{"1":"mlucifugus_homolog_perc_id","2":"%id. target Microbat gene identical to query gene","3":"homologs"},{"1":"mlucifugus_homolog_perc_id_r1","2":"%id. query gene identical to target Microbat gene","3":"homologs"},{"1":"mlucifugus_homolog_goc_score","2":"Microbat Gene-order conservation score","3":"homologs"},{"1":"mlucifugus_homolog_wga_coverage","2":"Microbat Whole-genome alignment coverage","3":"homologs"},{"1":"mlucifugus_homolog_orthology_confidence","2":"Microbat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"acitrinellus_homolog_ensembl_gene","2":"Midas cichlid gene stable ID","3":"homologs"},{"1":"acitrinellus_homolog_associated_gene_name","2":"Midas cichlid gene name","3":"homologs"},{"1":"acitrinellus_homolog_ensembl_peptide","2":"Midas cichlid protein or transcript stable ID","3":"homologs"},{"1":"acitrinellus_homolog_chromosome","2":"Midas cichlid chromosome/scaffold name","3":"homologs"},{"1":"acitrinellus_homolog_chrom_start","2":"Midas cichlid chromosome/scaffold start (bp)","3":"homologs"},{"1":"acitrinellus_homolog_chrom_end","2":"Midas cichlid chromosome/scaffold end (bp)","3":"homologs"},{"1":"acitrinellus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"acitrinellus_homolog_subtype","2":"Last common ancestor with Midas cichlid","3":"homologs"},{"1":"acitrinellus_homolog_orthology_type","2":"Midas cichlid homology type","3":"homologs"},{"1":"acitrinellus_homolog_perc_id","2":"%id. target Midas cichlid gene identical to query gene","3":"homologs"},{"1":"acitrinellus_homolog_perc_id_r1","2":"%id. query gene identical to target Midas cichlid gene","3":"homologs"},{"1":"acitrinellus_homolog_goc_score","2":"Midas cichlid Gene-order conservation score","3":"homologs"},{"1":"acitrinellus_homolog_wga_coverage","2":"Midas cichlid Whole-genome alignment coverage","3":"homologs"},{"1":"acitrinellus_homolog_orthology_confidence","2":"Midas cichlid orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmusculus_homolog_ensembl_gene","2":"Mouse gene stable ID","3":"homologs"},{"1":"mmusculus_homolog_associated_gene_name","2":"Mouse gene name","3":"homologs"},{"1":"mmusculus_homolog_ensembl_peptide","2":"Mouse protein or transcript stable ID","3":"homologs"},{"1":"mmusculus_homolog_chromosome","2":"Mouse chromosome/scaffold name","3":"homologs"},{"1":"mmusculus_homolog_chrom_start","2":"Mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmusculus_homolog_chrom_end","2":"Mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmusculus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmusculus_homolog_subtype","2":"Last common ancestor with Mouse","3":"homologs"},{"1":"mmusculus_homolog_orthology_type","2":"Mouse homology type","3":"homologs"},{"1":"mmusculus_homolog_perc_id","2":"%id. target Mouse gene identical to query gene","3":"homologs"},{"1":"mmusculus_homolog_perc_id_r1","2":"%id. query gene identical to target Mouse gene","3":"homologs"},{"1":"mmusculus_homolog_goc_score","2":"Mouse Gene-order conservation score","3":"homologs"},{"1":"mmusculus_homolog_wga_coverage","2":"Mouse Whole-genome alignment coverage","3":"homologs"},{"1":"mmusculus_homolog_orthology_confidence","2":"Mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmurinus_homolog_ensembl_gene","2":"Mouse Lemur gene stable ID","3":"homologs"},{"1":"mmurinus_homolog_associated_gene_name","2":"Mouse Lemur gene name","3":"homologs"},{"1":"mmurinus_homolog_ensembl_peptide","2":"Mouse Lemur protein or transcript stable ID","3":"homologs"},{"1":"mmurinus_homolog_chromosome","2":"Mouse Lemur chromosome/scaffold name","3":"homologs"},{"1":"mmurinus_homolog_chrom_start","2":"Mouse Lemur chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmurinus_homolog_chrom_end","2":"Mouse Lemur chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmurinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmurinus_homolog_subtype","2":"Last common ancestor with Mouse Lemur","3":"homologs"},{"1":"mmurinus_homolog_orthology_type","2":"Mouse Lemur homology type","3":"homologs"},{"1":"mmurinus_homolog_perc_id","2":"%id. target Mouse Lemur gene identical to query gene","3":"homologs"},{"1":"mmurinus_homolog_perc_id_r1","2":"%id. query gene identical to target Mouse Lemur gene","3":"homologs"},{"1":"mmurinus_homolog_goc_score","2":"Mouse Lemur Gene-order conservation score","3":"homologs"},{"1":"mmurinus_homolog_wga_coverage","2":"Mouse Lemur Whole-genome alignment coverage","3":"homologs"},{"1":"mmurinus_homolog_orthology_confidence","2":"Mouse Lemur orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"fheteroclitus_homolog_ensembl_gene","2":"Mummichog gene stable ID","3":"homologs"},{"1":"fheteroclitus_homolog_associated_gene_name","2":"Mummichog gene name","3":"homologs"},{"1":"fheteroclitus_homolog_ensembl_peptide","2":"Mummichog protein or transcript stable ID","3":"homologs"},{"1":"fheteroclitus_homolog_chromosome","2":"Mummichog chromosome/scaffold name","3":"homologs"},{"1":"fheteroclitus_homolog_chrom_start","2":"Mummichog chromosome/scaffold start (bp)","3":"homologs"},{"1":"fheteroclitus_homolog_chrom_end","2":"Mummichog chromosome/scaffold end (bp)","3":"homologs"},{"1":"fheteroclitus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"fheteroclitus_homolog_subtype","2":"Last common ancestor with Mummichog","3":"homologs"},{"1":"fheteroclitus_homolog_orthology_type","2":"Mummichog homology type","3":"homologs"},{"1":"fheteroclitus_homolog_perc_id","2":"%id. target Mummichog gene identical to query gene","3":"homologs"},{"1":"fheteroclitus_homolog_perc_id_r1","2":"%id. query gene identical to target Mummichog gene","3":"homologs"},{"1":"fheteroclitus_homolog_goc_score","2":"Mummichog Gene-order conservation score","3":"homologs"},{"1":"fheteroclitus_homolog_wga_coverage","2":"Mummichog Whole-genome alignment coverage","3":"homologs"},{"1":"fheteroclitus_homolog_orthology_confidence","2":"Mummichog orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"hgfemale_homolog_ensembl_gene","2":"Naked mole-rat female gene stable ID","3":"homologs"},{"1":"hgfemale_homolog_associated_gene_name","2":"Naked mole-rat female gene name","3":"homologs"},{"1":"hgfemale_homolog_ensembl_peptide","2":"Naked mole-rat female protein or transcript stable ID","3":"homologs"},{"1":"hgfemale_homolog_chromosome","2":"Naked mole-rat female chromosome/scaffold name","3":"homologs"},{"1":"hgfemale_homolog_chrom_start","2":"Naked mole-rat female chromosome/scaffold start (bp)","3":"homologs"},{"1":"hgfemale_homolog_chrom_end","2":"Naked mole-rat female chromosome/scaffold end (bp)","3":"homologs"},{"1":"hgfemale_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"hgfemale_homolog_subtype","2":"Last common ancestor with Naked mole-rat female","3":"homologs"},{"1":"hgfemale_homolog_orthology_type","2":"Naked mole-rat female homology type","3":"homologs"},{"1":"hgfemale_homolog_perc_id","2":"%id. target Naked mole-rat female gene identical to query gene","3":"homologs"},{"1":"hgfemale_homolog_perc_id_r1","2":"%id. query gene identical to target Naked mole-rat female gene","3":"homologs"},{"1":"hgfemale_homolog_goc_score","2":"Naked mole-rat female Gene-order conservation score","3":"homologs"},{"1":"hgfemale_homolog_wga_coverage","2":"Naked mole-rat female Whole-genome alignment coverage","3":"homologs"},{"1":"hgfemale_homolog_orthology_confidence","2":"Naked mole-rat female orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmonoceros_homolog_ensembl_gene","2":"Narwhal gene stable ID","3":"homologs"},{"1":"mmonoceros_homolog_associated_gene_name","2":"Narwhal gene name","3":"homologs"},{"1":"mmonoceros_homolog_ensembl_peptide","2":"Narwhal protein or transcript stable ID","3":"homologs"},{"1":"mmonoceros_homolog_chromosome","2":"Narwhal chromosome/scaffold name","3":"homologs"},{"1":"mmonoceros_homolog_chrom_start","2":"Narwhal chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmonoceros_homolog_chrom_end","2":"Narwhal chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmonoceros_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmonoceros_homolog_subtype","2":"Last common ancestor with Narwhal","3":"homologs"},{"1":"mmonoceros_homolog_orthology_type","2":"Narwhal homology type","3":"homologs"},{"1":"mmonoceros_homolog_perc_id","2":"%id. target Narwhal gene identical to query gene","3":"homologs"},{"1":"mmonoceros_homolog_perc_id_r1","2":"%id. query gene identical to target Narwhal gene","3":"homologs"},{"1":"mmonoceros_homolog_goc_score","2":"Narwhal Gene-order conservation score","3":"homologs"},{"1":"mmonoceros_homolog_wga_coverage","2":"Narwhal Whole-genome alignment coverage","3":"homologs"},{"1":"mmonoceros_homolog_orthology_confidence","2":"Narwhal orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"oniloticus_homolog_ensembl_gene","2":"Nile tilapia gene stable ID","3":"homologs"},{"1":"oniloticus_homolog_associated_gene_name","2":"Nile tilapia gene name","3":"homologs"},{"1":"oniloticus_homolog_ensembl_peptide","2":"Nile tilapia protein or transcript stable ID","3":"homologs"},{"1":"oniloticus_homolog_chromosome","2":"Nile tilapia chromosome/scaffold name","3":"homologs"},{"1":"oniloticus_homolog_chrom_start","2":"Nile tilapia chromosome/scaffold start (bp)","3":"homologs"},{"1":"oniloticus_homolog_chrom_end","2":"Nile tilapia chromosome/scaffold end (bp)","3":"homologs"},{"1":"oniloticus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"oniloticus_homolog_subtype","2":"Last common ancestor with Nile tilapia","3":"homologs"},{"1":"oniloticus_homolog_orthology_type","2":"Nile tilapia homology type","3":"homologs"},{"1":"oniloticus_homolog_perc_id","2":"%id. target Nile tilapia gene identical to query gene","3":"homologs"},{"1":"oniloticus_homolog_perc_id_r1","2":"%id. query gene identical to target Nile tilapia gene","3":"homologs"},{"1":"oniloticus_homolog_goc_score","2":"Nile tilapia Gene-order conservation score","3":"homologs"},{"1":"oniloticus_homolog_wga_coverage","2":"Nile tilapia Whole-genome alignment coverage","3":"homologs"},{"1":"oniloticus_homolog_orthology_confidence","2":"Nile tilapia orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pmbairdii_homolog_ensembl_gene","2":"Northern American deer mouse gene stable ID","3":"homologs"},{"1":"pmbairdii_homolog_associated_gene_name","2":"Northern American deer mouse gene name","3":"homologs"},{"1":"pmbairdii_homolog_ensembl_peptide","2":"Northern American deer mouse protein or transcript stable ID","3":"homologs"},{"1":"pmbairdii_homolog_chromosome","2":"Northern American deer mouse chromosome/scaffold name","3":"homologs"},{"1":"pmbairdii_homolog_chrom_start","2":"Northern American deer mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"pmbairdii_homolog_chrom_end","2":"Northern American deer mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"pmbairdii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pmbairdii_homolog_subtype","2":"Last common ancestor with Northern American deer mouse","3":"homologs"},{"1":"pmbairdii_homolog_orthology_type","2":"Northern American deer mouse homology type","3":"homologs"},{"1":"pmbairdii_homolog_perc_id","2":"%id. target Northern American deer mouse gene identical to query gene","3":"homologs"},{"1":"pmbairdii_homolog_perc_id_r1","2":"%id. query gene identical to target Northern American deer mouse gene","3":"homologs"},{"1":"pmbairdii_homolog_goc_score","2":"Northern American deer mouse Gene-order conservation score","3":"homologs"},{"1":"pmbairdii_homolog_wga_coverage","2":"Northern American deer mouse Whole-genome alignment coverage","3":"homologs"},{"1":"pmbairdii_homolog_orthology_confidence","2":"Northern American deer mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"elucius_homolog_ensembl_gene","2":"Northern pike gene stable ID","3":"homologs"},{"1":"elucius_homolog_associated_gene_name","2":"Northern pike gene name","3":"homologs"},{"1":"elucius_homolog_ensembl_peptide","2":"Northern pike protein or transcript stable ID","3":"homologs"},{"1":"elucius_homolog_chromosome","2":"Northern pike chromosome/scaffold name","3":"homologs"},{"1":"elucius_homolog_chrom_start","2":"Northern pike chromosome/scaffold start (bp)","3":"homologs"},{"1":"elucius_homolog_chrom_end","2":"Northern pike chromosome/scaffold end (bp)","3":"homologs"},{"1":"elucius_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"elucius_homolog_subtype","2":"Last common ancestor with Northern pike","3":"homologs"},{"1":"elucius_homolog_orthology_type","2":"Northern pike homology type","3":"homologs"},{"1":"elucius_homolog_perc_id","2":"%id. target Northern pike gene identical to query gene","3":"homologs"},{"1":"elucius_homolog_perc_id_r1","2":"%id. query gene identical to target Northern pike gene","3":"homologs"},{"1":"elucius_homolog_goc_score","2":"Northern pike Gene-order conservation score","3":"homologs"},{"1":"elucius_homolog_wga_coverage","2":"Northern pike Whole-genome alignment coverage","3":"homologs"},{"1":"elucius_homolog_orthology_confidence","2":"Northern pike orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"panubis_homolog_ensembl_gene","2":"Olive baboon gene stable ID","3":"homologs"},{"1":"panubis_homolog_associated_gene_name","2":"Olive baboon gene name","3":"homologs"},{"1":"panubis_homolog_ensembl_peptide","2":"Olive baboon protein or transcript stable ID","3":"homologs"},{"1":"panubis_homolog_chromosome","2":"Olive baboon chromosome/scaffold name","3":"homologs"},{"1":"panubis_homolog_chrom_start","2":"Olive baboon chromosome/scaffold start (bp)","3":"homologs"},{"1":"panubis_homolog_chrom_end","2":"Olive baboon chromosome/scaffold end (bp)","3":"homologs"},{"1":"panubis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"panubis_homolog_subtype","2":"Last common ancestor with Olive baboon","3":"homologs"},{"1":"panubis_homolog_orthology_type","2":"Olive baboon homology type","3":"homologs"},{"1":"panubis_homolog_perc_id","2":"%id. target Olive baboon gene identical to query gene","3":"homologs"},{"1":"panubis_homolog_perc_id_r1","2":"%id. query gene identical to target Olive baboon gene","3":"homologs"},{"1":"panubis_homolog_goc_score","2":"Olive baboon Gene-order conservation score","3":"homologs"},{"1":"panubis_homolog_wga_coverage","2":"Olive baboon Whole-genome alignment coverage","3":"homologs"},{"1":"panubis_homolog_orthology_confidence","2":"Olive baboon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mdomestica_homolog_ensembl_gene","2":"Opossum gene stable ID","3":"homologs"},{"1":"mdomestica_homolog_associated_gene_name","2":"Opossum gene name","3":"homologs"},{"1":"mdomestica_homolog_ensembl_peptide","2":"Opossum protein or transcript stable ID","3":"homologs"},{"1":"mdomestica_homolog_chromosome","2":"Opossum chromosome/scaffold name","3":"homologs"},{"1":"mdomestica_homolog_chrom_start","2":"Opossum chromosome/scaffold start (bp)","3":"homologs"},{"1":"mdomestica_homolog_chrom_end","2":"Opossum chromosome/scaffold end (bp)","3":"homologs"},{"1":"mdomestica_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mdomestica_homolog_subtype","2":"Last common ancestor with Opossum","3":"homologs"},{"1":"mdomestica_homolog_orthology_type","2":"Opossum homology type","3":"homologs"},{"1":"mdomestica_homolog_perc_id","2":"%id. target Opossum gene identical to query gene","3":"homologs"},{"1":"mdomestica_homolog_perc_id_r1","2":"%id. query gene identical to target Opossum gene","3":"homologs"},{"1":"mdomestica_homolog_goc_score","2":"Opossum Gene-order conservation score","3":"homologs"},{"1":"mdomestica_homolog_wga_coverage","2":"Opossum Whole-genome alignment coverage","3":"homologs"},{"1":"mdomestica_homolog_orthology_confidence","2":"Opossum orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"apercula_homolog_ensembl_gene","2":"Orange clownfish gene stable ID","3":"homologs"},{"1":"apercula_homolog_associated_gene_name","2":"Orange clownfish gene name","3":"homologs"},{"1":"apercula_homolog_ensembl_peptide","2":"Orange clownfish protein or transcript stable ID","3":"homologs"},{"1":"apercula_homolog_chromosome","2":"Orange clownfish chromosome/scaffold name","3":"homologs"},{"1":"apercula_homolog_chrom_start","2":"Orange clownfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"apercula_homolog_chrom_end","2":"Orange clownfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"apercula_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"apercula_homolog_subtype","2":"Last common ancestor with Orange clownfish","3":"homologs"},{"1":"apercula_homolog_orthology_type","2":"Orange clownfish homology type","3":"homologs"},{"1":"apercula_homolog_perc_id","2":"%id. target Orange clownfish gene identical to query gene","3":"homologs"},{"1":"apercula_homolog_perc_id_r1","2":"%id. query gene identical to target Orange clownfish gene","3":"homologs"},{"1":"apercula_homolog_goc_score","2":"Orange clownfish Gene-order conservation score","3":"homologs"},{"1":"apercula_homolog_wga_coverage","2":"Orange clownfish Whole-genome alignment coverage","3":"homologs"},{"1":"apercula_homolog_orthology_confidence","2":"Orange clownfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cpbellii_homolog_ensembl_gene","2":"Painted turtle gene stable ID","3":"homologs"},{"1":"cpbellii_homolog_associated_gene_name","2":"Painted turtle gene name","3":"homologs"},{"1":"cpbellii_homolog_ensembl_peptide","2":"Painted turtle protein or transcript stable ID","3":"homologs"},{"1":"cpbellii_homolog_chromosome","2":"Painted turtle chromosome/scaffold name","3":"homologs"},{"1":"cpbellii_homolog_chrom_start","2":"Painted turtle chromosome/scaffold start (bp)","3":"homologs"},{"1":"cpbellii_homolog_chrom_end","2":"Painted turtle chromosome/scaffold end (bp)","3":"homologs"},{"1":"cpbellii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cpbellii_homolog_subtype","2":"Last common ancestor with Painted turtle","3":"homologs"},{"1":"cpbellii_homolog_orthology_type","2":"Painted turtle homology type","3":"homologs"},{"1":"cpbellii_homolog_perc_id","2":"%id. target Painted turtle gene identical to query gene","3":"homologs"},{"1":"cpbellii_homolog_perc_id_r1","2":"%id. query gene identical to target Painted turtle gene","3":"homologs"},{"1":"cpbellii_homolog_goc_score","2":"Painted turtle Gene-order conservation score","3":"homologs"},{"1":"cpbellii_homolog_wga_coverage","2":"Painted turtle Whole-genome alignment coverage","3":"homologs"},{"1":"cpbellii_homolog_orthology_confidence","2":"Painted turtle orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pkingsleyae_homolog_ensembl_gene","2":"Paramormyrops kingsleyae gene stable ID","3":"homologs"},{"1":"pkingsleyae_homolog_associated_gene_name","2":"Paramormyrops kingsleyae gene name","3":"homologs"},{"1":"pkingsleyae_homolog_ensembl_peptide","2":"Paramormyrops kingsleyae protein or transcript stable ID","3":"homologs"},{"1":"pkingsleyae_homolog_chromosome","2":"Paramormyrops kingsleyae chromosome/scaffold name","3":"homologs"},{"1":"pkingsleyae_homolog_chrom_start","2":"Paramormyrops kingsleyae chromosome/scaffold start (bp)","3":"homologs"},{"1":"pkingsleyae_homolog_chrom_end","2":"Paramormyrops kingsleyae chromosome/scaffold end (bp)","3":"homologs"},{"1":"pkingsleyae_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pkingsleyae_homolog_subtype","2":"Last common ancestor with Paramormyrops kingsleyae","3":"homologs"},{"1":"pkingsleyae_homolog_orthology_type","2":"Paramormyrops kingsleyae homology type","3":"homologs"},{"1":"pkingsleyae_homolog_perc_id","2":"%id. target Paramormyrops kingsleyae gene identical to query gene","3":"homologs"},{"1":"pkingsleyae_homolog_perc_id_r1","2":"%id. query gene identical to target Paramormyrops kingsleyae gene","3":"homologs"},{"1":"pkingsleyae_homolog_goc_score","2":"Paramormyrops kingsleyae Gene-order conservation score","3":"homologs"},{"1":"pkingsleyae_homolog_wga_coverage","2":"Paramormyrops kingsleyae Whole-genome alignment coverage","3":"homologs"},{"1":"pkingsleyae_homolog_orthology_confidence","2":"Paramormyrops kingsleyae orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sscrofa_homolog_ensembl_gene","2":"Pig gene stable ID","3":"homologs"},{"1":"sscrofa_homolog_associated_gene_name","2":"Pig gene name","3":"homologs"},{"1":"sscrofa_homolog_ensembl_peptide","2":"Pig protein or transcript stable ID","3":"homologs"},{"1":"sscrofa_homolog_chromosome","2":"Pig chromosome/scaffold name","3":"homologs"},{"1":"sscrofa_homolog_chrom_start","2":"Pig chromosome/scaffold start (bp)","3":"homologs"},{"1":"sscrofa_homolog_chrom_end","2":"Pig chromosome/scaffold end (bp)","3":"homologs"},{"1":"sscrofa_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sscrofa_homolog_subtype","2":"Last common ancestor with Pig","3":"homologs"},{"1":"sscrofa_homolog_orthology_type","2":"Pig homology type","3":"homologs"},{"1":"sscrofa_homolog_perc_id","2":"%id. target Pig gene identical to query gene","3":"homologs"},{"1":"sscrofa_homolog_perc_id_r1","2":"%id. query gene identical to target Pig gene","3":"homologs"},{"1":"sscrofa_homolog_goc_score","2":"Pig Gene-order conservation score","3":"homologs"},{"1":"sscrofa_homolog_wga_coverage","2":"Pig Whole-genome alignment coverage","3":"homologs"},{"1":"sscrofa_homolog_orthology_confidence","2":"Pig orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mnemestrina_homolog_ensembl_gene","2":"Pig-tailed macaque gene stable ID","3":"homologs"},{"1":"mnemestrina_homolog_associated_gene_name","2":"Pig-tailed macaque gene name","3":"homologs"},{"1":"mnemestrina_homolog_ensembl_peptide","2":"Pig-tailed macaque protein or transcript stable ID","3":"homologs"},{"1":"mnemestrina_homolog_chromosome","2":"Pig-tailed macaque chromosome/scaffold name","3":"homologs"},{"1":"mnemestrina_homolog_chrom_start","2":"Pig-tailed macaque chromosome/scaffold start (bp)","3":"homologs"},{"1":"mnemestrina_homolog_chrom_end","2":"Pig-tailed macaque chromosome/scaffold end (bp)","3":"homologs"},{"1":"mnemestrina_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mnemestrina_homolog_subtype","2":"Last common ancestor with Pig-tailed macaque","3":"homologs"},{"1":"mnemestrina_homolog_orthology_type","2":"Pig-tailed macaque homology type","3":"homologs"},{"1":"mnemestrina_homolog_perc_id","2":"%id. target Pig-tailed macaque gene identical to query gene","3":"homologs"},{"1":"mnemestrina_homolog_perc_id_r1","2":"%id. query gene identical to target Pig-tailed macaque gene","3":"homologs"},{"1":"mnemestrina_homolog_goc_score","2":"Pig-tailed macaque Gene-order conservation score","3":"homologs"},{"1":"mnemestrina_homolog_wga_coverage","2":"Pig-tailed macaque Whole-genome alignment coverage","3":"homologs"},{"1":"mnemestrina_homolog_orthology_confidence","2":"Pig-tailed macaque orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"oprinceps_homolog_ensembl_gene","2":"Pika gene stable ID","3":"homologs"},{"1":"oprinceps_homolog_associated_gene_name","2":"Pika gene name","3":"homologs"},{"1":"oprinceps_homolog_ensembl_peptide","2":"Pika protein or transcript stable ID","3":"homologs"},{"1":"oprinceps_homolog_chromosome","2":"Pika chromosome/scaffold name","3":"homologs"},{"1":"oprinceps_homolog_chrom_start","2":"Pika chromosome/scaffold start (bp)","3":"homologs"},{"1":"oprinceps_homolog_chrom_end","2":"Pika chromosome/scaffold end (bp)","3":"homologs"},{"1":"oprinceps_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"oprinceps_homolog_subtype","2":"Last common ancestor with Pika","3":"homologs"},{"1":"oprinceps_homolog_orthology_type","2":"Pika homology type","3":"homologs"},{"1":"oprinceps_homolog_perc_id","2":"%id. target Pika gene identical to query gene","3":"homologs"},{"1":"oprinceps_homolog_perc_id_r1","2":"%id. query gene identical to target Pika gene","3":"homologs"},{"1":"oprinceps_homolog_goc_score","2":"Pika Gene-order conservation score","3":"homologs"},{"1":"oprinceps_homolog_wga_coverage","2":"Pika Whole-genome alignment coverage","3":"homologs"},{"1":"oprinceps_homolog_orthology_confidence","2":"Pika orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"slucioperca_homolog_ensembl_gene","2":"Pike-perch gene stable ID","3":"homologs"},{"1":"slucioperca_homolog_associated_gene_name","2":"Pike-perch gene name","3":"homologs"},{"1":"slucioperca_homolog_ensembl_peptide","2":"Pike-perch protein or transcript stable ID","3":"homologs"},{"1":"slucioperca_homolog_chromosome","2":"Pike-perch chromosome/scaffold name","3":"homologs"},{"1":"slucioperca_homolog_chrom_start","2":"Pike-perch chromosome/scaffold start (bp)","3":"homologs"},{"1":"slucioperca_homolog_chrom_end","2":"Pike-perch chromosome/scaffold end (bp)","3":"homologs"},{"1":"slucioperca_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"slucioperca_homolog_subtype","2":"Last common ancestor with Pike-perch","3":"homologs"},{"1":"slucioperca_homolog_orthology_type","2":"Pike-perch homology type","3":"homologs"},{"1":"slucioperca_homolog_perc_id","2":"%id. target Pike-perch gene identical to query gene","3":"homologs"},{"1":"slucioperca_homolog_perc_id_r1","2":"%id. query gene identical to target Pike-perch gene","3":"homologs"},{"1":"slucioperca_homolog_goc_score","2":"Pike-perch Gene-order conservation score","3":"homologs"},{"1":"slucioperca_homolog_wga_coverage","2":"Pike-perch Whole-genome alignment coverage","3":"homologs"},{"1":"slucioperca_homolog_orthology_confidence","2":"Pike-perch orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmurdjan_homolog_ensembl_gene","2":"Pinecone soldierfish gene stable ID","3":"homologs"},{"1":"mmurdjan_homolog_associated_gene_name","2":"Pinecone soldierfish gene name","3":"homologs"},{"1":"mmurdjan_homolog_ensembl_peptide","2":"Pinecone soldierfish protein or transcript stable ID","3":"homologs"},{"1":"mmurdjan_homolog_chromosome","2":"Pinecone soldierfish chromosome/scaffold name","3":"homologs"},{"1":"mmurdjan_homolog_chrom_start","2":"Pinecone soldierfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmurdjan_homolog_chrom_end","2":"Pinecone soldierfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmurdjan_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmurdjan_homolog_subtype","2":"Last common ancestor with Pinecone soldierfish","3":"homologs"},{"1":"mmurdjan_homolog_orthology_type","2":"Pinecone soldierfish homology type","3":"homologs"},{"1":"mmurdjan_homolog_perc_id","2":"%id. target Pinecone soldierfish gene identical to query gene","3":"homologs"},{"1":"mmurdjan_homolog_perc_id_r1","2":"%id. query gene identical to target Pinecone soldierfish gene","3":"homologs"},{"1":"mmurdjan_homolog_goc_score","2":"Pinecone soldierfish Gene-order conservation score","3":"homologs"},{"1":"mmurdjan_homolog_wga_coverage","2":"Pinecone soldierfish Whole-genome alignment coverage","3":"homologs"},{"1":"mmurdjan_homolog_orthology_confidence","2":"Pinecone soldierfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"abrachyrhynchus_homolog_ensembl_gene","2":"Pink-footed goose gene stable ID","3":"homologs"},{"1":"abrachyrhynchus_homolog_associated_gene_name","2":"Pink-footed goose gene name","3":"homologs"},{"1":"abrachyrhynchus_homolog_ensembl_peptide","2":"Pink-footed goose protein or transcript stable ID","3":"homologs"},{"1":"abrachyrhynchus_homolog_chromosome","2":"Pink-footed goose chromosome/scaffold name","3":"homologs"},{"1":"abrachyrhynchus_homolog_chrom_start","2":"Pink-footed goose chromosome/scaffold start (bp)","3":"homologs"},{"1":"abrachyrhynchus_homolog_chrom_end","2":"Pink-footed goose chromosome/scaffold end (bp)","3":"homologs"},{"1":"abrachyrhynchus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"abrachyrhynchus_homolog_subtype","2":"Last common ancestor with Pink-footed goose","3":"homologs"},{"1":"abrachyrhynchus_homolog_orthology_type","2":"Pink-footed goose homology type","3":"homologs"},{"1":"abrachyrhynchus_homolog_perc_id","2":"%id. target Pink-footed goose gene identical to query gene","3":"homologs"},{"1":"abrachyrhynchus_homolog_perc_id_r1","2":"%id. query gene identical to target Pink-footed goose gene","3":"homologs"},{"1":"abrachyrhynchus_homolog_goc_score","2":"Pink-footed goose Gene-order conservation score","3":"homologs"},{"1":"abrachyrhynchus_homolog_wga_coverage","2":"Pink-footed goose Whole-genome alignment coverage","3":"homologs"},{"1":"abrachyrhynchus_homolog_orthology_confidence","2":"Pink-footed goose orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"xmaculatus_homolog_ensembl_gene","2":"Platyfish gene stable ID","3":"homologs"},{"1":"xmaculatus_homolog_associated_gene_name","2":"Platyfish gene name","3":"homologs"},{"1":"xmaculatus_homolog_ensembl_peptide","2":"Platyfish protein or transcript stable ID","3":"homologs"},{"1":"xmaculatus_homolog_chromosome","2":"Platyfish chromosome/scaffold name","3":"homologs"},{"1":"xmaculatus_homolog_chrom_start","2":"Platyfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"xmaculatus_homolog_chrom_end","2":"Platyfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"xmaculatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"xmaculatus_homolog_subtype","2":"Last common ancestor with Platyfish","3":"homologs"},{"1":"xmaculatus_homolog_orthology_type","2":"Platyfish homology type","3":"homologs"},{"1":"xmaculatus_homolog_perc_id","2":"%id. target Platyfish gene identical to query gene","3":"homologs"},{"1":"xmaculatus_homolog_perc_id_r1","2":"%id. query gene identical to target Platyfish gene","3":"homologs"},{"1":"xmaculatus_homolog_goc_score","2":"Platyfish Gene-order conservation score","3":"homologs"},{"1":"xmaculatus_homolog_wga_coverage","2":"Platyfish Whole-genome alignment coverage","3":"homologs"},{"1":"xmaculatus_homolog_orthology_confidence","2":"Platyfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"oanatinus_homolog_ensembl_gene","2":"Platypus gene stable ID","3":"homologs"},{"1":"oanatinus_homolog_associated_gene_name","2":"Platypus gene name","3":"homologs"},{"1":"oanatinus_homolog_ensembl_peptide","2":"Platypus protein or transcript stable ID","3":"homologs"},{"1":"oanatinus_homolog_chromosome","2":"Platypus chromosome/scaffold name","3":"homologs"},{"1":"oanatinus_homolog_chrom_start","2":"Platypus chromosome/scaffold start (bp)","3":"homologs"},{"1":"oanatinus_homolog_chrom_end","2":"Platypus chromosome/scaffold end (bp)","3":"homologs"},{"1":"oanatinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"oanatinus_homolog_subtype","2":"Last common ancestor with Platypus","3":"homologs"},{"1":"oanatinus_homolog_orthology_type","2":"Platypus homology type","3":"homologs"},{"1":"oanatinus_homolog_perc_id","2":"%id. target Platypus gene identical to query gene","3":"homologs"},{"1":"oanatinus_homolog_perc_id_r1","2":"%id. query gene identical to target Platypus gene","3":"homologs"},{"1":"oanatinus_homolog_goc_score","2":"Platypus Gene-order conservation score","3":"homologs"},{"1":"oanatinus_homolog_wga_coverage","2":"Platypus Whole-genome alignment coverage","3":"homologs"},{"1":"oanatinus_homolog_orthology_confidence","2":"Platypus orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"umaritimus_homolog_ensembl_gene","2":"Polar bear gene stable ID","3":"homologs"},{"1":"umaritimus_homolog_associated_gene_name","2":"Polar bear gene name","3":"homologs"},{"1":"umaritimus_homolog_ensembl_peptide","2":"Polar bear protein or transcript stable ID","3":"homologs"},{"1":"umaritimus_homolog_chromosome","2":"Polar bear chromosome/scaffold name","3":"homologs"},{"1":"umaritimus_homolog_chrom_start","2":"Polar bear chromosome/scaffold start (bp)","3":"homologs"},{"1":"umaritimus_homolog_chrom_end","2":"Polar bear chromosome/scaffold end (bp)","3":"homologs"},{"1":"umaritimus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"umaritimus_homolog_subtype","2":"Last common ancestor with Polar bear","3":"homologs"},{"1":"umaritimus_homolog_orthology_type","2":"Polar bear homology type","3":"homologs"},{"1":"umaritimus_homolog_perc_id","2":"%id. target Polar bear gene identical to query gene","3":"homologs"},{"1":"umaritimus_homolog_perc_id_r1","2":"%id. query gene identical to target Polar bear gene","3":"homologs"},{"1":"umaritimus_homolog_goc_score","2":"Polar bear Gene-order conservation score","3":"homologs"},{"1":"umaritimus_homolog_wga_coverage","2":"Polar bear Whole-genome alignment coverage","3":"homologs"},{"1":"umaritimus_homolog_orthology_confidence","2":"Polar bear orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mochrogaster_homolog_ensembl_gene","2":"Prairie vole gene stable ID","3":"homologs"},{"1":"mochrogaster_homolog_associated_gene_name","2":"Prairie vole gene name","3":"homologs"},{"1":"mochrogaster_homolog_ensembl_peptide","2":"Prairie vole protein or transcript stable ID","3":"homologs"},{"1":"mochrogaster_homolog_chromosome","2":"Prairie vole chromosome/scaffold name","3":"homologs"},{"1":"mochrogaster_homolog_chrom_start","2":"Prairie vole chromosome/scaffold start (bp)","3":"homologs"},{"1":"mochrogaster_homolog_chrom_end","2":"Prairie vole chromosome/scaffold end (bp)","3":"homologs"},{"1":"mochrogaster_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mochrogaster_homolog_subtype","2":"Last common ancestor with Prairie vole","3":"homologs"},{"1":"mochrogaster_homolog_orthology_type","2":"Prairie vole homology type","3":"homologs"},{"1":"mochrogaster_homolog_perc_id","2":"%id. target Prairie vole gene identical to query gene","3":"homologs"},{"1":"mochrogaster_homolog_perc_id_r1","2":"%id. query gene identical to target Prairie vole gene","3":"homologs"},{"1":"mochrogaster_homolog_goc_score","2":"Prairie vole Gene-order conservation score","3":"homologs"},{"1":"mochrogaster_homolog_wga_coverage","2":"Prairie vole Whole-genome alignment coverage","3":"homologs"},{"1":"mochrogaster_homolog_orthology_confidence","2":"Prairie vole orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ocuniculus_homolog_ensembl_gene","2":"Rabbit gene stable ID","3":"homologs"},{"1":"ocuniculus_homolog_associated_gene_name","2":"Rabbit gene name","3":"homologs"},{"1":"ocuniculus_homolog_ensembl_peptide","2":"Rabbit protein or transcript stable ID","3":"homologs"},{"1":"ocuniculus_homolog_chromosome","2":"Rabbit chromosome/scaffold name","3":"homologs"},{"1":"ocuniculus_homolog_chrom_start","2":"Rabbit chromosome/scaffold start (bp)","3":"homologs"},{"1":"ocuniculus_homolog_chrom_end","2":"Rabbit chromosome/scaffold end (bp)","3":"homologs"},{"1":"ocuniculus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ocuniculus_homolog_subtype","2":"Last common ancestor with Rabbit","3":"homologs"},{"1":"ocuniculus_homolog_orthology_type","2":"Rabbit homology type","3":"homologs"},{"1":"ocuniculus_homolog_perc_id","2":"%id. target Rabbit gene identical to query gene","3":"homologs"},{"1":"ocuniculus_homolog_perc_id_r1","2":"%id. query gene identical to target Rabbit gene","3":"homologs"},{"1":"ocuniculus_homolog_goc_score","2":"Rabbit Gene-order conservation score","3":"homologs"},{"1":"ocuniculus_homolog_wga_coverage","2":"Rabbit Whole-genome alignment coverage","3":"homologs"},{"1":"ocuniculus_homolog_orthology_confidence","2":"Rabbit orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"omykiss_homolog_ensembl_gene","2":"Rainbow trout gene stable ID","3":"homologs"},{"1":"omykiss_homolog_associated_gene_name","2":"Rainbow trout gene name","3":"homologs"},{"1":"omykiss_homolog_ensembl_peptide","2":"Rainbow trout protein or transcript stable ID","3":"homologs"},{"1":"omykiss_homolog_chromosome","2":"Rainbow trout chromosome/scaffold name","3":"homologs"},{"1":"omykiss_homolog_chrom_start","2":"Rainbow trout chromosome/scaffold start (bp)","3":"homologs"},{"1":"omykiss_homolog_chrom_end","2":"Rainbow trout chromosome/scaffold end (bp)","3":"homologs"},{"1":"omykiss_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"omykiss_homolog_subtype","2":"Last common ancestor with Rainbow trout","3":"homologs"},{"1":"omykiss_homolog_orthology_type","2":"Rainbow trout homology type","3":"homologs"},{"1":"omykiss_homolog_perc_id","2":"%id. target Rainbow trout gene identical to query gene","3":"homologs"},{"1":"omykiss_homolog_perc_id_r1","2":"%id. query gene identical to target Rainbow trout gene","3":"homologs"},{"1":"omykiss_homolog_goc_score","2":"Rainbow trout Gene-order conservation score","3":"homologs"},{"1":"omykiss_homolog_orthology_confidence","2":"Rainbow trout orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"rnorvegicus_homolog_ensembl_gene","2":"Rat gene stable ID","3":"homologs"},{"1":"rnorvegicus_homolog_associated_gene_name","2":"Rat gene name","3":"homologs"},{"1":"rnorvegicus_homolog_ensembl_peptide","2":"Rat protein or transcript stable ID","3":"homologs"},{"1":"rnorvegicus_homolog_chromosome","2":"Rat chromosome/scaffold name","3":"homologs"},{"1":"rnorvegicus_homolog_chrom_start","2":"Rat chromosome/scaffold start (bp)","3":"homologs"},{"1":"rnorvegicus_homolog_chrom_end","2":"Rat chromosome/scaffold end (bp)","3":"homologs"},{"1":"rnorvegicus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"rnorvegicus_homolog_subtype","2":"Last common ancestor with Rat","3":"homologs"},{"1":"rnorvegicus_homolog_orthology_type","2":"Rat homology type","3":"homologs"},{"1":"rnorvegicus_homolog_perc_id","2":"%id. target Rat gene identical to query gene","3":"homologs"},{"1":"rnorvegicus_homolog_perc_id_r1","2":"%id. query gene identical to target Rat gene","3":"homologs"},{"1":"rnorvegicus_homolog_goc_score","2":"Rat Gene-order conservation score","3":"homologs"},{"1":"rnorvegicus_homolog_wga_coverage","2":"Rat Whole-genome alignment coverage","3":"homologs"},{"1":"rnorvegicus_homolog_orthology_confidence","2":"Rat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"vvulpes_homolog_ensembl_gene","2":"Red fox gene stable ID","3":"homologs"},{"1":"vvulpes_homolog_associated_gene_name","2":"Red fox gene name","3":"homologs"},{"1":"vvulpes_homolog_ensembl_peptide","2":"Red fox protein or transcript stable ID","3":"homologs"},{"1":"vvulpes_homolog_chromosome","2":"Red fox chromosome/scaffold name","3":"homologs"},{"1":"vvulpes_homolog_chrom_start","2":"Red fox chromosome/scaffold start (bp)","3":"homologs"},{"1":"vvulpes_homolog_chrom_end","2":"Red fox chromosome/scaffold end (bp)","3":"homologs"},{"1":"vvulpes_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"vvulpes_homolog_subtype","2":"Last common ancestor with Red fox","3":"homologs"},{"1":"vvulpes_homolog_orthology_type","2":"Red fox homology type","3":"homologs"},{"1":"vvulpes_homolog_perc_id","2":"%id. target Red fox gene identical to query gene","3":"homologs"},{"1":"vvulpes_homolog_perc_id_r1","2":"%id. query gene identical to target Red fox gene","3":"homologs"},{"1":"vvulpes_homolog_goc_score","2":"Red fox Gene-order conservation score","3":"homologs"},{"1":"vvulpes_homolog_wga_coverage","2":"Red fox Whole-genome alignment coverage","3":"homologs"},{"1":"vvulpes_homolog_orthology_confidence","2":"Red fox orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pnattereri_homolog_ensembl_gene","2":"Red-bellied piranha gene stable ID","3":"homologs"},{"1":"pnattereri_homolog_associated_gene_name","2":"Red-bellied piranha gene name","3":"homologs"},{"1":"pnattereri_homolog_ensembl_peptide","2":"Red-bellied piranha protein or transcript stable ID","3":"homologs"},{"1":"pnattereri_homolog_chromosome","2":"Red-bellied piranha chromosome/scaffold name","3":"homologs"},{"1":"pnattereri_homolog_chrom_start","2":"Red-bellied piranha chromosome/scaffold start (bp)","3":"homologs"},{"1":"pnattereri_homolog_chrom_end","2":"Red-bellied piranha chromosome/scaffold end (bp)","3":"homologs"},{"1":"pnattereri_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pnattereri_homolog_subtype","2":"Last common ancestor with Red-bellied piranha","3":"homologs"},{"1":"pnattereri_homolog_orthology_type","2":"Red-bellied piranha homology type","3":"homologs"},{"1":"pnattereri_homolog_perc_id","2":"%id. target Red-bellied piranha gene identical to query gene","3":"homologs"},{"1":"pnattereri_homolog_perc_id_r1","2":"%id. query gene identical to target Red-bellied piranha gene","3":"homologs"},{"1":"pnattereri_homolog_goc_score","2":"Red-bellied piranha Gene-order conservation score","3":"homologs"},{"1":"pnattereri_homolog_wga_coverage","2":"Red-bellied piranha Whole-genome alignment coverage","3":"homologs"},{"1":"pnattereri_homolog_orthology_confidence","2":"Red-bellied piranha orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ecalabaricus_homolog_ensembl_gene","2":"Reedfish gene stable ID","3":"homologs"},{"1":"ecalabaricus_homolog_associated_gene_name","2":"Reedfish gene name","3":"homologs"},{"1":"ecalabaricus_homolog_ensembl_peptide","2":"Reedfish protein or transcript stable ID","3":"homologs"},{"1":"ecalabaricus_homolog_chromosome","2":"Reedfish chromosome/scaffold name","3":"homologs"},{"1":"ecalabaricus_homolog_chrom_start","2":"Reedfish chromosome/scaffold start (bp)","3":"homologs"},{"1":"ecalabaricus_homolog_chrom_end","2":"Reedfish chromosome/scaffold end (bp)","3":"homologs"},{"1":"ecalabaricus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ecalabaricus_homolog_subtype","2":"Last common ancestor with Reedfish","3":"homologs"},{"1":"ecalabaricus_homolog_orthology_type","2":"Reedfish homology type","3":"homologs"},{"1":"ecalabaricus_homolog_perc_id","2":"%id. target Reedfish gene identical to query gene","3":"homologs"},{"1":"ecalabaricus_homolog_perc_id_r1","2":"%id. query gene identical to target Reedfish gene","3":"homologs"},{"1":"ecalabaricus_homolog_goc_score","2":"Reedfish Gene-order conservation score","3":"homologs"},{"1":"ecalabaricus_homolog_wga_coverage","2":"Reedfish Whole-genome alignment coverage","3":"homologs"},{"1":"ecalabaricus_homolog_orthology_confidence","2":"Reedfish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mcaroli_homolog_ensembl_gene","2":"Ryukyu mouse gene stable ID","3":"homologs"},{"1":"mcaroli_homolog_associated_gene_name","2":"Ryukyu mouse gene name","3":"homologs"},{"1":"mcaroli_homolog_ensembl_peptide","2":"Ryukyu mouse protein or transcript stable ID","3":"homologs"},{"1":"mcaroli_homolog_chromosome","2":"Ryukyu mouse chromosome/scaffold name","3":"homologs"},{"1":"mcaroli_homolog_chrom_start","2":"Ryukyu mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"mcaroli_homolog_chrom_end","2":"Ryukyu mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"mcaroli_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mcaroli_homolog_subtype","2":"Last common ancestor with Ryukyu mouse","3":"homologs"},{"1":"mcaroli_homolog_orthology_type","2":"Ryukyu mouse homology type","3":"homologs"},{"1":"mcaroli_homolog_perc_id","2":"%id. target Ryukyu mouse gene identical to query gene","3":"homologs"},{"1":"mcaroli_homolog_perc_id_r1","2":"%id. query gene identical to target Ryukyu mouse gene","3":"homologs"},{"1":"mcaroli_homolog_goc_score","2":"Ryukyu mouse Gene-order conservation score","3":"homologs"},{"1":"mcaroli_homolog_wga_coverage","2":"Ryukyu mouse Whole-genome alignment coverage","3":"homologs"},{"1":"mcaroli_homolog_orthology_confidence","2":"Ryukyu mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"scerevisiae_homolog_ensembl_gene","2":"Saccharomyces cerevisiae gene stable ID","3":"homologs"},{"1":"scerevisiae_homolog_associated_gene_name","2":"Saccharomyces cerevisiae gene name","3":"homologs"},{"1":"scerevisiae_homolog_ensembl_peptide","2":"Saccharomyces cerevisiae protein or transcript stable ID","3":"homologs"},{"1":"scerevisiae_homolog_chromosome","2":"Saccharomyces cerevisiae chromosome/scaffold name","3":"homologs"},{"1":"scerevisiae_homolog_chrom_start","2":"Saccharomyces cerevisiae chromosome/scaffold start (bp)","3":"homologs"},{"1":"scerevisiae_homolog_chrom_end","2":"Saccharomyces cerevisiae chromosome/scaffold end (bp)","3":"homologs"},{"1":"scerevisiae_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"scerevisiae_homolog_subtype","2":"Last common ancestor with Saccharomyces cerevisiae","3":"homologs"},{"1":"scerevisiae_homolog_orthology_type","2":"Saccharomyces cerevisiae homology type","3":"homologs"},{"1":"scerevisiae_homolog_perc_id","2":"%id. target Saccharomyces cerevisiae gene identical to query gene","3":"homologs"},{"1":"scerevisiae_homolog_perc_id_r1","2":"%id. query gene identical to target Saccharomyces cerevisiae gene","3":"homologs"},{"1":"scerevisiae_homolog_orthology_confidence","2":"Saccharomyces cerevisiae orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"platipinna_homolog_ensembl_gene","2":"Sailfin molly gene stable ID","3":"homologs"},{"1":"platipinna_homolog_associated_gene_name","2":"Sailfin molly gene name","3":"homologs"},{"1":"platipinna_homolog_ensembl_peptide","2":"Sailfin molly protein or transcript stable ID","3":"homologs"},{"1":"platipinna_homolog_chromosome","2":"Sailfin molly chromosome/scaffold name","3":"homologs"},{"1":"platipinna_homolog_chrom_start","2":"Sailfin molly chromosome/scaffold start (bp)","3":"homologs"},{"1":"platipinna_homolog_chrom_end","2":"Sailfin molly chromosome/scaffold end (bp)","3":"homologs"},{"1":"platipinna_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"platipinna_homolog_subtype","2":"Last common ancestor with Sailfin molly","3":"homologs"},{"1":"platipinna_homolog_orthology_type","2":"Sailfin molly homology type","3":"homologs"},{"1":"platipinna_homolog_perc_id","2":"%id. target Sailfin molly gene identical to query gene","3":"homologs"},{"1":"platipinna_homolog_perc_id_r1","2":"%id. query gene identical to target Sailfin molly gene","3":"homologs"},{"1":"platipinna_homolog_goc_score","2":"Sailfin molly Gene-order conservation score","3":"homologs"},{"1":"platipinna_homolog_wga_coverage","2":"Sailfin molly Whole-genome alignment coverage","3":"homologs"},{"1":"platipinna_homolog_orthology_confidence","2":"Sailfin molly orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"oarambouillet_homolog_ensembl_gene","2":"Sheep gene stable ID","3":"homologs"},{"1":"oarambouillet_homolog_associated_gene_name","2":"Sheep gene name","3":"homologs"},{"1":"oarambouillet_homolog_ensembl_peptide","2":"Sheep protein or transcript stable ID","3":"homologs"},{"1":"oarambouillet_homolog_chromosome","2":"Sheep chromosome/scaffold name","3":"homologs"},{"1":"oarambouillet_homolog_chrom_start","2":"Sheep chromosome/scaffold start (bp)","3":"homologs"},{"1":"oarambouillet_homolog_chrom_end","2":"Sheep chromosome/scaffold end (bp)","3":"homologs"},{"1":"oarambouillet_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"oarambouillet_homolog_subtype","2":"Last common ancestor with Sheep","3":"homologs"},{"1":"oarambouillet_homolog_orthology_type","2":"Sheep homology type","3":"homologs"},{"1":"oarambouillet_homolog_perc_id","2":"%id. target Sheep gene identical to query gene","3":"homologs"},{"1":"oarambouillet_homolog_perc_id_r1","2":"%id. query gene identical to target Sheep gene","3":"homologs"},{"1":"oarambouillet_homolog_goc_score","2":"Sheep Gene-order conservation score","3":"homologs"},{"1":"oarambouillet_homolog_wga_coverage","2":"Sheep Whole-genome alignment coverage","3":"homologs"},{"1":"oarambouillet_homolog_orthology_confidence","2":"Sheep orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cvariegatus_homolog_ensembl_gene","2":"Sheepshead minnow gene stable ID","3":"homologs"},{"1":"cvariegatus_homolog_associated_gene_name","2":"Sheepshead minnow gene name","3":"homologs"},{"1":"cvariegatus_homolog_ensembl_peptide","2":"Sheepshead minnow protein or transcript stable ID","3":"homologs"},{"1":"cvariegatus_homolog_chromosome","2":"Sheepshead minnow chromosome/scaffold name","3":"homologs"},{"1":"cvariegatus_homolog_chrom_start","2":"Sheepshead minnow chromosome/scaffold start (bp)","3":"homologs"},{"1":"cvariegatus_homolog_chrom_end","2":"Sheepshead minnow chromosome/scaffold end (bp)","3":"homologs"},{"1":"cvariegatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cvariegatus_homolog_subtype","2":"Last common ancestor with Sheepshead minnow","3":"homologs"},{"1":"cvariegatus_homolog_orthology_type","2":"Sheepshead minnow homology type","3":"homologs"},{"1":"cvariegatus_homolog_perc_id","2":"%id. target Sheepshead minnow gene identical to query gene","3":"homologs"},{"1":"cvariegatus_homolog_perc_id_r1","2":"%id. query gene identical to target Sheepshead minnow gene","3":"homologs"},{"1":"cvariegatus_homolog_goc_score","2":"Sheepshead minnow Gene-order conservation score","3":"homologs"},{"1":"cvariegatus_homolog_wga_coverage","2":"Sheepshead minnow Whole-genome alignment coverage","3":"homologs"},{"1":"cvariegatus_homolog_orthology_confidence","2":"Sheepshead minnow orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"saraneus_homolog_ensembl_gene","2":"Shrew gene stable ID","3":"homologs"},{"1":"saraneus_homolog_associated_gene_name","2":"Shrew gene name","3":"homologs"},{"1":"saraneus_homolog_ensembl_peptide","2":"Shrew protein or transcript stable ID","3":"homologs"},{"1":"saraneus_homolog_chromosome","2":"Shrew chromosome/scaffold name","3":"homologs"},{"1":"saraneus_homolog_chrom_start","2":"Shrew chromosome/scaffold start (bp)","3":"homologs"},{"1":"saraneus_homolog_chrom_end","2":"Shrew chromosome/scaffold end (bp)","3":"homologs"},{"1":"saraneus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"saraneus_homolog_subtype","2":"Last common ancestor with Shrew","3":"homologs"},{"1":"saraneus_homolog_orthology_type","2":"Shrew homology type","3":"homologs"},{"1":"saraneus_homolog_perc_id","2":"%id. target Shrew gene identical to query gene","3":"homologs"},{"1":"saraneus_homolog_perc_id_r1","2":"%id. query gene identical to target Shrew gene","3":"homologs"},{"1":"saraneus_homolog_goc_score","2":"Shrew Gene-order conservation score","3":"homologs"},{"1":"saraneus_homolog_wga_coverage","2":"Shrew Whole-genome alignment coverage","3":"homologs"},{"1":"saraneus_homolog_orthology_confidence","2":"Shrew orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mpahari_homolog_ensembl_gene","2":"Shrew mouse gene stable ID","3":"homologs"},{"1":"mpahari_homolog_associated_gene_name","2":"Shrew mouse gene name","3":"homologs"},{"1":"mpahari_homolog_ensembl_peptide","2":"Shrew mouse protein or transcript stable ID","3":"homologs"},{"1":"mpahari_homolog_chromosome","2":"Shrew mouse chromosome/scaffold name","3":"homologs"},{"1":"mpahari_homolog_chrom_start","2":"Shrew mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"mpahari_homolog_chrom_end","2":"Shrew mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"mpahari_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mpahari_homolog_subtype","2":"Last common ancestor with Shrew mouse","3":"homologs"},{"1":"mpahari_homolog_orthology_type","2":"Shrew mouse homology type","3":"homologs"},{"1":"mpahari_homolog_perc_id","2":"%id. target Shrew mouse gene identical to query gene","3":"homologs"},{"1":"mpahari_homolog_perc_id_r1","2":"%id. query gene identical to target Shrew mouse gene","3":"homologs"},{"1":"mpahari_homolog_goc_score","2":"Shrew mouse Gene-order conservation score","3":"homologs"},{"1":"mpahari_homolog_wga_coverage","2":"Shrew mouse Whole-genome alignment coverage","3":"homologs"},{"1":"mpahari_homolog_orthology_confidence","2":"Shrew mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bsplendens_homolog_ensembl_gene","2":"Siamese fighting fish gene stable ID","3":"homologs"},{"1":"bsplendens_homolog_associated_gene_name","2":"Siamese fighting fish gene name","3":"homologs"},{"1":"bsplendens_homolog_ensembl_peptide","2":"Siamese fighting fish protein or transcript stable ID","3":"homologs"},{"1":"bsplendens_homolog_chromosome","2":"Siamese fighting fish chromosome/scaffold name","3":"homologs"},{"1":"bsplendens_homolog_chrom_start","2":"Siamese fighting fish chromosome/scaffold start (bp)","3":"homologs"},{"1":"bsplendens_homolog_chrom_end","2":"Siamese fighting fish chromosome/scaffold end (bp)","3":"homologs"},{"1":"bsplendens_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bsplendens_homolog_subtype","2":"Last common ancestor with Siamese fighting fish","3":"homologs"},{"1":"bsplendens_homolog_orthology_type","2":"Siamese fighting fish homology type","3":"homologs"},{"1":"bsplendens_homolog_perc_id","2":"%id. target Siamese fighting fish gene identical to query gene","3":"homologs"},{"1":"bsplendens_homolog_perc_id_r1","2":"%id. query gene identical to target Siamese fighting fish gene","3":"homologs"},{"1":"bsplendens_homolog_goc_score","2":"Siamese fighting fish Gene-order conservation score","3":"homologs"},{"1":"bsplendens_homolog_wga_coverage","2":"Siamese fighting fish Whole-genome alignment coverage","3":"homologs"},{"1":"bsplendens_homolog_orthology_confidence","2":"Siamese fighting fish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mmoschiferus_homolog_ensembl_gene","2":"Siberian musk deer gene stable ID","3":"homologs"},{"1":"mmoschiferus_homolog_associated_gene_name","2":"Siberian musk deer gene name","3":"homologs"},{"1":"mmoschiferus_homolog_ensembl_peptide","2":"Siberian musk deer protein or transcript stable ID","3":"homologs"},{"1":"mmoschiferus_homolog_chromosome","2":"Siberian musk deer chromosome/scaffold name","3":"homologs"},{"1":"mmoschiferus_homolog_chrom_start","2":"Siberian musk deer chromosome/scaffold start (bp)","3":"homologs"},{"1":"mmoschiferus_homolog_chrom_end","2":"Siberian musk deer chromosome/scaffold end (bp)","3":"homologs"},{"1":"mmoschiferus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mmoschiferus_homolog_subtype","2":"Last common ancestor with Siberian musk deer","3":"homologs"},{"1":"mmoschiferus_homolog_orthology_type","2":"Siberian musk deer homology type","3":"homologs"},{"1":"mmoschiferus_homolog_perc_id","2":"%id. target Siberian musk deer gene identical to query gene","3":"homologs"},{"1":"mmoschiferus_homolog_perc_id_r1","2":"%id. query gene identical to target Siberian musk deer gene","3":"homologs"},{"1":"mmoschiferus_homolog_goc_score","2":"Siberian musk deer Gene-order conservation score","3":"homologs"},{"1":"mmoschiferus_homolog_wga_coverage","2":"Siberian musk deer Whole-genome alignment coverage","3":"homologs"},{"1":"mmoschiferus_homolog_orthology_confidence","2":"Siberian musk deer orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"choffmanni_homolog_ensembl_gene","2":"Sloth gene stable ID","3":"homologs"},{"1":"choffmanni_homolog_associated_gene_name","2":"Sloth gene name","3":"homologs"},{"1":"choffmanni_homolog_ensembl_peptide","2":"Sloth protein or transcript stable ID","3":"homologs"},{"1":"choffmanni_homolog_chromosome","2":"Sloth chromosome/scaffold name","3":"homologs"},{"1":"choffmanni_homolog_chrom_start","2":"Sloth chromosome/scaffold start (bp)","3":"homologs"},{"1":"choffmanni_homolog_chrom_end","2":"Sloth chromosome/scaffold end (bp)","3":"homologs"},{"1":"choffmanni_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"choffmanni_homolog_subtype","2":"Last common ancestor with Sloth","3":"homologs"},{"1":"choffmanni_homolog_orthology_type","2":"Sloth homology type","3":"homologs"},{"1":"choffmanni_homolog_perc_id","2":"%id. target Sloth gene identical to query gene","3":"homologs"},{"1":"choffmanni_homolog_perc_id_r1","2":"%id. query gene identical to target Sloth gene","3":"homologs"},{"1":"choffmanni_homolog_goc_score","2":"Sloth Gene-order conservation score","3":"homologs"},{"1":"choffmanni_homolog_wga_coverage","2":"Sloth Whole-genome alignment coverage","3":"homologs"},{"1":"choffmanni_homolog_orthology_confidence","2":"Sloth orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"catys_homolog_ensembl_gene","2":"Sooty mangabey gene stable ID","3":"homologs"},{"1":"catys_homolog_associated_gene_name","2":"Sooty mangabey gene name","3":"homologs"},{"1":"catys_homolog_ensembl_peptide","2":"Sooty mangabey protein or transcript stable ID","3":"homologs"},{"1":"catys_homolog_chromosome","2":"Sooty mangabey chromosome/scaffold name","3":"homologs"},{"1":"catys_homolog_chrom_start","2":"Sooty mangabey chromosome/scaffold start (bp)","3":"homologs"},{"1":"catys_homolog_chrom_end","2":"Sooty mangabey chromosome/scaffold end (bp)","3":"homologs"},{"1":"catys_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"catys_homolog_subtype","2":"Last common ancestor with Sooty mangabey","3":"homologs"},{"1":"catys_homolog_orthology_type","2":"Sooty mangabey homology type","3":"homologs"},{"1":"catys_homolog_perc_id","2":"%id. target Sooty mangabey gene identical to query gene","3":"homologs"},{"1":"catys_homolog_perc_id_r1","2":"%id. query gene identical to target Sooty mangabey gene","3":"homologs"},{"1":"catys_homolog_goc_score","2":"Sooty mangabey Gene-order conservation score","3":"homologs"},{"1":"catys_homolog_wga_coverage","2":"Sooty mangabey Whole-genome alignment coverage","3":"homologs"},{"1":"catys_homolog_orthology_confidence","2":"Sooty mangabey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pcatodon_homolog_ensembl_gene","2":"Sperm whale gene stable ID","3":"homologs"},{"1":"pcatodon_homolog_associated_gene_name","2":"Sperm whale gene name","3":"homologs"},{"1":"pcatodon_homolog_ensembl_peptide","2":"Sperm whale protein or transcript stable ID","3":"homologs"},{"1":"pcatodon_homolog_chromosome","2":"Sperm whale chromosome/scaffold name","3":"homologs"},{"1":"pcatodon_homolog_chrom_start","2":"Sperm whale chromosome/scaffold start (bp)","3":"homologs"},{"1":"pcatodon_homolog_chrom_end","2":"Sperm whale chromosome/scaffold end (bp)","3":"homologs"},{"1":"pcatodon_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pcatodon_homolog_subtype","2":"Last common ancestor with Sperm whale","3":"homologs"},{"1":"pcatodon_homolog_orthology_type","2":"Sperm whale homology type","3":"homologs"},{"1":"pcatodon_homolog_perc_id","2":"%id. target Sperm whale gene identical to query gene","3":"homologs"},{"1":"pcatodon_homolog_perc_id_r1","2":"%id. query gene identical to target Sperm whale gene","3":"homologs"},{"1":"pcatodon_homolog_goc_score","2":"Sperm whale Gene-order conservation score","3":"homologs"},{"1":"pcatodon_homolog_wga_coverage","2":"Sperm whale Whole-genome alignment coverage","3":"homologs"},{"1":"pcatodon_homolog_orthology_confidence","2":"Sperm whale orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"apolyacanthus_homolog_ensembl_gene","2":"Spiny chromis gene stable ID","3":"homologs"},{"1":"apolyacanthus_homolog_associated_gene_name","2":"Spiny chromis gene name","3":"homologs"},{"1":"apolyacanthus_homolog_ensembl_peptide","2":"Spiny chromis protein or transcript stable ID","3":"homologs"},{"1":"apolyacanthus_homolog_chromosome","2":"Spiny chromis chromosome/scaffold name","3":"homologs"},{"1":"apolyacanthus_homolog_chrom_start","2":"Spiny chromis chromosome/scaffold start (bp)","3":"homologs"},{"1":"apolyacanthus_homolog_chrom_end","2":"Spiny chromis chromosome/scaffold end (bp)","3":"homologs"},{"1":"apolyacanthus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"apolyacanthus_homolog_subtype","2":"Last common ancestor with Spiny chromis","3":"homologs"},{"1":"apolyacanthus_homolog_orthology_type","2":"Spiny chromis homology type","3":"homologs"},{"1":"apolyacanthus_homolog_perc_id","2":"%id. target Spiny chromis gene identical to query gene","3":"homologs"},{"1":"apolyacanthus_homolog_perc_id_r1","2":"%id. query gene identical to target Spiny chromis gene","3":"homologs"},{"1":"apolyacanthus_homolog_goc_score","2":"Spiny chromis Gene-order conservation score","3":"homologs"},{"1":"apolyacanthus_homolog_wga_coverage","2":"Spiny chromis Whole-genome alignment coverage","3":"homologs"},{"1":"apolyacanthus_homolog_orthology_confidence","2":"Spiny chromis orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"loculatus_homolog_ensembl_gene","2":"Spotted gar gene stable ID","3":"homologs"},{"1":"loculatus_homolog_associated_gene_name","2":"Spotted gar gene name","3":"homologs"},{"1":"loculatus_homolog_ensembl_peptide","2":"Spotted gar protein or transcript stable ID","3":"homologs"},{"1":"loculatus_homolog_chromosome","2":"Spotted gar chromosome/scaffold name","3":"homologs"},{"1":"loculatus_homolog_chrom_start","2":"Spotted gar chromosome/scaffold start (bp)","3":"homologs"},{"1":"loculatus_homolog_chrom_end","2":"Spotted gar chromosome/scaffold end (bp)","3":"homologs"},{"1":"loculatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"loculatus_homolog_subtype","2":"Last common ancestor with Spotted gar","3":"homologs"},{"1":"loculatus_homolog_orthology_type","2":"Spotted gar homology type","3":"homologs"},{"1":"loculatus_homolog_perc_id","2":"%id. target Spotted gar gene identical to query gene","3":"homologs"},{"1":"loculatus_homolog_perc_id_r1","2":"%id. query gene identical to target Spotted gar gene","3":"homologs"},{"1":"loculatus_homolog_goc_score","2":"Spotted gar Gene-order conservation score","3":"homologs"},{"1":"loculatus_homolog_wga_coverage","2":"Spotted gar Whole-genome alignment coverage","3":"homologs"},{"1":"loculatus_homolog_orthology_confidence","2":"Spotted gar orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"itridecemlineatus_homolog_ensembl_gene","2":"Squirrel gene stable ID","3":"homologs"},{"1":"itridecemlineatus_homolog_associated_gene_name","2":"Squirrel gene name","3":"homologs"},{"1":"itridecemlineatus_homolog_ensembl_peptide","2":"Squirrel protein or transcript stable ID","3":"homologs"},{"1":"itridecemlineatus_homolog_chromosome","2":"Squirrel chromosome/scaffold name","3":"homologs"},{"1":"itridecemlineatus_homolog_chrom_start","2":"Squirrel chromosome/scaffold start (bp)","3":"homologs"},{"1":"itridecemlineatus_homolog_chrom_end","2":"Squirrel chromosome/scaffold end (bp)","3":"homologs"},{"1":"itridecemlineatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"itridecemlineatus_homolog_subtype","2":"Last common ancestor with Squirrel","3":"homologs"},{"1":"itridecemlineatus_homolog_orthology_type","2":"Squirrel homology type","3":"homologs"},{"1":"itridecemlineatus_homolog_perc_id","2":"%id. target Squirrel gene identical to query gene","3":"homologs"},{"1":"itridecemlineatus_homolog_perc_id_r1","2":"%id. query gene identical to target Squirrel gene","3":"homologs"},{"1":"itridecemlineatus_homolog_goc_score","2":"Squirrel Gene-order conservation score","3":"homologs"},{"1":"itridecemlineatus_homolog_wga_coverage","2":"Squirrel Whole-genome alignment coverage","3":"homologs"},{"1":"itridecemlineatus_homolog_orthology_confidence","2":"Squirrel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mspicilegus_homolog_ensembl_gene","2":"Steppe mouse gene stable ID","3":"homologs"},{"1":"mspicilegus_homolog_associated_gene_name","2":"Steppe mouse gene name","3":"homologs"},{"1":"mspicilegus_homolog_ensembl_peptide","2":"Steppe mouse protein or transcript stable ID","3":"homologs"},{"1":"mspicilegus_homolog_chromosome","2":"Steppe mouse chromosome/scaffold name","3":"homologs"},{"1":"mspicilegus_homolog_chrom_start","2":"Steppe mouse chromosome/scaffold start (bp)","3":"homologs"},{"1":"mspicilegus_homolog_chrom_end","2":"Steppe mouse chromosome/scaffold end (bp)","3":"homologs"},{"1":"mspicilegus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mspicilegus_homolog_subtype","2":"Last common ancestor with Steppe mouse","3":"homologs"},{"1":"mspicilegus_homolog_orthology_type","2":"Steppe mouse homology type","3":"homologs"},{"1":"mspicilegus_homolog_perc_id","2":"%id. target Steppe mouse gene identical to query gene","3":"homologs"},{"1":"mspicilegus_homolog_perc_id_r1","2":"%id. query gene identical to target Steppe mouse gene","3":"homologs"},{"1":"mspicilegus_homolog_goc_score","2":"Steppe mouse Gene-order conservation score","3":"homologs"},{"1":"mspicilegus_homolog_wga_coverage","2":"Steppe mouse Whole-genome alignment coverage","3":"homologs"},{"1":"mspicilegus_homolog_orthology_confidence","2":"Steppe mouse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"gaculeatus_homolog_ensembl_gene","2":"Stickleback gene stable ID","3":"homologs"},{"1":"gaculeatus_homolog_associated_gene_name","2":"Stickleback gene name","3":"homologs"},{"1":"gaculeatus_homolog_ensembl_peptide","2":"Stickleback protein or transcript stable ID","3":"homologs"},{"1":"gaculeatus_homolog_chromosome","2":"Stickleback chromosome/scaffold name","3":"homologs"},{"1":"gaculeatus_homolog_chrom_start","2":"Stickleback chromosome/scaffold start (bp)","3":"homologs"},{"1":"gaculeatus_homolog_chrom_end","2":"Stickleback chromosome/scaffold end (bp)","3":"homologs"},{"1":"gaculeatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"gaculeatus_homolog_subtype","2":"Last common ancestor with Stickleback","3":"homologs"},{"1":"gaculeatus_homolog_orthology_type","2":"Stickleback homology type","3":"homologs"},{"1":"gaculeatus_homolog_perc_id","2":"%id. target Stickleback gene identical to query gene","3":"homologs"},{"1":"gaculeatus_homolog_perc_id_r1","2":"%id. query gene identical to target Stickleback gene","3":"homologs"},{"1":"gaculeatus_homolog_goc_score","2":"Stickleback Gene-order conservation score","3":"homologs"},{"1":"gaculeatus_homolog_wga_coverage","2":"Stickleback Whole-genome alignment coverage","3":"homologs"},{"1":"gaculeatus_homolog_orthology_confidence","2":"Stickleback orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"pabelii_homolog_ensembl_gene","2":"Sumatran orangutan gene stable ID","3":"homologs"},{"1":"pabelii_homolog_associated_gene_name","2":"Sumatran orangutan gene name","3":"homologs"},{"1":"pabelii_homolog_ensembl_peptide","2":"Sumatran orangutan protein or transcript stable ID","3":"homologs"},{"1":"pabelii_homolog_chromosome","2":"Sumatran orangutan chromosome/scaffold name","3":"homologs"},{"1":"pabelii_homolog_chrom_start","2":"Sumatran orangutan chromosome/scaffold start (bp)","3":"homologs"},{"1":"pabelii_homolog_chrom_end","2":"Sumatran orangutan chromosome/scaffold end (bp)","3":"homologs"},{"1":"pabelii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"pabelii_homolog_subtype","2":"Last common ancestor with Sumatran orangutan","3":"homologs"},{"1":"pabelii_homolog_orthology_type","2":"Sumatran orangutan homology type","3":"homologs"},{"1":"pabelii_homolog_perc_id","2":"%id. target Sumatran orangutan gene identical to query gene","3":"homologs"},{"1":"pabelii_homolog_perc_id_r1","2":"%id. query gene identical to target Sumatran orangutan gene","3":"homologs"},{"1":"pabelii_homolog_goc_score","2":"Sumatran orangutan Gene-order conservation score","3":"homologs"},{"1":"pabelii_homolog_wga_coverage","2":"Sumatran orangutan Whole-genome alignment coverage","3":"homologs"},{"1":"pabelii_homolog_orthology_confidence","2":"Sumatran orangutan orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"csyrichta_homolog_ensembl_gene","2":"Tarsier gene stable ID","3":"homologs"},{"1":"csyrichta_homolog_associated_gene_name","2":"Tarsier gene name","3":"homologs"},{"1":"csyrichta_homolog_ensembl_peptide","2":"Tarsier protein or transcript stable ID","3":"homologs"},{"1":"csyrichta_homolog_chromosome","2":"Tarsier chromosome/scaffold name","3":"homologs"},{"1":"csyrichta_homolog_chrom_start","2":"Tarsier chromosome/scaffold start (bp)","3":"homologs"},{"1":"csyrichta_homolog_chrom_end","2":"Tarsier chromosome/scaffold end (bp)","3":"homologs"},{"1":"csyrichta_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"csyrichta_homolog_subtype","2":"Last common ancestor with Tarsier","3":"homologs"},{"1":"csyrichta_homolog_orthology_type","2":"Tarsier homology type","3":"homologs"},{"1":"csyrichta_homolog_perc_id","2":"%id. target Tarsier gene identical to query gene","3":"homologs"},{"1":"csyrichta_homolog_perc_id_r1","2":"%id. query gene identical to target Tarsier gene","3":"homologs"},{"1":"csyrichta_homolog_goc_score","2":"Tarsier Gene-order conservation score","3":"homologs"},{"1":"csyrichta_homolog_wga_coverage","2":"Tarsier Whole-genome alignment coverage","3":"homologs"},{"1":"csyrichta_homolog_orthology_confidence","2":"Tarsier orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sharrisii_homolog_ensembl_gene","2":"Tasmanian devil gene stable ID","3":"homologs"},{"1":"sharrisii_homolog_associated_gene_name","2":"Tasmanian devil gene name","3":"homologs"},{"1":"sharrisii_homolog_ensembl_peptide","2":"Tasmanian devil protein or transcript stable ID","3":"homologs"},{"1":"sharrisii_homolog_chromosome","2":"Tasmanian devil chromosome/scaffold name","3":"homologs"},{"1":"sharrisii_homolog_chrom_start","2":"Tasmanian devil chromosome/scaffold start (bp)","3":"homologs"},{"1":"sharrisii_homolog_chrom_end","2":"Tasmanian devil chromosome/scaffold end (bp)","3":"homologs"},{"1":"sharrisii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sharrisii_homolog_subtype","2":"Last common ancestor with Tasmanian devil","3":"homologs"},{"1":"sharrisii_homolog_orthology_type","2":"Tasmanian devil homology type","3":"homologs"},{"1":"sharrisii_homolog_perc_id","2":"%id. target Tasmanian devil gene identical to query gene","3":"homologs"},{"1":"sharrisii_homolog_perc_id_r1","2":"%id. query gene identical to target Tasmanian devil gene","3":"homologs"},{"1":"sharrisii_homolog_goc_score","2":"Tasmanian devil Gene-order conservation score","3":"homologs"},{"1":"sharrisii_homolog_orthology_confidence","2":"Tasmanian devil orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"tnigroviridis_homolog_ensembl_gene","2":"Tetraodon gene stable ID","3":"homologs"},{"1":"tnigroviridis_homolog_associated_gene_name","2":"Tetraodon gene name","3":"homologs"},{"1":"tnigroviridis_homolog_ensembl_peptide","2":"Tetraodon protein or transcript stable ID","3":"homologs"},{"1":"tnigroviridis_homolog_chromosome","2":"Tetraodon chromosome/scaffold name","3":"homologs"},{"1":"tnigroviridis_homolog_chrom_start","2":"Tetraodon chromosome/scaffold start (bp)","3":"homologs"},{"1":"tnigroviridis_homolog_chrom_end","2":"Tetraodon chromosome/scaffold end (bp)","3":"homologs"},{"1":"tnigroviridis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"tnigroviridis_homolog_subtype","2":"Last common ancestor with Tetraodon","3":"homologs"},{"1":"tnigroviridis_homolog_orthology_type","2":"Tetraodon homology type","3":"homologs"},{"1":"tnigroviridis_homolog_perc_id","2":"%id. target Tetraodon gene identical to query gene","3":"homologs"},{"1":"tnigroviridis_homolog_perc_id_r1","2":"%id. query gene identical to target Tetraodon gene","3":"homologs"},{"1":"tnigroviridis_homolog_goc_score","2":"Tetraodon Gene-order conservation score","3":"homologs"},{"1":"tnigroviridis_homolog_wga_coverage","2":"Tetraodon Whole-genome alignment coverage","3":"homologs"},{"1":"tnigroviridis_homolog_orthology_confidence","2":"Tetraodon orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"tctriunguis_homolog_ensembl_gene","2":"Three-toed box turtle gene stable ID","3":"homologs"},{"1":"tctriunguis_homolog_associated_gene_name","2":"Three-toed box turtle gene name","3":"homologs"},{"1":"tctriunguis_homolog_ensembl_peptide","2":"Three-toed box turtle protein or transcript stable ID","3":"homologs"},{"1":"tctriunguis_homolog_chromosome","2":"Three-toed box turtle chromosome/scaffold name","3":"homologs"},{"1":"tctriunguis_homolog_chrom_start","2":"Three-toed box turtle chromosome/scaffold start (bp)","3":"homologs"},{"1":"tctriunguis_homolog_chrom_end","2":"Three-toed box turtle chromosome/scaffold end (bp)","3":"homologs"},{"1":"tctriunguis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"tctriunguis_homolog_subtype","2":"Last common ancestor with Three-toed box turtle","3":"homologs"},{"1":"tctriunguis_homolog_orthology_type","2":"Three-toed box turtle homology type","3":"homologs"},{"1":"tctriunguis_homolog_perc_id","2":"%id. target Three-toed box turtle gene identical to query gene","3":"homologs"},{"1":"tctriunguis_homolog_perc_id_r1","2":"%id. query gene identical to target Three-toed box turtle gene","3":"homologs"},{"1":"tctriunguis_homolog_goc_score","2":"Three-toed box turtle Gene-order conservation score","3":"homologs"},{"1":"tctriunguis_homolog_wga_coverage","2":"Three-toed box turtle Whole-genome alignment coverage","3":"homologs"},{"1":"tctriunguis_homolog_orthology_confidence","2":"Three-toed box turtle orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ptaltaica_homolog_ensembl_gene","2":"Tiger gene stable ID","3":"homologs"},{"1":"ptaltaica_homolog_associated_gene_name","2":"Tiger gene name","3":"homologs"},{"1":"ptaltaica_homolog_ensembl_peptide","2":"Tiger protein or transcript stable ID","3":"homologs"},{"1":"ptaltaica_homolog_chromosome","2":"Tiger chromosome/scaffold name","3":"homologs"},{"1":"ptaltaica_homolog_chrom_start","2":"Tiger chromosome/scaffold start (bp)","3":"homologs"},{"1":"ptaltaica_homolog_chrom_end","2":"Tiger chromosome/scaffold end (bp)","3":"homologs"},{"1":"ptaltaica_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ptaltaica_homolog_subtype","2":"Last common ancestor with Tiger","3":"homologs"},{"1":"ptaltaica_homolog_orthology_type","2":"Tiger homology type","3":"homologs"},{"1":"ptaltaica_homolog_perc_id","2":"%id. target Tiger gene identical to query gene","3":"homologs"},{"1":"ptaltaica_homolog_perc_id_r1","2":"%id. query gene identical to target Tiger gene","3":"homologs"},{"1":"ptaltaica_homolog_goc_score","2":"Tiger Gene-order conservation score","3":"homologs"},{"1":"ptaltaica_homolog_wga_coverage","2":"Tiger Whole-genome alignment coverage","3":"homologs"},{"1":"ptaltaica_homolog_orthology_confidence","2":"Tiger orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"hcomes_homolog_ensembl_gene","2":"Tiger tail seahorse gene stable ID","3":"homologs"},{"1":"hcomes_homolog_associated_gene_name","2":"Tiger tail seahorse gene name","3":"homologs"},{"1":"hcomes_homolog_ensembl_peptide","2":"Tiger tail seahorse protein or transcript stable ID","3":"homologs"},{"1":"hcomes_homolog_chromosome","2":"Tiger tail seahorse chromosome/scaffold name","3":"homologs"},{"1":"hcomes_homolog_chrom_start","2":"Tiger tail seahorse chromosome/scaffold start (bp)","3":"homologs"},{"1":"hcomes_homolog_chrom_end","2":"Tiger tail seahorse chromosome/scaffold end (bp)","3":"homologs"},{"1":"hcomes_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"hcomes_homolog_subtype","2":"Last common ancestor with Tiger tail seahorse","3":"homologs"},{"1":"hcomes_homolog_orthology_type","2":"Tiger tail seahorse homology type","3":"homologs"},{"1":"hcomes_homolog_perc_id","2":"%id. target Tiger tail seahorse gene identical to query gene","3":"homologs"},{"1":"hcomes_homolog_perc_id_r1","2":"%id. query gene identical to target Tiger tail seahorse gene","3":"homologs"},{"1":"hcomes_homolog_goc_score","2":"Tiger tail seahorse Gene-order conservation score","3":"homologs"},{"1":"hcomes_homolog_wga_coverage","2":"Tiger tail seahorse Whole-genome alignment coverage","3":"homologs"},{"1":"hcomes_homolog_orthology_confidence","2":"Tiger tail seahorse orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"csemilaevis_homolog_ensembl_gene","2":"Tongue sole gene stable ID","3":"homologs"},{"1":"csemilaevis_homolog_associated_gene_name","2":"Tongue sole gene name","3":"homologs"},{"1":"csemilaevis_homolog_ensembl_peptide","2":"Tongue sole protein or transcript stable ID","3":"homologs"},{"1":"csemilaevis_homolog_chromosome","2":"Tongue sole chromosome/scaffold name","3":"homologs"},{"1":"csemilaevis_homolog_chrom_start","2":"Tongue sole chromosome/scaffold start (bp)","3":"homologs"},{"1":"csemilaevis_homolog_chrom_end","2":"Tongue sole chromosome/scaffold end (bp)","3":"homologs"},{"1":"csemilaevis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"csemilaevis_homolog_subtype","2":"Last common ancestor with Tongue sole","3":"homologs"},{"1":"csemilaevis_homolog_orthology_type","2":"Tongue sole homology type","3":"homologs"},{"1":"csemilaevis_homolog_perc_id","2":"%id. target Tongue sole gene identical to query gene","3":"homologs"},{"1":"csemilaevis_homolog_perc_id_r1","2":"%id. query gene identical to target Tongue sole gene","3":"homologs"},{"1":"csemilaevis_homolog_goc_score","2":"Tongue sole Gene-order conservation score","3":"homologs"},{"1":"csemilaevis_homolog_wga_coverage","2":"Tongue sole Whole-genome alignment coverage","3":"homologs"},{"1":"csemilaevis_homolog_orthology_confidence","2":"Tongue sole orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"tbelangeri_homolog_ensembl_gene","2":"Tree Shrew gene stable ID","3":"homologs"},{"1":"tbelangeri_homolog_associated_gene_name","2":"Tree Shrew gene name","3":"homologs"},{"1":"tbelangeri_homolog_ensembl_peptide","2":"Tree Shrew protein or transcript stable ID","3":"homologs"},{"1":"tbelangeri_homolog_chromosome","2":"Tree Shrew chromosome/scaffold name","3":"homologs"},{"1":"tbelangeri_homolog_chrom_start","2":"Tree Shrew chromosome/scaffold start (bp)","3":"homologs"},{"1":"tbelangeri_homolog_chrom_end","2":"Tree Shrew chromosome/scaffold end (bp)","3":"homologs"},{"1":"tbelangeri_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"tbelangeri_homolog_subtype","2":"Last common ancestor with Tree Shrew","3":"homologs"},{"1":"tbelangeri_homolog_orthology_type","2":"Tree Shrew homology type","3":"homologs"},{"1":"tbelangeri_homolog_perc_id","2":"%id. target Tree Shrew gene identical to query gene","3":"homologs"},{"1":"tbelangeri_homolog_perc_id_r1","2":"%id. query gene identical to target Tree Shrew gene","3":"homologs"},{"1":"tbelangeri_homolog_goc_score","2":"Tree Shrew Gene-order conservation score","3":"homologs"},{"1":"tbelangeri_homolog_wga_coverage","2":"Tree Shrew Whole-genome alignment coverage","3":"homologs"},{"1":"tbelangeri_homolog_orthology_confidence","2":"Tree Shrew orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"xtropicalis_homolog_ensembl_gene","2":"Tropical clawed frog gene stable ID","3":"homologs"},{"1":"xtropicalis_homolog_associated_gene_name","2":"Tropical clawed frog gene name","3":"homologs"},{"1":"xtropicalis_homolog_ensembl_peptide","2":"Tropical clawed frog protein or transcript stable ID","3":"homologs"},{"1":"xtropicalis_homolog_chromosome","2":"Tropical clawed frog chromosome/scaffold name","3":"homologs"},{"1":"xtropicalis_homolog_chrom_start","2":"Tropical clawed frog chromosome/scaffold start (bp)","3":"homologs"},{"1":"xtropicalis_homolog_chrom_end","2":"Tropical clawed frog chromosome/scaffold end (bp)","3":"homologs"},{"1":"xtropicalis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"xtropicalis_homolog_subtype","2":"Last common ancestor with Tropical clawed frog","3":"homologs"},{"1":"xtropicalis_homolog_orthology_type","2":"Tropical clawed frog homology type","3":"homologs"},{"1":"xtropicalis_homolog_perc_id","2":"%id. target Tropical clawed frog gene identical to query gene","3":"homologs"},{"1":"xtropicalis_homolog_perc_id_r1","2":"%id. query gene identical to target Tropical clawed frog gene","3":"homologs"},{"1":"xtropicalis_homolog_goc_score","2":"Tropical clawed frog Gene-order conservation score","3":"homologs"},{"1":"xtropicalis_homolog_orthology_confidence","2":"Tropical clawed frog orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"spunctatus_homolog_ensembl_gene","2":"Tuatara gene stable ID","3":"homologs"},{"1":"spunctatus_homolog_associated_gene_name","2":"Tuatara gene name","3":"homologs"},{"1":"spunctatus_homolog_ensembl_peptide","2":"Tuatara protein or transcript stable ID","3":"homologs"},{"1":"spunctatus_homolog_chromosome","2":"Tuatara chromosome/scaffold name","3":"homologs"},{"1":"spunctatus_homolog_chrom_start","2":"Tuatara chromosome/scaffold start (bp)","3":"homologs"},{"1":"spunctatus_homolog_chrom_end","2":"Tuatara chromosome/scaffold end (bp)","3":"homologs"},{"1":"spunctatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"spunctatus_homolog_subtype","2":"Last common ancestor with Tuatara","3":"homologs"},{"1":"spunctatus_homolog_orthology_type","2":"Tuatara homology type","3":"homologs"},{"1":"spunctatus_homolog_perc_id","2":"%id. target Tuatara gene identical to query gene","3":"homologs"},{"1":"spunctatus_homolog_perc_id_r1","2":"%id. query gene identical to target Tuatara gene","3":"homologs"},{"1":"spunctatus_homolog_goc_score","2":"Tuatara Gene-order conservation score","3":"homologs"},{"1":"spunctatus_homolog_wga_coverage","2":"Tuatara Whole-genome alignment coverage","3":"homologs"},{"1":"spunctatus_homolog_orthology_confidence","2":"Tuatara orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"smaximus_homolog_ensembl_gene","2":"Turbot gene stable ID","3":"homologs"},{"1":"smaximus_homolog_associated_gene_name","2":"Turbot gene name","3":"homologs"},{"1":"smaximus_homolog_ensembl_peptide","2":"Turbot protein or transcript stable ID","3":"homologs"},{"1":"smaximus_homolog_chromosome","2":"Turbot chromosome/scaffold name","3":"homologs"},{"1":"smaximus_homolog_chrom_start","2":"Turbot chromosome/scaffold start (bp)","3":"homologs"},{"1":"smaximus_homolog_chrom_end","2":"Turbot chromosome/scaffold end (bp)","3":"homologs"},{"1":"smaximus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"smaximus_homolog_subtype","2":"Last common ancestor with Turbot","3":"homologs"},{"1":"smaximus_homolog_orthology_type","2":"Turbot homology type","3":"homologs"},{"1":"smaximus_homolog_perc_id","2":"%id. target Turbot gene identical to query gene","3":"homologs"},{"1":"smaximus_homolog_perc_id_r1","2":"%id. query gene identical to target Turbot gene","3":"homologs"},{"1":"smaximus_homolog_goc_score","2":"Turbot Gene-order conservation score","3":"homologs"},{"1":"smaximus_homolog_orthology_confidence","2":"Turbot orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mgallopavo_homolog_ensembl_gene","2":"Turkey gene stable ID","3":"homologs"},{"1":"mgallopavo_homolog_associated_gene_name","2":"Turkey gene name","3":"homologs"},{"1":"mgallopavo_homolog_ensembl_peptide","2":"Turkey protein or transcript stable ID","3":"homologs"},{"1":"mgallopavo_homolog_chromosome","2":"Turkey chromosome/scaffold name","3":"homologs"},{"1":"mgallopavo_homolog_chrom_start","2":"Turkey chromosome/scaffold start (bp)","3":"homologs"},{"1":"mgallopavo_homolog_chrom_end","2":"Turkey chromosome/scaffold end (bp)","3":"homologs"},{"1":"mgallopavo_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mgallopavo_homolog_subtype","2":"Last common ancestor with Turkey","3":"homologs"},{"1":"mgallopavo_homolog_orthology_type","2":"Turkey homology type","3":"homologs"},{"1":"mgallopavo_homolog_perc_id","2":"%id. target Turkey gene identical to query gene","3":"homologs"},{"1":"mgallopavo_homolog_perc_id_r1","2":"%id. query gene identical to target Turkey gene","3":"homologs"},{"1":"mgallopavo_homolog_goc_score","2":"Turkey Gene-order conservation score","3":"homologs"},{"1":"mgallopavo_homolog_orthology_confidence","2":"Turkey orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"nfurzeri_homolog_ensembl_gene","2":"Turquoise killifish gene stable ID","3":"homologs"},{"1":"nfurzeri_homolog_associated_gene_name","2":"Turquoise killifish gene name","3":"homologs"},{"1":"nfurzeri_homolog_ensembl_peptide","2":"Turquoise killifish protein or transcript stable ID","3":"homologs"},{"1":"nfurzeri_homolog_chromosome","2":"Turquoise killifish chromosome/scaffold name","3":"homologs"},{"1":"nfurzeri_homolog_chrom_start","2":"Turquoise killifish chromosome/scaffold start (bp)","3":"homologs"},{"1":"nfurzeri_homolog_chrom_end","2":"Turquoise killifish chromosome/scaffold end (bp)","3":"homologs"},{"1":"nfurzeri_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"nfurzeri_homolog_subtype","2":"Last common ancestor with Turquoise killifish","3":"homologs"},{"1":"nfurzeri_homolog_orthology_type","2":"Turquoise killifish homology type","3":"homologs"},{"1":"nfurzeri_homolog_perc_id","2":"%id. target Turquoise killifish gene identical to query gene","3":"homologs"},{"1":"nfurzeri_homolog_perc_id_r1","2":"%id. query gene identical to target Turquoise killifish gene","3":"homologs"},{"1":"nfurzeri_homolog_goc_score","2":"Turquoise killifish Gene-order conservation score","3":"homologs"},{"1":"nfurzeri_homolog_wga_coverage","2":"Turquoise killifish Whole-genome alignment coverage","3":"homologs"},{"1":"nfurzeri_homolog_orthology_confidence","2":"Turquoise killifish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"ngalili_homolog_ensembl_gene","2":"Upper Galilee mountains blind mole rat gene stable ID","3":"homologs"},{"1":"ngalili_homolog_associated_gene_name","2":"Upper Galilee mountains blind mole rat gene name","3":"homologs"},{"1":"ngalili_homolog_ensembl_peptide","2":"Upper Galilee mountains blind mole rat protein or transcript stable ID","3":"homologs"},{"1":"ngalili_homolog_chromosome","2":"Upper Galilee mountains blind mole rat chromosome/scaffold name","3":"homologs"},{"1":"ngalili_homolog_chrom_start","2":"Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)","3":"homologs"},{"1":"ngalili_homolog_chrom_end","2":"Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)","3":"homologs"},{"1":"ngalili_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"ngalili_homolog_subtype","2":"Last common ancestor with Upper Galilee mountains blind mole rat","3":"homologs"},{"1":"ngalili_homolog_orthology_type","2":"Upper Galilee mountains blind mole rat homology type","3":"homologs"},{"1":"ngalili_homolog_perc_id","2":"%id. target Upper Galilee mountains blind mole rat gene identical to query gene","3":"homologs"},{"1":"ngalili_homolog_perc_id_r1","2":"%id. query gene identical to target Upper Galilee mountains blind mole rat gene","3":"homologs"},{"1":"ngalili_homolog_goc_score","2":"Upper Galilee mountains blind mole rat Gene-order conservation score","3":"homologs"},{"1":"ngalili_homolog_wga_coverage","2":"Upper Galilee mountains blind mole rat Whole-genome alignment coverage","3":"homologs"},{"1":"ngalili_homolog_orthology_confidence","2":"Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"psinus_homolog_ensembl_gene","2":"Vaquita gene stable ID","3":"homologs"},{"1":"psinus_homolog_associated_gene_name","2":"Vaquita gene name","3":"homologs"},{"1":"psinus_homolog_ensembl_peptide","2":"Vaquita protein or transcript stable ID","3":"homologs"},{"1":"psinus_homolog_chromosome","2":"Vaquita chromosome/scaffold name","3":"homologs"},{"1":"psinus_homolog_chrom_start","2":"Vaquita chromosome/scaffold start (bp)","3":"homologs"},{"1":"psinus_homolog_chrom_end","2":"Vaquita chromosome/scaffold end (bp)","3":"homologs"},{"1":"psinus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"psinus_homolog_subtype","2":"Last common ancestor with Vaquita","3":"homologs"},{"1":"psinus_homolog_orthology_type","2":"Vaquita homology type","3":"homologs"},{"1":"psinus_homolog_perc_id","2":"%id. target Vaquita gene identical to query gene","3":"homologs"},{"1":"psinus_homolog_perc_id_r1","2":"%id. query gene identical to target Vaquita gene","3":"homologs"},{"1":"psinus_homolog_goc_score","2":"Vaquita Gene-order conservation score","3":"homologs"},{"1":"psinus_homolog_wga_coverage","2":"Vaquita Whole-genome alignment coverage","3":"homologs"},{"1":"psinus_homolog_orthology_confidence","2":"Vaquita orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"csabaeus_homolog_ensembl_gene","2":"Vervet-AGM gene stable ID","3":"homologs"},{"1":"csabaeus_homolog_associated_gene_name","2":"Vervet-AGM gene name","3":"homologs"},{"1":"csabaeus_homolog_ensembl_peptide","2":"Vervet-AGM protein or transcript stable ID","3":"homologs"},{"1":"csabaeus_homolog_chromosome","2":"Vervet-AGM chromosome/scaffold name","3":"homologs"},{"1":"csabaeus_homolog_chrom_start","2":"Vervet-AGM chromosome/scaffold start (bp)","3":"homologs"},{"1":"csabaeus_homolog_chrom_end","2":"Vervet-AGM chromosome/scaffold end (bp)","3":"homologs"},{"1":"csabaeus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"csabaeus_homolog_subtype","2":"Last common ancestor with Vervet-AGM","3":"homologs"},{"1":"csabaeus_homolog_orthology_type","2":"Vervet-AGM homology type","3":"homologs"},{"1":"csabaeus_homolog_perc_id","2":"%id. target Vervet-AGM gene identical to query gene","3":"homologs"},{"1":"csabaeus_homolog_perc_id_r1","2":"%id. query gene identical to target Vervet-AGM gene","3":"homologs"},{"1":"csabaeus_homolog_goc_score","2":"Vervet-AGM Gene-order conservation score","3":"homologs"},{"1":"csabaeus_homolog_wga_coverage","2":"Vervet-AGM Whole-genome alignment coverage","3":"homologs"},{"1":"csabaeus_homolog_orthology_confidence","2":"Vervet-AGM orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"neugenii_homolog_ensembl_gene","2":"Wallaby gene stable ID","3":"homologs"},{"1":"neugenii_homolog_associated_gene_name","2":"Wallaby gene name","3":"homologs"},{"1":"neugenii_homolog_ensembl_peptide","2":"Wallaby protein or transcript stable ID","3":"homologs"},{"1":"neugenii_homolog_chromosome","2":"Wallaby chromosome/scaffold name","3":"homologs"},{"1":"neugenii_homolog_chrom_start","2":"Wallaby chromosome/scaffold start (bp)","3":"homologs"},{"1":"neugenii_homolog_chrom_end","2":"Wallaby chromosome/scaffold end (bp)","3":"homologs"},{"1":"neugenii_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"neugenii_homolog_subtype","2":"Last common ancestor with Wallaby","3":"homologs"},{"1":"neugenii_homolog_orthology_type","2":"Wallaby homology type","3":"homologs"},{"1":"neugenii_homolog_perc_id","2":"%id. target Wallaby gene identical to query gene","3":"homologs"},{"1":"neugenii_homolog_perc_id_r1","2":"%id. query gene identical to target Wallaby gene","3":"homologs"},{"1":"neugenii_homolog_goc_score","2":"Wallaby Gene-order conservation score","3":"homologs"},{"1":"neugenii_homolog_wga_coverage","2":"Wallaby Whole-genome alignment coverage","3":"homologs"},{"1":"neugenii_homolog_orthology_confidence","2":"Wallaby orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"cjacchus_homolog_ensembl_gene","2":"White-tufted-ear marmoset gene stable ID","3":"homologs"},{"1":"cjacchus_homolog_associated_gene_name","2":"White-tufted-ear marmoset gene name","3":"homologs"},{"1":"cjacchus_homolog_ensembl_peptide","2":"White-tufted-ear marmoset protein or transcript stable ID","3":"homologs"},{"1":"cjacchus_homolog_chromosome","2":"White-tufted-ear marmoset chromosome/scaffold name","3":"homologs"},{"1":"cjacchus_homolog_chrom_start","2":"White-tufted-ear marmoset chromosome/scaffold start (bp)","3":"homologs"},{"1":"cjacchus_homolog_chrom_end","2":"White-tufted-ear marmoset chromosome/scaffold end (bp)","3":"homologs"},{"1":"cjacchus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"cjacchus_homolog_subtype","2":"Last common ancestor with White-tufted-ear marmoset","3":"homologs"},{"1":"cjacchus_homolog_orthology_type","2":"White-tufted-ear marmoset homology type","3":"homologs"},{"1":"cjacchus_homolog_perc_id","2":"%id. target White-tufted-ear marmoset gene identical to query gene","3":"homologs"},{"1":"cjacchus_homolog_perc_id_r1","2":"%id. query gene identical to target White-tufted-ear marmoset gene","3":"homologs"},{"1":"cjacchus_homolog_goc_score","2":"White-tufted-ear marmoset Gene-order conservation score","3":"homologs"},{"1":"cjacchus_homolog_wga_coverage","2":"White-tufted-ear marmoset Whole-genome alignment coverage","3":"homologs"},{"1":"cjacchus_homolog_orthology_confidence","2":"White-tufted-ear marmoset orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"bmutus_homolog_ensembl_gene","2":"Wild yak gene stable ID","3":"homologs"},{"1":"bmutus_homolog_associated_gene_name","2":"Wild yak gene name","3":"homologs"},{"1":"bmutus_homolog_ensembl_peptide","2":"Wild yak protein or transcript stable ID","3":"homologs"},{"1":"bmutus_homolog_chromosome","2":"Wild yak chromosome/scaffold name","3":"homologs"},{"1":"bmutus_homolog_chrom_start","2":"Wild yak chromosome/scaffold start (bp)","3":"homologs"},{"1":"bmutus_homolog_chrom_end","2":"Wild yak chromosome/scaffold end (bp)","3":"homologs"},{"1":"bmutus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"bmutus_homolog_subtype","2":"Last common ancestor with Wild yak","3":"homologs"},{"1":"bmutus_homolog_orthology_type","2":"Wild yak homology type","3":"homologs"},{"1":"bmutus_homolog_perc_id","2":"%id. target Wild yak gene identical to query gene","3":"homologs"},{"1":"bmutus_homolog_perc_id_r1","2":"%id. query gene identical to target Wild yak gene","3":"homologs"},{"1":"bmutus_homolog_goc_score","2":"Wild yak Gene-order conservation score","3":"homologs"},{"1":"bmutus_homolog_wga_coverage","2":"Wild yak Whole-genome alignment coverage","3":"homologs"},{"1":"bmutus_homolog_orthology_confidence","2":"Wild yak orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"chyarkandensis_homolog_ensembl_gene","2":"Yarkand deer gene stable ID","3":"homologs"},{"1":"chyarkandensis_homolog_associated_gene_name","2":"Yarkand deer gene name","3":"homologs"},{"1":"chyarkandensis_homolog_ensembl_peptide","2":"Yarkand deer protein or transcript stable ID","3":"homologs"},{"1":"chyarkandensis_homolog_chromosome","2":"Yarkand deer chromosome/scaffold name","3":"homologs"},{"1":"chyarkandensis_homolog_chrom_start","2":"Yarkand deer chromosome/scaffold start (bp)","3":"homologs"},{"1":"chyarkandensis_homolog_chrom_end","2":"Yarkand deer chromosome/scaffold end (bp)","3":"homologs"},{"1":"chyarkandensis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"chyarkandensis_homolog_subtype","2":"Last common ancestor with Yarkand deer","3":"homologs"},{"1":"chyarkandensis_homolog_orthology_type","2":"Yarkand deer homology type","3":"homologs"},{"1":"chyarkandensis_homolog_perc_id","2":"%id. target Yarkand deer gene identical to query gene","3":"homologs"},{"1":"chyarkandensis_homolog_perc_id_r1","2":"%id. query gene identical to target Yarkand deer gene","3":"homologs"},{"1":"chyarkandensis_homolog_goc_score","2":"Yarkand deer Gene-order conservation score","3":"homologs"},{"1":"chyarkandensis_homolog_wga_coverage","2":"Yarkand deer Whole-genome alignment coverage","3":"homologs"},{"1":"chyarkandensis_homolog_orthology_confidence","2":"Yarkand deer orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"sldorsalis_homolog_ensembl_gene","2":"Yellowtail amberjack gene stable ID","3":"homologs"},{"1":"sldorsalis_homolog_associated_gene_name","2":"Yellowtail amberjack gene name","3":"homologs"},{"1":"sldorsalis_homolog_ensembl_peptide","2":"Yellowtail amberjack protein or transcript stable ID","3":"homologs"},{"1":"sldorsalis_homolog_chromosome","2":"Yellowtail amberjack chromosome/scaffold name","3":"homologs"},{"1":"sldorsalis_homolog_chrom_start","2":"Yellowtail amberjack chromosome/scaffold start (bp)","3":"homologs"},{"1":"sldorsalis_homolog_chrom_end","2":"Yellowtail amberjack chromosome/scaffold end (bp)","3":"homologs"},{"1":"sldorsalis_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"sldorsalis_homolog_subtype","2":"Last common ancestor with Yellowtail amberjack","3":"homologs"},{"1":"sldorsalis_homolog_orthology_type","2":"Yellowtail amberjack homology type","3":"homologs"},{"1":"sldorsalis_homolog_perc_id","2":"%id. target Yellowtail amberjack gene identical to query gene","3":"homologs"},{"1":"sldorsalis_homolog_perc_id_r1","2":"%id. query gene identical to target Yellowtail amberjack gene","3":"homologs"},{"1":"sldorsalis_homolog_goc_score","2":"Yellowtail amberjack Gene-order conservation score","3":"homologs"},{"1":"sldorsalis_homolog_wga_coverage","2":"Yellowtail amberjack Whole-genome alignment coverage","3":"homologs"},{"1":"sldorsalis_homolog_orthology_confidence","2":"Yellowtail amberjack orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"tguttata_homolog_ensembl_gene","2":"Zebra finch gene stable ID","3":"homologs"},{"1":"tguttata_homolog_associated_gene_name","2":"Zebra finch gene name","3":"homologs"},{"1":"tguttata_homolog_ensembl_peptide","2":"Zebra finch protein or transcript stable ID","3":"homologs"},{"1":"tguttata_homolog_chromosome","2":"Zebra finch chromosome/scaffold name","3":"homologs"},{"1":"tguttata_homolog_chrom_start","2":"Zebra finch chromosome/scaffold start (bp)","3":"homologs"},{"1":"tguttata_homolog_chrom_end","2":"Zebra finch chromosome/scaffold end (bp)","3":"homologs"},{"1":"tguttata_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"tguttata_homolog_subtype","2":"Last common ancestor with Zebra finch","3":"homologs"},{"1":"tguttata_homolog_orthology_type","2":"Zebra finch homology type","3":"homologs"},{"1":"tguttata_homolog_perc_id","2":"%id. target Zebra finch gene identical to query gene","3":"homologs"},{"1":"tguttata_homolog_perc_id_r1","2":"%id. query gene identical to target Zebra finch gene","3":"homologs"},{"1":"tguttata_homolog_goc_score","2":"Zebra finch Gene-order conservation score","3":"homologs"},{"1":"tguttata_homolog_wga_coverage","2":"Zebra finch Whole-genome alignment coverage","3":"homologs"},{"1":"tguttata_homolog_orthology_confidence","2":"Zebra finch orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"mzebra_homolog_ensembl_gene","2":"Zebra mbuna gene stable ID","3":"homologs"},{"1":"mzebra_homolog_associated_gene_name","2":"Zebra mbuna gene name","3":"homologs"},{"1":"mzebra_homolog_ensembl_peptide","2":"Zebra mbuna protein or transcript stable ID","3":"homologs"},{"1":"mzebra_homolog_chromosome","2":"Zebra mbuna chromosome/scaffold name","3":"homologs"},{"1":"mzebra_homolog_chrom_start","2":"Zebra mbuna chromosome/scaffold start (bp)","3":"homologs"},{"1":"mzebra_homolog_chrom_end","2":"Zebra mbuna chromosome/scaffold end (bp)","3":"homologs"},{"1":"mzebra_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"mzebra_homolog_subtype","2":"Last common ancestor with Zebra mbuna","3":"homologs"},{"1":"mzebra_homolog_orthology_type","2":"Zebra mbuna homology type","3":"homologs"},{"1":"mzebra_homolog_perc_id","2":"%id. target Zebra mbuna gene identical to query gene","3":"homologs"},{"1":"mzebra_homolog_perc_id_r1","2":"%id. query gene identical to target Zebra mbuna gene","3":"homologs"},{"1":"mzebra_homolog_goc_score","2":"Zebra mbuna Gene-order conservation score","3":"homologs"},{"1":"mzebra_homolog_wga_coverage","2":"Zebra mbuna Whole-genome alignment coverage","3":"homologs"},{"1":"mzebra_homolog_orthology_confidence","2":"Zebra mbuna orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"drerio_homolog_ensembl_gene","2":"Zebrafish gene stable ID","3":"homologs"},{"1":"drerio_homolog_associated_gene_name","2":"Zebrafish gene name","3":"homologs"},{"1":"drerio_homolog_ensembl_peptide","2":"Zebrafish protein or transcript stable ID","3":"homologs"},{"1":"drerio_homolog_chromosome","2":"Zebrafish chromosome/scaffold name","3":"homologs"},{"1":"drerio_homolog_chrom_start","2":"Zebrafish chromosome/scaffold start (bp)","3":"homologs"},{"1":"drerio_homolog_chrom_end","2":"Zebrafish chromosome/scaffold end (bp)","3":"homologs"},{"1":"drerio_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"drerio_homolog_subtype","2":"Last common ancestor with Zebrafish","3":"homologs"},{"1":"drerio_homolog_orthology_type","2":"Zebrafish homology type","3":"homologs"},{"1":"drerio_homolog_perc_id","2":"%id. target Zebrafish gene identical to query gene","3":"homologs"},{"1":"drerio_homolog_perc_id_r1","2":"%id. query gene identical to target Zebrafish gene","3":"homologs"},{"1":"drerio_homolog_goc_score","2":"Zebrafish Gene-order conservation score","3":"homologs"},{"1":"drerio_homolog_wga_coverage","2":"Zebrafish Whole-genome alignment coverage","3":"homologs"},{"1":"drerio_homolog_orthology_confidence","2":"Zebrafish orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"marmatus_homolog_ensembl_gene","2":"Zig-zag eel gene stable ID","3":"homologs"},{"1":"marmatus_homolog_associated_gene_name","2":"Zig-zag eel gene name","3":"homologs"},{"1":"marmatus_homolog_ensembl_peptide","2":"Zig-zag eel protein or transcript stable ID","3":"homologs"},{"1":"marmatus_homolog_chromosome","2":"Zig-zag eel chromosome/scaffold name","3":"homologs"},{"1":"marmatus_homolog_chrom_start","2":"Zig-zag eel chromosome/scaffold start (bp)","3":"homologs"},{"1":"marmatus_homolog_chrom_end","2":"Zig-zag eel chromosome/scaffold end (bp)","3":"homologs"},{"1":"marmatus_homolog_canonical_transcript_protein","2":"Query protein or transcript ID","3":"homologs"},{"1":"marmatus_homolog_subtype","2":"Last common ancestor with Zig-zag eel","3":"homologs"},{"1":"marmatus_homolog_orthology_type","2":"Zig-zag eel homology type","3":"homologs"},{"1":"marmatus_homolog_perc_id","2":"%id. target Zig-zag eel gene identical to query gene","3":"homologs"},{"1":"marmatus_homolog_perc_id_r1","2":"%id. query gene identical to target Zig-zag eel gene","3":"homologs"},{"1":"marmatus_homolog_goc_score","2":"Zig-zag eel Gene-order conservation score","3":"homologs"},{"1":"marmatus_homolog_orthology_confidence","2":"Zig-zag eel orthology confidence [0 low, 1 high]","3":"homologs"},{"1":"hsapiens_paralog_ensembl_gene","2":"Human paralogue gene stable ID","3":"homologs"},{"1":"hsapiens_paralog_associated_gene_name","2":"Human paralogue associated gene name","3":"homologs"},{"1":"hsapiens_paralog_ensembl_peptide","2":"Human paralogue protein or transcript ID","3":"homologs"},{"1":"hsapiens_paralog_chromosome","2":"Human paralogue chromosome/scaffold name","3":"homologs"},{"1":"hsapiens_paralog_chrom_start","2":"Human paralogue chromosome/scaffold start (bp)","3":"homologs"},{"1":"hsapiens_paralog_chrom_end","2":"Human paralogue chromosome/scaffold end (bp)","3":"homologs"},{"1":"hsapiens_paralog_canonical_transcript_protein","2":"Paralogue query protein or transcript ID","3":"homologs"},{"1":"hsapiens_paralog_subtype","2":"Paralogue last common ancestor with Human","3":"homologs"},{"1":"hsapiens_paralog_orthology_type","2":"Human paralogue homology type","3":"homologs"},{"1":"hsapiens_paralog_perc_id","2":"Paralogue %id. target Human gene identical to query gene","3":"homologs"},{"1":"hsapiens_paralog_perc_id_r1","2":"Paralogue %id. query gene identical to target Human gene","3":"homologs"},{"1":"ensembl_gene_id","2":"Gene stable ID","3":"snp"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"snp"},{"1":"version","2":"Version (gene)","3":"snp"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"snp"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"snp"},{"1":"transcript_version","2":"Version (transcript)","3":"snp"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"snp"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"snp"},{"1":"peptide_version","2":"Version (protein)","3":"snp"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"snp"},{"1":"start_position","2":"Gene start (bp)","3":"snp"},{"1":"end_position","2":"Gene end (bp)","3":"snp"},{"1":"strand","2":"Strand","3":"snp"},{"1":"band","2":"Karyotype band","3":"snp"},{"1":"external_gene_name","2":"Gene name","3":"snp"},{"1":"external_gene_source","2":"Source of gene name","3":"snp"},{"1":"transcript_count","2":"Transcript count","3":"snp"},{"1":"percentage_gene_gc_content","2":"Gene % GC content","3":"snp"},{"1":"description","2":"Gene description","3":"snp"},{"1":"variation_name","2":"Variant name","3":"snp"},{"1":"germ_line_variation_source","2":"Variant source","3":"snp"},{"1":"source_description","2":"Variant source description","3":"snp"},{"1":"allele","2":"Variant alleles","3":"snp"},{"1":"validated","2":"Variant supporting evidence","3":"snp"},{"1":"mapweight","2":"Mapweight","3":"snp"},{"1":"minor_allele","2":"Minor allele","3":"snp"},{"1":"minor_allele_freq","2":"Minor allele frequency","3":"snp"},{"1":"minor_allele_count","2":"Minor allele count","3":"snp"},{"1":"clinical_significance","2":"Clinical significance","3":"snp"},{"1":"transcript_location","2":"Transcript location (bp)","3":"snp"},{"1":"snp_chromosome_strand","2":"Variant chromosome Strand","3":"snp"},{"1":"peptide_location","2":"Protein location (aa)","3":"snp"},{"1":"chromosome_start","2":"chromosome/scaffold position start (bp)","3":"snp"},{"1":"chromosome_end","2":"Chromosome/scaffold position end (bp)","3":"snp"},{"1":"polyphen_prediction_2076","2":"PolyPhen prediction","3":"snp"},{"1":"polyphen_score_2076","2":"PolyPhen score","3":"snp"},{"1":"sift_prediction_2076","2":"SIFT prediction","3":"snp"},{"1":"sift_score_2076","2":"SIFT score","3":"snp"},{"1":"distance_to_transcript_2076","2":"Distance to transcript","3":"snp"},{"1":"cds_start_2076","2":"CDS start","3":"snp"},{"1":"cds_end_2076","2":"CDS end","3":"snp"},{"1":"peptide_shift","2":"Protein allele","3":"snp"},{"1":"synonymous_status","2":"Variant consequence","3":"snp"},{"1":"allele_string_2076","2":"Consequence specific allele","3":"snp"},{"1":"ensembl_gene_id","2":"Gene stable ID","3":"snp_somatic"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"snp_somatic"},{"1":"version","2":"Version (gene)","3":"snp_somatic"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"snp_somatic"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"snp_somatic"},{"1":"transcript_version","2":"Version (transcript)","3":"snp_somatic"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"snp_somatic"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"snp_somatic"},{"1":"peptide_version","2":"Version (protein)","3":"snp_somatic"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"snp_somatic"},{"1":"start_position","2":"Gene start (bp)","3":"snp_somatic"},{"1":"end_position","2":"Gene end (bp)","3":"snp_somatic"},{"1":"strand","2":"Strand","3":"snp_somatic"},{"1":"band","2":"Karyotype band","3":"snp_somatic"},{"1":"external_gene_name","2":"Gene name","3":"snp_somatic"},{"1":"external_gene_source","2":"Source of gene name","3":"snp_somatic"},{"1":"transcript_count","2":"Transcript count","3":"snp_somatic"},{"1":"percentage_gene_gc_content","2":"Gene % GC content","3":"snp_somatic"},{"1":"description","2":"Gene description","3":"snp_somatic"},{"1":"somatic_variation_name","2":"Variant name","3":"snp_somatic"},{"1":"somatic_source_name","2":"Variant source","3":"snp_somatic"},{"1":"somatic_source_description","2":"Variant source description","3":"snp_somatic"},{"1":"somatic_allele","2":"Variant alleles","3":"snp_somatic"},{"1":"somatic_validated","2":"Variant supporting evidence","3":"snp_somatic"},{"1":"somatic_mapweight","2":"Mapweight","3":"snp_somatic"},{"1":"somatic_transcript_location","2":"Transcript location (bp)","3":"snp_somatic"},{"1":"somatic_snp_chromosome_strand","2":"Variant chromosome/scaffold strand","3":"snp_somatic"},{"1":"somatic_peptide_location","2":"Protein location (aa)","3":"snp_somatic"},{"1":"somatic_chromosome_start","2":"Chromosome/scaffold position start (bp)","3":"snp_somatic"},{"1":"somatic_chromosome_end","2":"Chromosome/scaffold position end (bp)","3":"snp_somatic"},{"1":"mart_transcript_variation_som__dm_distance_to_transcript_2076","2":"Distance to transcript","3":"snp_somatic"},{"1":"somatic_cds_start_2076","2":"CDS start","3":"snp_somatic"},{"1":"somatic_cds_end_2076","2":"CDS end","3":"snp_somatic"},{"1":"somatic_synonymous_status","2":"Variant consequence","3":"snp_somatic"},{"1":"mart_transcript_variation_som__dm_allele_string_2076","2":"Consequence specific allele","3":"snp_somatic"},{"1":"transcript_exon_intron","2":"Unspliced (Transcript)","3":"sequences"},{"1":"gene_exon_intron","2":"Unspliced (Gene)","3":"sequences"},{"1":"transcript_flank","2":"Flank (Transcript)","3":"sequences"},{"1":"gene_flank","2":"Flank (Gene)","3":"sequences"},{"1":"coding_transcript_flank","2":"Flank-coding region (Transcript)","3":"sequences"},{"1":"coding_gene_flank","2":"Flank-coding region (Gene)","3":"sequences"},{"1":"5utr","2":"5' UTR","3":"sequences"},{"1":"3utr","2":"3' UTR","3":"sequences"},{"1":"gene_exon","2":"Exon sequences","3":"sequences"},{"1":"cdna","2":"cDNA sequences","3":"sequences"},{"1":"coding","2":"Coding sequence","3":"sequences"},{"1":"peptide","2":"Peptide","3":"sequences"},{"1":"upstream_flank","2":"upstream_flank","3":"sequences"},{"1":"downstream_flank","2":"downstream_flank","3":"sequences"},{"1":"ensembl_gene_id","2":"Gene stable ID","3":"sequences"},{"1":"ensembl_gene_id_version","2":"Gene stable ID version","3":"sequences"},{"1":"description","2":"Gene description","3":"sequences"},{"1":"external_gene_name","2":"Gene name","3":"sequences"},{"1":"external_gene_source","2":"Source of gene name","3":"sequences"},{"1":"chromosome_name","2":"Chromosome/scaffold name","3":"sequences"},{"1":"start_position","2":"Gene start (bp)","3":"sequences"},{"1":"end_position","2":"Gene end (bp)","3":"sequences"},{"1":"gene_biotype","2":"Gene type","3":"sequences"},{"1":"version","2":"Version (gene)","3":"sequences"},{"1":"uniparc","2":"UniParc ID","3":"sequences"},{"1":"uniprotswissprot","2":"UniProtKB/Swiss-Prot ID","3":"sequences"},{"1":"uniprotsptrembl","2":"UniProtKB/TrEMBL ID","3":"sequences"},{"1":"cdna_coding_start","2":"CDS start (within cDNA)","3":"sequences"},{"1":"cdna_coding_end","2":"CDS end (within cDNA)","3":"sequences"},{"1":"5_utr_start","2":"5' UTR start","3":"sequences"},{"1":"5_utr_end","2":"5' UTR end","3":"sequences"},{"1":"3_utr_start","2":"3' UTR start","3":"sequences"},{"1":"3_utr_end","2":"3' UTR end","3":"sequences"},{"1":"ensembl_transcript_id","2":"Transcript stable ID","3":"sequences"},{"1":"ensembl_transcript_id_version","2":"Transcript stable ID version","3":"sequences"},{"1":"ensembl_peptide_id","2":"Protein stable ID","3":"sequences"},{"1":"ensembl_peptide_id_version","2":"Protein stable ID version","3":"sequences"},{"1":"transcript_biotype","2":"Transcript type","3":"sequences"},{"1":"transcript_version","2":"Version (transcript)","3":"sequences"},{"1":"peptide_version","2":"Version (protein)","3":"sequences"},{"1":"strand","2":"Strand","3":"sequences"},{"1":"transcript_start","2":"Transcript start (bp)","3":"sequences"},{"1":"transcript_end","2":"Transcript end (bp)","3":"sequences"},{"1":"transcription_start_site","2":"Transcription start site (TSS)","3":"sequences"},{"1":"transcript_length","2":"Transcript length (including UTRs and CDS)","3":"sequences"},{"1":"cds_length","2":"CDS Length","3":"sequences"},{"1":"cds_start","2":"CDS start","3":"sequences"},{"1":"cds_end","2":"CDS end","3":"sequences"},{"1":"ensembl_exon_id","2":"Exon stable ID","3":"sequences"},{"1":"exon_chrom_start","2":"Exon region start (bp)","3":"sequences"},{"1":"exon_chrom_end","2":"Exon region end (bp)","3":"sequences"},{"1":"strand","2":"Strand","3":"sequences"},{"1":"rank","2":"Exon rank in transcript","3":"sequences"},{"1":"phase","2":"Start phase","3":"sequences"},{"1":"end_phase","2":"End phase","3":"sequences"},{"1":"cdna_coding_start","2":"cDNA coding start","3":"sequences"},{"1":"cdna_coding_end","2":"cDNA coding end","3":"sequences"},{"1":"genomic_coding_start","2":"Genomic coding start","3":"sequences"},{"1":"genomic_coding_end","2":"Genomic coding end","3":"sequences"},{"1":"is_constitutive","2":"Constitutive exon","3":"sequences"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="biomart-4" class="slide level2">
<h2>BioMart</h2>
<h3 id="query-with-getbm">Query with <code>getBM()</code></h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a><span class="fu">getBM</span>(<span class="at">attributes =</span> <span class="fu">c</span>(<span class="st">"ensembl_gene_id"</span>,<span class="st">"entrezgene_id"</span>,<span class="st">"hgnc_symbol"</span>),</span>
<span id="cb10-2"><a href="#cb10-2"></a>      <span class="at">filters =</span> <span class="st">"ensembl_gene_id"</span>,</span>
<span id="cb10-3"><a href="#cb10-3"></a>      <span class="at">values =</span> de_genes<span class="sc">$</span>rowname, </span>
<span id="cb10-4"><a href="#cb10-4"></a>      <span class="at">mart =</span> ensembl)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>    ensembl_gene_id entrezgene_id hgnc_symbol
1   ENSG00000006459         80853       KDM7A
2   ENSG00000008513          6482     ST3GAL1
3   ENSG00000010295         25900       IFFO1
4   ENSG00000018408         25937       WWTR1
5   ENSG00000023445           330       BIRC3
6   ENSG00000026103           355         FAS
7   ENSG00000031081         57514    ARHGAP31
8   ENSG00000047365        116984       ARAP2
9   ENSG00000049130          4254       KITLG
10  ENSG00000049759         23327      NEDD4L
11  ENSG00000056558          7185       TRAF1
12  ENSG00000058335          5923     RASGRF1
13  ENSG00000067798         89795        NAV3
14  ENSG00000067955           865        CBFB
15  ENSG00000069020        375449       MAST4
16  ENSG00000071575         28951       TRIB2
17  ENSG00000075618          6624       FSCN1
18  ENSG00000076356          5362      PLXNA2
19  ENSG00000078269          8871       SYNJ2
20  ENSG00000080546         27244       SESN1
21  ENSG00000090020          6548      SLC9A1
22  ENSG00000090776          1947       EFNB1
23  ENSG00000095752          3589        IL11
24  ENSG00000099250          8829        NRP1
25  ENSG00000100036        339665     SLC35E4
26  ENSG00000100505        114088       TRIM9
27  ENSG00000100985          4318        MMP9
28  ENSG00000101134         55816        DOK5
29  ENSG00000101187         28231     SLCO4A1
30  ENSG00000101236         11237       RNF24
31  ENSG00000102003          6855         SYP
32  ENSG00000102265          7076       TIMP1
33  ENSG00000103257          8140      SLC7A5
34  ENSG00000103647         10391      CORO2B
35  ENSG00000104856          5971        RELB
36  ENSG00000104998          9466      IL27RA
37  ENSG00000105327         27113        BBC3
38  ENSG00000105855          3696       ITGB8
39  ENSG00000106366          5054    SERPINE1
40  ENSG00000107554         23268       DNMBP
41  ENSG00000107719         27143       PALD1
42  ENSG00000108018        114815      SORCS1
43  ENSG00000108691          6347        CCL2
44  ENSG00000109220         26511       CHIC2
45  ENSG00000110047         10938        EHD1
46  ENSG00000111252         10019       SH2B3
47  ENSG00000111262          3736       KCNA1
48  ENSG00000111860        387119      CEP85L
49  ENSG00000112149          9308        CD83
50  ENSG00000112559          4188        MDFI
51  ENSG00000113070          1839       HBEGF
52  ENSG00000113721          5159      PDGFRB
53  ENSG00000115009          6364       CCL20
54  ENSG00000115641          2274        FHL2
55  ENSG00000116016          2034       EPAS1
56  ENSG00000116106          2043       EPHA4
57  ENSG00000116285         54206      ERRFI1
58  ENSG00000117152          5999        RGS4
59  ENSG00000118503          7128     TNFAIP3
60  ENSG00000118922         11278       KLF12
61  ENSG00000119681          4053       LTBP2
62  ENSG00000119772          1788      DNMT3A
63  ENSG00000120093          3213       HOXB3
64  ENSG00000120217         29126       CD274
65  ENSG00000122861          5328        PLAU
66  ENSG00000122863          9469       CHST3
67  ENSG00000123358          3164       NR4A1
68  ENSG00000123908         27161        AGO2
69  ENSG00000124145          6385        SDC4
70  ENSG00000124762          1026      CDKN1A
71  ENSG00000125089         54436      SH3TC1
72  ENSG00000125148          4502        MT2A
73  ENSG00000125398          6662        SOX9
74  ENSG00000128342          3976         LIF
75  ENSG00000128512          9732       DOCK4
76  ENSG00000128805         58504    ARHGAP22
77  ENSG00000130294           547       KIF1A
78  ENSG00000130766         83667       SESN2
79  ENSG00000131016          9590      AKAP12
80  ENSG00000131370          9467      SH3BP5
81  ENSG00000131378         23180       RFTN1
82  ENSG00000131435          8572      PDLIM4
83  ENSG00000131459          9945       GFPT2
84  ENSG00000131650         79412     KREMEN2
85  ENSG00000132334          5791       PTPRE
86  ENSG00000133083          9201       DCLK1
87  ENSG00000133639           694        BTG1
88  ENSG00000134531          2012        EMP1
89  ENSG00000134853          5156      PDGFRA
90  ENSG00000134954          2113        ETS1
91  ENSG00000135048         23670      CEMIP2
92  ENSG00000135111          6926        TBX3
93  ENSG00000135185         79161     TMEM243
94  ENSG00000135679          4193        MDM2
95  ENSG00000136842          7111       TMOD1
96  ENSG00000137801          7057       THBS1
97  ENSG00000137819         54852       PAQR5
98  ENSG00000138166          1847       DUSP5
99  ENSG00000138271         53836       GPR87
100 ENSG00000138623          8482      SEMA7A
101 ENSG00000138675          2250        FGF5
102 ENSG00000138771         57619     SHROOM3
103 ENSG00000138795         51176        LEF1
104 ENSG00000139289         22822      PHLDA1
105 ENSG00000139624         91012       CERS5
106 ENSG00000140379           597      BCL2A1
107 ENSG00000141524         11322        TMC6
108 ENSG00000142627          1969       EPHA2
109 ENSG00000143067         90874      ZNF697
110 ENSG00000144136          6574     SLC20A1
111 ENSG00000146232          4794      NFKBIE
112 ENSG00000146376         93663    ARHGAP18
113 ENSG00000146648          1956        EGFR
114 ENSG00000148516          6935        ZEB1
115 ENSG00000148841         85450      ITPRIP
116 ENSG00000149256         26011       TENM4
117 ENSG00000149948          8091       HMGA2
118 ENSG00000150961          9871      SEC24D
119 ENSG00000150967         23457       ABCB9
120 ENSG00000151062         93589    CACNA2D4
121 ENSG00000151067           775     CACNA1C
122 ENSG00000151067     100874369     CACNA1C
123 ENSG00000151474         55691      FRMD4A
124 ENSG00000153162           654        BMP6
125 ENSG00000153317         50807       ASAP1
126 ENSG00000153814        221895       JAZF1
127 ENSG00000153815         80790        CMIP
128 ENSG00000154783        152273        FGD5
129 ENSG00000156804        114907      FBXO32
130 ENSG00000157933          6497         SKI
131 ENSG00000159128          3460      IFNGR2
132 ENSG00000159388          7832        BTG2
133 ENSG00000160223         23308      ICOSLG
134 ENSG00000160223     102723996      ICOSLG
135 ENSG00000161509          2905      GRIN2C
136 ENSG00000162545         55450     CAMK2N1
137 ENSG00000162711        114548       NLRP3
138 ENSG00000162892         11009        IL24
139 ENSG00000163046        554226   ANKRD30BL
140 ENSG00000163235          7039        TGFA
141 ENSG00000163293        152519      NIPAL1
142 ENSG00000163297        118429      ANTXR2
143 ENSG00000163347          9076       CLDN1
144 ENSG00000163359          1293      COL6A3
145 ENSG00000163513          7048      TGFBR2
146 ENSG00000163661          5806        PTX3
147 ENSG00000164086          1849       DUSP7
148 ENSG00000164171          3673       ITGA2
149 ENSG00000164283         11082        ESM1
150 ENSG00000164484        114801    TMEM200A
151 ENSG00000166025        154810      AMOTL1
152 ENSG00000166401          5271    SERPINB8
153 ENSG00000166922          6447        SCG5
154 ENSG00000167977         54442       KCTD5
155 ENSG00000169252           154       ADRB2
156 ENSG00000169429          3576       CXCL8
157 ENSG00000169436        169044     COL22A1
158 ENSG00000169851          5099       PCDH7
159 ENSG00000169991        126917       IFFO2
160 ENSG00000170006        201799     TMEM154
161 ENSG00000170421          3856        KRT8
162 ENSG00000170439        196410     METTL7B
163 ENSG00000171310         50515      CHST11
164 ENSG00000171617          8507        ENC1
165 ENSG00000171903         57834     CYP4F11
166 ENSG00000173166         65059       RAPH1
167 ENSG00000173221          2745        GLRX
168 ENSG00000173227         91683       SYT12
169 ENSG00000173846          1263        PLK3
170 ENSG00000174343         55584      CHRNA9
171 ENSG00000174791          9610        RIN1
172 ENSG00000175040          9435       CHST2
173 ENSG00000175294        117144    CATSPER1
174 ENSG00000175592          8061       FOSL1
175 ENSG00000175832          2118        ETV4
176 ENSG00000175984        163259     DENND2C
177 ENSG00000176170          8877       SPHK1
178 ENSG00000177076        340485       ACER2
179 ENSG00000177311        253461      ZBTB38
180 ENSG00000177614         79605       PGBD5
181 ENSG00000178104          9659     PDE4DIP
182 ENSG00000178104     124904395     PDE4DIP
183 ENSG00000179542        139065     SLITRK4
184 ENSG00000180921        286077      FAM83H
185 ENSG00000181218         92815      H2AC25
186 ENSG00000181790           575      ADGRB1
187 ENSG00000182261        338322      NLRP10
188 ENSG00000184156          3786       KCNQ3
189 ENSG00000184384         84441       MAML2
190 ENSG00000184702          5413     SEPTIN5
191 ENSG00000184792         23762       OSBP2
192 ENSG00000185022         23764        MAFF
193 ENSG00000187678         81848       SPRY4
194 ENSG00000187801         65243      ZFP69B
195 ENSG00000188825            NA   LINC00910
196 ENSG00000188910          2707        GJB3
197 ENSG00000189362     100131211       NEMP2
198 ENSG00000196083          3556      IL1RAP
199 ENSG00000196526         60312       AFAP1
200 ENSG00000196576         23654      PLXNB2
201 ENSG00000196878          3914       LAMB3
202 ENSG00000196935         57522      SRGAP1
203 ENSG00000197019         29950     SERTAD1
204 ENSG00000197467          1305     COL13A1
205 ENSG00000197852         55924       INKA2
206 ENSG00000198142         65124      SOWAHC
207 ENSG00000198576         23237         ARC
208 ENSG00000198959          7052        TGM2
209 ENSG00000200312            NA   RN7SKP255
210 ENSG00000200434     124907486   RNA5-8SP2
211 ENSG00000200434     124908528   RNA5-8SP2
212 ENSG00000201098          6084        RNY1
213 ENSG00000203485         64423        INF2
214 ENSG00000204131        340527       NHSL2
215 ENSG00000206538        389136       VGLL3
216 ENSG00000212724        730755    KRTAP2-3
217 ENSG00000213626         81606         LBH
218 ENSG00000213694          1903       S1PR3
219 ENSG00000219438         25817       TAFA5
220 ENSG00000222032            NA            
221 ENSG00000222626            NA    RNU2-48P
222 ENSG00000225721            NA            
223 ENSG00000225968        392617       ELFN1
224 ENSG00000227195        284801   MIR663AHG
225 ENSG00000229807          7503        XIST
226 ENSG00000234814            NA      SVIL2P
227 ENSG00000234961            NA            
228 ENSG00000235884            NA   LINC00941
229 ENSG00000236609         54753      ZNF853
230 ENSG00000238266     100507127   LINC00707
231 ENSG00000240602            NA     AADACP1
232 ENSG00000240891        257068      PLCXD2
233 ENSG00000248323            NA      LUCAT1
234 ENSG00000250337        643401       PURPL
235 ENSG00000251011            NA TMEM108-AS1
236 ENSG00000254741            NA            
237 ENSG00000254838        387751      GVINP1
238 ENSG00000255248        399959    MIR100HG
239 ENSG00000258590            NA      NBEAP1
240 ENSG00000260912            NA            
241 ENSG00000260920            NA            
242 ENSG00000264462     103504731   MIR3648-2
243 ENSG00000266088            NA            
244 ENSG00000269968            NA            
245 ENSG00000271225            NA     BNIP3P4</code></pre>
</div>
</div>
</section>
<section id="using-archived-versions-of-ensembl" class="slide level2">
<h2>Using archived versions of Ensembl</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a><span class="fu">listEnsemblArchives</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>             name     date                                 url version
1  Ensembl GRCh37 Feb 2014          https://grch37.ensembl.org  GRCh37
2     Ensembl 109 Feb 2023 https://feb2023.archive.ensembl.org     109
3     Ensembl 108 Oct 2022 https://oct2022.archive.ensembl.org     108
4     Ensembl 107 Jul 2022 https://jul2022.archive.ensembl.org     107
5     Ensembl 106 Apr 2022 https://apr2022.archive.ensembl.org     106
6     Ensembl 105 Dec 2021 https://dec2021.archive.ensembl.org     105
7     Ensembl 104 May 2021 https://may2021.archive.ensembl.org     104
8     Ensembl 103 Feb 2021 https://feb2021.archive.ensembl.org     103
9     Ensembl 102 Nov 2020 https://nov2020.archive.ensembl.org     102
10    Ensembl 101 Aug 2020 https://aug2020.archive.ensembl.org     101
11    Ensembl 100 Apr 2020 https://apr2020.archive.ensembl.org     100
12     Ensembl 99 Jan 2020 https://jan2020.archive.ensembl.org      99
13     Ensembl 98 Sep 2019 https://sep2019.archive.ensembl.org      98
14     Ensembl 97 Jul 2019 https://jul2019.archive.ensembl.org      97
15     Ensembl 96 Apr 2019 https://apr2019.archive.ensembl.org      96
16     Ensembl 95 Jan 2019 https://jan2019.archive.ensembl.org      95
17     Ensembl 94 Oct 2018 https://oct2018.archive.ensembl.org      94
18     Ensembl 93 Jul 2018 https://jul2018.archive.ensembl.org      93
19     Ensembl 92 Apr 2018 https://apr2018.archive.ensembl.org      92
20     Ensembl 80 May 2015 https://may2015.archive.ensembl.org      80
21     Ensembl 77 Oct 2014 https://oct2014.archive.ensembl.org      77
22     Ensembl 75 Feb 2014 https://feb2014.archive.ensembl.org      75
23     Ensembl 54 May 2009 https://may2009.archive.ensembl.org      54
   current_release
1                 
2                *
3                 
4                 
5                 
6                 
7                 
8                 
9                 
10                
11                
12                
13                
14                
15                
16                
17                
18                
19                
20                
21                
22                
23                </code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a>ensembl.grch37 <span class="ot">&lt;-</span> <span class="fu">useEnsembl</span>(<span class="at">biomart =</span> <span class="st">'genes'</span>, </span>
<span id="cb14-2"><a href="#cb14-2"></a>                       <span class="at">dataset =</span> <span class="st">'hsapiens_gene_ensembl'</span>,</span>
<span id="cb14-3"><a href="#cb14-3"></a>                       <span class="at">host =</span> <span class="fu">listEnsemblArchives</span>()[<span class="dv">1</span>,]<span class="sc">$</span>url)</span>
<span id="cb14-4"><a href="#cb14-4"></a>ensembl.grch37</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Object of class 'Mart':
  Using the ENSEMBL_MART_ENSEMBL BioMart database
  Using the hsapiens_gene_ensembl dataset</code></pre>
</div>
</div>
</section>
<section id="annotationhub" class="slide level2">
<h2>AnnotationHub</h2>
<h3 id="get-proper-annotation-for-annotationhub-using-biochubsshiny">Get proper annotation for <code>AnnotationHub</code> using <code>BiocHubsShiny</code>:</h3>
<p>The <code>BiocHubsShiny</code> package allows users to visually explore the <code>AnnotationHub</code> and <code>ExperimentHub</code> resources via shiny. It provides a tabular display of the available resources with the ability to filter and search through the column fields.<sup>1</sup></p>
<div class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>BiocHubsShiny<span class="sc">::</span><span class="fu">BiocHubsShiny</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<aside><ol class="aside-footnotes"><li id="fn1"><p>In case of alternative hypothesis as <code>greater</code>.</p></li></ol></aside></section>
<section id="annotationhub-1" class="slide level2">
<h2>AnnotationHub</h2>
<p>Paste the code from <code>BiocHubsShiny</code> and get the <code>EnsemblDB</code> object</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1"></a><span class="do">## Make sure BiocManager is installed</span></span>
<span id="cb17-2"><a href="#cb17-2"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">'BiocManager'</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb17-3"><a href="#cb17-3"></a>    <span class="fu">install.packages</span>(<span class="st">'BiocManager'</span>)</span>
<span id="cb17-4"><a href="#cb17-4"></a></span>
<span id="cb17-5"><a href="#cb17-5"></a><span class="do">## Make sure AnnotationHub is installed</span></span>
<span id="cb17-6"><a href="#cb17-6"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(<span class="st">"AnnotationHub"</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb17-7"><a href="#cb17-7"></a>    BiocManager<span class="sc">::</span><span class="fu">install</span>(<span class="st">"AnnotationHub"</span>)</span>
<span id="cb17-8"><a href="#cb17-8"></a></span>
<span id="cb17-9"><a href="#cb17-9"></a><span class="do">## Use this code to download the resource</span></span>
<span id="cb17-10"><a href="#cb17-10"></a><span class="fu">library</span>(<span class="st">"AnnotationHub"</span>)</span>
<span id="cb17-11"><a href="#cb17-11"></a>hub <span class="ot">&lt;-</span> <span class="fu">AnnotationHub</span>()</span>
<span id="cb17-12"><a href="#cb17-12"></a></span>
<span id="cb17-13"><a href="#cb17-13"></a><span class="do">## Select rows in the table</span></span>
<span id="cb17-14"><a href="#cb17-14"></a>ens.db <span class="ot">&lt;-</span> hub[[<span class="st">'AH109336'</span>]]</span>
<span id="cb17-15"><a href="#cb17-15"></a>ens.db</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>EnsDb for Ensembl:
|Backend: SQLite
|Db type: EnsDb
|Type of Gene ID: Ensembl Gene ID
|Supporting package: ensembldb
|Db created by: ensembldb package from Bioconductor
|script_version: 0.3.7
|Creation time: Fri Oct 28 05:24:43 2022
|ensembl_version: 108
|ensembl_host: localhost
|Organism: Homo sapiens
|taxonomy_id: 9606
|genome_build: GRCh38
|DBSCHEMAVERSION: 2.2
| No. of genes: 70616.
| No. of transcripts: 275721.
|Protein data available.</code></pre>
</div>
</div>
</section>
<section id="annotationhub-2" class="slide level2">
<h2>AnnotationHub</h2>
<h3 id="object-description">Object description</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1"></a><span class="fu">columns</span>(ens.db)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] "CANONICALTRANSCRIPT" "DESCRIPTION"         "ENTREZID"           
 [4] "EXONID"              "EXONIDX"             "EXONSEQEND"         
 [7] "EXONSEQSTART"        "GCCONTENT"           "GENEBIOTYPE"        
[10] "GENEID"              "GENEIDVERSION"       "GENENAME"           
[13] "GENESEQEND"          "GENESEQSTART"        "INTERPROACCESSION"  
[16] "ISCIRCULAR"          "PROTDOMEND"          "PROTDOMSTART"       
[19] "PROTEINDOMAINID"     "PROTEINDOMAINSOURCE" "PROTEINID"          
[22] "PROTEINSEQUENCE"     "SEQCOORDSYSTEM"      "SEQLENGTH"          
[25] "SEQNAME"             "SEQSTRAND"           "SYMBOL"             
[28] "TXBIOTYPE"           "TXCDSSEQEND"         "TXCDSSEQSTART"      
[31] "TXEXTERNALNAME"      "TXID"                "TXIDVERSION"        
[34] "TXISCANONICAL"       "TXNAME"              "TXSEQEND"           
[37] "TXSEQSTART"          "TXSUPPORTLEVEL"      "UNIPROTDB"          
[40] "UNIPROTID"           "UNIPROTMAPPINGTYPE" </code></pre>
</div>
</div>
</section>
<section id="annotationhub-3" class="slide level2">
<h2>AnnotationHub</h2>
<h3 id="get-full-gene-annotation-gtf-like">Get full gene annotation (gtf like)</h3>
<div class="columns">
<div class="column" style="width:40%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1"></a><span class="fu">genes</span>(ens.db)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>GRanges object with 70616 ranges and 9 metadata columns:
                  seqnames            ranges strand |         gene_id
                     &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt;
  ENSG00000290825        1       11869-14409      + | ENSG00000290825
  ENSG00000223972        1       12010-13670      + | ENSG00000223972
  ENSG00000227232        1       14404-29570      - | ENSG00000227232
  ENSG00000278267        1       17369-17436      - | ENSG00000278267
  ENSG00000243485        1       29554-31109      + | ENSG00000243485
              ...      ...               ...    ... .             ...
  ENSG00000224240        Y 26549425-26549743      + | ENSG00000224240
  ENSG00000227629        Y 26586642-26591601      - | ENSG00000227629
  ENSG00000237917        Y 26594851-26634652      - | ENSG00000237917
  ENSG00000231514        Y 26626520-26627159      - | ENSG00000231514
  ENSG00000235857        Y 56855244-56855488      + | ENSG00000235857
                    gene_name           gene_biotype seq_coord_system
                  &lt;character&gt;            &lt;character&gt;      &lt;character&gt;
  ENSG00000290825     DDX11L2                 lncRNA       chromosome
  ENSG00000223972     DDX11L1 transcribed_unproces..       chromosome
  ENSG00000227232      WASH7P unprocessed_pseudogene       chromosome
  ENSG00000278267   MIR6859-1                  miRNA       chromosome
  ENSG00000243485 MIR1302-2HG                 lncRNA       chromosome
              ...         ...                    ...              ...
  ENSG00000224240     CYCSP49   processed_pseudogene       chromosome
  ENSG00000227629  SLC25A15P1 unprocessed_pseudogene       chromosome
  ENSG00000237917     PARP4P1 unprocessed_pseudogene       chromosome
  ENSG00000231514      CCNQP2   processed_pseudogene       chromosome
  ENSG00000235857     CTBP2P1   processed_pseudogene       chromosome
                             description   gene_id_version canonical_transcript
                             &lt;character&gt;       &lt;character&gt;          &lt;character&gt;
  ENSG00000290825 DEAD/H-box helicase .. ENSG00000290825.1      ENST00000456328
  ENSG00000223972 DEAD/H-box helicase .. ENSG00000223972.6      ENST00000450305
  ENSG00000227232 WASP family homolog .. ENSG00000227232.5      ENST00000488147
  ENSG00000278267 microRNA 6859-1 [Sou.. ENSG00000278267.1      ENST00000619216
  ENSG00000243485 MIR1302-2 host gene .. ENSG00000243485.5      ENST00000473358
              ...                    ...               ...                  ...
  ENSG00000224240 CYCS pseudogene 49 [.. ENSG00000224240.1      ENST00000420810
  ENSG00000227629 solute carrier famil.. ENSG00000227629.1      ENST00000456738
  ENSG00000237917 poly(ADP-ribose) pol.. ENSG00000237917.1      ENST00000435945
  ENSG00000231514 CCNQ pseudogene 2 [S.. ENSG00000231514.1      ENST00000435741
  ENSG00000235857 CTBP2 pseudogene 1 [.. ENSG00000235857.1      ENST00000431853
                       symbol                   entrezid
                  &lt;character&gt;                     &lt;list&gt;
  ENSG00000290825     DDX11L2 84771,727856,100287102,...
  ENSG00000223972     DDX11L1                       &lt;NA&gt;
  ENSG00000227232      WASH7P                       &lt;NA&gt;
  ENSG00000278267   MIR6859-1                  102466751
  ENSG00000243485 MIR1302-2HG                       &lt;NA&gt;
              ...         ...                        ...
  ENSG00000224240     CYCSP49                       &lt;NA&gt;
  ENSG00000227629  SLC25A15P1                       &lt;NA&gt;
  ENSG00000237917     PARP4P1                       &lt;NA&gt;
  ENSG00000231514      CCNQP2                       &lt;NA&gt;
  ENSG00000235857     CTBP2P1                       &lt;NA&gt;
  -------
  seqinfo: 457 sequences (1 circular) from GRCh38 genome</code></pre>
</div>
</div>
</div><div class="column" style="width:60%;">
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a><span class="fu">transcripts</span>(ens.db)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>GRanges object with 275721 ranges and 11 metadata columns:
                  seqnames            ranges strand |           tx_id
                     &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; |     &lt;character&gt;
  ENST00000456328        1       11869-14409      + | ENST00000456328
  ENST00000450305        1       12010-13670      + | ENST00000450305
  ENST00000488147        1       14404-29570      - | ENST00000488147
  ENST00000619216        1       17369-17436      - | ENST00000619216
  ENST00000473358        1       29554-31097      + | ENST00000473358
              ...      ...               ...    ... .             ...
  ENST00000420810        Y 26549425-26549743      + | ENST00000420810
  ENST00000456738        Y 26586642-26591601      - | ENST00000456738
  ENST00000435945        Y 26594851-26634652      - | ENST00000435945
  ENST00000435741        Y 26626520-26627159      - | ENST00000435741
  ENST00000431853        Y 56855244-56855488      + | ENST00000431853
                              tx_biotype tx_cds_seq_start tx_cds_seq_end
                             &lt;character&gt;        &lt;integer&gt;      &lt;integer&gt;
  ENST00000456328                 lncRNA             &lt;NA&gt;           &lt;NA&gt;
  ENST00000450305 transcribed_unproces..             &lt;NA&gt;           &lt;NA&gt;
  ENST00000488147 unprocessed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
  ENST00000619216                  miRNA             &lt;NA&gt;           &lt;NA&gt;
  ENST00000473358                 lncRNA             &lt;NA&gt;           &lt;NA&gt;
              ...                    ...              ...            ...
  ENST00000420810   processed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
  ENST00000456738 unprocessed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
  ENST00000435945 unprocessed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
  ENST00000435741   processed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
  ENST00000431853   processed_pseudogene             &lt;NA&gt;           &lt;NA&gt;
                          gene_id tx_support_level     tx_id_version gc_content
                      &lt;character&gt;        &lt;integer&gt;       &lt;character&gt;  &lt;numeric&gt;
  ENST00000456328 ENSG00000290825                1 ENST00000456328.2    55.3410
  ENST00000450305 ENSG00000223972             &lt;NA&gt; ENST00000450305.2    58.0696
  ENST00000488147 ENSG00000227232             &lt;NA&gt; ENST00000488147.1    61.1399
  ENST00000619216 ENSG00000278267             &lt;NA&gt; ENST00000619216.1    61.7647
  ENST00000473358 ENSG00000243485                5 ENST00000473358.1    58.5674
              ...             ...              ...               ...        ...
  ENST00000420810 ENSG00000224240             &lt;NA&gt; ENST00000420810.1    41.6928
  ENST00000456738 ENSG00000227629             &lt;NA&gt; ENST00000456738.1    44.7368
  ENST00000435945 ENSG00000237917             &lt;NA&gt; ENST00000435945.1    44.2875
  ENST00000435741 ENSG00000231514             &lt;NA&gt; ENST00000435741.1    55.1562
  ENST00000431853 ENSG00000235857             &lt;NA&gt; ENST00000431853.1    53.0612
                  tx_external_name tx_is_canonical         tx_name
                       &lt;character&gt;       &lt;integer&gt;     &lt;character&gt;
  ENST00000456328      DDX11L2-202               1 ENST00000456328
  ENST00000450305      DDX11L1-201               1 ENST00000450305
  ENST00000488147       WASH7P-201               1 ENST00000488147
  ENST00000619216    MIR6859-1-201               1 ENST00000619216
  ENST00000473358  MIR1302-2HG-202               1 ENST00000473358
              ...              ...             ...             ...
  ENST00000420810      CYCSP49-201               1 ENST00000420810
  ENST00000456738   SLC25A15P1-201               1 ENST00000456738
  ENST00000435945      PARP4P1-201               1 ENST00000435945
  ENST00000435741       CCNQP2-201               1 ENST00000435741
  ENST00000431853      CTBP2P1-201               1 ENST00000431853
  -------
  seqinfo: 457 sequences (1 circular) from GRCh38 genome</code></pre>
</div>
</div>
</div>
</div>
</section>
<section id="annotationhub-4" class="slide level2">
<h2>AnnotationHub</h2>
<h3 id="gene-id-conversion-from-ensembl-to-symbol-and-entrez">Gene ID conversion from <code>Ensembl</code> To <code>Symbol</code> and <code>ENTREZ</code></h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a><span class="fu">select</span>(ens.db, <span class="at">keys=</span>de_genes<span class="sc">$</span>rowname,<span class="at">columns=</span><span class="fu">c</span>(<span class="st">"SYMBOL"</span>, <span class="st">"GENEID"</span>,<span class="st">"ENTREZID"</span>,<span class="st">"GENENAME"</span>,<span class="st">"GENEBIOTYPE"</span>),<span class="at">keytype =</span> <span class="st">"GENEID"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>         SYMBOL          GENEID  ENTREZID    GENENAME
1         KDM7A ENSG00000006459     80853       KDM7A
2       ST3GAL1 ENSG00000008513      6482     ST3GAL1
3         IFFO1 ENSG00000010295     25900       IFFO1
4         WWTR1 ENSG00000018408     25937       WWTR1
5         BIRC3 ENSG00000023445       330       BIRC3
6           FAS ENSG00000026103       355         FAS
7      ARHGAP31 ENSG00000031081     57514    ARHGAP31
8         ARAP2 ENSG00000047365    116984       ARAP2
9         KITLG ENSG00000049130      4254       KITLG
10       NEDD4L ENSG00000049759     23327      NEDD4L
11        TRAF1 ENSG00000056558      7185       TRAF1
12      RASGRF1 ENSG00000058335      5923     RASGRF1
13         NAV3 ENSG00000067798     89795        NAV3
14         CBFB ENSG00000067955       865        CBFB
15        MAST4 ENSG00000069020    375449       MAST4
16        TRIB2 ENSG00000071575     28951       TRIB2
17        FSCN1 ENSG00000075618      6624       FSCN1
18       PLXNA2 ENSG00000076356      5362      PLXNA2
19        SYNJ2 ENSG00000078269      8871       SYNJ2
20        SESN1 ENSG00000080546     27244       SESN1
21       SLC9A1 ENSG00000090020      6548      SLC9A1
22        EFNB1 ENSG00000090776      1947       EFNB1
23         IL11 ENSG00000095752      3589        IL11
24         NRP1 ENSG00000099250      8829        NRP1
25      SLC35E4 ENSG00000100036    339665     SLC35E4
26        TRIM9 ENSG00000100505    114088       TRIM9
27         MMP9 ENSG00000100985      4318        MMP9
28         DOK5 ENSG00000101134     55816        DOK5
29      SLCO4A1 ENSG00000101187     28231     SLCO4A1
30        RNF24 ENSG00000101236     11237       RNF24
31          SYP ENSG00000102003      6855         SYP
32        TIMP1 ENSG00000102265      7076       TIMP1
33       SLC7A5 ENSG00000103257      8140      SLC7A5
34       CORO2B ENSG00000103647     10391      CORO2B
35         RELB ENSG00000104856      5971        RELB
36       IL27RA ENSG00000104998      9466      IL27RA
37         BBC3 ENSG00000105327     27113        BBC3
38        ITGB8 ENSG00000105855      3696       ITGB8
39     SERPINE1 ENSG00000106366      5054    SERPINE1
40        DNMBP ENSG00000107554     23268       DNMBP
41        PALD1 ENSG00000107719     27143       PALD1
42       SORCS1 ENSG00000108018    114815      SORCS1
43         CCL2 ENSG00000108691      6347        CCL2
44        CHIC2 ENSG00000109220     26511       CHIC2
45         EHD1 ENSG00000110047     10938        EHD1
46        SH2B3 ENSG00000111252     10019       SH2B3
47        KCNA1 ENSG00000111262      3736       KCNA1
48       CEP85L ENSG00000111860    387119      CEP85L
49         CD83 ENSG00000112149      9308        CD83
50         MDFI ENSG00000112559      4188        MDFI
51        HBEGF ENSG00000113070      1839       HBEGF
52       PDGFRB ENSG00000113721      5159      PDGFRB
53        CCL20 ENSG00000115009      6364       CCL20
54         FHL2 ENSG00000115641      2274        FHL2
55        EPAS1 ENSG00000116016      2034       EPAS1
56        EPHA4 ENSG00000116106      2043       EPHA4
57       ERRFI1 ENSG00000116285     54206      ERRFI1
58         RGS4 ENSG00000117152      5999        RGS4
59      TNFAIP3 ENSG00000118503      7128     TNFAIP3
60        KLF12 ENSG00000118922     11278       KLF12
61        LTBP2 ENSG00000119681      4053       LTBP2
62       DNMT3A ENSG00000119772      1788      DNMT3A
63        HOXB3 ENSG00000120093      3213       HOXB3
64        CD274 ENSG00000120217     29126       CD274
65         PLAU ENSG00000122861      5328        PLAU
66        CHST3 ENSG00000122863      9469       CHST3
67        NR4A1 ENSG00000123358      3164       NR4A1
68         AGO2 ENSG00000123908     27161        AGO2
69         SDC4 ENSG00000124145      6385        SDC4
70       CDKN1A ENSG00000124762      1026      CDKN1A
71       SH3TC1 ENSG00000125089     54436      SH3TC1
72         MT2A ENSG00000125148      4502        MT2A
73         SOX9 ENSG00000125398      6662        SOX9
74          LIF ENSG00000128342      3976         LIF
75        DOCK4 ENSG00000128512      9732       DOCK4
76     ARHGAP22 ENSG00000128805     58504    ARHGAP22
77        KIF1A ENSG00000130294       547       KIF1A
78        SESN2 ENSG00000130766     83667       SESN2
79       AKAP12 ENSG00000131016      9590      AKAP12
80       SH3BP5 ENSG00000131370      9467      SH3BP5
81        RFTN1 ENSG00000131378     23180       RFTN1
82       PDLIM4 ENSG00000131435      8572      PDLIM4
83        GFPT2 ENSG00000131459      9945       GFPT2
84      KREMEN2 ENSG00000131650     79412     KREMEN2
85        PTPRE ENSG00000132334      5791       PTPRE
86        DCLK1 ENSG00000133083      9201       DCLK1
87         BTG1 ENSG00000133639       694        BTG1
88         EMP1 ENSG00000134531      2012        EMP1
89       PDGFRA ENSG00000134853      5156      PDGFRA
90         ETS1 ENSG00000134954      2113        ETS1
91       CEMIP2 ENSG00000135048     23670      CEMIP2
92         TBX3 ENSG00000135111      6926        TBX3
93      TMEM243 ENSG00000135185     79161     TMEM243
94         MDM2 ENSG00000135679      4193        MDM2
95        TMOD1 ENSG00000136842      7111       TMOD1
96        THBS1 ENSG00000137801      7057       THBS1
97        PAQR5 ENSG00000137819     54852       PAQR5
98        DUSP5 ENSG00000138166      1847       DUSP5
99        GPR87 ENSG00000138271     53836       GPR87
100      SEMA7A ENSG00000138623      8482      SEMA7A
101        FGF5 ENSG00000138675      2250        FGF5
102     SHROOM3 ENSG00000138771     57619     SHROOM3
103        LEF1 ENSG00000138795     51176        LEF1
104      PHLDA1 ENSG00000139289     22822      PHLDA1
105       CERS5 ENSG00000139624     91012       CERS5
106      BCL2A1 ENSG00000140379       597      BCL2A1
107        TMC6 ENSG00000141524     11322        TMC6
108       EPHA2 ENSG00000142627      1969       EPHA2
109      ZNF697 ENSG00000143067     90874      ZNF697
110     SLC20A1 ENSG00000144136      6574     SLC20A1
111      NFKBIE ENSG00000146232      4794      NFKBIE
112    ARHGAP18 ENSG00000146376     93663    ARHGAP18
113        EGFR ENSG00000146648      1956        EGFR
114        ZEB1 ENSG00000148516      6935        ZEB1
115      ITPRIP ENSG00000148841     85450      ITPRIP
116       TENM4 ENSG00000149256     26011       TENM4
117       HMGA2 ENSG00000149948      8091       HMGA2
118      SEC24D ENSG00000150961      9871      SEC24D
119       ABCB9 ENSG00000150967     23457       ABCB9
120    CACNA2D4 ENSG00000151062     93589    CACNA2D4
121     CACNA1C ENSG00000151067       775     CACNA1C
122     CACNA1C ENSG00000151067 100874369     CACNA1C
123      FRMD4A ENSG00000151474     55691      FRMD4A
124        BMP6 ENSG00000153162       654        BMP6
125       ASAP1 ENSG00000153317     50807       ASAP1
126       JAZF1 ENSG00000153814    221895       JAZF1
127        CMIP ENSG00000153815     80790        CMIP
128        FGD5 ENSG00000154783    152273        FGD5
129      FBXO32 ENSG00000156804    114907      FBXO32
130         SKI ENSG00000157933      6497         SKI
131      IFNGR2 ENSG00000159128      3460      IFNGR2
132        BTG2 ENSG00000159388      7832        BTG2
133      ICOSLG ENSG00000160223     23308      ICOSLG
134      ICOSLG ENSG00000160223 102723996      ICOSLG
135      GRIN2C ENSG00000161509      2905      GRIN2C
136     CAMK2N1 ENSG00000162545     55450     CAMK2N1
137       NLRP3 ENSG00000162711    114548       NLRP3
138        IL24 ENSG00000162892     11009        IL24
139   ANKRD30BL ENSG00000163046    554226   ANKRD30BL
140        TGFA ENSG00000163235      7039        TGFA
141      NIPAL1 ENSG00000163293    152519      NIPAL1
142      ANTXR2 ENSG00000163297    118429      ANTXR2
143       CLDN1 ENSG00000163347      9076       CLDN1
144      COL6A3 ENSG00000163359      1293      COL6A3
145      TGFBR2 ENSG00000163513      7048      TGFBR2
146        PTX3 ENSG00000163661      5806        PTX3
147       DUSP7 ENSG00000164086      1849       DUSP7
148       ITGA2 ENSG00000164171      3673       ITGA2
149        ESM1 ENSG00000164283     11082        ESM1
150    TMEM200A ENSG00000164484    114801    TMEM200A
151      AMOTL1 ENSG00000166025    154810      AMOTL1
152    SERPINB8 ENSG00000166401      5271    SERPINB8
153        SCG5 ENSG00000166922      6447        SCG5
154       KCTD5 ENSG00000167977     54442       KCTD5
155       ADRB2 ENSG00000169252       154       ADRB2
156       CXCL8 ENSG00000169429      3576       CXCL8
157     COL22A1 ENSG00000169436    169044     COL22A1
158       PCDH7 ENSG00000169851      5099       PCDH7
159       IFFO2 ENSG00000169991    126917       IFFO2
160     TMEM154 ENSG00000170006    201799     TMEM154
161        KRT8 ENSG00000170421      3856        KRT8
162     METTL7B ENSG00000170439    196410     METTL7B
163      CHST11 ENSG00000171310     50515      CHST11
164        ENC1 ENSG00000171617      8507        ENC1
165     CYP4F11 ENSG00000171903     57834     CYP4F11
166       RAPH1 ENSG00000173166     65059       RAPH1
167        GLRX ENSG00000173221      2745        GLRX
168       SYT12 ENSG00000173227     91683       SYT12
169        PLK3 ENSG00000173846      1263        PLK3
170      CHRNA9 ENSG00000174343     55584      CHRNA9
171        RIN1 ENSG00000174791      9610        RIN1
172       CHST2 ENSG00000175040      9435       CHST2
173    CATSPER1 ENSG00000175294    117144    CATSPER1
174       FOSL1 ENSG00000175592      8061       FOSL1
175        ETV4 ENSG00000175832      2118        ETV4
176     DENND2C ENSG00000175984    163259     DENND2C
177       SPHK1 ENSG00000176170      8877       SPHK1
178       ACER2 ENSG00000177076    340485       ACER2
179      ZBTB38 ENSG00000177311    253461      ZBTB38
180       PGBD5 ENSG00000177614     79605       PGBD5
181     PDE4DIP ENSG00000178104      9659     PDE4DIP
182     PDE4DIP ENSG00000178104 124904395     PDE4DIP
183     SLITRK4 ENSG00000179542    139065     SLITRK4
184      FAM83H ENSG00000180921    286077      FAM83H
185        H2AW ENSG00000181218     92815        H2AW
186      ADGRB1 ENSG00000181790       575      ADGRB1
187      NLRP10 ENSG00000182261    338322      NLRP10
188       KCNQ3 ENSG00000184156      3786       KCNQ3
189       MAML2 ENSG00000184384     84441       MAML2
190     SEPTIN5 ENSG00000184702      5413     SEPTIN5
191       OSBP2 ENSG00000184792     23762       OSBP2
192        MAFF ENSG00000185022     23764        MAFF
193       SPRY4 ENSG00000187678     81848       SPRY4
194      ZFP69B ENSG00000187801     65243      ZFP69B
195   LINC00910 ENSG00000188825        NA   LINC00910
196        GJB3 ENSG00000188910      2707        GJB3
197       NEMP2 ENSG00000189362 100131211       NEMP2
198      IL1RAP ENSG00000196083      3556      IL1RAP
199       AFAP1 ENSG00000196526     60312       AFAP1
200      PLXNB2 ENSG00000196576     23654      PLXNB2
201       LAMB3 ENSG00000196878      3914       LAMB3
202      SRGAP1 ENSG00000196935     57522      SRGAP1
203     SERTAD1 ENSG00000197019     29950     SERTAD1
204     COL13A1 ENSG00000197467      1305     COL13A1
205       INKA2 ENSG00000197852     55924       INKA2
206      SOWAHC ENSG00000198142     65124      SOWAHC
207         ARC ENSG00000198576     23237         ARC
208        TGM2 ENSG00000198959      7052        TGM2
209   RN7SKP255 ENSG00000200312        NA   RN7SKP255
210   RNA5-8SP2 ENSG00000200434 124907486   RNA5-8SP2
211   RNA5-8SP2 ENSG00000200434 124908528   RNA5-8SP2
212        RNY1 ENSG00000201098      6084        RNY1
213        INF2 ENSG00000203485     64423        INF2
214       NHSL2 ENSG00000204131    340527       NHSL2
215       VGLL3 ENSG00000206538    389136       VGLL3
216    KRTAP2-3 ENSG00000212724    730755    KRTAP2-3
217         LBH ENSG00000213626     81606         LBH
218       S1PR3 ENSG00000213694      1903       S1PR3
219       TAFA5 ENSG00000219438     25817       TAFA5
220             ENSG00000222032        NA            
221    RNU2-48P ENSG00000222626        NA    RNU2-48P
222             ENSG00000225721        NA            
223       ELFN1 ENSG00000225968    392617       ELFN1
224   MIR663AHG ENSG00000227195    284801   MIR663AHG
225        XIST ENSG00000229807      7503        XIST
226      SVIL2P ENSG00000234814        NA      SVIL2P
227             ENSG00000234961        NA            
228   LINC00941 ENSG00000235884        NA   LINC00941
229      ZNF853 ENSG00000236609     54753      ZNF853
230   LINC00707 ENSG00000238266 100507127   LINC00707
231     AADACP1 ENSG00000240602        NA     AADACP1
232      PLCXD2 ENSG00000240891    257068      PLCXD2
233      LUCAT1 ENSG00000248323        NA      LUCAT1
234       PURPL ENSG00000250337    643401       PURPL
235 TMEM108-AS1 ENSG00000251011        NA TMEM108-AS1
236             ENSG00000254741        NA            
237      GVINP1 ENSG00000254838    387751      GVINP1
238    MIR100HG ENSG00000255248    399959    MIR100HG
239      NBEAP1 ENSG00000258590        NA      NBEAP1
240             ENSG00000260912        NA            
241             ENSG00000260920        NA            
242   MIR3648-2 ENSG00000264462 103504731   MIR3648-2
243             ENSG00000266088        NA            
244             ENSG00000269968        NA            
245     BNIP3P4 ENSG00000271225        NA     BNIP3P4
                           GENEBIOTYPE
1                       protein_coding
2                       protein_coding
3                       protein_coding
4                       protein_coding
5                       protein_coding
6                       protein_coding
7                       protein_coding
8                       protein_coding
9                       protein_coding
10                      protein_coding
11                      protein_coding
12                      protein_coding
13                      protein_coding
14                      protein_coding
15                      protein_coding
16                      protein_coding
17                      protein_coding
18                      protein_coding
19                      protein_coding
20                      protein_coding
21                      protein_coding
22                      protein_coding
23                      protein_coding
24                      protein_coding
25                      protein_coding
26                      protein_coding
27                      protein_coding
28                      protein_coding
29                      protein_coding
30                      protein_coding
31                      protein_coding
32                      protein_coding
33                      protein_coding
34                      protein_coding
35                      protein_coding
36                      protein_coding
37                      protein_coding
38                      protein_coding
39                      protein_coding
40                      protein_coding
41                      protein_coding
42                      protein_coding
43                      protein_coding
44                      protein_coding
45                      protein_coding
46                      protein_coding
47                      protein_coding
48                      protein_coding
49                      protein_coding
50                      protein_coding
51                      protein_coding
52                      protein_coding
53                      protein_coding
54                      protein_coding
55                      protein_coding
56                      protein_coding
57                      protein_coding
58                      protein_coding
59                      protein_coding
60                      protein_coding
61                      protein_coding
62                      protein_coding
63                      protein_coding
64                      protein_coding
65                      protein_coding
66                      protein_coding
67                      protein_coding
68                      protein_coding
69                      protein_coding
70                      protein_coding
71                      protein_coding
72                      protein_coding
73                      protein_coding
74                      protein_coding
75                      protein_coding
76                      protein_coding
77                      protein_coding
78                      protein_coding
79                      protein_coding
80                      protein_coding
81                      protein_coding
82                      protein_coding
83                      protein_coding
84                      protein_coding
85                      protein_coding
86                      protein_coding
87                      protein_coding
88                      protein_coding
89                      protein_coding
90                      protein_coding
91                      protein_coding
92                      protein_coding
93                      protein_coding
94                      protein_coding
95                      protein_coding
96                      protein_coding
97                      protein_coding
98                      protein_coding
99                      protein_coding
100                     protein_coding
101                     protein_coding
102                     protein_coding
103                     protein_coding
104                     protein_coding
105                     protein_coding
106                     protein_coding
107                     protein_coding
108                     protein_coding
109                     protein_coding
110                     protein_coding
111                     protein_coding
112                     protein_coding
113                     protein_coding
114                     protein_coding
115                     protein_coding
116                     protein_coding
117                     protein_coding
118                     protein_coding
119                     protein_coding
120                     protein_coding
121                     protein_coding
122                     protein_coding
123                     protein_coding
124                     protein_coding
125                     protein_coding
126                     protein_coding
127                     protein_coding
128                     protein_coding
129                     protein_coding
130                     protein_coding
131                     protein_coding
132                     protein_coding
133                     protein_coding
134                     protein_coding
135                     protein_coding
136                     protein_coding
137                     protein_coding
138                     protein_coding
139                     protein_coding
140                     protein_coding
141                     protein_coding
142                     protein_coding
143                     protein_coding
144                     protein_coding
145                     protein_coding
146                     protein_coding
147                     protein_coding
148                     protein_coding
149                     protein_coding
150                     protein_coding
151                     protein_coding
152                     protein_coding
153                     protein_coding
154                     protein_coding
155                     protein_coding
156                     protein_coding
157                     protein_coding
158                     protein_coding
159                     protein_coding
160                     protein_coding
161                     protein_coding
162                     protein_coding
163                     protein_coding
164                     protein_coding
165                     protein_coding
166                     protein_coding
167                     protein_coding
168                     protein_coding
169                     protein_coding
170                     protein_coding
171                     protein_coding
172                     protein_coding
173                     protein_coding
174                     protein_coding
175                     protein_coding
176                     protein_coding
177                     protein_coding
178                     protein_coding
179                     protein_coding
180                     protein_coding
181                     protein_coding
182                     protein_coding
183                     protein_coding
184                     protein_coding
185                     protein_coding
186                     protein_coding
187                     protein_coding
188                     protein_coding
189                     protein_coding
190                     protein_coding
191                     protein_coding
192                     protein_coding
193                     protein_coding
194                     protein_coding
195                             lncRNA
196                     protein_coding
197                     protein_coding
198                     protein_coding
199                     protein_coding
200                     protein_coding
201                     protein_coding
202                     protein_coding
203                     protein_coding
204                     protein_coding
205                     protein_coding
206                     protein_coding
207                     protein_coding
208                     protein_coding
209                           misc_RNA
210                    rRNA_pseudogene
211                    rRNA_pseudogene
212                           misc_RNA
213                     protein_coding
214                     protein_coding
215                     protein_coding
216                     protein_coding
217                     protein_coding
218                     protein_coding
219                     protein_coding
220                             lncRNA
221                              snRNA
222                             lncRNA
223                     protein_coding
224                             lncRNA
225                             lncRNA
226 transcribed_unprocessed_pseudogene
227                             lncRNA
228                             lncRNA
229                     protein_coding
230                             lncRNA
231 transcribed_unprocessed_pseudogene
232                     protein_coding
233                             lncRNA
234                             lncRNA
235                             lncRNA
236                             lncRNA
237 transcribed_unprocessed_pseudogene
238                             lncRNA
239 transcribed_unprocessed_pseudogene
240                             lncRNA
241                             lncRNA
242                              miRNA
243                             lncRNA
244                             lncRNA
245               processed_pseudogene</code></pre>
</div>
</div>
<blockquote>
<p>Note: If you don’t have a gene list and want everything, provide <code>keys=keys(ens.db, keytype="GENEID")</code> to the <code>select()</code> function. ## AnnotationHub</p>
</blockquote>
</section>
<section id="annotationhub-5" class="slide level2">
<h2>AnnotationHub</h2>
<h3 id="work-as-well-for-ncbi">Work as well for <code>NCBI</code>:</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1"></a>hub <span class="ot">&lt;-</span> <span class="fu">AnnotationHub</span>()</span>
<span id="cb27-2"><a href="#cb27-2"></a>hub[[<span class="st">'AH111575'</span>]]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>OrgDb object:
| DBSCHEMAVERSION: 2.1
| Db type: OrgDb
| Supporting package: AnnotationDbi
| DBSCHEMA: HUMAN_DB
| ORGANISM: Homo sapiens
| SPECIES: Human
| EGSOURCEDATE: 2023-Mar05
| EGSOURCENAME: Entrez Gene
| EGSOURCEURL: ftp://ftp.ncbi.nlm.nih.gov/gene/DATA
| CENTRALID: EG
| TAXID: 9606
| GOSOURCENAME: Gene Ontology
| GOSOURCEURL: http://current.geneontology.org/ontology/go-basic.obo
| GOSOURCEDATE: 2023-01-01
| GOEGSOURCEDATE: 2023-Mar05
| GOEGSOURCENAME: Entrez Gene
| GOEGSOURCEURL: ftp://ftp.ncbi.nlm.nih.gov/gene/DATA
| KEGGSOURCENAME: KEGG GENOME
| KEGGSOURCEURL: ftp://ftp.genome.jp/pub/kegg/genomes
| KEGGSOURCEDATE: 2011-Mar15
| GPSOURCENAME: UCSC Genome Bioinformatics (Homo sapiens)
| GPSOURCEURL: 
| GPSOURCEDATE: 2023-Feb8
| ENSOURCEDATE: 2023-Feb10
| ENSOURCENAME: Ensembl
| ENSOURCEURL: ftp://ftp.ensembl.org/pub/current_fasta
| UPSOURCENAME: Uniprot
| UPSOURCEURL: http://www.UniProt.org/
| UPSOURCEDATE: Wed Mar 15 15:58:07 2023</code></pre>
</div>
</div>
</section>
<section id="retrieve-other-annotations-using-annotationhub-or-biomart" class="slide level2">
<h2>Retrieve other annotations using <code>AnnotationHub</code> or <code>biomaRt</code></h2>
</section></section>
<section>
<section id="functionnal-analysis" class="title-slide slide level1 center">
<h1>Functionnal analysis</h1>

</section>
<section id="pathways-collections-for-genes" class="slide level2">
<h2>Pathways collections for Genes</h2>
<ol type="1">
<li><strong>Gene Ontology (GO)</strong></li>
<li><strong>KEGG (Kyoto Encyclopedia of Genes and Genomes)</strong>:</li>
<li><strong>WikiPathways</strong></li>
<li><strong>Disease Ontology</strong></li>
<li><strong>Reactome</strong></li>
<li><strong>Molecular Signatures Database (MSigDB)</strong></li>
<li><strong>Medical Subject Headings (MeSH)</strong></li>
</ol>
<p>These pathway collections databases serve as valuable resources for researchers to explore and analyze biological pathways, functional annotations, and disease associations. They facilitate the interpretation of gene expression data, identify key pathways related to specific biological processes or diseases, and contribute to a deeper understanding of biological systems.</p>
</section>
<section id="gene-ontology-go" class="slide level2">
<h2><strong>Gene Ontology (GO)</strong></h2>
<p>GO is a widely used database that categorizes genes into three main ontologies: molecular function, biological process, and cellular component. It provides a standardized vocabulary to describe gene functions and interactions, allowing researchers to annotate and analyze gene sets in a structured manner.</p>
</section>
<section id="kegg-kyoto-encyclopedia-of-genes-and-genomes" class="slide level2">
<h2><strong>KEGG (Kyoto Encyclopedia of Genes and Genomes)</strong></h2>
<p>KEGG is a comprehensive database that integrates biological pathways, genomic information, and functional annotations. It covers various organisms and provides detailed pathway maps, as well as information on metabolic pathways, signaling networks, diseases, and drugs.</p>
</section>
<section id="wikipathways" class="slide level2">
<h2><strong>WikiPathways</strong></h2>
<p>WikiPathways is a community-curated database that focuses on biological pathways. It provides a platform for researchers to collaboratively create, annotate, and update pathway information. WikiPathways covers a wide range of species and offers pathway diagrams, annotations, and links to external resources.</p>
</section>
<section id="disease-ontology" class="slide level2">
<h2><strong>Disease Ontology</strong></h2>
<p>The Disease Ontology (DO) is a structured and standardized ontology that describes human diseases and their relationships. It provides a hierarchical framework to classify and annotate diseases, enabling researchers to study disease-associated genes, pathways, and biological processes.</p>
</section>
<section id="reactome" class="slide level2">
<h2><strong>Reactome</strong></h2>
<p>Reactome is a curated database of biological pathways and reactions. It covers a wide range of species and provides detailed pathway information, including molecular events, regulatory processes, and disease associations. Reactome also offers advanced visualization tools and data analysis features.</p>
</section>
<section id="molecular-signatures-database-msigdb" class="slide level2">
<h2><strong>Molecular Signatures Database (MSigDB)</strong></h2>
<p>MSigDB is a collection of gene sets that represent various biological concepts and pathways. It includes curated gene sets from different sources, such as pathways, gene ontology terms, and gene expression signatures. MSigDB is widely used for gene set enrichment analysis and pathway-based functional analysis.</p>
</section>
<section id="medical-subject-headings-mesh" class="slide level2">
<h2><strong>Medical Subject Headings (MeSH)</strong></h2>
<p>MeSH is a controlled vocabulary database developed by the National Library of Medicine (NLM). It provides standardized terminology for biomedical research and categorizes articles in PubMed. MeSH includes hierarchical relationships between terms and enables the identification of relevant articles based on specific topics or concepts.</p>
</section>
<section id="over-representation-analysis" class="slide level2">
<h2>Over Representation analysis</h2>
<p>Over Representation Analysis (ORA) is a widely used approach to determine whether known biological functions or processes are over-represented (= enriched) in an experimentally-derived gene list, e.g.&nbsp;a list of differentially expressed genes (DEGs).</p>

<img data-src="index_files/imgs/genesetoverlap.svg" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="over-representation-analysis-1" class="slide level2">
<h2>Over Representation analysis</h2>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="index_files/imgs/contingency.png" width="400" height="300"></p>
<p></p><figcaption>From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</figcaption><p></p>
</figure>
</div>
</section>
<section id="fisher-exact-test" class="slide level2">
<h2>Fisher Exact Test</h2>
<ul>
<li>The Fisher Exact Test is a statistical method used to determine whether there is a significant association between two categorical variables, such as the presence or absence of a particular gene and a particular phenotype or disease.</li>
<li>In the context of gene enrichment analysis, the Fisher Exact Test is used to determine whether there is a significant association between a set of genes and a particular biological pathway or process.</li>
</ul>
<p>Hypothesis:</p>
<ul>
<li><span class="math inline">\(H_0\)</span>: The categories are independent.</li>
<li><span class="math inline">\(H_1\)</span>: The categories are dependent.</li>
</ul>
<blockquote>
<p>In Gene Enrichment, we want to see as genes as possible in the pathway that are DE, so we want a one sided test with greater hypothesis.</p>
</blockquote>
</section>
<section id="fisher-exact-test-1" class="slide level2">
<h2>Fisher Exact Test</h2>
<p>The p-value can be calculated by hypergeometric distribution<sup>1</sup>:</p>
<p><span class="math display">\[
p = 1 - \displaystyle\sum_{i = 0}^{k-1}\frac{{M \choose i}{{N-M} \choose {n-i}}} {{N \choose n}}
\]</span></p>
<ul>
<li><span class="math inline">\(N\)</span> is the total number of genes in the background distribution</li>
<li><span class="math inline">\(M\)</span> is the number of genes within that distribution that are annotated (either directly or indirectly) to the gene set of interest</li>
<li><span class="math inline">\(n\)</span> is the size of the list of genes of interest</li>
<li><span class="math inline">\(k\)</span> is the number of genes within that list which are annotated to the gene set.</li>
</ul>
<aside><ol class="aside-footnotes"><li id="fn2"><p>In case of alternative hypothesis as <code>greater</code>.</p></li></ol></aside></section>
<section id="fisher-exact-test-2" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="with-an-false-example">With an (false) example</h3>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="index_files/imgs/table_1.png" width="400" height="300"></p>
<p></p><figcaption>From https://www.pathwaycommons.org/guide/primers/statistics/fishers_exact_test/</figcaption><p></p>
</figure>
</div>
</section>
<section id="fisher-exact-test-3" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="with-an-false-example-1">With an (false) example</h3>

<img data-src="index_files/imgs/figure_1.png" class="r-stretch quarto-figure-center"><p class="caption">From https://www.pathwaycommons.org/guide/primers/statistics/fishers_exact_test/</p></section>
<section id="fisher-exact-test-4" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="with-an-false-example-2">With an (false) example</h3>
<div class="quarto-figure quarto-figure-center">
<figure>
<p><img data-src="index_files/imgs/table_2.png" width="400" height="300"></p>
<p></p><figcaption>From https://www.pathwaycommons.org/guide/primers/statistics/fishers_exact_test/</figcaption><p></p>
</figure>
</div>
</section>
<section id="fisher-exact-test-5" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="using-the-hypergeometric-distribution">Using the Hypergeometric distribution</h3>
<p><span class="math display">\[
\begin{equation*}
  \begin{split}
    \frac{(\text{15 Choose 1})\cdot(\text{15 Choose 14})}{(\text{30 Choose 15})}&amp; = \frac{15\cdot15}{155 117 520}\\
    &amp; = 1.45\cdot10^{-6}
  \end{split}
\end{equation*}
\]</span></p>
</section>
<section id="fisher-exact-test-6" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="with-an-false-example-3">With an (false) example</h3>

<img data-src="index_files/imgs/figure_3.png" class="r-stretch quarto-figure-center"><p class="caption">From https://www.pathwaycommons.org/guide/primers/statistics/fishers_exact_test/</p></section>
<section id="fisher-exact-test-7" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="solution">Solution</h3>
<p><span class="math display">\[
\begin{equation*}
  \begin{split}
    p&amp; = \sum\limits_{\text{DE &amp; IN}\geq12}p_i\\
     &amp; = 1.33\cdot10^{-3}+7.11\cdot10^{-5}+1.45\cdot10^{-6}+6.45\cdot10^{-9}\\
     &amp; = 0.001407159\\
     &amp; \simeq 0.0014
  \end{split}
\end{equation*}
\]</span></p>
</section>
<section id="fisher-exact-test-8" class="slide level2">
<h2>Fisher Exact Test</h2>
<h3 id="with-a-real-dataset">With a real dataset</h3>
<div class="cell">

</div>
<p>For a gene set regulation of cell adhesion (GO:0030155), I have :</p>
<ul>
<li>10096 (<span class="math inline">\(N\)</span>) number of genes in the analysis (background distribution).</li>
<li>203 (<span class="math inline">\(M\)</span>) genes within that distribution that are in the pathway of interest.</li>
<li>448 (<span class="math inline">\(n\)</span>) genes of interest (DE).</li>
<li>32 (<span class="math inline">\(k\)</span>) genes that are DE AND in the pathway of interest.</li>
</ul>
<p>And we can draw this contingency table:</p>
<div class="cell">
<div class="cell-output-display">

<div id="najvjgfqog" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#najvjgfqog table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#najvjgfqog thead, #najvjgfqog tbody, #najvjgfqog tfoot, #najvjgfqog tr, #najvjgfqog td, #najvjgfqog th {
  border-style: none;
}

#najvjgfqog p {
  margin: 0;
  padding: 0;
}

#najvjgfqog .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#najvjgfqog .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#najvjgfqog .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#najvjgfqog .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#najvjgfqog .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#najvjgfqog .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#najvjgfqog .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#najvjgfqog .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#najvjgfqog .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#najvjgfqog .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#najvjgfqog .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#najvjgfqog .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#najvjgfqog .gt_spanner_row {
  border-bottom-style: hidden;
}

#najvjgfqog .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#najvjgfqog .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#najvjgfqog .gt_from_md > :first-child {
  margin-top: 0;
}

#najvjgfqog .gt_from_md > :last-child {
  margin-bottom: 0;
}

#najvjgfqog .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#najvjgfqog .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#najvjgfqog .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#najvjgfqog .gt_row_group_first td {
  border-top-width: 2px;
}

#najvjgfqog .gt_row_group_first th {
  border-top-width: 2px;
}

#najvjgfqog .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#najvjgfqog .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#najvjgfqog .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#najvjgfqog .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#najvjgfqog .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#najvjgfqog .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#najvjgfqog .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#najvjgfqog .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#najvjgfqog .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#najvjgfqog .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#najvjgfqog .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#najvjgfqog .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#najvjgfqog .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#najvjgfqog .gt_left {
  text-align: left;
}

#najvjgfqog .gt_center {
  text-align: center;
}

#najvjgfqog .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#najvjgfqog .gt_font_normal {
  font-weight: normal;
}

#najvjgfqog .gt_font_bold {
  font-weight: bold;
}

#najvjgfqog .gt_font_italic {
  font-style: italic;
}

#najvjgfqog .gt_super {
  font-size: 65%;
}

#najvjgfqog .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#najvjgfqog .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#najvjgfqog .gt_indent_1 {
  text-indent: 5px;
}

#najvjgfqog .gt_indent_2 {
  text-indent: 10px;
}

#najvjgfqog .gt_indent_3 {
  text-indent: 15px;
}

#najvjgfqog .gt_indent_4 {
  text-indent: 20px;
}

#najvjgfqog .gt_indent_5 {
  text-indent: 25px;
}
</style>
<table class="gt_table" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
  <thead>
    <tr class="gt_heading">
      <td colspan="4" class="gt_heading gt_title gt_font_normal gt_bottom_border" style="">Contingency table: DE genes against regulation of cell adhesion (GO:0030155)</td>
    </tr>
    
    <tr class="gt_col_headings gt_spanner_row">
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="2" colspan="1" scope="col" id=""></th>
      <th class="gt_center gt_columns_top_border gt_column_spanner_outer" rowspan="1" colspan="1" scope="col" id="<strong>Genes of interest</strong>">
        <span class="gt_column_spanner"><strong>Genes of interest</strong></span>
      </th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="2" colspan="1" scope="col" id="No">No</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="2" colspan="1" scope="col" id="Total">Total</th>
    </tr>
    <tr class="gt_col_headings">
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1" scope="col" id="Yes">Yes</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><th id="stub_1_1" scope="row" class="gt_row gt_center gt_stub"><div class="gt_from_md"><p><strong>Pathway of interest</strong>: yes</p>
</div></th>
<td headers="stub_1_1 Of_interest_yes" class="gt_row gt_right"><div class="gt_from_md"><p>32</p>
</div></td>
<td headers="stub_1_1 Of_interest_no" class="gt_row gt_right"><div class="gt_from_md"><p>171</p>
</div></td>
<td headers="stub_1_1 Total" class="gt_row gt_right"><div class="gt_from_md"><p>203</p>
</div></td></tr>
    <tr><th id="stub_1_2" scope="row" class="gt_row gt_center gt_stub"><div class="gt_from_md"><p><strong>Pathway of interest</strong>: no</p>
</div></th>
<td headers="stub_1_2 Of_interest_yes" class="gt_row gt_right"><div class="gt_from_md"><p>416</p>
</div></td>
<td headers="stub_1_2 Of_interest_no" class="gt_row gt_right"><div class="gt_from_md"><p>9477</p>
</div></td>
<td headers="stub_1_2 Total" class="gt_row gt_right"><div class="gt_from_md"><p>9893</p>
</div></td></tr>
    <tr><th id="stub_1_3" scope="row" class="gt_row gt_center gt_stub"><div class="gt_from_md"><p><strong>Pathway of interest</strong>: Total</p>
</div></th>
<td headers="stub_1_3 Of_interest_yes" class="gt_row gt_right"><div class="gt_from_md"><p>448</p>
</div></td>
<td headers="stub_1_3 Of_interest_no" class="gt_row gt_right"><div class="gt_from_md"><p>9648</p>
</div></td>
<td headers="stub_1_3 Total" class="gt_row gt_right"><div class="gt_from_md"><p>10096</p>
</div></td></tr>
  </tbody>
  
  
</table>
</div>
</div>
</div>
</section>
<section id="tests-with-r" class="slide level2">
<h2>Tests with <code>R</code></h2>
<h3 id="using-hypergeometric-distribution">Using Hypergeometric distribution</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1"></a>all_probas <span class="ot">&lt;-</span> <span class="fu">dhyper</span>(<span class="dv">0</span><span class="sc">:</span><span class="dv">203</span>, M, N<span class="sc">-</span>M, n,<span class="at">log=</span>F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>

<img data-src="index_files/figure-revealjs/unnamed-chunk-24-1.png" width="960" class="r-stretch"></section>
<section id="tests-with-r-1" class="slide level2">
<h2>Tests with <code>R</code></h2>
<h3 id="using-hypergeometric-distribution-1">Using Hypergeometric distribution</h3>
<div class="columns">
<div class="column" style="width:50%;">
<h4 id="summing-all-probability-having-k32-or-more">Summing all probability having <span class="math inline">\(k=32\)</span> or more :</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1"></a><span class="fu">dhyper</span>(k<span class="sc">:</span>M, M, N<span class="sc">-</span>M, n,<span class="at">log=</span>F) <span class="sc">%&gt;%</span> <span class="fu">sum</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 3.050019e-10</code></pre>
</div>
</div>
</div><div class="column" style="width:50%;">
<h4 id="using-the-distribution-function-directly">Using the distribution function directly</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1"></a><span class="fu">phyper</span>(k<span class="dv">-1</span>, M, N<span class="sc">-</span>M, n,<span class="at">lower.tail =</span>F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 3.050019e-10</code></pre>
</div>
</div>
<p>Or even if we want to replicate the formula :</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1"></a><span class="dv">1</span><span class="sc">-</span><span class="fu">phyper</span>(k<span class="dv">-1</span>, M, N<span class="sc">-</span>M, n)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 3.050019e-10</code></pre>
</div>
</div>
</div>
</div>
<h3 id="using-fisher-test-directly">Using Fisher Test directly</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1"></a><span class="fu">fisher.test</span>(d, <span class="at">alternative =</span> <span class="st">"greater"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Fisher's Exact Test for Count Data

data:  d
p-value = 3.05e-10
alternative hypothesis: true odds ratio is greater than 1
95 percent confidence interval:
 2.986321      Inf
sample estimates:
odds ratio 
  4.262014 </code></pre>
</div>
</div>
</section>
<section id="gene-set-enrichment-analysis-gsea" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>
<ul>
<li>GSEA is a statistical method for analyzing gene expression data that aims to identify sets of genes or pathways that are significantly differentially expressed between two or more biological conditions.</li>
<li>GSEA does not rely on predefined gene sets, but rather uses all genes in the dataset and ranks them based on their differential expression between the conditions. The method then tests whether genes in a given set are more likely to be located at the top or bottom of the ranked list.</li>
<li>GSEA can provide insights into the biological processes or pathways that are involved in a specific disease or phenotype, and can help identify potential therapeutic targets.</li>
</ul>
<p>They may consist in expression levels, p-values, correlations, fold-changes (FC), t-statistics, signal-to-noise ratios, etc.</p>
</section>
<section id="gene-set-enrichment-analysis-gsea-1" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>

<img data-src="index_files/imgs/gsea1.png" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="gene-set-enrichment-analysis-gsea-2" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>

<img data-src="index_files/imgs/gsea2.png" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="gene-set-enrichment-analysis-gsea-3" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>

<img data-src="index_files/imgs/gsea3.png" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="gene-set-enrichment-analysis-gsea-4" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>

<img data-src="index_files/imgs/gsea4.gif" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="gene-set-enrichment-analysis-gsea-5" class="slide level2">
<h2>Gene Set Enrichment Analysis (GSEA)</h2>

<img data-src="index_files/imgs/gsea5.gif" class="r-stretch quarto-figure-center"><p class="caption">From bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22</p></section>
<section id="gene-set-enrichment-analysis" class="slide level2">
<h2>Gene Set Enrichment Analysis</h2>
<h3 id="with-a-real-dataset-1">With a real dataset</h3>

<img data-src="index_files/figure-revealjs/unnamed-chunk-29-1.png" width="960" class="r-stretch"></section>
<section id="gene-set-enrichment-analysis-1" class="slide level2">
<h2>Gene Set Enrichment Analysis</h2>
<h3 id="against-random-walks">Against random walks</h3>

<img data-src="index_files/figure-revealjs/unnamed-chunk-30-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Real ES score computation against random walks by permutation (100)</p></section>
<section id="gene-set-enrichment-analysis-2" class="slide level2">
<h2>Gene Set Enrichment Analysis</h2>
<h3 id="against-random-walks-1">Against random walks</h3>

<img data-src="index_files/figure-revealjs/unnamed-chunk-31-1.png" width="960" class="r-stretch quarto-figure-center"><p class="caption">Distribution of NULL distribution using permutation</p></section>
<section id="fisher-exact-test-9" class="slide level2">
<h2>Fisher Exact Test</h2>
<ul>
<li>The Fisher Exact Test is a statistical method used to determine whether there is a significant association between two categorical variables, such as the presence or absence of a particular gene and a particular phenotype or disease.</li>
<li>In the context of gene enrichment analysis, the Fisher Exact Test is used to determine whether there is a significant association between a set of genes and a particular biological pathway or process.</li>
<li>The Fisher Exact Test can provide valuable information about the biological processes or pathways that are involved in a particular disease or phenotype, and can help identify potential drug targets or therapeutic interventions.</li>
</ul>
</section>
<section id="weighted-kolmogorov-smirnov-statistics" class="slide level2">
<h2>Weighted Kolmogorov-Smirnov Statistics</h2>
<ul>
<li>Definition and principle of WKSS</li>
<li>Advantages over other methods</li>
<li>Example of WKSS results</li>
</ul>
</section>
<section id="tools-for-gene-enrichment-analysis" class="slide level2">
<h2>Tools for Gene Enrichment Analysis</h2>
<ul>
<li>R packages:
<ul>
<li>gprofiler2</li>
<li>clusterProfiler</li>
<li>GOstats</li>
</ul></li>
<li>Online tools:
<ul>
<li>Enrichr</li>
<li>DAVID</li>
<li>WebGestalt</li>
</ul></li>
</ul>
</section>
<section id="using-gprofiler2" class="slide level2">
<h2>Using gprofiler2</h2>
<ul>
<li>Example of gene enrichment analysis using gprofiler2</li>
<li>Input: list of genes</li>
<li>Output: enriched GO terms and KEGG pathways</li>
</ul>
</section>
<section id="conclusion" class="slide level2">
<h2>Conclusion</h2>
<ul>
<li>Summary of gene enrichment analysis methods and tools</li>
<li>Importance of carefully selecting appropriate methods and tools</li>
<li>Future directions in gene enrichment analysis</li>
</ul>
</section>
<section id="references" class="slide level2">
<h2>References</h2>
<ul>
<li>List of relevant articles and resources used in the presentation.</li>
</ul>
<h3 id="gene-ontology">Gene Ontology</h3>
<ul>
<li>Gene Ontology (GO) is a widely used standardized vocabulary for describing gene function and localization across different organisms.</li>
<li>GO categories classify genes into three main categories: molecular function, biological process, and cellular component, based on their functions and locations within a cell or organism.</li>
<li>Each GO term is a specific functional annotation that describes the activity or role of a gene in a biological process or pathway. For example, “cellular respiration” is a GO term that describes the process of converting nutrients into energy in the form of ATP within a cell.</li>
</ul>
<h3 id="fisher-exact-test-10">Fisher Exact Test</h3>
<ul>
<li>The Fisher Exact Test is a statistical method used to determine whether there is a significant association between two categorical variables, such as the presence or absence of a particular gene and a particular phenotype or disease.</li>
<li>In the context of gene enrichment analysis, the Fisher Exact Test is used to determine whether there is a significant association between a set of genes and a particular biological pathway or process.</li>
<li>The Fisher Exact Test can provide valuable information about the biological processes or pathways that are involved in a particular disease or phenotype, and can help identify potential drug targets or therapeutic interventions.</li>
</ul>
<h3 id="weighted-kolmogorov-smirnov-statistics-1">Weighted Kolmogorov-Smirnov Statistics</h3>
<ul>
<li>The Weighted Kolmogorov-Smirnov Statistics (WKSS) is a method for measuring the similarity between two sets of ranked genes, such as those obtained from a GSEA analysis.</li>
<li>WKSS takes into account the weights or scores assigned to each gene in the ranked list, and can provide a more accurate measure of similarity between two sets of genes compared to other methods that only consider the rankings.</li>
<li>WKSS can be useful for identifying the biological pathways or processes that are common between different gene expression datasets or between different stages of a disease.</li>
</ul>
<p>I hope this provides more clarity on the topics covered in the gene enrichment analysis presentation template.</p>
<div class="footer footer-default">

</div>
</section></section>

    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="index_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="index_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="index_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="index_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="index_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="index_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        target: function(trigger) {
          return trigger.previousElementSibling;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'quarto-reveal',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>