<!DOCTYPE html>
<html lang="en"><head>
<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/tabby.min.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-html.min.css" rel="stylesheet" data-mode="light">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles"><meta charset="utf-8">
  <meta name="generator" content="quarto-1.2.475">

  <title>TP metabolites</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reset.css">
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      { color: #003b4f; background-color: #f1f3f5; }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span { color: #003b4f; } /* Normal */
    code span.al { color: #ad0000; } /* Alert */
    code span.an { color: #5e5e5e; } /* Annotation */
    code span.at { color: #657422; } /* Attribute */
    code span.bn { color: #ad0000; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #003b4f; } /* ControlFlow */
    code span.ch { color: #20794d; } /* Char */
    code span.cn { color: #8f5902; } /* Constant */
    code span.co { color: #5e5e5e; } /* Comment */
    code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
    code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
    code span.dt { color: #ad0000; } /* DataType */
    code span.dv { color: #ad0000; } /* DecVal */
    code span.er { color: #ad0000; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #ad0000; } /* Float */
    code span.fu { color: #4758ab; } /* Function */
    code span.im { color: #00769e; } /* Import */
    code span.in { color: #5e5e5e; } /* Information */
    code span.kw { color: #003b4f; } /* Keyword */
    code span.op { color: #5e5e5e; } /* Operator */
    code span.ot { color: #003b4f; } /* Other */
    code span.pp { color: #ad0000; } /* Preprocessor */
    code span.sc { color: #5e5e5e; } /* SpecialChar */
    code span.ss { color: #20794d; } /* SpecialString */
    code span.st { color: #20794d; } /* String */
    code span.va { color: #111111; } /* Variable */
    code span.vs { color: #20794d; } /* VerbatimString */
    code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
  </style>
  <link rel="stylesheet" href="index_files/libs/revealjs/dist/theme/quarto.css" id="theme">
  <link href="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.css" rel="stylesheet">
  <link href="index_files/libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">
  <style type="text/css">

  .callout {
    margin-top: 1em;
    margin-bottom: 1em;  
    border-radius: .25rem;
  }

  .callout.callout-style-simple { 
    padding: 0em 0.5em;
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
    display: flex;
  }

  .callout.callout-style-default {
    border-left: solid #acacac .3rem;
    border-right: solid 1px silver;
    border-top: solid 1px silver;
    border-bottom: solid 1px silver;
  }

  .callout .callout-body-container {
    flex-grow: 1;
  }

  .callout.callout-style-simple .callout-body {
    font-size: 1rem;
    font-weight: 400;
  }

  .callout.callout-style-default .callout-body {
    font-size: 0.9rem;
    font-weight: 400;
  }

  .callout.callout-captioned.callout-style-simple .callout-body {
    margin-top: 0.2em;
  }

  .callout:not(.callout-captioned) .callout-body {
      display: flex;
  }

  .callout:not(.no-icon).callout-captioned.callout-style-simple .callout-content {
    padding-left: 1.6em;
  }

  .callout.callout-captioned .callout-header {
    padding-top: 0.2em;
    margin-bottom: -0.2em;
  }

  .callout.callout-captioned .callout-caption  p {
    margin-top: 0.5em;
    margin-bottom: 0.5em;
  }
    
  .callout.callout-captioned.callout-style-simple .callout-content  p {
    margin-top: 0;
  }

  .callout.callout-captioned.callout-style-default .callout-content  p {
    margin-top: 0.7em;
  }

  .callout.callout-style-simple div.callout-caption {
    border-bottom: none;
    font-size: .9rem;
    font-weight: 600;
    opacity: 75%;
  }

  .callout.callout-style-default  div.callout-caption {
    border-bottom: none;
    font-weight: 600;
    opacity: 85%;
    font-size: 0.9rem;
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-default div.callout-content {
    padding-left: 0.5em;
    padding-right: 0.5em;
  }

  .callout.callout-style-simple .callout-icon::before {
    height: 1rem;
    width: 1rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 1rem 1rem;
  }

  .callout.callout-style-default .callout-icon::before {
    height: 0.9rem;
    width: 0.9rem;
    display: inline-block;
    content: "";
    background-repeat: no-repeat;
    background-size: 0.9rem 0.9rem;
  }

  .callout-caption {
    display: flex
  }
    
  .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  .callout.no-icon::before {
    display: none !important;
  }

  .callout.callout-captioned .callout-body > .callout-content > :last-child {
    margin-bottom: 0.5rem;
  }

  .callout.callout-captioned .callout-icon::before {
    margin-top: .5rem;
    padding-right: .5rem;
  }

  .callout:not(.callout-captioned) .callout-icon::before {
    margin-top: 1rem;
    padding-right: .5rem;
  }

  /* Callout Types */

  div.callout-note {
    border-left-color: #4582ec !important;
  }

  div.callout-note .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEU0lEQVRYCcVXTWhcVRQ+586kSUMMxkyaElstCto2SIhitS5Ek8xUKV2poatCcVHtUlFQk8mbaaziwpWgglJwVaquitBOfhQXFlqlzSJpFSpIYyXNjBNiTCck7x2/8/LeNDOZxDuEkgOXe++553zfefee+/OYLOXFk3+1LLrRdiO81yNqZ6K9cG0P3MeFaMIQjXssE8Z1JzLO9ls20MBZX7oG8w9GxB0goaPrW5aNMp1yOZIa7Wv6o2ykpLtmAPs/vrG14Z+6d4jpbSKuhdcSyq9wGMPXjonwmESXrriLzFGOdDBLB8Y6MNYBu0dRokSygMA/mrun8MGFN3behm6VVAwg4WR3i6FvYK1T7MHo9BK7ydH+1uurECoouk5MPRyVSBrBHMYwVobG2aOXM07sWrn5qgB60rc6mcwIDJtQrnrEr44kmy+UO9r0u9O5/YbkS9juQckLed3DyW2XV/qWBBB3ptvI8EUY3I9p/67OW+g967TNr3Sotn3IuVlfMLVnsBwH4fsnebJvyGm5GeIUA3jljERmrv49SizPYuq+z7c2H/jlGC+Ghhupn/hcapqmcudB9jwJ/3jvnvu6vu5lVzF1fXyZuZZ7U8nRmVzytvT+H3kilYvH09mLWrQdwFSsFEsxFVs5fK7A0g8gMZjbif4ACpKbjv7gNGaD8bUrlk8x+KRflttr22JEMRUbTUwwDQScyzPgedQHZT0xnx7ujw2jfVfExwYHwOsDTjLdJ2ebmeQIlJ7neo41s/DrsL3kl+W2lWvAga0tR3zueGr6GL78M3ifH0rGXrBC2aAR8uYcIA5gwV8zIE8onoh8u0Fca/ciF7j1uOzEnqcIm59sEXoGc0+z6+H45V1CvAvHcD7THztu669cnp+L0okAeIc6zjbM/24LgGM1gZk7jnRu1aQWoU9sfUOuhrmtaPIO3YY1KLLWZaEO5TKUbMY5zx8W9UJ6elpLwKXbsaZ4EFl7B4bMtDv0iRipKoDQT2sNQI9b1utXFdYisi+wzZ/ri/1m7QfDgEuvgUUEIJPq3DhX/5DWNqIXDOweC2wvIR90Oq3lDpdMIgD2r0dXvGdsEW5H6x6HLRJYU7C69VefO1x8Gde1ZFSJLfWS1jbCnhtOPxmpfv2LXOA2Xk2tvnwKKPFuZ/oRmwBwqRQDcKNeVQkYcOjtWVBuM/JuYw5b6isojIkYxyYAFn5K7ZBF10fea52y8QltAg6jnMqNHFBmGkQ1j+U43HMi2xMar1Nv0zGsf1s8nUsmUtPOOrbFIR8bHFDMB5zL13Gmr/kGlCkUzedTzzmzsaJXhYawnA3UmARpiYj5ooJZiUoxFRtK3X6pgNPv+IZVPcnwbOl6f+aBaO1CNvPW9n9LmCp01nuSaTRF2YxHqZ8DYQT6WsXT+RD6eUztwYLZ8rM+rcPxamv1VQzFUkzFXvkiVrySGQgJNvXHJAxiU3/NwiC03rSf05VBaPtu/Z7/B8Yn/w7eguloAAAAAElFTkSuQmCC');
  }

  div.callout-note.callout-style-default .callout-caption {
    background-color: #dae6fb
  }

  div.callout-important {
    border-left-color: #d9534f !important;
  }

  div.callout-important .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAEKklEQVRYCcVXTWhcVRS+575MJym48A+hSRFr00ySRQhURRfd2HYjk2SSTokuBCkU2o0LoSKKraKIBTcuFCoidGFD08nkBzdREbpQ1EDNIv8qSGMFUboImMSZd4/f9zJv8ibJMC8xJQfO3HPPPef7zrvvvnvviIkpC9nsw0UttFunbUhpFzFtarSd6WJkStVMw5xyVqYTvkwfzuf/5FgtkVoB0729j1rjXwThS7Vio+Mo6DNnvLfahoZ+i/o32lULuJ3NNiz7q6+pyAUkJaFF6JwaM2lUJlV0MlnQn5aTRbEu0SEqHUa0A4AdiGuB1kFXRfVyg5d87+Dg4DL6m2TLAub60ilj7A1Ec4odSAc8X95sHh7+ZRPCFo6Fnp7HfU/fBng/hi10CjCnWnJjsxvDNxWw0NfV6Rv5GgP3I3jGWXumdTD/3cbEOP2ZbOZp69yniG3FQ9z1jD7bnBu9Fc2tKGC2q+uAJOQHBDRiZX1x36o7fWBs7J9ownbtO+n0/qWkvW7UPIfc37WgT6ZGR++EOJyeQDSb9UB+DZ1G6DdLDzyS+b/kBCYGsYgJbSQHuThGKRcw5xdeQf8YdNHsc6ePXrlSYMBuSIAFTGAtQo+VuALo4BX83N190NWZWbynBjhOHsmNfFWLeL6v+ynsA58zDvvAC8j5PkbOcXCMg2PZFk3q8MjI7WAG/Dp9AwP7jdGBOOQkAvlFUB+irtm16I1Zw9YBcpGTGXYmk3kQIC/Cds55l+iMI3jqhjAuaoe+am2Jw5GT3Nbz3CkE12NavmzN5+erJW7046n/CH1RO/RVa8lBLozXk9uqykkGAyRXLWlLv5jyp4RFsG5vGVzpDLnIjTWgnRy2Rr+tDKvRc7Y8AyZq10jj8DqXdnIRNtFZb+t/ZRtXcDiVnzpqx8mPcDWxgARUqx0W1QB9MeUZiNrV4qP+Ehc+BpNgATsTX8ozYKL2NtFYAHc84fG7ndxUPr+AR/iQSns7uSUufAymwDOb2+NjK27lEFocm/EE2WpyIy/Hi66MWuMKJn8RvxIcj87IM5Vh9663ziW36kR0HNenXuxmfaD8JC7tfKbrhFr7LiZCrMjrzTeGx+PmkosrkNzW94ObzwocJ7A1HokLolY+AvkTiD/q1H0cN48c5EL8Crkttsa/AXQVDmutfyku0E7jShx49XqV3MFK8IryDhYVbj7Sj2P2eBxwcXoe8T8idsKKPRcnZw1b+slFTubwUwhktrfnAt7J++jwQtLZcm3sr9LQrjRzz6cfMv9aLvgmnAGvpoaGLxM4mAEaLV7iAzQ3oU0IvD5x9ix3yF2RAAuYAOO2f7PEFWCXZ4C9Pb2UsgDeVnFSpbFK7/IWu7TPTvBqzbGdCHOJQSxiEjt6IyZmxQyEJHv6xyQsYk//moVFsN2zP6fRImjfq7/n/wFDguUQFNEwugAAAABJRU5ErkJggg==');
  }

  div.callout-important.callout-style-default .callout-caption {
    background-color: #f7dddc
  }

  div.callout-warning {
    border-left-color: #f0ad4e !important;
  }

  div.callout-warning .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAAETklEQVRYCeVWW2gcVRg+58yaTUnizqbipZeX4uWhBEniBaoUX1Ioze52t7sRq6APio9V9MEaoWlVsFasRq0gltaAPuxms8lu0gcviE/FFOstVbSIxgcv6SU7EZqmdc7v9+9mJtNks51NTUH84ed889/PP+cmxP+d5FIbMJmNbpREu4WUkiTtCicKny0l1pIKmBzovF2S+hIJHX8iEu3hZJ5lNZGqyRrGSIQpq15AzF28jgpeY6yk6GVdrfFqdrD6Iw+QlB8g0YS2g7dyQmXM/IDhBhT0UCiRf59lfqmmDvzRt6kByV/m4JjtzuaujMUM2c5Z2d6JdKrRb3K2q6mA+oYVz8JnDdKPmmNthzkAk/lN63sYPgevrguc72aZX/L9C6x09GYyxBgCX4NlvyGUHOKELlm5rXeR1kchuChJt4SSwyddZRXgvwMGvYo4QSlk3/zkHD8UHxwVJA6zjZZqP8v8kK8OWLnIZtLyCAJagYC4rTGW/9Pqj92N/c+LUaAj27movwbi19tk/whRCIE7Q9vyI6yvRpftAKVTdUjOW40X3h5OXsKCdmFcx0xlLJoSuQngnrJe7Kcjm4OMq9FlC7CMmScQANuNvjfP3PjGXDBaUQmbp296S5L4DrpbrHN1T87ZVEZVCzg1FF0Ft+dKrlLukI+/c9ENo+TvlTDbYFvuKPtQ9+l052rXrgKoWkDAFnvh0wTOmYn8R5f4k/jN/fZiCM1tQx9jQQ4ANhqG4hiL0qIFTGViG9DKB7GYzgubnpofgYRwO+DFjh0Zin2m4b/97EDkXkc+f6xYAPX0KK2I/7fUQuwzuwo/L3AkcjugPNixC8cHf0FyPjWlItmLxWw4Ou9YsQCr5fijMGoD/zpdRy95HRysyXA74MWOnscpO4j2y3HAVisw85hX5+AFBRSHt4ShfLFkIMXTqyKFc46xdzQM6XbAi702a7sy04J0+feReMFKp5q9esYLCqAZYw/k14E/xcLLsFElaornTuJB0svMuJINy8xkIYuL+xPAlWRceH6+HX7THJ0djLUom46zREu7tTkxwmf/FdOZ/sh6Q8qvEAiHpm4PJ4a/doJe0gH1t+aHRgCzOvBvJedEK5OFE5jpm4AGP2a8Dxe3gGJ/pAutug9Gp6he92CsSsWBaEcxGx0FHytmIpuqGkOpldqNYQK8cSoXvd+xLxXADw0kf6UkJNFtdo5MOgaLjiQOQHcn+A6h5NuL2s0qsC2LOM75PcF3yr5STuBSAcGG+meA14K/CI21HcS4LBT6tv0QAh8Dr5l93AhZzG5ZJ4VxAqdZUEl9z7WJ4aN+svMvwHHL21UKTd1mqvChH7/Za5xzXBBKrUcB0TQ+Ulgkfbi/H/YT5EptrGzsEK7tR1B7ln9BBwckYfMiuSqklSznIuoIIOM42MQO+QnduCoFCI0bpkzjCjddHPN/F+2Yu+sd9bKNpVwHhbS3LluK/0zgfwD0xYI5dXuzlQAAAABJRU5ErkJggg==');
  }

  div.callout-warning.callout-style-default .callout-caption {
    background-color: #fcefdc
  }

  div.callout-tip {
    border-left-color: #02b875 !important;
  }

  div.callout-tip .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAADr0lEQVRYCe1XTWgTQRj9ZjZV8a9SPIkKgj8I1bMHsUWrqYLVg4Ue6v9BwZOxSYsIerFao7UiUryIqJcqgtpimhbBXoSCVxUFe9CTiogUrUp2Pt+3aUI2u5vdNh4dmMzOzHvvezuz8xNFM0mjnbXaNu1MvFWRXkXEyE6aYOYJpdW4IXuA4r0fo8qqSMDBU0v1HJUgVieAXxzCsdE/YJTdFcVIZQNMyhruOMJKXYFoLfIfIvVIMWdsrd+Rpd86ZmyzzjJmLStqRn0v8lzkb4rVIXvnpScOJuAn2ACC65FkPzEdEy4TPWRLJ2h7z4cArXzzaOdKlbOvKKX25Wl00jSnrwVxAg3o4dRxhO13RBSdNvH0xSARv3adTXbBdTf64IWO2vH0LT+cv4GR1DJt+DUItaQogeBX/chhbTBxEiZ6gftlDNXTrvT7co4ub5A6gp9HIcHvzTa46OS5fBeP87Qm0fQkr4FsYgVQ7Qg+ZayaDg9jhg1GkWj8RG6lkeSacrrHgDaxdoBiZPg+NXV/KifMuB6//JmYH4CntVEHy/keA6x4h4CU5oFy8GzrBS18cLJMXcljAKB6INjWsRcuZBWVaS3GDrqB7rdapVIeA+isQ57Eev9eCqzqOa81CY05VLd6SamW2wA2H3SiTbnbSxmzfp7WtKZkqy4mdyAlGx7ennghYf8voqp9cLSgKdqNfa6RdRsAAkPwRuJZNbpByn+RrJi1RXTwdi8RQF6ymDwGMAtZ6TVE+4uoKh+MYkcLsT0Hk8eAienbiGdjJHZTpmNjlbFJNKDVAp2fJlYju6IreQxQ08UJDNYdoLSl6AadO+fFuCQqVMB1NJwPm69T04Wv5WhfcWyfXQB+wXRs1pt+nCknRa0LVzSA/2B+a9+zQJadb7IyyV24YAxKp2Jqs3emZTuNnKxsah+uabKbMk7CbTgJx/zIgQYErIeTKRQ9yD9wxVof5YolPHqaWo7TD6tJlh7jQnK5z2n3+fGdggIOx2kaa2YI9QWarc5Ce1ipNWMKeSG4DysFF52KBmTNMmn5HqCFkwy34rDg05gDwgH3bBi+sgFhN/e8QvRn8kbamCOhgrZ9GJhFDgfcMHzFb6BAtjKpFhzTjwv1KCVuxHvCbsSiEz4CANnj84cwHdFXAbAOJ4LTSAawGWFn5tDhLMYz6nWeU2wJfIhmIJBefcd/A5FWQWGgrWzyORZ3Q6HuV+Jf0Bj+BTX69fm1zWgK7By1YTXchFDORywnfQ7GpzOo6S+qECrsx2ifVQAAAABJRU5ErkJggg==');
  }

  div.callout-tip.callout-style-default .callout-caption {
    background-color: #ccf1e3
  }

  div.callout-caution {
    border-left-color: #fd7e14 !important;
  }

  div.callout-caution .callout-icon::before {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACshmLzAAACV0lEQVRYCdVWzWoUQRCuqp2ICBLJXgITZL1EfQDBW/bkzUMUD7klD+ATSHBEfAIfQO+iXsWDxJsHL96EHAwhgzlkg8nBg25XWb0zIb0zs9muYYWkoKeru+vn664fBqElyZNuyh167NXJ8Ut8McjbmEraKHkd7uAnAFku+VWdb3reSmRV8PKSLfZ0Gjn3a6Xlcq9YGb6tADjn+lUfTXtVmaZ1KwBIvFI11rRXlWlatwIAAv2asaa9mlB9wwygiDX26qaw1yYPzFXg2N1GgG0FMF8Oj+VIx7E/03lHx8UhvYyNZLN7BwSPgekXXLribw7w5/c8EF+DBK5idvDVYtEEwMeYefjjLAdEyQ3M9nfOkgnPTEkYU+sxMq0BxNR6jExrAI31H1rzvLEfRIdgcv1XEdj6QTQAS2wtstEALLG1yEZ3QhH6oDX7ExBSFEkFINXH98NTrme5IOaaA7kIfiu2L8A3qhH9zRbukdCqdsA98TdElyeMe5BI8Rs2xHRIsoTSSVFfCFCWGPn9XHb4cdobRIWABNf0add9jakDjQJpJ1bTXOJXnnRXHRf+dNL1ZV1MBRCXhMbaHqGI1JkKIL7+i8uffuP6wVQAzO7+qVEbF6NbS0LJureYcWXUUhH66nLR5rYmva+2tjRFtojkM2aD76HEGAD3tPtKM309FJg5j/K682ywcWJ3PASCcycH/22u+Bh7Aa0ehM2Fu4z0SAE81HF9RkB21c5bEn4Dzw+/qNOyXr3DCTQDMBOdhi4nAgiFDGCinIa2owCEChUwD8qzd03PG+qdW/4fDzjUMcE1ZpIAAAAASUVORK5CYII=');
  }

  div.callout-caution.callout-style-default .callout-caption {
    background-color: #ffe5d0
  }

  </style>
  <style type="text/css">
    .reveal div.sourceCode {
      margin: 0;
      overflow: auto;
    }
    .reveal div.hanging-indent {
      margin-left: 1em;
      text-indent: -1em;
    }
    .reveal .slide:not(.center) {
      height: 100%;
    }
    .reveal .slide.scrollable {
      overflow-y: auto;
    }
    .reveal .footnotes {
      height: 100%;
      overflow-y: auto;
    }
    .reveal .slide .absolute {
      position: absolute;
      display: block;
    }
    .reveal .footnotes ol {
      counter-reset: ol;
      list-style-type: none; 
      margin-left: 0;
    }
    .reveal .footnotes ol li:before {
      counter-increment: ol;
      content: counter(ol) ". "; 
    }
    .reveal .footnotes ol li > p:first-child {
      display: inline-block;
    }
    .reveal .slide ul,
    .reveal .slide ol {
      margin-bottom: 0.5em;
    }
    .reveal .slide ul li,
    .reveal .slide ol li {
      margin-top: 0.4em;
      margin-bottom: 0.2em;
    }
    .reveal .slide ul[role="tablist"] li {
      margin-bottom: 0;
    }
    .reveal .slide ul li > *:first-child,
    .reveal .slide ol li > *:first-child {
      margin-block-start: 0;
    }
    .reveal .slide ul li > *:last-child,
    .reveal .slide ol li > *:last-child {
      margin-block-end: 0;
    }
    .reveal .slide .columns:nth-child(3) {
      margin-block-start: 0.8em;
    }
    .reveal blockquote {
      box-shadow: none;
    }
    .reveal .tippy-content>* {
      margin-top: 0.2em;
      margin-bottom: 0.7em;
    }
    .reveal .tippy-content>*:last-child {
      margin-bottom: 0.2em;
    }
    .reveal .slide > img.stretch.quarto-figure-center,
    .reveal .slide > img.r-stretch.quarto-figure-center {
      display: block;
      margin-left: auto;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-left,
    .reveal .slide > img.r-stretch.quarto-figure-left  {
      display: block;
      margin-left: 0;
      margin-right: auto; 
    }
    .reveal .slide > img.stretch.quarto-figure-right,
    .reveal .slide > img.r-stretch.quarto-figure-right  {
      display: block;
      margin-left: auto;
      margin-right: 0; 
    }
  </style>
</head>
<body class="quarto-light">
  <div class="reveal">
    <div class="slides">

<section id="title-slide" class="quarto-title-block center">
  <h1 class="title">TP metabolites</h1>

<div class="quarto-title-authors">
</div>

</section>
<section id="goals" class="title-slide slide level1 center">
<h1>Goals</h1>
<ul>
<li><p>Read in &amp; format sample data</p></li>
<li><p>Match IDs so they can be used with different pathway databases</p></li>
<li><p>Highlight enriched pathways using different methods &amp; visualisations</p></li>
<li><p>Choose and understand parameters and their impact on the results</p></li>
</ul>
</section>

<section>
<section id="input-data" class="title-slide slide level1 center">
<h1>Input data</h1>
<p><code>read.table</code> is a generic function: specify the separator with <code>sep=</code> , and if there are column names, use <code>header = T</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="co"># Pre-filtered metabolite inputs</span></span>
<span id="cb1-2"><a href="#cb1-2"></a>geno.mother.raw <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/ML_add_geno_mother11_05_2023.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T)</span>
<span id="cb1-3"><a href="#cb1-3"></a>gestation.raw <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/ML_add_gestation_stage_11_05_2023.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T)</span>
<span id="cb1-4"><a href="#cb1-4"></a></span>
<span id="cb1-5"><a href="#cb1-5"></a><span class="co"># Annotation database</span></span>
<span id="cb1-6"><a href="#cb1-6"></a>annot.db <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/ASICS_annotationID_2022_02.csv"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">header =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="data-types" class="slide level2">
<h2>Data types</h2>
<p>The two types of data we will be using are:</p>
<ul>
<li><p>vector</p></li>
<li><p>dataframe</p></li>
</ul>
<p>Vectors are lists of elements. Vector elements can be named.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a>v <span class="ot">=</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">3</span>)</span>
<span id="cb2-2"><a href="#cb2-2"></a>v</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 1 2 3</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="fu">names</span>(v) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"a"</span>, <span class="st">"b"</span>, <span class="st">"c"</span>)</span>
<span id="cb4-2"><a href="#cb4-2"></a>v</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>a b c 
1 2 3 </code></pre>
</div>
</div>
<p>Dataframes are tables: they have columns, rows, column names and can have row names.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a>df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="st">"cola"</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">2</span>,<span class="dv">3</span>), <span class="st">"colb"</span> <span class="ot">=</span> <span class="fu">c</span>(<span class="dv">4</span>,<span class="dv">5</span>,<span class="dv">6</span>), <span class="at">row.names =</span> <span class="fu">c</span>(<span class="st">"row1"</span>, <span class="st">"row2"</span>, <span class="st">"row3"</span>))</span>
<span id="cb6-2"><a href="#cb6-2"></a>df</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>     cola colb
row1    1    4
row2    2    5
row3    3    6</code></pre>
</div>
</div>
</section>
<section id="tidyverse" class="slide level2">
<h2>Tidyverse</h2>
<p>Tidyverse is a R package which loads a collection of other packages designed to work well together for data manipulation. Some packages it contains are <code>tidyr</code>, <code>dplyr</code>, <code>ggplot2</code>.</p>
<p>The syntax it is based on uses the pipe symbol: <code>%&gt;%</code> , useful for chaining different operations.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>df <span class="sc">%&gt;%</span> <span class="fu">print</span>() <span class="co"># same as print(df)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>     cola colb
row1    1    4
row2    2    5
row3    3    6</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<p>The main operations used in these exercises are:</p>
<ul>
<li><p><code>select()</code>: for selecting columns by name, can also select everything except a column using <code>select(-colname)</code></p></li>
<li><p><code>filter()</code>: for filtering rows by a condition, usually based on a column</p></li>
<li><p><code>left_join()</code>: for joining two dataframes on a common ID column</p></li>
<li><p><code>mutate()</code>: for editing the values of a column</p></li>
<li><p><code>enframe()</code>: for converting a named vector into a two-column dataframe, useful for getting KEGG pathways into the correct format in our case</p></li>
<li><p><code>pull()</code>: converts a dataframe column into a vector, does the same thing as <code>df$colname</code></p></li>
<li><p><code>rename()</code>: for renaming column(s)</p></li>
<li><p><code>relocate()</code>: specify a column name to reorder and place it as the first column</p></li>
</ul>
<p>You can also chain standard functions like <code>unique()</code>, <code>na.omit()</code>.</p>
</section>
<section class="slide level2">

<h3 id="examples">Examples</h3>
<p>Here is an example using the background set:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a><span class="co"># Background set = all measurable metabolites</span></span>
<span id="cb10-2"><a href="#cb10-2"></a>background_unfiltered <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/metabolites_present_endometre.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T)</span>
<span id="cb10-3"><a href="#cb10-3"></a><span class="fu">tail</span>(background_unfiltered)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>    Metabolite names_asics Match HMDB PubChem KEGG SMILES Comment ChEBI METLIN
132                                        NA                  NA    NA     NA
133                                        NA                  NA    NA     NA
134                                        NA                  NA    NA     NA
135                                        NA                  NA    NA     NA
136                                        NA                  NA    NA     NA
137                                        NA                  NA    NA     NA
     X X.1 X.2 X.3 X.4 X.5 X.6 X.7 X.8 X.9 X.10 X.11 X.12 X.13 X.14 X.15 X.16
132 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA
133 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA
134 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA
135 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA
136 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA
137 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA   NA   NA   NA   NA</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a>background <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/metabolites_present_endometre.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T) <span class="sc">%&gt;%</span> </span>
<span id="cb12-2"><a href="#cb12-2"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">if_any</span>(PubChem, is.na))  <span class="co"># filter out rows that contain NA in column PubChem</span></span>
<span id="cb12-3"><a href="#cb12-3"></a><span class="fu">tail</span>(background)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>      Metabolite  names_asics                       Match        HMDB  PubChem
40 Saccaric acid SaccaricAcid               Glucaric acid HMDB0000663    33037
41     Succinate    Succinate               Succinic acid HMDB0000254     1110
42       Taurine      Taurine                     Taurine HMDB0000251     1123
43          TMAO         TMAO      Trimethylamine N-oxide HMDB0000925     1145
44          UDPG         UDPG Uridine diphosphate glucose HMDB0000286 53477679
45 Vanillic acid VanillicAcid               Vanillic acid HMDB0000484     8468
     KEGG
40 C00818
41 C00042
42 C00245
43 C01104
44 C00029
45 C06672
                                                                                                          SMILES
40                                                              [C@H]([C@@H]([C@@H](C(=O)O)O)O)([C@H](C(=O)O)O)O
41                                                                                              C(CC(=O)O)C(=O)O
42                                                                                               C(CS(=O)(=O)O)N
43                                                                                               C[N+](C)(C)[O-]
44 C1=CN(C(=O)NC1=O)[C@@H]2[C@H]([C@H]([C@@H](O2)COP(=O)(O)OP(=O)(O)OC3[C@H]([C@@H]([C@H]([C@@H](O3)CO)O)O)O)O)O
45                                                                                      COC1=C(C=CC(=C1)C(=O)O)O
   Comment ChEBI METLIN  X X.1 X.2 X.3 X.4 X.5 X.6 X.7 X.8 X.9 X.10 X.11 X.12
40       1 16002   5633 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
41       1 15741    114 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
42       1 15891     31 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
43       1 15724   5876 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
44       1 46229   5278 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
45       1 30816   5471 NA  NA  NA  NA  NA  NA  NA  NA  NA  NA   NA   NA   NA
   X.13 X.14 X.15 X.16
40   NA   NA   NA   NA
41   NA   NA   NA   NA
42   NA   NA   NA   NA
43   NA   NA   NA   NA
44   NA   NA   NA   NA
45   NA   NA   NA   NA</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<p>And an example using the raw unfiltered fold changes:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="co"># Join by metabolite names to get all fold changes with the ID columns (ChEBI, KEGG..)</span></span>
<span id="cb14-2"><a href="#cb14-2"></a>all.fc.file <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/log2_FC_endometrium_metabolites_present.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T) </span>
<span id="cb14-3"><a href="#cb14-3"></a><span class="fu">head</span>(all.fc.file)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>   metabolites fc_D110vsD90   fc_LWvsMS
1      Lactate   0.20843228  0.07893974
2 Myo-Inositol   0.10919475 -0.20458874
3    L-Glycine   0.04401215 -0.22991271
4      Taurine  -0.07949751 -0.08002337
5    L-Alanine   0.12676021  0.07203598
6     Methanol   0.09152747 -0.21221730</code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>all.fc <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/log2_FC_endometrium_metabolites_present.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T) <span class="sc">%&gt;%</span> </span>
<span id="cb16-2"><a href="#cb16-2"></a>  <span class="fu">left_join</span>(., background <span class="sc">%&gt;%</span> <span class="fu">select</span>(names_asics, KEGG), </span>
<span id="cb16-3"><a href="#cb16-3"></a>            <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"metabolites"</span> <span class="ot">=</span> <span class="st">"names_asics"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb16-4"><a href="#cb16-4"></a>  <span class="fu">rename</span>(<span class="st">"gestation"</span> <span class="ot">=</span> fc_D110vsD90, <span class="st">"geno.mother"</span> <span class="ot">=</span> fc_LWvsMS)</span>
<span id="cb16-5"><a href="#cb16-5"></a><span class="fu">head</span>(all.fc)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>   metabolites   gestation geno.mother   KEGG
1      Lactate  0.20843228  0.07893974 C00186
2 Myo-Inositol  0.10919475 -0.20458874   &lt;NA&gt;
3    L-Glycine  0.04401215 -0.22991271 C00037
4      Taurine -0.07949751 -0.08002337 C00245
5    L-Alanine  0.12676021  0.07203598 C00041
6     Methanol  0.09152747 -0.21221730 C00132</code></pre>
</div>
</div>
</section></section>
<section>
<section id="pathway-sets" class="title-slide slide level1 center">
<h1>Pathway sets</h1>
<p>Pathway sets are metabolites grouped together by a common function. We can get pathway sets from KEGG, Reactome, and using metabolic networks.</p>
</section>
<section id="kegg" class="slide level2">
<h2>KEGG</h2>
<p>Using the KEGG API, we can access the database directly inside R. We can get a list of all pathways for a given organism, and a list of all compounds per pathway. We can then match them together to get all metabolites organised into pathways for our organism.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a><span class="co"># Pig KEGG pathways</span></span>
<span id="cb18-2"><a href="#cb18-2"></a>pig.pathways <span class="ot">=</span> <span class="fu">keggList</span>(<span class="st">"pathway"</span>, <span class="st">"ssc"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb18-3"><a href="#cb18-3"></a>  <span class="fu">enframe</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb18-4"><a href="#cb18-4"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_remove</span>(name,<span class="st">"^ssc"</span>), </span>
<span id="cb18-5"><a href="#cb18-5"></a>         <span class="at">value =</span> <span class="fu">str_remove</span>(value, <span class="st">" </span><span class="sc">\\</span><span class="st">- Sus scrofa </span><span class="sc">\\</span><span class="st">(pig</span><span class="sc">\\</span><span class="st">)"</span>)) </span>
<span id="cb18-6"><a href="#cb18-6"></a></span>
<span id="cb18-7"><a href="#cb18-7"></a><span class="co"># Compounds per pathways</span></span>
<span id="cb18-8"><a href="#cb18-8"></a>pathway.compounds <span class="ot">=</span> <span class="fu">keggLink</span>(<span class="st">"cpd"</span>, <span class="st">"pathway"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb18-9"><a href="#cb18-9"></a>  <span class="fu">enframe</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb18-10"><a href="#cb18-10"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_remove</span>(name,<span class="st">"^path:map"</span>), </span>
<span id="cb18-11"><a href="#cb18-11"></a>         <span class="at">value =</span> <span class="fu">str_remove</span>(value, <span class="st">"^cpd:"</span>))</span>
<span id="cb18-12"><a href="#cb18-12"></a></span>
<span id="cb18-13"><a href="#cb18-13"></a><span class="co"># Join the pathways we want to use with the compounds, filter out non metabolic pathways</span></span>
<span id="cb18-14"><a href="#cb18-14"></a>kegg.pathways <span class="ot">=</span> pig.pathways <span class="sc">%&gt;%</span> </span>
<span id="cb18-15"><a href="#cb18-15"></a>  <span class="fu">left_join</span>(., pathway.compounds, <span class="at">by =</span> <span class="st">"name"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb18-16"><a href="#cb18-16"></a>  <span class="fu">filter</span>(<span class="fu">str_starts</span>(name, <span class="st">"01|00"</span>), </span>
<span id="cb18-17"><a href="#cb18-17"></a>         name <span class="sc">!=</span> <span class="st">"01100"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section class="slide level2">

<p>Other organisms, and a table to convert IDs:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1"></a><span class="co"># Human KEGG pathways</span></span>
<span id="cb19-2"><a href="#cb19-2"></a>human.pathways <span class="ot">=</span> <span class="fu">keggList</span>(<span class="st">"pathway"</span>, <span class="st">"hsa"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb19-3"><a href="#cb19-3"></a>  <span class="fu">enframe</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb19-4"><a href="#cb19-4"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_remove</span>(name,<span class="st">"^hsa"</span>), </span>
<span id="cb19-5"><a href="#cb19-5"></a>         <span class="at">value =</span> <span class="fu">str_remove</span>(value, <span class="st">" </span><span class="sc">\\</span><span class="st">- Homo sapiens </span><span class="sc">\\</span><span class="st">(human</span><span class="sc">\\</span><span class="st">)"</span>))</span>
<span id="cb19-6"><a href="#cb19-6"></a></span>
<span id="cb19-7"><a href="#cb19-7"></a><span class="co"># All KEGG pathways</span></span>
<span id="cb19-8"><a href="#cb19-8"></a>all.pathways <span class="ot">=</span> <span class="fu">keggList</span>(<span class="st">"pathway"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb19-9"><a href="#cb19-9"></a>  <span class="fu">enframe</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb19-10"><a href="#cb19-10"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_remove</span>(name,<span class="st">"^map"</span>))</span>
<span id="cb19-11"><a href="#cb19-11"></a></span>
<span id="cb19-12"><a href="#cb19-12"></a><span class="co"># We can also use the KEGG API to extract the ChEBI IDs for compounds</span></span>
<span id="cb19-13"><a href="#cb19-13"></a>chebi.to.kegg <span class="ot">=</span> <span class="fu">keggConv</span>(<span class="st">"cpd"</span>, <span class="st">"chebi"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb19-14"><a href="#cb19-14"></a>  <span class="fu">enframe</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb19-15"><a href="#cb19-15"></a>  <span class="fu">mutate</span>(<span class="at">name =</span> <span class="fu">str_remove</span>(name,<span class="st">"^chebi:"</span>), </span>
<span id="cb19-16"><a href="#cb19-16"></a>         <span class="at">value =</span> <span class="fu">str_remove</span>(value, <span class="st">"^cpd:"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb19-17"><a href="#cb19-17"></a>  <span class="fu">rename</span>(<span class="st">"ChEBI"</span> <span class="ot">=</span> <span class="st">"name"</span>, <span class="st">"KEGG"</span> <span class="ot">=</span> <span class="st">"value"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb19-18"><a href="#cb19-18"></a>  <span class="fu">unique</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="reactome" class="slide level2">
<h2>Reactome</h2>
<p>Reactome pathways can be downloaded <a href="https://reactome.org/download-data">here</a>, in <code>Physical Entity (PE) Identifier mapping files</code> &gt; <code>ChEBI to pathways</code>.</p>
<p>Reactome has many more pathways than KEGG meaning it is harder to enrich them. The database contains many pathways not pertaining to metabolism meaning we would need to filter them (manually?).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a><span class="co"># Create colnames</span></span>
<span id="cb20-2"><a href="#cb20-2"></a>reactome.colnames <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ChEBI"</span>, <span class="st">"RID"</span>, <span class="st">"URL"</span>, <span class="st">"PathwayName"</span>, <span class="st">"EvidenceCode"</span>, <span class="st">"Species"</span>)</span>
<span id="cb20-3"><a href="#cb20-3"></a></span>
<span id="cb20-4"><a href="#cb20-4"></a><span class="co"># Lowest level pathways</span></span>
<span id="cb20-5"><a href="#cb20-5"></a>reactome.pathways <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/ChEBI2Reactome_Pathway.txt"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>,<span class="at">comment.char =</span> <span class="st">""</span>, <span class="at">quote=</span><span class="st">""</span>, <span class="at">col.names =</span> reactome.colnames) <span class="sc">%&gt;%</span> </span>
<span id="cb20-6"><a href="#cb20-6"></a>  <span class="fu">filter</span>(Species <span class="sc">==</span> <span class="st">"Sus scrofa"</span>) <span class="sc">%&gt;%</span> <span class="co"># Filter only pig pathways</span></span>
<span id="cb20-7"><a href="#cb20-7"></a>  <span class="fu">select</span>(ChEBI, PathwayName)<span class="sc">%&gt;%</span> <span class="co"># Select relevant columns</span></span>
<span id="cb20-8"><a href="#cb20-8"></a>  <span class="fu">relocate</span>(PathwayName) <span class="sc">%&gt;%</span> <span class="co"># Reorder</span></span>
<span id="cb20-9"><a href="#cb20-9"></a>  <span class="fu">mutate</span>(<span class="at">ChEBI =</span> <span class="fu">as.character</span>(ChEBI)) <span class="sc">%&gt;%</span> <span class="co"># Fix ChEBI IDs being read as numbers</span></span>
<span id="cb20-10"><a href="#cb20-10"></a>  <span class="fu">left_join</span>(., chebi.to.kegg, <span class="at">by =</span> <span class="st">"ChEBI"</span>) <span class="sc">%&gt;%</span> <span class="co"># Join with ChEBI to KEGG df to get KEGG IDs</span></span>
<span id="cb20-11"><a href="#cb20-11"></a>  <span class="fu">select</span>(<span class="sc">-</span>ChEBI) <span class="sc">%&gt;%</span> <span class="co"># Remove ChEBI column</span></span>
<span id="cb20-12"><a href="#cb20-12"></a>  <span class="fu">na.omit</span>()</span>
<span id="cb20-13"><a href="#cb20-13"></a></span>
<span id="cb20-14"><a href="#cb20-14"></a><span class="co"># All level pathways</span></span>
<span id="cb20-15"><a href="#cb20-15"></a>reactome.all.pathways <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/ChEBI2Reactome_All_Levels.txt"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>,<span class="at">comment.char =</span> <span class="st">""</span>, <span class="at">quote=</span><span class="st">""</span>, <span class="at">col.names =</span> reactome.colnames) <span class="sc">%&gt;%</span> </span>
<span id="cb20-16"><a href="#cb20-16"></a>  <span class="fu">filter</span>(Species <span class="sc">==</span> <span class="st">"Sus scrofa"</span>) <span class="sc">%&gt;%</span> <span class="co"># Filter only pig pathways</span></span>
<span id="cb20-17"><a href="#cb20-17"></a>  <span class="fu">select</span>(ChEBI, PathwayName)<span class="sc">%&gt;%</span> <span class="co"># Select relevant columns</span></span>
<span id="cb20-18"><a href="#cb20-18"></a>  <span class="fu">relocate</span>(PathwayName) <span class="sc">%&gt;%</span> <span class="co"># Reorder</span></span>
<span id="cb20-19"><a href="#cb20-19"></a>  <span class="fu">mutate</span>(<span class="at">ChEBI =</span> <span class="fu">as.character</span>(ChEBI)) <span class="sc">%&gt;%</span> <span class="co"># Fix ChEBI IDs being read as numbers</span></span>
<span id="cb20-20"><a href="#cb20-20"></a>  <span class="fu">left_join</span>(., chebi.to.kegg, <span class="at">by =</span> <span class="st">"ChEBI"</span>) <span class="sc">%&gt;%</span> <span class="co"># Join with ChEBI to KEGG df to get KEGG IDs</span></span>
<span id="cb20-21"><a href="#cb20-21"></a>  <span class="fu">select</span>(<span class="sc">-</span>ChEBI) <span class="sc">%&gt;%</span> <span class="co"># Remove ChEBI column</span></span>
<span id="cb20-22"><a href="#cb20-22"></a>  <span class="fu">na.omit</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="metabolic-network" class="slide level2">
<h2>Metabolic network</h2>
<p>Metabolic networks can be used for various flux-based simulations of metabolism, but they are also databases of knowledge. We can use pathway-metabolite information extracted from a network in the same way as any other pathway set.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1"></a><span class="co"># Metabolites have network-specific IDs, this is a mapped ID to KEGG and ChEBI table</span></span>
<span id="cb21-2"><a href="#cb21-2"></a>metab.network.ids <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/metabolite_ids.tsv"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">header =</span> T, <span class="at">quote =</span> <span class="st">""</span>)</span>
<span id="cb21-3"><a href="#cb21-3"></a><span class="co"># The original file contains nan instead of NA, so replace them:</span></span>
<span id="cb21-4"><a href="#cb21-4"></a>metab.network.ids<span class="sc">$</span>kegg[metab.network.ids<span class="sc">$</span>kegg <span class="sc">==</span> <span class="st">"nan"</span>] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb21-5"><a href="#cb21-5"></a>metab.network.ids<span class="sc">$</span>chebi[metab.network.ids<span class="sc">$</span>chebi <span class="sc">==</span> <span class="st">"nan"</span>] <span class="ot">=</span> <span class="cn">NA</span></span>
<span id="cb21-6"><a href="#cb21-6"></a></span>
<span id="cb21-7"><a href="#cb21-7"></a><span class="co"># Create a filter to filter out irrelevant pathways (specific to flux simulation)</span></span>
<span id="cb21-8"><a href="#cb21-8"></a>metab.network.filter <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"Artificial reactions"</span>, <span class="st">"Isolated"</span>, <span class="st">"Miscellaneous"</span>, <span class="st">"Pool reactions"</span>, <span class="st">"Exchange/demand reactions"</span>, <span class="st">"Transport reactions"</span>)</span>
<span id="cb21-9"><a href="#cb21-9"></a></span>
<span id="cb21-10"><a href="#cb21-10"></a><span class="co"># Read in the pathway data, map to KEGG and ChEBI, and filter</span></span>
<span id="cb21-11"><a href="#cb21-11"></a>metab.network.pathways <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/metabolite_pathways.tsv"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">header =</span> T) <span class="sc">%&gt;%</span> </span>
<span id="cb21-12"><a href="#cb21-12"></a>  <span class="fu">unique</span>() <span class="sc">%&gt;%</span></span>
<span id="cb21-13"><a href="#cb21-13"></a>  <span class="fu">left_join</span>(., metab.network.ids, <span class="at">by =</span> <span class="st">"metabolite"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb21-14"><a href="#cb21-14"></a>  <span class="fu">filter</span>(<span class="sc">!</span>(subsystem <span class="sc">%in%</span> metab.network.filter))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="cpdb" class="slide level2">
<h2>CPDB</h2>
<p>ConsensusPathDB is a compiled database of various pathway sets, including Reactome, KEGG, BioCyc and others. We are using it as an example of a very large pathway set database.</p>
<p>The database can be downloaded from <a href="http://cpdb.molgen.mpg.de/" class="uri">http://cpdb.molgen.mpg.de/</a>: &gt; “Biological pathways (as defined by source databases) with their metabolites identified with KEGG / ChEBI / Pubchem-compound accession numbers”</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1"></a>cpdb <span class="ot">=</span> <span class="fu">read.table</span>(<span class="st">"data/CPDB_pathways_metabolites.tab"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">comment.char =</span> <span class="st">""</span>, <span class="at">header =</span> T) <span class="sc">%&gt;%</span></span>
<span id="cb22-2"><a href="#cb22-2"></a>  <span class="fu">cSplit</span>(<span class="st">"metabolites"</span>, <span class="st">","</span>,<span class="at">direction =</span> <span class="st">"long"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb22-3"><a href="#cb22-3"></a>  <span class="fu">unique</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb22-4"><a href="#cb22-4"></a>  <span class="fu">mutate</span>(<span class="at">metabolites =</span> <span class="fu">str_remove</span>(metabolites, <span class="st">"^kegg:"</span>),</span>
<span id="cb22-5"><a href="#cb22-5"></a>         <span class="at">pathway =</span> <span class="fu">str_remove</span>(pathway, <span class="st">" - Homo sapiens </span><span class="sc">\\</span><span class="st">(human</span><span class="sc">\\</span><span class="st">)$"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section></section>
<section>
<section id="ora" class="title-slide slide level1 center">
<h1>ORA</h1>
<p>Cluster profiler is normally used with gene enrichment and therefore has direct access to gene pathway set databases. For metabolites we will need to specify a custom pathway set.</p>
<p>Important parameters:</p>
<ul>
<li><p><code>gene</code>: a vector of IDs = pre-filtered metabolite IDs</p></li>
<li><p><code>TERM2GENE</code>: a two column data.frame of pathway names and metabolite IDs = custom pathway set</p></li>
<li><p><code>universe</code>: a vector of IDs = background dataset</p></li>
<li><p><code>pAdjustMethod</code>: “BH” by default, one of “holm”, “hochberg”, “hommel”, “bonferroni”, “BH”, “BY”, “fdr”, “none”</p></li>
<li><p><code>pvalueCutoff</code> and <code>qvalueCutoff</code>: default = 0.05 and 0.2 respectively</p></li>
</ul>
</section>
<section id="basic-ora" class="slide level2">
<h2>Basic ORA</h2>
<p>We will use the KEGG IDs in the input data and perform ORA using the KEGG pathway sets. By default, the background set is the entire list of metabolites in the pathway set.</p>
<h3 id="geno-mother-dataset">Geno mother dataset</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a>ora.geno.mother.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name))</span>
<span id="cb23-2"><a href="#cb23-2"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-14-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="gestation-data-set">Gestation data set</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1"></a>ora.gestation.kegg <span class="ot">=</span> <span class="fu">enricher</span>(gestation.raw<span class="sc">$</span>KEGG, <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name))</span>
<span id="cb24-2"><a href="#cb24-2"></a><span class="fu">dotplot</span>(ora.gestation.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-15-1.png" width="960" class="r-stretch"></section>
<section id="varying-background-set" class="slide level2">
<h2>Varying background set</h2>
<p>Changing the background set will completely change the resulting pathway significance, usually reducing the number of significant pathways. In metabolomics, we should use the set of measurable metabolites during the experiment. Depending on if it was targeted or untargeted, this set may be larger or smaller.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a><span class="co"># Using the background set</span></span>
<span id="cb25-2"><a href="#cb25-2"></a>ora.geno.mother.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb25-3"><a href="#cb25-3"></a>                                <span class="at">universe =</span> background <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG))</span>
<span id="cb25-4"><a href="#cb25-4"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-16-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>Using a restrictive background set means that we get fewer enriched pathways. However, biologically and statistically we must use a background set reflective of what could have been measured.</p>

<img data-src="images/ora.PNG" width="333" class="r-stretch quarto-figure-center"><p>N = background set</p>
<p>n = input metabolites of interest</p>
<p>M = metabolites in pathway being tested against</p>
<p>k = overlap between n and M</p>
</section>
<section class="slide level2">

<h3 id="using-the-annotation-set-as-an-example">Using the annotation set as an example</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1"></a>ora.geno.mother.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb26-2"><a href="#cb26-2"></a>                                <span class="at">universe =</span> annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG))</span>
<span id="cb26-3"><a href="#cb26-3"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-17-1.png" width="960" class="r-stretch"></section>
<section id="varying-pathway-database" class="slide level2">
<h2>Varying pathway database</h2>
<p>Instead of KEGG pathways, we can use the metabolic network pathways or Reactome pathways.</p>
<h3 id="metabolic-network-1">Metabolic network</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1"></a>ora.geno.mother.metnet <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> metab.network.pathways <span class="sc">%&gt;%</span> </span>
<span id="cb27-2"><a href="#cb27-2"></a>                                    <span class="fu">select</span>(subsystem, kegg) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>(), </span>
<span id="cb27-3"><a href="#cb27-3"></a>                                  <span class="at">universe =</span> annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG))</span>
<span id="cb27-4"><a href="#cb27-4"></a><span class="fu">dotplot</span>(ora.geno.mother.metnet)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-18-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>Using the second larger dataset, it is harder to gain significantly enriched pathways. Using the metabolic network pathways, we are unable to get any enriched pathways even with no p-value cutoff. Only by removing the background set are we able to get enrichment.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1"></a>ora.gestation.metnet <span class="ot">=</span> <span class="fu">enricher</span>(gestation.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> metab.network.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(subsystem, kegg) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>())</span>
<span id="cb28-2"><a href="#cb28-2"></a><span class="fu">dotplot</span>(ora.gestation.metnet)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-19-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="reactome-1">Reactome</h3>
<p>Reactome is a larger pathway set, meaning we need to use no background set (!) to get significant pathways.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1"></a>ora.gestation.reactome <span class="ot">=</span> <span class="fu">enricher</span>(gestation.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> reactome.pathways)</span>
<span id="cb29-2"><a href="#cb29-2"></a><span class="fu">dotplot</span>(ora.gestation.reactome)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-20-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>As you can see, even though the p-values are low, having only one metabolite enrich each pathway is not necessarily relevant.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1"></a>ora.geno.mother.reactome <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> reactome.pathways,</span>
<span id="cb30-2"><a href="#cb30-2"></a>                                    <span class="at">pvalueCutoff =</span> <span class="fl">0.1</span>)</span>
<span id="cb30-3"><a href="#cb30-3"></a><span class="fu">dotplot</span>(ora.geno.mother.reactome)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-21-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="cpdb-1">CPDB</h3>
<p>Here is an example of using a non filtered pathway set database. We obtain many non-metabolic pathways that are enriched but are meaningless in terms of our input data.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1"></a>ora.gestation.cpdb <span class="ot">=</span> <span class="fu">enricher</span>(gestation.raw<span class="sc">$</span>KEGG, </span>
<span id="cb31-2"><a href="#cb31-2"></a>                              <span class="at">TERM2GENE =</span> cpdb <span class="sc">%&gt;%</span> <span class="fu">filter</span>(source <span class="sc">==</span> <span class="st">"Reactome"</span>) <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>source))</span>
<span id="cb31-3"><a href="#cb31-3"></a><span class="fu">dotplot</span>(ora.gestation.cpdb)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-22-1.png" width="960" class="r-stretch"></section>
<section id="varying-significant-metabolites" class="slide level2">
<h2>Varying significant metabolites</h2>
<p>The initial filtering of measured metabolites can also change the resulting enrichment. Here we can use a strict filtering of the fold changes in the initial dataset. Since it is stricter, we can’t use a background set in this example since we wouldn’t get any results.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1"></a>gestation.filtered <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>geno.mother) <span class="sc">%&gt;%</span></span>
<span id="cb32-2"><a href="#cb32-2"></a>  <span class="fu">filter</span>(<span class="fu">abs</span>(gestation) <span class="sc">&gt;</span> <span class="dv">1</span>)</span>
<span id="cb32-3"><a href="#cb32-3"></a>ora.gestation.filtered.kegg <span class="ot">=</span> <span class="fu">enricher</span>(gestation.filtered<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name))</span>
<span id="cb32-4"><a href="#cb32-4"></a><span class="fu">dotplot</span>(ora.gestation.filtered.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-23-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>We can also use a more relaxed threshold on this dataset’s fold changes, again without a background set for illustration purposes.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1"></a>gestation.filtered <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>geno.mother) <span class="sc">%&gt;%</span></span>
<span id="cb33-2"><a href="#cb33-2"></a>  <span class="fu">filter</span>(<span class="fu">abs</span>(gestation) <span class="sc">&gt;</span> <span class="fl">0.1</span>)</span>
<span id="cb33-3"><a href="#cb33-3"></a>ora.gestation.filtered.kegg <span class="ot">=</span> <span class="fu">enricher</span>(gestation.filtered<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name))</span>
<span id="cb33-4"><a href="#cb33-4"></a><span class="fu">dotplot</span>(ora.gestation.filtered.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-24-1.png" width="960" class="r-stretch"></section>
<section id="varying-mingssize" class="slide level2">
<h2>Varying minGSSize</h2>
<p>By default, ORA filters out pathways smaller than 10 metabolites when enriching. This parameter can be varied depending on the pathway sets: if you know that they are smaller in general, lowering this threshold may be more relevant.</p>
<p>A lower minGSSize means more pathways to test against, which usually means higher p-values but more possibilities for small pathways to be enriched.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1"></a>pathway.sizes <span class="ot">=</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">group_by</span>(value.x) <span class="sc">%&gt;%</span> <span class="fu">summarise</span>(<span class="at">count =</span> <span class="fu">n</span>()) </span>
<span id="cb34-2"><a href="#cb34-2"></a>pathway.counts <span class="ot">=</span> pathway.sizes <span class="sc">%&gt;%</span> <span class="fu">pull</span>(count)</span>
<span id="cb34-3"><a href="#cb34-3"></a><span class="fu">hist</span>(pathway.counts, <span class="at">breaks =</span> <span class="dv">50</span>)</span>
<span id="cb34-4"><a href="#cb34-4"></a><span class="fu">abline</span>(<span class="at">v=</span><span class="dv">10</span>,<span class="at">col=</span><span class="st">"red"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-25-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1"></a>ora.gestation.filtered.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , </span>
<span id="cb35-2"><a href="#cb35-2"></a>                                       <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name), </span>
<span id="cb35-3"><a href="#cb35-3"></a>                                       <span class="at">minGSSize =</span> <span class="dv">10</span>, </span>
<span id="cb35-4"><a href="#cb35-4"></a>                                       <span class="at">universe =</span> annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG), <span class="at">pvalueCutoff =</span> <span class="fl">0.1</span>)</span>
<span id="cb35-5"><a href="#cb35-5"></a><span class="fu">dotplot</span>(ora.gestation.filtered.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-26-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>In this case, by setting <code>minGSSize</code> to 3, we gain some new enriched pathways at the cost of higher adjusted p-values. These pathways weren’t enriched before because there were less total pathways, so their enrichments weren’t as significant. We most likely added many small pathways with no enrichment by lowering the <code>minGSSize</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1"></a>ora.gestation.filtered.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , </span>
<span id="cb36-2"><a href="#cb36-2"></a>                                       <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name), </span>
<span id="cb36-3"><a href="#cb36-3"></a>                                       <span class="at">minGSSize =</span> <span class="dv">3</span>, </span>
<span id="cb36-4"><a href="#cb36-4"></a>                                       <span class="at">universe =</span>  annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG), <span class="at">pvalueCutoff =</span> <span class="fl">0.1</span>)</span>
<span id="cb36-5"><a href="#cb36-5"></a><span class="fu">dotplot</span>(ora.gestation.filtered.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-27-1.png" width="960" class="r-stretch"></section>
<section id="bonus-varying-p-value-cutoff" class="slide level2">
<h2>Bonus: varying p-value cutoff</h2>
<p>We can also vary the p-value and p-value adjust cutoff to determine what is and isn’t a significantly enriched pathway. This must be done with caution as a high cutoff will provide many false positives.</p>
<p>We can use the previous example with a <code>pvalueCutoff</code> of 0.05 (the default).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1"></a>ora.gestation.filtered.kegg <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw<span class="sc">$</span>KEGG , </span>
<span id="cb37-2"><a href="#cb37-2"></a>                                       <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name), </span>
<span id="cb37-3"><a href="#cb37-3"></a>                                       <span class="at">minGSSize =</span> <span class="dv">3</span>, </span>
<span id="cb37-4"><a href="#cb37-4"></a>                                       <span class="at">universe =</span>  annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG), <span class="at">pvalueCutoff =</span> <span class="fl">0.05</span>)</span>
<span id="cb37-5"><a href="#cb37-5"></a><span class="fu">dotplot</span>(ora.gestation.filtered.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-28-1.png" width="960" class="r-stretch"></section>
<section id="ora-on-up-vs-down-regulated-in-dataset" class="slide level2">
<h2>ORA on up vs down regulated in dataset</h2>
<p>In addition to using the entire filtered fold changes, we can split the dataset into up and down regulated metabolites.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1"></a>geno.mother.raw.up <span class="ot">=</span> geno.mother.raw <span class="sc">%&gt;%</span> <span class="fu">filter</span>(log2FC_mg <span class="sc">&gt;</span> <span class="dv">0</span>)</span>
<span id="cb38-2"><a href="#cb38-2"></a>geno.mother.raw.down <span class="ot">=</span> geno.mother.raw <span class="sc">%&gt;%</span> <span class="fu">filter</span>(log2FC_mg <span class="sc">&lt;</span> <span class="dv">0</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Using a higher <code>pvalueCutoff</code> to show enriched pathways: again, because we are being stringent with the background set and by splitting the input set into two, we can be less strict with the p.adjust.</p>
<h3 id="upregulated-metabolites">Upregulated metabolites</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1"></a>ora.geno.mother.kegg.up <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw.up<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb39-2"><a href="#cb39-2"></a>                                <span class="at">universe =</span> annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG), <span class="at">pvalueCutoff =</span> <span class="fl">0.2</span>, <span class="at">minGSSize =</span> <span class="dv">3</span>)</span>
<span id="cb39-3"><a href="#cb39-3"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg.up)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-30-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>We can plot using the p-value as a the colour to show that these unadjusted p-values are still significant.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg.up, <span class="at">color =</span> <span class="st">"pvalue"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-31-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="downregulated-metabolites">Downregulated metabolites:</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1"></a>ora.geno.mother.kegg.down <span class="ot">=</span> <span class="fu">enricher</span>(geno.mother.raw.down<span class="sc">$</span>KEGG , <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb41-2"><a href="#cb41-2"></a>                                <span class="at">universe =</span> annot.db <span class="sc">%&gt;%</span> <span class="fu">pull</span>(KEGG), <span class="at">pvalueCutoff =</span> <span class="fl">0.1</span>, <span class="at">minGSSize =</span> <span class="dv">3</span>)</span>
<span id="cb41-3"><a href="#cb41-3"></a><span class="fu">dotplot</span>(ora.geno.mother.kegg.down)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-32-1.png" width="960" class="r-stretch"><p>This results in pathways that we can define as up or down regulated in addition to being enriched.</p>
</section></section>
<section>
<section id="gsea" class="title-slide slide level1 center">
<h1>GSEA</h1>
<p>In this section, we remove set multiple correction to <code>"none"</code> as GSEA will not return anything significant with a small input dataset. This is illustrated using the gseaplots.</p>
</section>
<section id="gestation-data-set-1" class="slide level2">
<h2>Gestation data set</h2>
<h3 id="using-fold-changes">Using fold changes</h3>
<p>For illustrative purposes, we set the <code>pvalueCutoff</code> to be 1 to show all pathway p-values.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb42"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1"></a>gsea.input.gestation <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span> </span>
<span id="cb42-2"><a href="#cb42-2"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(gestation)) <span class="sc">%&gt;%</span> </span>
<span id="cb42-3"><a href="#cb42-3"></a>  <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span></span>
<span id="cb42-4"><a href="#cb42-4"></a>  <span class="fu">pull</span>(gestation,KEGG) </span>
<span id="cb42-5"><a href="#cb42-5"></a></span>
<span id="cb42-6"><a href="#cb42-6"></a>gsea.gestation.kegg <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.gestation, </span>
<span id="cb42-7"><a href="#cb42-7"></a>                           <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb42-8"><a href="#cb42-8"></a>                           <span class="at">minGSSize =</span> <span class="dv">3</span>, <span class="at">pvalueCutoff =</span> <span class="dv">1</span>,</span>
<span id="cb42-9"><a href="#cb42-9"></a>                           <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>)</span>
<span id="cb42-10"><a href="#cb42-10"></a><span class="fu">tibble</span>(gsea.gestation.kegg<span class="sc">@</span>result)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 27 × 11
   ID     Description setSize enrichmentScore   NES pvalue p.adjust qvalue  rank
   &lt;chr&gt;  &lt;chr&gt;         &lt;int&gt;           &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt;    &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt;
 1 Purin… Purine met…       7          -0.731 -1.32  0.117    0.117  0.885     9
 2 Nucle… Nucleotide…       6          -0.740 -1.29  0.148    0.148  0.885     9
 3 Cyste… Cysteine a…       5           0.652  1.25  0.256    0.256  0.885     1
 4 Tyros… Tyrosine m…       3           0.726  1.20  0.293    0.293  0.885     6
 5 Pheny… Phenylalan…       4           0.670  1.18  0.321    0.321  0.885    10
 6 Argin… Arginine b…       3          -0.732 -1.12  0.370    0.370  0.885    15
 7 Valin… Valine, le…       3           0.683  1.13  0.372    0.372  0.885    16
 8 Valin… Valine, le…       3           0.683  1.13  0.372    0.372  0.885    16
 9 Sulfu… Sulfur met…       4           0.632  1.11  0.377    0.377  0.885     6
10 2-Oxo… 2-Oxocarbo…       8           0.467  1.04  0.407    0.407  0.885    11
# ℹ 17 more rows
# ℹ 2 more variables: leading_edge &lt;chr&gt;, core_enrichment &lt;chr&gt;</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<p>This gseaplot shows that even if Purine metabolism seems to be enriched, with so few metabolites it doesn’t reach a high enough enrichment score and p-value to be classed as significant when correcting the p-value. Even the raw p-value is fairly high at 0.169.</p>
<p>The gseaplot also reveals that we have a split of strong fold changed metabolites: 1 up and 4 down.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb44"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.gestation.kegg, <span class="at">geneSetID =</span> <span class="dv">1</span>, <span class="at">title =</span> gsea.gestation.kegg<span class="sc">$</span>Description[<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-34-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>This dotplot uses the unadjusted pvalue as the colour legend. Only the very red pathways could be considered as enriched.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1"></a><span class="fu">dotplot</span>(gsea.gestation.kegg, <span class="at">color =</span> <span class="st">"pvalue"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-35-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="using-absolute-fold-changes">Using absolute fold changes</h3>
<p>In genomics, when genes are upregulated together this usually means that the enriched pathway is upregulated. With metabolites, pathways can be enriched regardless of an increase or a decrease in metabolite concentration: this means we can look for enriched pathways that are affected using the metabolites, without being able to conclude on its up or down regulation.</p>
<p>To do this, we use the absolute value of the fold changes instead of the raw fold changes. We also need to add a parameter to GSEA: <code>scoreType = "pos"</code>, to do a one tailed test, since all our values will be positive.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1"></a>gsea.input.gestation <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span></span>
<span id="cb46-2"><a href="#cb46-2"></a>  <span class="fu">mutate</span>(<span class="at">gestation =</span> <span class="fu">abs</span>(gestation)) <span class="sc">%&gt;%</span> </span>
<span id="cb46-3"><a href="#cb46-3"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(gestation)) <span class="sc">%&gt;%</span> </span>
<span id="cb46-4"><a href="#cb46-4"></a>  <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span></span>
<span id="cb46-5"><a href="#cb46-5"></a>  <span class="fu">pull</span>(gestation,KEGG) </span>
<span id="cb46-6"><a href="#cb46-6"></a></span>
<span id="cb46-7"><a href="#cb46-7"></a>gsea.gestation.kegg <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.gestation, </span>
<span id="cb46-8"><a href="#cb46-8"></a>                           <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb46-9"><a href="#cb46-9"></a>                           <span class="at">minGSSize =</span> <span class="dv">3</span>, <span class="at">pvalueCutoff =</span> <span class="dv">1</span>,</span>
<span id="cb46-10"><a href="#cb46-10"></a>                           <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>, </span>
<span id="cb46-11"><a href="#cb46-11"></a>                           <span class="at">scoreType=</span><span class="st">"pos"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.gestation.kegg, <span class="at">geneSetID =</span> <span class="dv">1</span>, <span class="at">title =</span> gsea.gestation.kegg<span class="sc">$</span>Description[<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-37-1.png" width="960" class="r-stretch"><p>In the gseaplot, we can see that we have 5 highly ranked metabolites for Purine metabolism, instead of 1 up and 4 down as seen in the previous gseaplot. This groups together the metabolites affected by the condition in order to better enrich affected pathways.</p>
</section>
<section id="geno-mother-data-set" class="slide level2">
<h2>Geno mother data set</h2>
<p>We can do the same analyses with the second data set.</p>
<h3 id="using-fold-changes-1">Using fold changes</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb48"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1"></a>gsea.input.geno <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span></span>
<span id="cb48-2"><a href="#cb48-2"></a>  <span class="fu">mutate</span>(<span class="at">geno.mother =</span> geno.mother) <span class="sc">%&gt;%</span> </span>
<span id="cb48-3"><a href="#cb48-3"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(geno.mother))<span class="sc">%&gt;%</span> <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span></span>
<span id="cb48-4"><a href="#cb48-4"></a>  <span class="fu">pull</span>(geno.mother,KEGG) </span>
<span id="cb48-5"><a href="#cb48-5"></a></span>
<span id="cb48-6"><a href="#cb48-6"></a>gsea.geno.kegg <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.geno, </span>
<span id="cb48-7"><a href="#cb48-7"></a>                      <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb48-8"><a href="#cb48-8"></a>                      <span class="at">minGSSize =</span> <span class="dv">3</span>, </span>
<span id="cb48-9"><a href="#cb48-9"></a>                      <span class="at">pvalueCutoff =</span> <span class="dv">1</span>, </span>
<span id="cb48-10"><a href="#cb48-10"></a>                      <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>)</span>
<span id="cb48-11"><a href="#cb48-11"></a><span class="fu">tibble</span>(gsea.geno.kegg<span class="sc">@</span>result)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 27 × 11
   ID     Description setSize enrichmentScore   NES pvalue p.adjust qvalue  rank
   &lt;chr&gt;  &lt;chr&gt;         &lt;int&gt;           &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt;    &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt;
 1 Biosy… Biosynthes…      14          -0.718 -1.65 0.0119   0.0119  0.321    12
 2 Thiam… Thiamine m…       3          -0.902 -1.43 0.0513   0.0513  0.520     8
 3 Cyste… Cysteine a…       5          -0.808 -1.45 0.0604   0.0604  0.520     3
 4 Gluta… Glutathion…       5          -0.783 -1.41 0.0771   0.0771  0.520    12
 5 Carbo… Carbon met…       9          -0.587 -1.22 0.218    0.218   0.964    13
 6 Glyce… Glyceropho…       5           0.688  1.20 0.294    0.294   0.964    15
 7 Oxida… Oxidative …       3          -0.749 -1.19 0.310    0.310   0.964     8
 8 Nucle… Nucleotide…       6           0.610  1.11 0.403    0.403   0.964     4
 9 Purin… Purine met…       7           0.571  1.08 0.423    0.423   0.964     4
10 Argin… Arginine b…       3           0.671  1.08 0.434    0.434   0.964     3
# ℹ 17 more rows
# ℹ 2 more variables: leading_edge &lt;chr&gt;, core_enrichment &lt;chr&gt;</code></pre>
</div>
</div>
</section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb50"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1"></a><span class="fu">dotplot</span>(gsea.geno.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-39-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<p>The most enriched pathway for this dataset is Biosynthesis of cofactors. In the gseaplot, many metabolites have a strong negative fold change.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.geno.kegg, <span class="at">geneSetID =</span> <span class="dv">1</span>, <span class="at">title =</span> gsea.geno.kegg<span class="sc">$</span>Description[<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-40-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<h3 id="using-absolute-fold-changes-1">Using absolute fold changes</h3>
<p>By using the absolute values of fold changes, we can reveal pathways that weren’t enriched before simply because their metabolites were split over up and down regulated metabolites.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb52"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1"></a>gsea.input.geno.abs <span class="ot">=</span> all.fc <span class="sc">%&gt;%</span></span>
<span id="cb52-2"><a href="#cb52-2"></a>  <span class="fu">mutate</span>(<span class="at">geno.mother =</span> <span class="fu">abs</span>(geno.mother)) <span class="sc">%&gt;%</span> </span>
<span id="cb52-3"><a href="#cb52-3"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(geno.mother)) <span class="sc">%&gt;%</span> </span>
<span id="cb52-4"><a href="#cb52-4"></a>  <span class="fu">na.omit</span>() <span class="sc">%&gt;%</span></span>
<span id="cb52-5"><a href="#cb52-5"></a>  <span class="fu">pull</span>(geno.mother,KEGG) </span>
<span id="cb52-6"><a href="#cb52-6"></a></span>
<span id="cb52-7"><a href="#cb52-7"></a>gsea.geno.kegg.abs <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.geno.abs, </span>
<span id="cb52-8"><a href="#cb52-8"></a>                           <span class="at">TERM2GENE =</span> kegg.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>name),</span>
<span id="cb52-9"><a href="#cb52-9"></a>                           <span class="at">minGSSize =</span> <span class="dv">3</span>, <span class="at">pvalueCutoff =</span> <span class="dv">1</span>,</span>
<span id="cb52-10"><a href="#cb52-10"></a>                           <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>, </span>
<span id="cb52-11"><a href="#cb52-11"></a>                           <span class="at">scoreType=</span><span class="st">"pos"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.geno.kegg, <span class="at">geneSetID =</span> <span class="dv">9</span>, <span class="at">title =</span> gsea.geno.kegg<span class="sc">$</span>Description[<span class="dv">9</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img data-src="index_files/figure-revealjs/unnamed-chunk-42-1.png" width="960"></p>
</div>
<div class="sourceCode cell-code" id="cb54"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.geno.kegg.abs, <span class="at">geneSetID =</span> <span class="dv">1</span>, <span class="at">title =</span> gsea.geno.kegg.abs<span class="sc">$</span>Description[<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img data-src="index_files/figure-revealjs/unnamed-chunk-42-2.png" width="960"></p>
</div>
</div>
</section>
<section id="metabolic-network-pathways-instead-of-kegg-pathways" class="slide level2">
<h2>Metabolic network pathways instead of KEGG pathways</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1"></a>gsea.gestation.network <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.gestation, </span>
<span id="cb55-2"><a href="#cb55-2"></a>                      <span class="at">TERM2GENE =</span> metab.network.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(subsystem, kegg) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>(),</span>
<span id="cb55-3"><a href="#cb55-3"></a>                      <span class="at">minGSSize =</span> <span class="dv">3</span>, <span class="at">pvalueCutoff =</span> <span class="fl">0.1</span>, <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>)</span>
<span id="cb55-4"><a href="#cb55-4"></a><span class="fu">tibble</span>(gsea.gestation.network<span class="sc">@</span>result)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 1 × 11
  ID      Description setSize enrichmentScore   NES pvalue p.adjust qvalue  rank
  &lt;chr&gt;   &lt;chr&gt;         &lt;int&gt;           &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt;    &lt;dbl&gt;  &lt;dbl&gt; &lt;int&gt;
1 Purine… Purine met…      11           0.709  1.31 0.0867   0.0867  0.967    11
# ℹ 2 more variables: leading_edge &lt;chr&gt;, core_enrichment &lt;chr&gt;</code></pre>
</div>
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1"></a><span class="fu">dotplot</span>(gsea.gestation.network)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-43-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb58"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1"></a>enrichplot<span class="sc">::</span><span class="fu">gseaplot2</span>(gsea.gestation.network, <span class="at">geneSetID =</span> <span class="dv">1</span>, <span class="at">title =</span> gsea.gestation.network<span class="sc">$</span>Description[<span class="dv">1</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-44-1.png" width="960" class="r-stretch"></section>
<section class="slide level2">

<div class="cell">
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1"></a>gsea.geno.kegg <span class="ot">=</span> <span class="fu">GSEA</span>(gsea.input.geno, </span>
<span id="cb59-2"><a href="#cb59-2"></a>                      <span class="at">TERM2GENE =</span> metab.network.pathways <span class="sc">%&gt;%</span> <span class="fu">select</span>(subsystem, kegg) <span class="sc">%&gt;%</span> <span class="fu">na.omit</span>(),</span>
<span id="cb59-3"><a href="#cb59-3"></a>                      <span class="at">minGSSize =</span> <span class="dv">3</span>, <span class="at">pvalueCutoff =</span> <span class="fl">0.05</span>, <span class="at">pAdjustMethod =</span> <span class="st">"none"</span>)</span>
<span id="cb59-4"><a href="#cb59-4"></a><span class="fu">tibble</span>(gsea.geno.kegg<span class="sc">@</span>result)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 3 × 11
  ID      Description setSize enrichmentScore   NES pvalue p.adjust qvalue  rank
  &lt;chr&gt;   &lt;chr&gt;         &lt;int&gt;           &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt;    &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt;
1 Glutat… Glutathion…       8          -0.817 -1.67 0.0122   0.0122  0.249    12
2 Ascorb… Ascorbate …       4          -0.9   -1.52 0.0175   0.0175  0.249     9
3 Glycin… Glycine, s…      11          -0.736 -1.60 0.0178   0.0178  0.249    14
# ℹ 2 more variables: leading_edge &lt;chr&gt;, core_enrichment &lt;chr&gt;</code></pre>
</div>
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode numberSource r number-lines code-with-copy"><code class="sourceCode r"><span id="cb61-1"><a href="#cb61-1"></a><span class="fu">dotplot</span>(gsea.geno.kegg)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>

</div>
<img data-src="index_files/figure-revealjs/unnamed-chunk-45-1.png" width="960" class="r-stretch"></section></section>
<section>
<section id="metexplore" class="title-slide slide level1 center">
<h1>MetExplore</h1>
<p>MetExplore can be found <a href="https://metexplore.toulouse.inrae.fr/index.html/">here</a>. Click on START MetExplore to start.</p>
<p>It can be used without an account.</p>
</section>
<section id="biosource" class="slide level2">
<h2>Biosource</h2>
<p>In the Network Data tab, you can browse all available BioSources. BioSources are metabolic networks. We recommend using the Human1 / HumanGEM network as it is the most recent network.</p>
<p>On the right of the website, you can start typing the name of a network and select the network you wish to use. You can also right click in the BioSources table and Select Biosource.</p>
</section>
<section id="data-in-the-network" class="slide level2">
<h2>Data in the network</h2>
<p>The tabs along the top of the window in the Network Data tab show the different types of information included in the network.</p>
<p><img data-src="images/metexplore_network_data.png"></p>
<p>Compartments represent cellular compartments such as cytosol, nucleus…</p>
<p>Pathways are pathway sets defined in the model.</p>
<p>Reactions usually have a name and a GPR (gene protein reaction) relationship, which is a rule defining the activity of a reaction based on the activity of one or more genes.</p>
<p><img data-src="images/metexplore_GPR.png"></p>
<p>Metabolites have a name and an Identifier, either specific to the network or sometimes an external ID (KEGG for example). They also usually have a compartment, meaning there are multiple versions of some metabolites with different compartments.</p>
</section>
<section id="mapping-your-data" class="slide level2">
<h2>Mapping your data</h2>
<p>If you have reaction or metabolite IDs, you can map them to a network, run pathway enrichment and visualise them. This is done by clicking the Omics button &gt; Mapping &gt; From Omics.</p>
<p>Here you can upload a file or copy paste directly from Excel into the grid. In this example, we pasted the KEGG IDs from the geno mother dataset.</p>

<img data-src="images/metexplore_id_pasted.png" class="r-stretch"></section>
<section class="slide level2">

<p>By checking “Taking into account chemical library” you can provide a background set for pathway enrichment. Once you have uploaded or pasted the data, you can select the Feature type depending on the ID you provided (e.g.&nbsp;KEGG). Then, click Map, and it will tell you how many IDs were successfully mapped to the network.</p>

<img data-src="images/metexplore_mapping_results.png" class="r-stretch"></section>
<section id="pathway-enrichment" class="slide level2">
<h2>Pathway enrichment</h2>
<p>When data is mapped, pathway enrichment is automatically calculated on the input data.</p>
<p>You can find the results in Network Data &gt; Pathways as new columns. You can then click on the column names to order by p-value, coverage etc.</p>

<img data-src="images/metexplore_top_PE.png" class="r-stretch"></section>
<section id="visualise-the-selected-network" class="slide level2">
<h2>Visualise the selected network</h2>
<p>By selecting one or more pathways, you can visualise your mapped data among all metabolites of the pathway(s). To do this, select multiple pathways by Shift + clicking or Ctrl + clicking. Once you have your selection, right click &gt; Filter on selection. This will filter the reactions in the network to only show those in the pathways you have filtered.</p>

<img data-src="images/metexplore_filter_on_selection.png" class="r-stretch"></section>
<section class="slide level2">

<h3 id="cart">Cart</h3>
<div class="columns">
<div class="column">
<p>You can then go into the Reactions tab and right click on any reaction &gt; Copy All to cart. The cart on the right of the screen is what contains the reactions you will visualise in the visualisation section.</p>
</div><div class="column">
<p><img data-src="images/metexplore_cart.png"></p>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="create-a-network-viz">Create a network viz</h3>
<p>Right click on any reaction in the cart and click Create network in viz from cart, or go directly to the Network Viz tab and click MetExplore selection.</p>
<p>This should show the selected reactions in the visualisation tab. Here, metabolites are circles and reactions are squares.</p>

<img data-src="images/metexplore_viz_1.png" class="r-stretch quarto-figure-center"></section>
<section class="slide level2">

<h3 id="side-compounds">Side compounds</h3>
<p>Sometimes, there are some areas that are very densely connected. This is due to side compounds being very highly connected in the network. Side compounds are metabolites such as ATP, water, H+ used in many reactions. In MetExplore, the Human1 metabolic network has a predefined list of side compounds. Click on Drawing &gt; Remove side compounds to remove them from the visualisation, and click the Play button to let the layout algorithm calculate a new layout.</p>

<img data-src="images/metexplore_viz_2.png" class="r-stretch quarto-figure-center"></section>
<section class="slide level2">

<h3 id="compartment-colours">Compartment colours</h3>
<p>On the left, we have the Network Manager which helps us change various styles of the network. Click on Compartments and click highlight compartments on links to see the compartments for each reaction.</p>

<img data-src="images/metexplore_viz_compartments.png" class="r-stretch quarto-figure-center"></section>
<section id="visualise-mapped-data" class="slide level2">
<h2>Visualise mapped data</h2>
<p>Go to Styles &gt; Metabolite and click on the middle (Mapping) column of the Node background. Click on Select a condition and choose your mapping (Mapping / undefined). In Type of data, select Identified in mapping. You can change the colour to red for example.</p>
<div class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img data-src="images/metexplore_node_bg_select.png"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img data-src="images/metexplore_node_bg_mapped.png"></p>
</div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="input-metabolites-highlighted-in-red">Input metabolites highlighted in red</h3>

<img data-src="images/metexplore_viz_node_bg.png" class="r-stretch quarto-figure-center"></section>
<section class="slide level2">

<h3 id="changing-widthheight-and-roundness">Changing width/height and roundness</h3>
<p>You can further enhance your visualisation by changing the width and height of mapped metabolites in a similar fashion.</p>
<div class="quarto-layout-panel">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img data-src="images/metexplore_width_height.png"></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: center;">
<p><img data-src="images/metexplore_rx_ry.png"></p>
</div>
</div>
</div>
</section>
<section class="slide level2">

<h3 id="larger-nodes-for-input-metabolites">Larger nodes for input metabolites</h3>

<img data-src="images/metexplore_viz_large.png" class="r-stretch quarto-figure-center"></section>
<section id="other-mapping" class="slide level2">
<h2>Other mapping</h2>
<p>Instead of mapping all of your metabolomics dataset, you could map the leading edge metabolites from GSEA (which is a subset of your data for one pathway).</p>
<p>You can visualise more than one pathway at once, but too many will be very slow to load!</p>
<div class="footer footer-default">

</div>
</section></section>
    </div>
  </div>

  <script>window.backupDefine = window.define; window.define = undefined;</script>
  <script src="index_files/libs/revealjs/dist/reveal.js"></script>
  <!-- reveal.js plugins -->
  <script src="index_files/libs/revealjs/plugin/quarto-line-highlight/line-highlight.js"></script>
  <script src="index_files/libs/revealjs/plugin/pdf-export/pdfexport.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/reveal-menu/quarto-menu.js"></script>
  <script src="index_files/libs/revealjs/plugin/quarto-support/support.js"></script>
  

  <script src="index_files/libs/revealjs/plugin/notes/notes.js"></script>
  <script src="index_files/libs/revealjs/plugin/search/search.js"></script>
  <script src="index_files/libs/revealjs/plugin/zoom/zoom.js"></script>
  <script src="index_files/libs/revealjs/plugin/math/math.js"></script>
  <script>window.define = window.backupDefine; window.backupDefine = undefined;</script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
'controlsAuto': true,
'previewLinksAuto': false,
'smaller': false,
'pdfSeparateFragments': false,
'autoAnimateEasing': "ease",
'autoAnimateDuration': 1,
'autoAnimateUnmatched': true,
'menu': {"side":"left","useTextContentForMissingTitles":true,"markers":false,"loadIcons":false,"custom":[{"title":"Tools","icon":"<i class=\"fas fa-gear\"></i>","content":"<ul class=\"slide-menu-items\">\n<li class=\"slide-tool-item active\" data-item=\"0\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.fullscreen(event)\"><kbd>f</kbd> Fullscreen</a></li>\n<li class=\"slide-tool-item\" data-item=\"1\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.speakerMode(event)\"><kbd>s</kbd> Speaker View</a></li>\n<li class=\"slide-tool-item\" data-item=\"2\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>o</kbd> Slide Overview</a></li>\n<li class=\"slide-tool-item\" data-item=\"3\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.overview(event)\"><kbd>e</kbd> PDF Export Mode</a></li>\n<li class=\"slide-tool-item\" data-item=\"4\"><a href=\"#\" onclick=\"RevealMenuToolHandlers.keyboardHelp(event)\"><kbd>?</kbd> Keyboard Help</a></li>\n</ul>"}],"openButton":true},
'smaller': false,
 
        // Display controls in the bottom right corner
        controls: false,

        // Help the user learn the controls by providing hints, for example by
        // bouncing the down arrow when they first encounter a vertical slide
        controlsTutorial: false,

        // Determines where controls appear, "edges" or "bottom-right"
        controlsLayout: 'edges',

        // Visibility rule for backwards navigation arrows; "faded", "hidden"
        // or "visible"
        controlsBackArrows: 'faded',

        // Display a presentation progress bar
        progress: true,

        // Display the page number of the current slide
        slideNumber: false,

        // 'all', 'print', or 'speaker'
        showSlideNumber: 'all',

        // Add the current slide number to the URL hash so that reloading the
        // page/copying the URL will return you to the same slide
        hash: true,

        // Start with 1 for the hash rather than 0
        hashOneBasedIndex: false,

        // Flags if we should monitor the hash and change slides accordingly
        respondToHashChanges: true,

        // Push each slide change to the browser history
        history: true,

        // Enable keyboard shortcuts for navigation
        keyboard: true,

        // Enable the slide overview mode
        overview: true,

        // Disables the default reveal.js slide layout (scaling and centering)
        // so that you can use custom CSS layout
        disableLayout: false,

        // Vertical centering of slides
        center: false,

        // Enables touch navigation on devices with touch input
        touch: true,

        // Loop the presentation
        loop: false,

        // Change the presentation direction to be RTL
        rtl: false,

        // see https://revealjs.com/vertical-slides/#navigation-mode
        navigationMode: 'linear',

        // Randomizes the order of slides each time the presentation loads
        shuffle: false,

        // Turns fragments on and off globally
        fragments: true,

        // Flags whether to include the current fragment in the URL,
        // so that reloading brings you to the same fragment position
        fragmentInURL: false,

        // Flags if the presentation is running in an embedded mode,
        // i.e. contained within a limited portion of the screen
        embedded: false,

        // Flags if we should show a help overlay when the questionmark
        // key is pressed
        help: true,

        // Flags if it should be possible to pause the presentation (blackout)
        pause: true,

        // Flags if speaker notes should be visible to all viewers
        showNotes: false,

        // Global override for autoplaying embedded media (null/true/false)
        autoPlayMedia: null,

        // Global override for preloading lazy-loaded iframes (null/true/false)
        preloadIframes: null,

        // Number of milliseconds between automatically proceeding to the
        // next slide, disabled when set to 0, this value can be overwritten
        // by using a data-autoslide attribute on your slides
        autoSlide: 0,

        // Stop auto-sliding after user input
        autoSlideStoppable: true,

        // Use this method for navigation when auto-sliding
        autoSlideMethod: null,

        // Specify the average time in seconds that you think you will spend
        // presenting each slide. This is used to show a pacing timer in the
        // speaker view
        defaultTiming: null,

        // Enable slide navigation via mouse wheel
        mouseWheel: false,

        // The display mode that will be used to show slides
        display: 'block',

        // Hide cursor if inactive
        hideInactiveCursor: true,

        // Time before the cursor is hidden (in ms)
        hideCursorTime: 5000,

        // Opens links in an iframe preview overlay
        previewLinks: false,

        // Transition style (none/fade/slide/convex/concave/zoom)
        transition: 'none',

        // Transition speed (default/fast/slow)
        transitionSpeed: 'default',

        // Transition style for full page slide backgrounds
        // (none/fade/slide/convex/concave/zoom)
        backgroundTransition: 'none',

        // Number of slides away from the current that are visible
        viewDistance: 3,

        // Number of slides away from the current that are visible on mobile
        // devices. It is advisable to set this to a lower number than
        // viewDistance in order to save resources.
        mobileViewDistance: 2,

        // The "normal" size of the presentation, aspect ratio will be preserved
        // when the presentation is scaled to fit different resolutions. Can be
        // specified using percentage units.
        width: 1050,

        height: 700,

        // Factor of the display size that should remain empty around the content
        margin: 0.1,

        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [QuartoLineHighlight, PdfExport, RevealMenu, QuartoSupport,

          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    
    <script>
      // htmlwidgets need to know to resize themselves when slides are shown/hidden.
      // Fire the "slideenter" event (handled by htmlwidgets.js) when the current
      // slide changes (different for each slide format).
      (function () {
        // dispatch for htmlwidgets
        function fireSlideEnter() {
          const event = window.document.createEvent("Event");
          event.initEvent("slideenter", true, true);
          window.document.dispatchEvent(event);
        }

        function fireSlideChanged(previousSlide, currentSlide) {
          fireSlideEnter();

          // dispatch for shiny
          if (window.jQuery) {
            if (previousSlide) {
              window.jQuery(previousSlide).trigger("hidden");
            }
            if (currentSlide) {
              window.jQuery(currentSlide).trigger("shown");
            }
          }
        }

        // hookup for slidy
        if (window.w3c_slidy) {
          window.w3c_slidy.add_observer(function (slide_num) {
            // slide_num starts at position 1
            fireSlideChanged(null, w3c_slidy.slides[slide_num - 1]);
          });
        }

      })();
    </script>

    <script id="quarto-html-after-body" type="application/javascript">
    window.document.addEventListener("DOMContentLoaded", function (event) {
      const toggleBodyColorMode = (bsSheetEl) => {
        const mode = bsSheetEl.getAttribute("data-mode");
        const bodyEl = window.document.querySelector("body");
        if (mode === "dark") {
          bodyEl.classList.add("quarto-dark");
          bodyEl.classList.remove("quarto-light");
        } else {
          bodyEl.classList.add("quarto-light");
          bodyEl.classList.remove("quarto-dark");
        }
      }
      const toggleBodyColorPrimary = () => {
        const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
        if (bsSheetEl) {
          toggleBodyColorMode(bsSheetEl);
        }
      }
      toggleBodyColorPrimary();  
      const tabsets =  window.document.querySelectorAll(".panel-tabset-tabby")
      tabsets.forEach(function(tabset) {
        const tabby = new Tabby('#' + tabset.id);
      });
      const clipboard = new window.ClipboardJS('.code-copy-button', {
        target: function(trigger) {
          return trigger.previousElementSibling;
        }
      });
      clipboard.on('success', function(e) {
        // button target
        const button = e.trigger;
        // don't keep focus
        button.blur();
        // flash "checked"
        button.classList.add('code-copy-button-checked');
        var currentTitle = button.getAttribute("title");
        button.setAttribute("title", "Copied!");
        let tooltip;
        if (window.bootstrap) {
          button.setAttribute("data-bs-toggle", "tooltip");
          button.setAttribute("data-bs-placement", "left");
          button.setAttribute("data-bs-title", "Copied!");
          tooltip = new bootstrap.Tooltip(button, 
            { trigger: "manual", 
              customClass: "code-copy-button-tooltip",
              offset: [0, -8]});
          tooltip.show();    
        }
        setTimeout(function() {
          if (tooltip) {
            tooltip.hide();
            button.removeAttribute("data-bs-title");
            button.removeAttribute("data-bs-toggle");
            button.removeAttribute("data-bs-placement");
          }
          button.setAttribute("title", currentTitle);
          button.classList.remove('code-copy-button-checked');
        }, 1000);
        // clear code selection
        e.clearSelection();
      });
      function tippyHover(el, contentFn) {
        const config = {
          allowHTML: true,
          content: contentFn,
          maxWidth: 500,
          delay: 100,
          arrow: false,
          appendTo: function(el) {
              return el.closest('section.slide') || el.parentElement;
          },
          interactive: true,
          interactiveBorder: 10,
          theme: 'quarto-reveal',
          placement: 'bottom-start'
        };
          config['offset'] = [0,0];
          config['maxWidth'] = 700;
        window.tippy(el, config); 
      }
      const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
      for (var i=0; i<noterefs.length; i++) {
        const ref = noterefs[i];
        tippyHover(ref, function() {
          // use id or data attribute instead here
          let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
          try { href = new URL(href).hash; } catch {}
          const id = href.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          return note.innerHTML;
        });
      }
      const findCites = (el) => {
        const parentEl = el.parentElement;
        if (parentEl) {
          const cites = parentEl.dataset.cites;
          if (cites) {
            return {
              el,
              cites: cites.split(' ')
            };
          } else {
            return findCites(el.parentElement)
          }
        } else {
          return undefined;
        }
      };
      var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
      for (var i=0; i<bibliorefs.length; i++) {
        const ref = bibliorefs[i];
        const citeInfo = findCites(ref);
        if (citeInfo) {
          tippyHover(citeInfo.el, function() {
            var popup = window.document.createElement('div');
            citeInfo.cites.forEach(function(cite) {
              var citeDiv = window.document.createElement('div');
              citeDiv.classList.add('hanging-indent');
              citeDiv.classList.add('csl-entry');
              var biblioDiv = window.document.getElementById('ref-' + cite);
              if (biblioDiv) {
                citeDiv.innerHTML = biblioDiv.innerHTML;
              }
              popup.appendChild(citeDiv);
            });
            return popup.innerHTML;
          });
        }
      }
    });
    </script>
    

</body></html>